% This is the Oregon State University LaTeX template. To the best of my
% knowledge, most of the work was done by those acknowledged in beavtex.cls.

%%
%% Preamble
%%
% \documentclass{<something>} must begin each LaTeX document
\documentclass[double,12pt]{beavtex}
  \renewcommand{\familydefault}{\sfdefault}
% Added by CII
\usepackage{graphicx,latexsym}
\usepackage{amsmath}
\usepackage{amssymb,amsthm}
\usepackage{longtable,booktabs,setspace}
\usepackage[hyphens]{url}
\usepackage[colorlinks = true,
			urlcolor = blue,
			linkcolor = black,
			citecolor = black,
			anchorcolor = black]{hyperref}
\usepackage{lmodern}
\usepackage{float}
\floatplacement{figure}{H}
% End of CII addition
\usepackage{rotating} % Package added to allow the rotation of figures and chart
                      % on a page, {sidewaysfigure} command
\usepackage{tablefootnote} % Packaged added to allow footnotes in the tabular
                           % environment, use \tablefootnote command

% This has to do with a default pandoc thing
% http://tex.stackexchange.com/a/258486/77699
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

% Added by CII (Thanks, Hadley!)
% Use ref for internal links
\renewcommand{\hyperref}[2][???]{\autoref{#1}}
\def\chapterautorefname{Chapter}
\def\sectionautorefname{Section}
\def\subsectionautorefname{Subsection}
% End of CII addition

% Added by CII
\usepackage{caption}
\captionsetup{width=5in}
% End of CII addition

\title{Development and Application of Tools for Analysis of Clonal Population
Genetics} % {An Analysis of Something}
\author{Zhian N. Kamvar} % {Joseph A. Student}
\degree{Doctor of Philosophy} % {Master of Science}
\doctype{Dissertation}
\submitdate{December 6, 2016} % {January 1, 2013}
\commencementyear{2017} % {2013}

\department{Botany and Plant Pathology} % {Nuclear Engineering and Radiation Health Physics}

\depttype{Department} % {School}

\depthead{Head} % {Director}

\major{Botany and Plant Pathology} % {Radiation Health Physics}

\advisor{Niklaus J. Grünwald} % {Jane R. Professor}

\abstract{This is a \LaTeX template derived from the \beavtex template found on
overleaf
(\url{https://www.overleaf.com/latex/templates/oregon-state-university-thesis-and-dissertation/wnvzcdhqshxf})
and cloned using
\texttt{git\ clone\ https://git.overleaf.com/5973847rnpkdf} I have made
the following changes

\begin{itemize}
\tightlist
\item
  modified \texttt{beavtex.cls} from v1.1 to include support for
  verbatim font styles in R code
\item
  stripped the template of its original content to create a pandoc
  template
\item
  placed this into a bookdown (\url{https://bookdown.org/}) framework
\item
  added packages latexsym, amsmath amssymb,amsthm, longtable, booktabs,
  setspace, url, hyperref, lmodern, caption, and float
\end{itemize}

This is still a WIP, but it seems to work for the moment. Zhian N.
Kamvar 2016-08-25}
\acknowledgements{I would like to acknowledge\ldots{}Lorem ipsum dolor sit amet,
consectetur adipiscing elit. Maecenas vel eros sed mauris porttitor
semper nec a orci. Nullam vestibulum mi nec condimentum posuere.
Pellentesque eget diam id sapien aliquet ullamcorper. Pellentesque
blandit nec lectus ut mollis. Praesent in facilisis justo. Vestibulum
ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia
Curae; Sed eget congue leo, sed consequat libero. In rutrum malesuada
nisi. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices
posuere cubilia Curae; Morbi sollicitudin tortor ut sem facilisis
mollis.}


	\contributors{The following people contributed to this dissertation:

\subsubsection*{Chapter 2}\label{chapter-2}
\addcontentsline{toc}{subsubsection}{Chapter 2}

Javier F. Tabima assisted in writing the code and design, Niklaus J.
Grünwald assisted in design, and editing of the manuscript.

\subsubsection*{Chapter 3}\label{chapter-3}
\addcontentsline{toc}{subsubsection}{Chapter 3}

Zhian N. Kamvar, Meredeth M. Larsen, Alan M. Kanaskie, Everett M.
Hansen, and Niklaus J. Grünwald\ldots{}

\subsubsection*{Chapter 4}\label{chapter-4}
\addcontentsline{toc}{subsubsection}{Chapter 4}

Jonah C. Brooks assisted in writing and testing the code. Niklaus J.
Grünwald assisted in the design, coördination of the collaborative
effort, and editing the manuscript.

\subsubsection*{Chapter 5}\label{chapter-5}
\addcontentsline{toc}{subsubsection}{Chapter 5}

Niklaus J. Grünwald assisted in the design, and editing of the
manuscript.}


	\dedication{This dissertation is for my grandfather, Franklin Hepner.}




\begin{document}

\maketitle
\mainmatter


  \chapter{Introduction}\label{introduction}
  
  \begin{itemize}
  \tightlist
  \item
    Plant Pathogens
  
    \begin{itemize}
    \tightlist
    \item
      Evolution and Adaptation
    \item
      Understanding clonal population dynamics (\emph{Phytophthora} and
      \emph{Zymoseptoria} examples)
    \end{itemize}
  \item
    Tools for analysis
  
    \begin{itemize}
    \tightlist
    \item
      Tools for clonal populations always come after sexual populations
    \item
      We wrote our own tools
    \end{itemize}
  \item
    Goals of my work
  
    \begin{enumerate}
    \def\labelenumi{\arabic{enumi}.}
    \tightlist
    \item
      development of computational tools to characterize populations
    \item
      application of these tools to address empirical and theoretical
      questions
    \end{enumerate}
  \end{itemize}
  
  \section{Population genetics of clonal
  organisms}\label{population-genetics-of-clonal-organisms}
  
  \begin{itemize}
  \tightlist
  \item
    Population genetics is traditionally centered around HWE
  \item
    Clonal populations violate basic statistical inference
  
    \begin{itemize}
    \tightlist
    \item
      non-independent samples (clones)
    \item
      excess heterozygosity
    \item
      Meselson effect (Balloux et al.,
      \protect\hyperlink{ref-balloux2003population}{2003}; Butlin,
      \protect\hyperlink{ref-butlin2000virgin}{2000}; Welch and Meselson,
      \protect\hyperlink{ref-welch2000evidence}{2000},
      \protect\hyperlink{ref-welch2001rates}{2001})
    \end{itemize}
  \end{itemize}
  
  \section{\texorpdfstring{The Genus
  \emph{Phytophthora}}{The Genus Phytophthora}}\label{the-genus-phytophthora}
  
  The genus \emph{Phytophthora}, translating to ``plant destroyer'' in
  Greek, contains over 100 species (Kroon et al.,
  \protect\hyperlink{ref-kroon2012genus}{2012}), many of which have
  significant impact on US agriculture. \emph{P. sojae} is a major problem
  on soybean, causing \$1-2 billion in losses each year (Tyler,
  \protect\hyperlink{ref-tyler2007phytophthora}{2007}). \emph{P. ramorum}
  is changing the landscape of the North American West due to its wide
  host range (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2008phytophthora}{2008}\protect\hyperlink{ref-grunwald2008phytophthora}{a}),
  which results in devastating losses for the US forestry and nursery
  industries. And \emph{P. infestans}, which was a root cause of over a
  million deaths during the Irish Potato Famine and continues to be a
  problem on tomato and potato crops, resulting in losses exceeding \$6
  billion, annually (Haas et al.,
  \protect\hyperlink{ref-haas2009genome}{2009}). \emph{Phytophthora spp.}
  are water molds characterized by production of oospores and biflagellate
  zoospores that place them into the Stramenopiles (Baldauf,
  \protect\hyperlink{ref-baldauf2003deep}{2003}). They are most closely
  related to golden brown algae and quite diverged from fungi.
  
  \subsection{life cycle}\label{life-cycle}
  
  \subsection{Sex and mating types}\label{sex-and-mating-types}
  
  \subsection{\texorpdfstring{Heterothallic, clonal: \emph{P.
  ramorum}}{Heterothallic, clonal: P. ramorum}}\label{heterothallic-clonal-p.-ramorum}
  
  \begin{itemize}
  \tightlist
  \item
    Sudden Oak Death
  \item
    Population genetics in US nurseries (Goss et al.,
    \protect\hyperlink{ref-goss2009population}{2009})
  \end{itemize}
  
  \subsection{\texorpdfstring{Homothallic, partially clonal: \emph{P.
  syringae}}{Homothallic, partially clonal: P. syringae}}\label{homothallic-partially-clonal-p.-syringae}
  
  \begin{itemize}
  \tightlist
  \item
    Abundant in OR Nurseries (found in foliar isolates) (Parke et al.,
    \protect\hyperlink{ref-parke2014phytophthora}{2014})
  \item
    Genetic structure uncharacterized
  \end{itemize}
  
  \section{Tools for analysis of clonal population
  genetics}\label{tools-for-analysis-of-clonal-population-genetics}
  
  Recommendations have been made for analysis (Arnaud-Hanod et al.,
  \protect\hyperlink{ref-arnaud2007standardizing}{2007})
  
  \begin{itemize}
  \tightlist
  \item
    MLG diversity
  \item
    Genotype Accumulation Curve
  \item
    \(P_{sex}\) and \(P_{gen}\)
  \end{itemize}
  
  \subsection{Index of Association}\label{index-of-association}
  
  \begin{itemize}
  \tightlist
  \item
    Standardized Version (Agapow and Burt,
    \protect\hyperlink{ref-Agapowux5f2001}{2001})
  \item
    Previous Simulation Analyses (de Meeûs and Balloux,
    \protect\hyperlink{ref-de2004clonal}{2004})
  \item
    What's missing
  
    \begin{itemize}
    \tightlist
    \item
      Sympatric clonal lineages
    \item
      Analysis of significance testing
    \item
      HTS markers
    \end{itemize}
  \end{itemize}
  
  \subsection{Software Limitations}\label{software-limitations}
  
  Plethora of tools, most designed for sexual populations except:
  
  \begin{itemize}
  \tightlist
  \item
    GenClone
  \item
    GenoDive
  \end{itemize}
  
  Problems with file formatting, time, and reproducibility.
  
  \subsection{poppr}\label{poppr}
  
  \begin{itemize}
  \tightlist
  \item
    R
  \item
    poppr
  \end{itemize}
  
  \section{Applications of novel tools for analysis of partially-clonal
  populations}\label{applications-of-novel-tools-for-analysis-of-partially-clonal-populations}
  
  \begin{itemize}
  \tightlist
  \item
    Simulation analysis of \(\bar{r}_d\)
  \item
    SSR analysis of \emph{P. ramorum}
  \item
    GBS analysis of \emph{P. syringae}
  \end{itemize}
  
  \section{Conclusion}\label{conclusion}
  
  \begin{itemize}
  \tightlist
  \item
    Open Source Scientific Software Development of Poppr
  
    \begin{itemize}
    \tightlist
    \item
      Related tools
    \end{itemize}
  \item
    Simulation Analysis
  \item
    Pop gen info for two \emph{Phytophthoras}
  \item
    Major results of your work in 2-4 sentences \ldots{}
  \end{itemize}
  
  \chapter{\texorpdfstring{\emph{Poppr}: an R Package For Genetic Analysis
  of Populations With Clonal, Partially Clonal, and/or Sexual
  Reproduction}{Poppr: an R Package For Genetic Analysis of Populations With Clonal, Partially Clonal, and/or Sexual Reproduction}}\label{poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction}
  
  \singlespacing
  
  \begin{center}
  
  Zhian N. Kamvar, Javier F. Tabima, and Niklaus J. Grünwald
  
  
  
  \end{center}\vspace*{\fill}
  
  Journal: \textbf{PeerJ}
  
  PO Box 614, Corte Madera, CA 94976, USA
  
  Published 2014-03-04, Issue: \textbf{2}:e281, DOI:
  \href{https://dx.doi.org/10.7717/peerj.281}{10.7717/peerj.281}
  
  \doublespacing
  \newpage
  
  \section{Abstract}\label{abstract}
  
  Many microbial, fungal, or oomcyete populations violate assumptions for
  population genetic analysis because these populations are clonal,
  admixed, partially clonal, and/or sexual. Furthermore, few tools exist
  that are specifically designed for analyzing data from clonal
  populations, making analysis difficult and haphazard. We developed the R
  package poppr providing unique tools for analysis of data from admixed,
  clonal, mixed, and/or sexual populations. Currently, poppr can be used
  for dominant/codominant and haploid/diploid genetic data. Data can be
  imported from several formats including GenAlEx formatted text files and
  can be analyzed on a user-defined hierarchy that includes unlimited
  levels of subpopulation structure and clone censoring. New functions
  include calculation of Bruvo's distance for microsatellites,
  batch-analysis of the index of association with several indices of
  genotypic diversity, and graphing including dendrograms with bootstrap
  support and minimum spanning networks. While functions for genotypic
  diversity and clone censoring are specific for clonal populations,
  several functions found in poppr are also valuable to analysis of any
  populations. A manual with documentation and examples is provided. Poppr
  is open source and major releases are available on CRAN:
  \url{http://cran.r-project.org/package=poppr}. More supporting
  documentation and tutorials can be found under `resources' at:
  \url{http://grunwaldlab.cgrb.oregonstate.edu/}.
  
  \chapter{Novel R Tools For Analysis of Genome-Wide Population Genetic
  Data With Emphasis on
  Clonality}\label{novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality}
  
  \singlespacing
  
  \begin{center}
  
  Zhian N. Kamvar, Jonah C. Brooks, and Niklaus J. Grünwald
  
  
  
  \end{center}\vspace*{\fill}
  
  Journal: \textbf{Frontiers in Genetics}
  
  EPFL Innovation Park, Building I, CH -- 1015 Lausanne Switzerland
  
  Published 2015-06-10. Issue: \textbf{6}, DOI:
  \href{http://dx.doi.org/10.3389/fgene.2015.00208}{10.3389/fgene.2015.00208}
  
  \doublespacing
  \newpage
  
  \section{Abstract}\label{abstract-1}
  
  To gain a detailed understanding of how plant microbes evolve and adapt
  to hosts, pesticides, and other factors, knowledge of the population
  dynamics and evolutionary history of populations is crucial. Plant
  pathogen populations are often clonal or partially clonal which requires
  different analytical tools. With the advent of high throughput
  sequencing technologies, obtaining genome-wide population genetic data
  has become easier than ever before. We previously contributed the R
  package \emph{poppr} specifically addressing issues with analysis of
  clonal populations. In this paper we provide several significant
  extensions to \emph{poppr} with a focus on large, genome-wide SNP data.
  Specifically, we provide several new functionalities including the new
  function \texttt{mlg.filter} to define clone boundaries allowing for
  inspection and definition of what is a clonal lineage, minimum spanning
  networks with reticulation, a sliding-window analysis of the index of
  association, modular bootstrapping of any genetic distance, and analyses
  across any level of hierarchies.
  
  \section{Introduction}\label{introduction-1}
  
  To paraphrase Dobzhansky, nothing in the field of plant-microbe
  interactions makes sense except in the light of population genetics
  (Dobzhansky, \protect\hyperlink{ref-dobzhansky2013nothing}{1973}).
  Genetic forces such as selection and drift act on alleles in a
  population. Thus, a true understanding of how plant pathogens emerge,
  evolve and adapt to crops, fungicides, or other factors, can only be
  elucidated in the context of population level phenomena given the
  demographic history of populations (Grünwald and Goss,
  \protect\hyperlink{ref-grunwald2011evolution}{2011}; McDonald and Linde,
  \protect\hyperlink{ref-Mcdonald2002}{2002}; Milgroom et al.,
  \protect\hyperlink{ref-milgroom1989population}{1989}). The field of
  population genetics, in the era of whole genome resequencing, provides
  unprecedented power to describe the evolutionary history and population
  processes that drive coevolution between pathogens and hosts. This
  powerful field thus critically enables effective deployment of R genes,
  design of pathogen informed plant resistance breeding programs, and
  implementation of fungicide rotations that minimize emergence of
  resistance.
  
  Most computational tools for population genetics are based on concepts
  developed for sexual model organisms. Populations that reproduce
  clonally or are polyploid are thus difficult to characterize using
  classical population genetic tools because theoretical assumptions
  underlying the theory are violated. Yet, many plant pathogen populations
  are at least partially clonal if not completely clonal (Anderson and
  Kohn, \protect\hyperlink{ref-anderson1995clonality}{1995}; Milgroom,
  \protect\hyperlink{ref-milgroom1996recombination}{1996}). Thus,
  development of tools for analysis of clonal or polyploid populations is
  needed.
  
  Genotyping by sequencing and whole genome resequencing provide the
  unprecedented ability to identify thousands of single nucleotide
  polymorphisms (SNPs) in populations (Davey et al.,
  \protect\hyperlink{ref-davey2011genome}{2011}; Elshire et al.,
  \protect\hyperlink{ref-elshire2011robust}{2011}; Luikart et al.,
  \protect\hyperlink{ref-luikart2003power}{2003}). With traditional marker
  data (e.g., SSR, AFLP) a clone was typically defined as a unique
  multilocus genotype (MLG) (Cooke et al.,
  \protect\hyperlink{ref-cooke2012genome}{2012}; Falush et al.,
  \protect\hyperlink{ref-Falush01082003}{2003}; Goss et al.,
  \protect\hyperlink{ref-goss2009population}{2009}; Grünwald and Hoheisel,
  \protect\hyperlink{ref-grunwald2006hierarchical}{2006}; Taylor and
  Fisher, \protect\hyperlink{ref-taylor2003fungal}{2003}). Availability of
  large SNP data sets provides new challenges for data analysis. These
  data are based on reduced representation libraries and high throughput
  sequencing with moderate sequencing depth which invariably results in
  substantial missing data, error in SNP calling due to sequencing error,
  lack of read depth or other sources of spurious allele calls
  (Mastretta-Yanes et al.,
  \protect\hyperlink{ref-mastretta2015restriction}{2014}). It is thus not
  clear what a clone is in large SNP data sets and novel tools are
  required for definition of clone boundaries.
  
  The research community using the R statistical and computing language (R
  Core Team, \protect\hyperlink{ref-R}{2015}) has developed a plethora of
  new resources for population genetic analysis. R is particularly
  appealing because all code is open source and functions can be evaluated
  and modified by any user. Recently, we introduced the R package
  \emph{poppr} specifically developed for analysis of clonal populations
  (Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b}).
  \emph{Poppr} previously introduced several novel features including the
  ability to conduct a hierarchical analysis across unlimited hierarchies,
  test for linkage association, graph minimum spanning networks or provide
  bootstrap support for Bruvo's distance in resulting trees. \emph{Poppr}
  has been rapidly adopted and applied to a range of studies including for
  example horizontal transmission in leukemia of clams (Metzger et al.,
  \protect\hyperlink{ref-metzger2015horizontal}{2015}), study of the
  vector-mediated parent-to-offspring transmission in an avian
  malaria-like parasite (Chakarov et al.,
  \protect\hyperlink{ref-chakarov2015apparent}{2015}), and
  characterization of the emergence of the invasive forest pathogen
  \emph{Hymenoscyphus pseudoalbidus} (Gross et al.,
  \protect\hyperlink{ref-gross2014population}{2014}). It has also been
  used to implement real-time, online R based tools for visualizing
  relationships among unknown MLGs in reference databases\\
  (\url{http://phytophthora-id.org/}) (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2011phytophthora}{2011}).
  
  Here, we introduce \emph{poppr} 2.0, which provides a major update to
  \emph{poppr} (Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b})
  including novel tools for analysis of clonal populations specifically
  addressing large SNP data. Significant novel tools include functions for
  calculating clone boundaries and collapsing individuals into clonal
  groups based on a user-specified genetic distance threshold, sliding
  window analyses, genotype accumulation curves, reticulations in minimum
  spanning networks, and bootstrapping for any genetic distance.
  
  \section{Implementations and
  Examples}\label{implementations-and-examples}
  
  \subsection{Clonal identification}\label{clonal-identification}
  
  As highlighted in previous work, clone correction is an important
  component of population genetic analysis of organisms that are known to
  reproduce asexually (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2003analysis}{2003}; Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b};
  Milgroom, \protect\hyperlink{ref-milgroom1996recombination}{1996}). This
  method is a partial correction for bias that affects metrics that rely
  on allele frequencies assuming panmixia and was initially designed for
  data with only a handful of markers. With the advent of large-scale
  sequencing and reduced- representation libraries, it has become easier
  to sequence tens of thousands of markers from hundreds of individuals
  (Davey and Blaxter, \protect\hyperlink{ref-davey2010rad}{2010}; Davey et
  al., \protect\hyperlink{ref-davey2011genome}{2011}; Elshire et al.,
  \protect\hyperlink{ref-elshire2011robust}{2011}). With this larger
  number of markers, the genetic resolution is much greater, but the
  chance of genotyping error is also greatly increased and missing data is
  frequent (Mastretta-Yanes et al.,
  \protect\hyperlink{ref-mastretta2015restriction}{2014}). Taking this
  fact and occasional somatic mutations into account, it would be
  impossible to separate true clones from independent individuals by just
  comparing what MLGs are different. We introduce a new method for
  collapsing unique multilocus genotypes determined by naive string
  comparison into multilocus lineages utilizing any genetic distance given
  three different clustering algorithms: farthest neighbor, nearest
  neighbor, and UPGMA (average neighbor) (Sokal,
  \protect\hyperlink{ref-sokal1958statistical}{1958}).
  
  These clustering algorithms act on a distance matrix that is either
  provided by the user or generated via a function that will calculate a
  distance from genetic data such as \texttt{bruvo.dist}, which in
  particular applies to any level of ploidy (Bruvo et al.,
  \protect\hyperlink{ref-bruvo2004simple}{2004}). All algorithms have been
  implemented in C and utilize the OpenMP framework for optional parallel
  processing (Dagum and Menon,
  \protect\hyperlink{ref-dagum1998openmp}{1998}). Default is the
  conservative farthest neighbor algorithm (Fig. \ref{fig:Figure1}A),
  which will only cluster samples together if all samples in the cluster
  are at a distance less than the given threshold. By contrast, the
  nearest neighbor algorithm will have a chaining effect that will cluster
  samples akin to adding links on a chain where a sample can be included
  in a cluster if all of the samples have at least one connection below a
  given threshold (Fig. \ref{fig:Figure1}C). The UPGMA, or average
  neighbor clustering algorithm is the one most familiar to biologists as
  it is often used to generate ultra-metric trees based on genetic
  distance (Fig. \ref{fig:Figure1}B). This algorithm will cluster by
  creating a representative sample per cluster and joining clusters if
  these representative samples are closer than the given threshold.
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.5\linewidth]{figure/frontiers/Figure-1} 
  
  }
  
  \caption[Diagrammatic representation of the three clustering algorithms
  implemented in \texttt{mlg.filter}.]{Diagrammatic representation of the three clustering algorithms
  implemented in \texttt{mlg.filter}. (\textbf{A-C}) Represent different
  clustering algorithms on the same imaginary network with a threshold of
  0.451. Edge weights are represented in arbitrary units noted by the line
  thickness and numerical values next to the lines. All outer angles are
  90 degrees, so the un-labeled edge weights can be obtained with simply
  geometry. Colored circles represent clusters of genotypes. (\textbf{A})
  Farthest neighbor clustering does not cluster nodes B and C because
  nodes A and C are more than a distance of 0.451 apart. (\textbf{B})
  UPGMA (average neighbor) clustering clusters nodes A, B, and C together
  because the average distance between them and C is \textless{} 0.451.
  (\textbf{C}) Nearest neighbor clustering clusters all nodes together
  because the minimum distance between them is always \textless{} 0.451.}\label{fig:Figure1}
  \end{figure}
  
  We utilize data from the microbe \emph{Phytophthora infestans} to show
  how the \texttt{mlg.filter} function collapses multilocus genotypes with
  Bruvo's distance assuming a genome addition model (Bruvo et al.,
  \protect\hyperlink{ref-bruvo2004simple}{2004}). \emph{P. infestans} is
  the causal agent of potato late blight originating from Mexico that
  spread to Europe in the mid 19th century (Goss et al.,
  \protect\hyperlink{ref-goss2014irish}{2014}; Yoshida et al.,
  \protect\hyperlink{ref-yoshida2013rise}{2013}). \emph{P. infestans}
  reproduces both clonally and sexually. The clonal lineages of \emph{P.
  infestans} have been formally defined into 18 separate clonal lineages
  using a combination of various molecular methods including AFLP and
  microsatellite markers (Lees et al.,
  \protect\hyperlink{ref-lees2006novel}{2006}; Li et al.,
  \protect\hyperlink{ref-li2013efficient}{2013}). For these data, we used
  \texttt{mlg.filter} to detect all of the distance thresholds at which 18
  multilocus lineages would be resolved. We used these thresholds to
  define multilocus lineages and create contingency tables and dendrograms
  to determine how well the multilocus lineages were detected.
  
  For the \emph{P. infestans} population, the three algorithms were able
  to detect 18 multilocus lineages at different distance thresholds (Fig.
  \ref{fig:Figure-2}). Contingency tables between the described multilocus
  genotypes and the genotypes defined by distance show that most of the 18
  lineages were resolved, except for US-8, which is polytomic (Table
  \ref{pinftable}).
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{thesis_files/figure-latex/Figure-2-1} 
  
  }
  
  \caption[Graphical representation of three different clustering algorithms
  collapsing multilocus genotypes for 12 SSR loci from \emph{Phytophthora
  infestans} representing 18 clonal lineages.]{Graphical representation of three different clustering algorithms
  collapsing multilocus genotypes for 12 SSR loci from \emph{Phytophthora
  infestans} representing 18 clonal lineages. The horizontal axis is
  Bruvo's genetic distance assuming the genome addition model. The
  vertical axis represents the number of multilocus lineages observed.
  Each point shows the threshold at which one would observe a given number
  of multilocus genotypes. The horizontal black line represents 18
  multilocus genotypes and vertical dashed lines mark the thresholds used
  to collapse the multilocus genotypes into 18 multilocus lineages.}\label{fig:Figure-2}
  \end{figure}
  
  \begin{sidewaystable}[ph!]
  \centering
  \begin{tabular}{l|cccccccccccccccccc}
    & \textbf{3} & \textbf{4} & \textbf{5} & \textbf{6} & \textbf{8} & \textbf{10} & \textbf{12} & \textbf{15} & \textbf{16} & \textbf{17} & \textbf{18} & \textbf{20} & \textbf{21} & \textbf{22} & \textbf{24} & \textbf{25} & \textbf{27} & \textbf{28} \\ 
    \midrule
  \textbf{B} & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & 1 & . & . \\ 
    \textbf{C} & . & . & . & . & . & . & . & . & . & . & . & . & . & . & 1 & . & . & . \\ 
    \textbf{D.1} & . & . & . & . & . & . & . & . & . & . & . & . & . & 1 & . & . & . & . \\ 
    \textbf{D.2} & . & . & . & . & . & . & . & . & . & . & . & . & . & 1 & . & . & . & . \\ 
    \textbf{EU-13} & . & . & . & . & . & . & . & . & 1 & . & . & . & . & . & . & . & . & . \\ 
    \textbf{EU-4} & . & . & . & . & . & . & . & . & . & 1 & . & . & . & . & . & . & . & . \\ 
    \textbf{EU-5} & . & . & . & . & . & . & . & . & . & . & 2 & . & . & . & . & . & . & . \\ 
    \textbf{EU-8} & . & . & . & . & . & . & 1 & . & . & . & . & . & . & . & . & . & . & . \\ 
    \textbf{US-11} & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & 2 \\ 
    \textbf{US-12} & . & 1 & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . \\ 
    \textbf{US-14} & . & . & . & . & . & 1 & . & . & . & . & . & . & . & . & . & . & . & . \\ 
    \textbf{US-17} & . & . & . & . & . & . & . & . & . & . & . & 1 & . & . & . & . & . & . \\ 
    \textbf{US-20} & 2 & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . \\ 
    \textbf{US-21} & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & . & 2 & . \\ 
    \textbf{US-22} & . & . & . & . & . & . & . & . & . & . & . & . & 2 & . & . & . & . & . \\ 
    \textbf{US-23} & . & . & . & . & . & . & . & 3 & . & . & . & . & . & . & . & . & . & . \\ 
    \textbf{US-24} & . & . & . & . & 3 & . & . & . & . & . & . & . & . & . & . & . & . & . \\ 
    \textbf{US-8} & . & . & 1 & 1 & . & 2 & . & . & . & . & . & . & . & . & . & . & . & . \\ 
     \bottomrule
  \end{tabular}
  \caption[Contingency table comparing multilocus lineages (MLL)]{Contingency table comparing multilocus lineages (MLL) defined in Li et
  al. (\protect\hyperlink{ref-li2013efficient}{2013}) and Lees et al.
  (\protect\hyperlink{ref-lees2006novel}{2006}) (rows) to MLLs inferred
  from Bruvo's genetic distance (columns) at a threshold of 0.07 with the
  average neighbor algorithm (Bruvo et al.,
  \protect\hyperlink{ref-bruvo2004simple}{2004}; Sokal,
  \protect\hyperlink{ref-sokal1958statistical}{1958}). Values in the table
  represent the number of times any given inferred MLL matches with a
  previously defined MLL. For example, in our original data set, there
  were three genotypes previously defined as the US-24 MLL. All three
  genotypes were also determined to cluster into a single MLL by
  filtering. In contrast, US-8 was determined to cluster into three
  different MLLs by filtering.} 
  \label{pinftable}
  \end{sidewaystable}
  
  \newpage
  
  We utilized simulated data to evaluate the effect of sequencing error
  and missing data on MLG calling. We constructed the data using the
  \texttt{glSim} function in \emph{adegenet} (Jombart and Ahmed,
  \protect\hyperlink{ref-jombart2011adegenet}{2011}) to obtain a SNP data
  set for demonstration. Two diploid data sets were created, each with 10k
  SNPs (25\% structured into two groups) and 200 samples with 10 ancestral
  populations of even sizes. Clones were created in one data set by
  marking each sample with a unique identifier and then randomly sampling
  with replacement. It is well documented that reduced- representation
  sequencing can introduce several erroneous calls and missing data
  (Mastretta-Yanes et al.,
  \protect\hyperlink{ref-mastretta2015restriction}{2014}). To reflect
  this, we mutated SNPs at a rate of 10\% and inserted an average of 10\%
  missing data for each sample after clones were created, ensuring that no
  two sequences were alike. The number of mutations and missing data per
  sample were determined by sampling from a Poisson distribution with
  \(\lambda = 1000\). After pooling, 20\% of the data set was randomly
  sampled for analysis. Genetic distance was obtained with the function
  \texttt{bitwise.dist}, which calculates the fraction of different sites
  between samples equivalent to Provesti's distance, counting missing data
  as equivalent in comparison (Prevosti et al.,
  \protect\hyperlink{ref-prevosti1975distances}{1975}).
  
  All three filtering algorithms were run with a threshold of 1, returning
  a numeric vector of length \(n - 1\) where each element represented a
  threshold at which two samples/clusters would join. Since each data set
  would have varying distances between samples, the clonal boundary
  threshold was defined as the midpoint of the largest gap between two
  thresholds that collapsed less than 50\% of the data.
  
  Out of the 100 simulations run, we found that across all methods,
  detection of duplicated samples had \(\sim\) 98\% true positive fraction
  and \(\sim\) 0.8\% false positive fraction indicating that this method
  is robust to simulated populations (supplementary materials\footnote{Supplementary
    data available at
    \url{https://github.com/grunwaldlab/supplementary-poppr-2.0}; DOI:
    \href{http://dx.doi.org/10.5281/zenodo.17424}{10.5281/zenodo.17424}}).
  
  \subsection{Minimum Spanning Networks with
  Reticulation}\label{minimum-spanning-networks-with-reticulation}
  
  In its original iteration, \emph{poppr} introduced minimum spanning
  networks that were based on the \emph{igraph} function
  \texttt{minimum.spanning.tree} (Csardi and Nepusz,
  \protect\hyperlink{ref-csardi2006igraph}{2006}). This algorithm produces
  a minimum spanning tree with no reticulations where nodes represent
  individual MLGs. In other minimum spanning network programs,
  reticulation is obtained by calculating the minimum spanning tree
  several times and returning the set of all edges included in the trees.
  Due to the way \emph{igraph} has implemented Prim's algorithm, it is not
  possible to utilize this strategy, thus we implemented an internal C
  function to walk the space of minimum spanning trees based on genetic
  distance to connect groups of nodes with edges of equal weight.
  
  To demonstrate the utility of minimum spanning networks with
  reticulation, we used two clonal data sets: the H3N2 flu virus data from
  the \emph{adegenet} package using years of each epidemic as the
  population factor, and \emph{Phytophthora ramorum} data from Nurseries
  and Oregon forests (Jombart et al.,
  \protect\hyperlink{ref-jombart2010discriminant}{2010}; Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014sudden}{2014}\protect\hyperlink{ref-kamvar2014sudden}{a}).
  Minimum spanning networks were created with and without reticulation
  using the \emph{poppr} functions \texttt{diss.dist} and
  \texttt{bruvo.msn} for the H3N2 and \emph{P. ramorum} data, respectively
  (Bruvo et al., \protect\hyperlink{ref-bruvo2004simple}{2004}; Kamvar et
  al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b}).
  To detect mlg clusters, the infoMAP community detection algorithm was
  applied with 10,000 trials as implemented in the R package \emph{igraph}
  version 0.7.1 utilizing genetic distance as edge weights and number of
  samples in each MLG as vertex weights (Csardi and Nepusz,
  \protect\hyperlink{ref-csardi2006igraph}{2006}; Rosvall and Bergstrom,
  \protect\hyperlink{ref-rosvall2008maps}{2008}).
  
  To evaluate the results, we compared the number, size, and entropy
  (\(H\)) of the resulting communities as we expect a highly clonal
  organism with low genetic diversity to result in a few, large
  communities. We also created contingency tables of the community
  assignments with the defined populations and used those to calculate
  entropy using Shannon's index with the function \texttt{diversity} from
  the R package \emph{vegan} version 2.2-1 (Oksanen et al.,
  \protect\hyperlink{ref-oksanen2015vegan}{2015}; Shannon,
  \protect\hyperlink{ref-shannon2001mathematical}{2001}). A low entropy
  indicates presence of a few large communities whereas high entropy
  indicates presence of many small communities.
  
  The infoMAP algorithm revealed 63 communities with a maximum community
  size of 77 and \(H = 3.56\) for the reticulate network of the H3N2 data
  and 117 communities with a maximum community size of 26 and \(H = 4.65\)
  for the minimum spanning tree. The entropy across years was greatly
  decreased for all populations with the reticulate network compared to
  the minimum spanning tree (Fig. \ref{fig:Figure-3}). Note that the
  reticulated network (Fig. \ref{fig:Figure-3}B) showed patterns
  corresponding with those resulting from a discriminant analysis of
  principal components (Fig. \ref{fig:Figure-3}D) (Jombart et al.,
  \protect\hyperlink{ref-jombart2010discriminant}{2010}).
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/frontiers/Figure-3} 
  
  }
  
  \caption[Minimum Spanning Networks with Reticulation]{(\textbf{A-B}) Minimum spanning networks of the hemagglutinin (HA)
  segment of H3N2 viral DNA from the \emph{adegenet} package representing
  flu epidemics from 2001 to 2006 without reticulation (\textbf{A}) and
  with reticulation (\textbf{B}) (Jombart,
  \protect\hyperlink{ref-Jombartux5f2008}{2008}; Jombart et al.,
  \protect\hyperlink{ref-jombart2010discriminant}{2010}). Each node
  represents a unique multilocus genotype, colors represent epidemic year,
  and edge color represents absolute genetic distance. (\textbf{C})
  Shannon entropy values for population assignments compared with
  communities determined by the infoMAP algorithm on (\textbf{A}) and
  (\textbf{B}). (\textbf{D}) Graphic reproduced from Jombart et al.
  (\protect\hyperlink{ref-jombart2010discriminant}{2010}) showing that the
  2006 epidemic does not cluster neatly with the other years via
  Discriminant Analysis of Principal Components. Horizontal axis
  represents the first discriminant component. Vertical axis represents
  the second discriminant component.}\label{fig:Figure-3}
  \end{figure}
  
  Graph walking of the reticulated minimum spanning network of \emph{P.
  ramorum} by the infoMAP algorithm revealed 16 communities with a maximum
  community size of 13 and \(H = 2.60\). The un-reticulated minimum
  spanning tree revealed 20 communities with a maximum community size of 7
  and \(H = 2.96\). In the ability to predict Hunter Creek as belonging to
  a single community, the reticulated network was successful whereas the
  minimum spanning tree separated one genotype from that community. The
  entropy for the reticulated network was lower for all populations except
  for the coast population (supplementary materials\footnote{Supplementary
    data available at
    \url{https://github.com/grunwaldlab/supplementary-poppr-2.0}; DOI:
    \href{http://dx.doi.org/10.5281/zenodo.17424}{10.5281/zenodo.17424}}).
  
  \subsection{Bootstrapping}\label{bootstrapping}
  
  Assessing population differentiation through methods such as \(G_{st}\),
  AMOVA, and Mantel tests relies on comparing samples within and across
  populations (Excoffier et al.,
  \protect\hyperlink{ref-excoffier1992analysis}{1992}; Mantel,
  \protect\hyperlink{ref-mantel1967detection}{1967}; Nei,
  \protect\hyperlink{ref-nei1973analysis}{1973}). Confidence in distance
  metrics is related to the confidence in the markers to accurately
  represent the diversity of the data. Especially true with microsatellite
  markers, a single hyper-diverse locus can make a population appear to
  have more diversity based on genetic distance. Using a bootstrapping
  procedure of randomly sampling loci with replacement when calculating a
  distance matrix provides support for clades in hierarchical clustering.
  
  Data in genind and genpop objects are represented as matrices with
  individuals in rows and alleles in columns (Jombart,
  \protect\hyperlink{ref-Jombartux5f2008}{2008}). This gives the advantage
  of being able to use R's matrix algebra capabilities to efficiently
  calculate genetic distance. Unfortunately, this also means that
  bootstrapping is a non- trivial task as all alleles at a single locus
  need to be sampled together. To remedy this, we have created an internal
  S4 class called ``bootgen'', which extends the internal ``gen'' class
  from \emph{adegenet}. This class can be created from any genind,
  genclone, or genpop object, and allows loci to be sampled with
  replacement. To further facilitate bootstrapping, a function called
  \texttt{aboot}, which stands for ``any boot'', is introduced that will
  bootstrap any genclone, genind, or genpop object with any genetic
  distance that can be calculated from it.
  
  To demonstrate calculating a dendrogram with bootstrap support, we used
  the \emph{poppr} function \texttt{aboot} on population allelic
  frequencies derived from the data set \texttt{microbov} in the
  \emph{adegenet} package with 1000 bootstrap replicates (Jombart,
  \protect\hyperlink{ref-Jombartux5f2008}{2008}; Laloë et al.,
  \protect\hyperlink{ref-laloe2007consensus}{2007}). The resulting
  dendrogram shows bootstrap support values \(>50\%\) (Fig.
  \ref{fig:microboot}) and used the following code:
  
  \begin{Shaded}
  \begin{Highlighting}[]
  \KeywordTok{library}\NormalTok{(}\StringTok{"poppr"}\NormalTok{);}
  \KeywordTok{data}\NormalTok{(}\StringTok{"microbov"}\NormalTok{, }\DataTypeTok{package =} \StringTok{"adegenet"}\NormalTok{);}
  \KeywordTok{strata}\NormalTok{(microbov) <-}\StringTok{ }\KeywordTok{data.frame}\NormalTok{(}\KeywordTok{other}\NormalTok{(microbov));}
  \KeywordTok{setPop}\NormalTok{(microbov) <-}\StringTok{ }\ErrorTok{~}\NormalTok{coun/spe/breed;}
  \NormalTok{bov_pop <-}\StringTok{ }\KeywordTok{genind2genpop}\NormalTok{(microbov);}
  
  \KeywordTok{set.seed}\NormalTok{(}\DecValTok{20150428}\NormalTok{);}
  \NormalTok{pop_tree <-}\StringTok{ }\KeywordTok{aboot}\NormalTok{(bov_pop, }\DataTypeTok{sample =} \DecValTok{1000}\NormalTok{, }\DataTypeTok{cutoff =} \DecValTok{50}\NormalTok{);}
  \end{Highlighting}
  \end{Shaded}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{thesis_files/figure-latex/microboot-1} 
  
  }
  
  \caption[UPGMA dendrogram generated from Nei's genetic distance]{UPGMA dendrogram generated from Nei's genetic distance on 15 breeds of
  \emph{Bos taurus} (BT) or \emph{Bos indicus} (BI) from Africa (AF) or
  France (FR). These data are from Laloë et al.
  (\protect\hyperlink{ref-laloe2007consensus}{2007}). Node labels
  represent bootstrap support \(>50\%\) out of 1,000 bootstrap replicates.}\label{fig:microboot}
  \end{figure}
  
  \subsection{Genotype Accumulation
  Curve}\label{genotype-accumulation-curve}
  
  Analysis of population genetics of clonal organisms often borrows from
  ecological methods such as analysis of diversity within populations
  (Arnaud-Hanod et al.,
  \protect\hyperlink{ref-arnaud2007standardizing}{2007}; Grünwald et al.,
  \protect\hyperlink{ref-grunwald2003analysis}{2003}; Milgroom,
  \protect\hyperlink{ref-milgroom1996recombination}{1996}). When choosing
  markers for analysis, it is important to make sure that the observed
  diversity in your sample will not appreciably increase if an additional
  marker is added (Arnaud-Hanod et al.,
  \protect\hyperlink{ref-arnaud2007standardizing}{2007}). This concept is
  analogous to a species accumulation curve, obtained by rarefaction. The
  genotype accumulation curve in \emph{poppr} is implemented in the
  function \texttt{genotype\_curve}. The curve is constructed by randomly
  sampling \(x\) loci and counting the number of observed MLGs. This
  repeated \(r\) times for 1 locus up to \(n-1\) loci, creating \(n-1\)
  distributions of observed MLGs.
  
  The following code example demonstrates the genotype accumulation curve
  for data from Everhart and Scherm
  (\protect\hyperlink{ref-everhart2014fine}{2015}) showing that these data
  reach a small plateau and have a greatly decreased variance with 12
  markers, indicating that there are enough markers such that adding more
  markers to the analysis will not create very many new genotypes (Fig.
  \ref{fig:moniliniacurve}).
  
  \begin{Shaded}
  \begin{Highlighting}[]
  \KeywordTok{library}\NormalTok{(}\StringTok{"poppr"}\NormalTok{);}
  \KeywordTok{library}\NormalTok{(}\StringTok{"ggplot2"}\NormalTok{);}
  \KeywordTok{data}\NormalTok{(}\StringTok{"monpop"}\NormalTok{, }\DataTypeTok{package =} \StringTok{"poppr"}\NormalTok{);}
  
  \KeywordTok{set.seed}\NormalTok{(}\DecValTok{20150428}\NormalTok{);}
  \KeywordTok{genotype_curve}\NormalTok{(monpop, }\DataTypeTok{sample =} \DecValTok{1000}\NormalTok{);}
  \NormalTok{p <-}\StringTok{ }\KeywordTok{last_plot}\NormalTok{() +}\StringTok{ }\KeywordTok{theme_bw}\NormalTok{();   }\CommentTok{# get the last plot}
  \NormalTok{p +}\StringTok{ }\KeywordTok{geom_smooth}\NormalTok{(}\KeywordTok{aes}\NormalTok{(}\DataTypeTok{group =} \DecValTok{1}\NormalTok{)); }\CommentTok{# plot with a trendline}
  \end{Highlighting}
  \end{Shaded}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{thesis_files/figure-latex/moniliniacurve-1} 
  
  }
  
  \caption[Genotype accumulation curve]{Genotype accumulation curve for 694 isolates of the peach brown rot
  pathogen, \emph{Monilinia fructicola} genotyped over 13 loci from
  Everhart and Scherm (\protect\hyperlink{ref-everhart2014fine}{2015}).
  The horizontal axis represents the number of loci randomly sampled
  without replacement up to \emph{n - 1} loci, the vertical axis shows the
  number of multilocus genotypes observed, up to 262, the number of unique
  multilocus genotypes in the data set. The red dashed line represents
  90\% of the total observed multilocus genotypes. A trendline (blue) has
  been added using the \emph{ggplot2} function \texttt{stat\_smooth}.}\label{fig:moniliniacurve}
  \end{figure}
  
  \subsection{Index of association}\label{index-of-association-1}
  
  The index of association (\(I_A\)) is a measure of multilocus linkage
  disequilibrium that is most often used to detect clonal reproduction
  within organisms that have the ability to reproduce via sexual or
  asexual processes (Brown et al.,
  \protect\hyperlink{ref-brown1980multilocus}{1980}; Milgroom,
  \protect\hyperlink{ref-milgroom1996recombination}{1996}; Smith et al.,
  \protect\hyperlink{ref-smith1993how}{1993}). It was standardized in 2001
  as \(\bar{r}_d\) by Agapow and Burt
  (\protect\hyperlink{ref-Agapowux5f2001}{2001}) to address the issue of
  scaling with increasing number of loci. This metric is typically applied
  to traditional dominant and co-dominant markers such as AFLPs, SNPs, or
  microsatellite markers. With the advent of high throughput sequencing,
  SNP data is now available in a genome-wide context and in very large
  matrices including thousands of SNPs. For this reason, we devised two
  approaches using the index of association for large numbers of markers
  typical for population genomic studies. Both functions utilize
  \emph{adegenet}'s ``genlight'' object class, which efficiently stores 8
  binary alleles in a single byte (Jombart and Ahmed,
  \protect\hyperlink{ref-jombart2011adegenet}{2011}). As calculation of
  the \(\bar{r}_d\) requires distance matrices of absolute number of
  differences, we utilize a function that calculates these distances
  directly from the compressed data called \texttt{bitwise.dist}.
  
  The first approach is a sliding window analysis implemented in the
  function \texttt{win.ia}. It utilizes the position of markers in the
  genome to calculate \(\bar{r}_d\) among any number of SNPs found within
  a user-specified windowed region. It is important that this calculation
  utilize \(\bar{r}_d\) as the number of loci will be different within
  each window (Agapow and Burt,
  \protect\hyperlink{ref-Agapowux5f2001}{2001}). This approach would be
  suited for a quick calculation of linkage disequilibrium across the
  genome that can detect potential hotspots of LD that could be
  investigated further with more computationally intensive methods
  assuming that the number of samples \textless{}\textless{} the number of
  loci.
  
  As it would necessarily focus on loci within a short section of the
  genome that may or may not be recombining, a sliding window approach
  would not be good for utilizing \(\bar{r}_d\) as a test for clonal
  reproduction. A remedy for this is implemented in the function
  \texttt{samp.ia}, which will randomly sample \(m\) loci, calculate
  \(\bar{r}_d\), and repeat \(r\) times, thus creating a distribution of
  expected values of \(\bar{r}_d\).
  
  To demonstrate the sliding window and random sampling of \(\bar{r}_d\)
  with respect to clonal populations, we simulated two populations
  containing 1,100 neutral SNPs for 100 diploid individuals under the same
  initial seed. One population had individuals randomly sampled with
  replacement, representing the clonal population. After sampling, both
  populations had 5\% random error and 1\% missing data independently
  propagated across all samples. On average, we obtained a higher value of
  \(\bar{r}_d\) for the clonal population compared to the sexual
  population for both methods (Fig. \ref{fig:Figure-6}).
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{thesis_files/figure-latex/Figure-6-1} 
  
  }
  
  \caption[Sliding window analysis of the standardized index of association
  (\(\bar{r}_d\))]{\textbf{(A)} Sliding window analysis of the standardized index of
  association (\(\bar{r}_d\)) across a simulated \(1.1 \times 10^4\) nt
  chromosome containing 1,100 variants among 100 individuals. Each window
  analyzed variants within 500nt chunks. The black line refers to the
  clonal and the blue line to the sexual populations. \textbf{(B)}
  boxplots showing 100 random samples of 50 variants to calculate a
  distribution of \(\bar{r}_d\) for the clonal (red) and sexual (blue)
  populations. Each box is centered around the mean, with whiskers
  extending out to 1.5 times the interquartile range. The median is
  indicated by the center line. \textbf{(A)} and \textbf{(B)} are plotted
  on the same y-axis.}\label{fig:Figure-6}
  \end{figure}
  
  \subsection{Data format updates: population strata and
  hierarchies}\label{data-format-updates-population-strata-and-hierarchies}
  
  Assessments of population structure through methods such as hierarchical
  \(F_{st}\) (Goudet, \protect\hyperlink{ref-goudet2005hierfstat}{2005})
  and AMOVA (Michalakis and Excoffier,
  \protect\hyperlink{ref-michalakis1996generic}{1996}) require
  hierarchical sampling of populations across space or time (Everhart and
  Scherm, \protect\hyperlink{ref-everhart2014fine}{2015}; Grünwald and
  Hoheisel, \protect\hyperlink{ref-grunwald2006hierarchical}{2006}; Linde
  et al., \protect\hyperlink{ref-linde2002population}{2002}). With clonal
  organisms, basic practice has been to clone-censor data to avoid
  downward bias in diversity due to duplicated genotypes that may or may
  not represent different samples (Milgroom,
  \protect\hyperlink{ref-milgroom1996recombination}{1996}). This
  correction should be performed with respect to a population hierarchy to
  accurately reflect the biology of the organism. Traditional data
  structures for population genetic data in most analysis tools allow for
  only one level of hierarchical definition. The investigator thus had to
  provide the data set for analysis at each hierarchical level.
  
  To facilitate handling hierarchical and mutlilocus genotypic metadata,
  \emph{poppr} version 1.1 introduced a new S4 data object called
  ``genclone'', extending \emph{adegenet}'s ``genind'' object (Kamvar and
  Grünwald, unpublished). The genclone object formalized the definitions
  of multilocus genotypes and population hierarchies by adding two slots
  called ``mlg'' and ``hierarchy'' that carried a numeric vector and a
  data frame, respectively. These new slots allow for increased efficiency
  and ease of use by allowing these metadata to travel with the genetic
  data. The hierarchy slot in particular contains a data frame where each
  column represents a separate hierarchical level. This is then used to
  set the population factor of the data by supplying a hierarchical
  formula containing one or more column names of the data frame in the
  hierarchy slot.
  
  The functionality represented by the hierarchy slot has now been
  migrated from the \emph{poppr} to the \emph{adegenet} package version
  2.0 to allow hierarchical analysis in \emph{adegenet}, \emph{poppr}, and
  other dependent packages. The prior \emph{poppr} \texttt{hierarchy} slot
  and methods have now been renamed \texttt{strata} in \emph{adegenet}. A
  short example of the utility of these methods can be seen in the code
  segment under \textbf{Bootstrapping}, above. This migration provides end
  users with a broader ability to analyze data hierarchically in R across
  packages.
  
  \section{Availability}\label{availability}
  
  As of this writing, the \emph{poppr} R package version 2.0 containing
  all of the features described here is located at
  \url{https://github.com/grunwaldlab/poppr/tree/2.0-rc}. It is necessary
  to install \emph{adegenet} 2.0 before installing \emph{poppr}. It can be
  found at \url{https://github.com/thibautjombart/adegenet}. Both of these
  can be installed via the R package \emph{devtools} (Wickham and Chang,
  \protect\hyperlink{ref-wickham2015devtools}{2015}). More information and
  example code can be found in the supplementary materials\footnote{Supplementary
    data available at
    \url{https://github.com/grunwaldlab/supplementary-poppr-2.0}; DOI:
    \href{http://dx.doi.org/10.5281/zenodo.17424}{10.5281/zenodo.17424}}.
  
  \subsection{Requirements}\label{requirements}
  
  \begin{itemize}
  \tightlist
  \item
    R version 3.0 or better
  \item
    A C compiler. For windows, it can be obtained via Rtools
    (\url{http://cran.r-project.org/bin/windows/Rtools/}). On OSX, it can
    be obtained via Xcode. For parallel support, gcc version 4.6 or better
    is needed.
  \end{itemize}
  
  \newpage
  
  \subsection{Installation}\label{installation}
  
  From within R, \emph{poppr} can be installed via:
  
  \begin{Shaded}
  \begin{Highlighting}[]
  \KeywordTok{install.packages}\NormalTok{(}\StringTok{"devtools"}\NormalTok{)}
  \KeywordTok{library}\NormalTok{(}\StringTok{"devtools"}\NormalTok{)}
  \KeywordTok{install_github}\NormalTok{(}\StringTok{"thibautjombart/adegenet"}\NormalTok{)}
  \KeywordTok{install_github}\NormalTok{(}\StringTok{"grunwaldlab/poppr@2.0-rc"}\NormalTok{)}
  \end{Highlighting}
  \end{Shaded}
  
  Several population genetics packages in R are currently going through a
  major upgrade following the 2015 R hackathon on population genetics
  (\url{https://github.com/NESCent/r-popgen-hackathon}) and have not yet
  been updated in CRAN. We will upload \emph{poppr} 2.0 to CRAN once all
  other reverse dependent packages have been updated.
  
  \section{Discussion}\label{discussion}
  
  Given low cost and high throughput of current sequencing technologies we
  are entering a new era of population genetics where large SNP data sets
  with thousands of markers are becoming available for large populations
  in a genome- wide context. This data provides new possibilities and
  challenges for population genetic analyses. We provide novel tools that
  enable analysis of this data in R with a particular emphasis on clonal
  organisms.
  
  Particularly useful is the implementation of \(\bar{r}_d\) in a genomic
  context (Agapow and Burt, \protect\hyperlink{ref-Agapowux5f2001}{2001}).
  Random sampling of loci across the genome can give an expected
  distribution of \(\bar{r}_d\), which is expected to have a mean of zero
  for panmictic populations. This metric is not affected by the number of
  loci sampled, is model free, and has the ability to detect population
  structure. \(\bar{r}_d\) is also implemented for sliding window analyses
  that are useful to detect candidate regions of linkage disequilibrium
  for further analysis.
  
  Clustering multilocus genotypes into multilocus lineages based on
  genetic distances is a non-trivial task given large SNP data sets.
  Moreover, this has not previously been implemented for genomic data for
  clonal populations. Clonal assignment has previously been available in
  the programs \textsc{GenClone} and \textsc{Genodive} for classical
  markers (Arnaud-Hanod et al.,
  \protect\hyperlink{ref-arnaud2007standardizing}{2007}; Meirmans and Van
  Tienderen, \protect\hyperlink{ref-meirmans2004genotype}{2004}). Our
  method with \texttt{mlg.filter} builds upon this idea and allows the
  user to choose between three different approaches for clustering MLGs.
  The choice of clustering algorithm has an impact on the data (Fig.
  \ref{fig:Figure1}, \ref{fig:Figure-2}), where for example a genetic
  distance cutoff of 0.1 would be the difference between 14 multilocus
  lineages (MLLs) and 17 MLLs for nearest neighbor and UPGMA clustering,
  respectively (Fig. \ref{fig:Figure-2}). The option to choose the
  clustering algorithm gives the user the ability to choose what is
  biologically relevant to their populations. While there is not one
  optimal procedure for defining boundaries in clonal lineages, our tool
  provides a means of exploring the potential MLG or MLL boundary space.
  
  Minimum spanning networks are a useful tool to analyze the relationships
  between individuals in a population, because it reduces the complexity
  of a distance matrix to the connections that are strongest. By default,
  these networks are drawn without reticulations, but for clonal organisms
  where many of the connections between samples are equivalent, the
  minimum spanning network appears as a chain and reduces the information
  that can be communicated. This is problematic because the ability to
  detect population structure with one instance of a minimum spanning
  network is limited. Adding reticulation into the minimum spanning
  network thus presents all equivalent connections and allows population
  structure to be more readily detectable. As shown in Fig.
  \ref{fig:Figure-3}, population structure is apparent both visually and
  by graph community detection algorithms such as the infoMAP algorithm
  (Rosvall and Bergstrom, \protect\hyperlink{ref-rosvall2008maps}{2008}).
  Additionally, the current implementation in \emph{poppr} has been
  successfully used in analyses such as reconstruction of the \emph{P.
  ramorum} epidemic in Oregon forests (Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014sudden}{2014}\protect\hyperlink{ref-kamvar2014sudden}{a},
  \protect\hyperlink{ref-kamvar2015spatial}{2015}\protect\hyperlink{ref-kamvar2015spatial}{c}).
  
  \emph{Poppr} 2.0 is open source and available on GitHub. Members of the
  community are invited to contribute by raising issues or pull requests
  on our repository at \url{https://github.com/grunwaldlab/poppr/issues}.
  
  \newpage
  
  \section{Acknowledgements}\label{acknowledgements}
  
  We thank Ignazio Carbone for discussions on the index of association;
  David Cooke, Sanmohan Baby, and Jens Hansen for beta testing; and
  Thibaut Jombart for allowing us to incorporate the \texttt{strata} slot
  and related methods in \emph{adegenet}. We also thank all the members of
  the 2015 R hackathon on population genetics in Durham, NC for their
  advice and input (\url{https://github.com/NESCent/r-popgen-hackathon}).
  This work was supported in part by US Department of Agriculture (USDA)
  Agricultural Research Service Grant 5358-22000-039-00D, USDA National
  Institute of Food and Agriculture Grant 2011-68004-30154, USDA APHIS,
  the USDA-ARS Floriculture Nursery Initiative, and the USDA-Forest
  Service Forest Health Monitoring Program (to NJG).
  
  \chapter{Spatial and Temporal Analysis of Populations of the Sudden Oak
  Death Pathogen in Oregon
  Forests}\label{spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests}
  
  \singlespacing
  
  \begin{center}
  Zhian N. Kamvar, Meredeth M. Larsen, Alan M. Kanaskie, Everett M.
  Hansen, and Niklaus J. Grünwald
  
  
  
  \end{center}\vspace*{\fill}
  
  Journal: \textbf{Phytopathology}
  
  3340 Pilot Knob Rd, St Paul, MN 55121, USA
  
  Published 2015-07, Volume 105, Issue: \textbf{7}, DOI:
  \href{http://dx.doi.org/10.1094/PHYTO-12-14-0350-FI}{10.1094/PHYTO-12-14-0350-FI}
  
  \doublespacing
  \newpage
  
  \section{Abstract}\label{abstract-2}
  
  Sudden oak death caused by the oomycete \emph{Phytophthora ramorum} was
  first discovered in California toward the end of the 20th century and
  subsequently emerged on tanoak forests in Oregon before its first
  detection in 2001 by aerial surveys. The Oregon Department of Forestry
  has since monitored the epidemic and sampled symptomatic tanoak trees
  from 2001 to the present. Populations sampled over this period were
  genotyped using microsatellites and studied to infer the population
  genetic history. To date, only the NA1 clonal lineage is established in
  this region, although three lineages exist on the North American west
  coast. The original introduction into the Joe Hall area eventually
  spread to several regions: mostly north but also east and southwest. A
  new introduction into Hunter Creek appears to correspond to a second
  introduction not clustering with the early introduction. Our data are
  best explained by both introductions originating from nursery
  populations in California or Oregon and resulting from two distinct
  introduction events. Continued vigilance and eradication of nursery
  populations of \emph{P. ramorum} are important to avoid further
  emergence and potential introduction of other clonal lineages.
  
  \section{Introduction}\label{introduction-2}
  
  Sudden oak death (SOD) emerged as a severe epidemic disease on coast
  live oak (\emph{Quercus agrifolia}) and tanoak (\emph{Notholithocarpus
  densiflorus}) in California in the mid 1990s and reemerged shortly
  thereafter on tanoak in Oregon in the early 2000s (Everhart et al.,
  \protect\hyperlink{ref-everhart2014phytophthora}{2014}; Grünwald et al.,
  \protect\hyperlink{ref-grunwald2008phytophthora}{2008}\protect\hyperlink{ref-grunwald2008phytophthora}{a};
  Hansen et al., \protect\hyperlink{ref-hansen2008epidemiology}{2008};
  Rizzo et al., \protect\hyperlink{ref-rizzo2005phytophthora}{2005}). SOD
  is caused by \emph{Phytophthora ramorum} Werres, De Cock \& Man in't
  Veld, and is considered to be one of the top two oomycete pathogens
  based on its scientific and economic importance (Kamoun et al.,
  \protect\hyperlink{ref-kamoun2014top}{2014}; Werres et al.,
  \protect\hyperlink{ref-werres2001phytophthora}{2001}). The Oregon
  epidemic was first detected during aerial surveys in 2001 on tanoak but
  likely derived from initial introductions in the late 1990s. The Oregon
  Department of Forestry has since monitored the epidemic and sampled
  symptomatic tanoaks since 2001 (Hansen et al.,
  \protect\hyperlink{ref-hansen2008epidemiology}{2008}). Strains sampled
  from infected sites in forest or nursery environments have been
  genotyped in several labs using a range of microsatellite loci (Grünwald
  et al., \protect\hyperlink{ref-grunwald2009standardizing}{2009}; Ivors
  et al., \protect\hyperlink{ref-ivors2006microsatellite}{2006}; Prospero
  et al., \protect\hyperlink{ref-prospero2004isolation}{2004},
  \protect\hyperlink{ref-prospero2009migration}{2009},
  \protect\hyperlink{ref-prospero2007population}{2007}).
  
  \emph{P. ramorum} has emerged repeatedly around the world as 4 distinct
  clonal lineages found in North America (lineages NA1, NA2, and EU1) and
  Europe (EU1 and EU2) (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2012emergence}{2012}; Ivors et al.,
  \protect\hyperlink{ref-ivors2006microsatellite}{2006}; Poucke et al.,
  \protect\hyperlink{ref-vanpoucke2012discovery}{2012}). The lineages have
  been named by the continent on which they first appeared, i.e.~North
  America (= NA) or Europe (= EU) and are numbered in order of discovery
  (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2009standardizing}{2009}). The NA1 clonal
  lineage was first discovered in California causing SOD on tanoak and
  coast live oak and is the one currently found in Curry County, Oregon,
  USA (Mascheretti et al.,
  \protect\hyperlink{ref-mascheretti2008reconstruction}{2008}). The EU1
  and NA2 populations were discovered later in nursery environments and
  are currently only found in California, Oregon, Washington and British
  Columbia while the NA1 clone has been shipped with nursery plants from
  the West to the Southern and Southwestern US (Goss et al.,
  \protect\hyperlink{ref-goss2009population}{2009},
  \protect\hyperlink{ref-goss2011phytophthora}{2011}; Grünwald et al.,
  \protect\hyperlink{ref-grunwald2012emergence}{2012}; Ivors et al.,
  \protect\hyperlink{ref-ivors2006microsatellite}{2006}; Mascheretti et
  al., \protect\hyperlink{ref-mascheretti2008reconstruction}{2008};
  Prospero et al., \protect\hyperlink{ref-prospero2009migration}{2009}).
  The EU1 clonal lineage is the one first discovered in Europe, but in
  2007 the new EU2 lineage emerged in Northern Ireland and since migrated
  to Western Scotland (Poucke et al.,
  \protect\hyperlink{ref-vanpoucke2012discovery}{2012}; Werres et al.,
  \protect\hyperlink{ref-werres2001phytophthora}{2001}). EU1 was first
  introduced to Europe and eventually migrated to the Pacific Northwest of
  North America (Goss et al.,
  \protect\hyperlink{ref-goss2011phytophthora}{2011}).
  
  \emph{P. ramorum} populations sampled in Oregon forests to date belong
  exclusively to the NA1 clonal lineage (Hansen et al.,
  \protect\hyperlink{ref-hansen2008epidemiology}{2008}; Prospero et al.,
  \protect\hyperlink{ref-prospero2007population}{2007}). Given that NA2
  and/or EU1 clones have been found in California, Oregon, Washington,
  and/or British Columbia in association with nursery plant movements,
  introduction of NA2 or EU1 from nursery environments to Curry County
  forests is a plausible scenario (Goss et al.,
  \protect\hyperlink{ref-goss2009population}{2009},
  \protect\hyperlink{ref-goss2011phytophthora}{2011}; Grünwald et al.,
  \protect\hyperlink{ref-grunwald2012emergence}{2012}; Prospero et al.,
  \protect\hyperlink{ref-prospero2009migration}{2009},
  \protect\hyperlink{ref-prospero2007population}{2007}). Our present work
  thus monitors populations and potential emergence of novel lineages in
  Oregon forests.
  
  Our main objectives here are to describe the spatial and temporal
  pattern of the populations and clonal dynamic of the SOD pathogen in
  Curry County in southwestern Oregon from 2001 to the present.
  Specifically, we asked (1) if novel lineages have been introduced into
  the forests in Curry County, (2) if multiple introductions occurred, and
  (3) whether introduction might have come from nursery populations. We
  sampled infected tanoaks between 2001-2014 and characterized populations
  using microsatellite analysis.
  
  \section{Materials and Methods}\label{materials-and-methods}
  
  \subsection{Location}\label{location}
  
  The SOD infested areas are located in the Siskiyou Mountains of Curry
  County in south western Oregon near the town of Brookings
  (42.0575\(^{\circ}\) N, 124.2864\(^{\circ}\) W) on the coast (Figure
  \ref{fig:ramorum1}) (Prospero et al.,
  \protect\hyperlink{ref-prospero2007population}{2007}). The Siskiyou
  mountains form part of the Klamath Mountain range (Franklin and Dyrness
  1988). The vegetation in SOD infested areas is a mosaic of different
  vegetation types including mixed-evergreen, redwood (\emph{Sequoia
  sempervirens}) and Douglas-fir (\emph{Pseudotsuga menziesii}) forests
  with tanoak as the dominant SOD host.
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figure_1} 
  
  }
  
  \caption[Spatial distribution of the SOD epidemic and multilocus genotypes of
  \emph{Phytophthora ramorum} in Curry County, Oregon.]{Spatial distribution of the SOD epidemic and multilocus genotypes of
  \emph{Phytophthora ramorum} in Curry County, Oregon. The yellow crosses
  mark tanoak trees found positive for \emph{P. ramorum} during aerial
  surveys. A total of 70 multilocus genotypes have been identified between
  2001-2014 and are marked by color as shown in the legend. The
  abbreviations for regions shown in the map are explained in Table
  \ref{tab:ramorum1}. The inset shows the placement of Curry county (red
  dot) in SW Oregon.}\label{fig:ramorum1}
  \end{figure}
  
  \newpage
  
  \subsection{Sampling}\label{sampling}
  
  Commencing in 2001, 2-4 aerial surveys per year were conducted over the
  tanoak range by the Oregon Department of Forestry and the USDA Forest
  Service in Curry County. The survey detects recently killed tanoaks
  based on the reddish-brown color of foliage (Hansen et al.,
  \protect\hyperlink{ref-hansen2008epidemiology}{2008}). All trees
  identified by aerial surveys were ground checked and geographically
  referenced using a hand-held GPS instrument (Garmin GPS 12XL or 60CX,
  Garmin International, Olathe, KS). Bark or foliage samples were
  collected for determination of \emph{P. ramorum} presence by culturing
  in the field and laboratory. Host plants within the area of the
  delimitation survey, generally 300 feet, were also inspected and sampled
  if they were symptomatic. Maps of distribution were prepared using
  ArcView GIS version 3.3 and ArcMap version 10.2 (Environmental Systems
  Research Institute, Redlands, CA).
  
  \subsection{Isolation, identification and DNA
  extraction}\label{isolation-identification-and-dna-extraction}
  
  \begin{sidewaystable}[ph!]
  \begin{tabular}{@{}lllrc@{}}
  \toprule
  \textbf{Abbreviation} & \textbf{Region name} & \textbf{Year} &
  \textbf{Number of} & \textbf{MLGs detected}\tabularnewline
   & & & \textbf{isolates} & \textbf{(region specific)}\tabularnewline
  \midrule
  JHallCr & Joe Hall Creek & 2001, 2002, 2003, 2004, & 244 & 30 (19)\tabularnewline
   &  & 2005, 2013, 2014 &  & \tabularnewline
  NFChetHigh & North Fork Chetco & 2003, 2012, 2013, 2014 & 114 & 35 (19)\tabularnewline
  Coast & Coastal Region & 2006, 2010, 2011, 2012, & 34 & 12 (7)\tabularnewline
   &  & 2013, 2014 &  & \tabularnewline
  HunterCr & Hunter Creek; Cape Sebastian & 2011 & 66 & 4 (4)\tabularnewline
  Winchuck &              ...             & 2012, 2013 & 35 & 9 (3)\tabularnewline
  ChetcoMain &              ...             & 2013, 2014 & 16 & 7 (1)\tabularnewline
  PistolRSF & Pistol River South Fork & 2013 & 4 & 2 (0)\tabularnewline
  \textbf{Total} & \textbf{-} & \textbf{2001-2014} & \textbf{513} &
  \textbf{70 (53)}\tabularnewline
  \bottomrule
  \end{tabular}
  \caption{Summary of \emph{P. ramorum} isolates sampled in Oregon forests and multilocus genotypes (MLG)observed across regions and years.
  }
  \label{tab:ramorum1}
  \end{sidewaystable}
  
  Isolations were made from symptomatic plant tissue onto selective CARP
  agar (Difco corn meal agar, 10 ppm natamycin, 200pm NA-ampicillin, and
  10 ppm rifampicin) (Prospero et al.,
  \protect\hyperlink{ref-prospero2007population}{2007}). Candidate
  \emph{Phytophthora} cultures were transferred onto corn meal agar with
  30 ppm \(\beta\)-sitosterol. \emph{P. ramorum} identification was
  confirmed by microscopic inspection for presence of characteristic
  chlamydospores and deciduous sporangia (Werres et al.,
  \protect\hyperlink{ref-werres2001phytophthora}{2001}). Genomic DNA was
  extracted using either the FastDNA SPIN kit (MP Biomedicals, LLC;
  116540600) (Goss et al.,
  \protect\hyperlink{ref-goss2009population}{2009}) or the cetyltrimethyl
  ammonium bromide (CTAB )-chloroform-isopropanol method (Winton and
  Hansen, \protect\hyperlink{ref-winton2001molecular}{2001}). Table
  \ref{tab:ramorum1} provides an overview of strains collected by year and
  region following regions as shown in figure \ref{fig:ramorum1}.
  
  \subsection{Genotyping, data validation, and
  harmonization}\label{genotyping-data-validation-and-harmonization}
  
  Five microsatellite loci were utilized in this analysis: PrMS6, Pr9C3,
  PrMS39, PrMS45, and PrMS43 (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2009standardizing}{2009},
  \protect\hyperlink{ref-grunwald2008susceptibility}{2008}\protect\hyperlink{ref-grunwald2008susceptibility}{b};
  Prospero et al., \protect\hyperlink{ref-prospero2004isolation}{2004},
  \protect\hyperlink{ref-prospero2007population}{2007}). Genotyping (see
  specific protocols in supplementary text) of \emph{P. ramorum} strains
  collected 2001-2012 occurred over several years and in several
  laboratories, with different protocols and sequencers. Consequently, a
  concerted effort was made to create a comprehensive dataset with
  identical allele calls. To detect errors, allele calls from all five
  genotyped loci were generated for a subsample of 40 isolates
  representing the most common multilocus genotypes from the culture
  collection, and then compared to data from participating laboratories.
  Three of the five loci, PrMS6, Pr9C3, and PrMS39 had identical allele
  calls between laboratories for the subsampled isolates. The remaining
  two loci, PrMS45 and PrMS43, had allele calls that differed by a single
  bp between laboratories. Data from PrMS45 and PrMS43 were therefore
  corrected to allow consistent comparisons of allele calls. Given the
  varied nature of the genotyping data described above genotyping of
  \emph{P. ramorum} strains consists of two datasets including either 5
  loci (2001-14) or a newly developed, multiplexed method including 14
  loci (samples 2013-14) (Table \ref{tab:ramorum2}). Details on both
  genotyping methods can be found in the supplementary text \ref{text:S1}
  and figure S\ref{fig:ramS1}.
  
  \begin{sidewaystable}[ph!]
  \begin{tabular}{@{}llllll@{}}
  \toprule
  \textbf{SSR Locus} & \textbf{Dye} & \textbf{Product} & \textbf{Primer sequence\textsuperscript{b}} &
  \textbf{Final conc.} & \textbf{Rxn}\tabularnewline
   &  & \textbf{(bp)\textsuperscript{c}} &  &
  \textbf{($\mu$M)} & \tabularnewline
  \midrule
  ILVOPrMS145abc\textsuperscript{f} & 6-FAM & 167-257 &
  \vtop{\hbox{\strut Fwd6FAM-TGGCAGTGTTCTTCAACAGC}\hbox{\strut Rev-\emph{GTTT}ATTCCCGTGAACAGCGTATC}}
  & 0.04 & 8-plex\tabularnewline
  PrMS39\textsuperscript{d} & NED & 130-258 &
  \vtop{\hbox{\strut FwdNED-GCACGGCCAGAGATTGATAG}\hbox{\strut Rev-\emph{GTTT}ATCTGCCGACGTGAAGAAGT}}
  & 0.07 & 8-plex\tabularnewline
  PrMS9C3\textsuperscript{d} & PET & 210-226 &
  \vtop{\hbox{\strut FwdVIC-TCACACGAAGCAGCAACTCT}\hbox{\strut Rev-\emph{GTTT}AGCGGCACTACGGAATACAT}}
  & 0.04 & 8-plex\tabularnewline
  ILVOPrMS79\textsuperscript{af} & 6-FAM & 342-396 &
  \vtop{\hbox{\strut Fwd6FAM-AGGCGGAAAACGTCAGAAC}\hbox{\strut Rev-\emph{GTTT}CTCGAGAGGCTGGAAGTACG}}
  & 0.15 & 8-plex\tabularnewline
  KI18\textsuperscript{e} & VIC & 217-279 &
  \vtop{\hbox{\strut FwdPET-TGCCATCACAACACAAATCC}\hbox{\strut Rev-\emph{GTT}TGTGCTATCTTTCCTGAACGG}}
  & 1.0 & 8-plex\tabularnewline
  KI64\textsuperscript{e} & NED & 342-401 &
  \vtop{\hbox{\strut FwdNED-GCGCTAAGAAAGACACTCCG}\hbox{\strut Rev-\emph{GTTT}CAACATGTAGCCATTGCAGG}}
  & 0.35 & 8-plex\tabularnewline
  PrMS45\textsuperscript{d} & VIC & 138-186 &
  \vtop{\hbox{\strut FwdVIC-CGTGCTGCATCTGGTGTAGT}\hbox{\strut Rev-GAAAGTCCGGATTTGCGTTA}}
  & 0.15 & 8-plex\tabularnewline
  PrMS6\textsuperscript{d} & PET & 165-168 &
  \vtop{\hbox{\strut FwdPET-AATCGATCTCTCGGCTTTGA}\hbox{\strut Rev-TATAGCCCCAGCTGCAACA}}
  & 0.15 & 8-plex\tabularnewline
  ILVOPrMS131\textsuperscript{f} & VIC & 146-414 &
  \vtop{\hbox{\strut FwdVIC-CGGCCGTTTTTGTAAGTTTG}\hbox{\strut Rev-\emph{GTTT}CAGATCAAACCAAAATCTGCTC}}
  & 0.2 & 2-plex\tabularnewline
  KI82ab\textsuperscript{e} & NED & 95-243 &
  \vtop{\hbox{\strut FwdNED-CCACGTCATTGGGTGACTTC}\hbox{\strut Rev-\emph{GTTT}CGTACAAGTCACGACTCCCC}}
  & 0.2 & 2-plex\tabularnewline
  PrMS43\textsuperscript{d} & 6-FAM & 122-493 &
  \vtop{\hbox{\strut Fwd6FAM-AAATATGCAAAAAGGCAGGA}\hbox{\strut Rev-\emph{GTTT}CCGCGTAACCTAGTCTGCTC}}
  & 0.3 & Simplex\tabularnewline
  \bottomrule
  \end{tabular}
  
  \caption[Newly multiplexed protocol for \emph{P. ramorum}
  primer sequences of simple sequence repeat (SSR) loci and final
  concentrations used to determine multilocus genotypes for four clonal
  lineages.]{(Caption on Next Page)}
  \label{tab:ramorum2}
  \end{sidewaystable}
  
  \newpage
  
  \vspace*{\fill}
  
  \begin{table}[ph!]
  \caption[Caption for Table \ref{tab:ramorum2}]{(Caption for Table 
  \ref{tab:ramorum2}) Newly multiplexed protocol for \emph{P. ramorum}
  primer sequences of simple sequence repeat (SSR) loci and final
  concentrations used to determine multilocus genotypes for four clonal
  lineages. PrMS6, Pr9C3, PrMS39, PrMS45, and PrMS43 were utilized in this
  study as they were commonly genotyped across all laboratories.
  \textsuperscript{a} ILVOPrMS79 amplifies three alleles in the NA1 lineage. The first two alleles are fixed and the third is polymorphic.
  \textsuperscript{b} Reverse (Rev) primer includes PIG tail addition except for PrMS45 and PrMS6. Indicated in italic.
  \textsuperscript{c} Product size range is for four lineages (EU1, EU2, NA1, NA2). Only the NA1 lineage has been reported in Curry County,OR forests.
  \textsuperscript{d} Described by Prospero et al. (\protect\hyperlink{ref-prospero2004isolation}{2004}) and/or Prospero et al. (\protect\hyperlink{ref-prospero2007population}{2007}).
  \textsuperscript{e} Described by Ivors et al. (\protect\hyperlink{ref-ivors2006microsatellite}{2006}).
  \textsuperscript{f} Described by Vercauteren et al. (\protect\hyperlink{ref-vercauteren2010clonal}{2010}), and Vercauteren et al.
  (\protect\hyperlink{ref-vercauteren2011identification}{2011}).}
  \label{cap:ramorum2}
  \end{table}\vspace*{\fill}
  
  \newpage
  
  \subsection{Nursery populations}\label{nursery-populations}
  
  To determine if forest populations cluster with different nursery
  populations from Oregon or California, we used previously published data
  from our work to determine relationships among nursery and Curry County
  forest populations (Goss et al.,
  \protect\hyperlink{ref-goss2009population}{2009},
  \protect\hyperlink{ref-goss2011phytophthora}{2011}; Grünwald et al.,
  \protect\hyperlink{ref-grunwald2009standardizing}{2009}; Prospero et
  al., \protect\hyperlink{ref-prospero2009migration}{2009},
  \protect\hyperlink{ref-prospero2007population}{2007}).
  
  \subsection{Data analysis}\label{data-analysis}
  
  All individuals genotyped for this effort belonged to the NA1 clonal
  lineage (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2009standardizing}{2009}). Thus, all
  analyses presented here focused on describing the clonal dynamic using
  model-free approaches that avoid violation of population genetic theory.
  Samples were grouped into different multilocus genotypes (MLGs) defined
  by the unique combination of alleles across all observed loci from the
  consensus five SSR loci genotyped across all years. For identification
  purposes, unique MLGs were then assigned an arbitrary number from 1 to
  the total number of observed MLGs. Population genetic analysis was
  conducted using the computer and statistical language R (R Core Team,
  \protect\hyperlink{ref-R2014}{2014}) using various packages as well as R
  functions written specifically for this project (see github link below).
  Graphs and figures were created using the R packages \emph{ggplot2, ape,
  igraph, ggmap,} and \emph{poppr} (Csardi and Nepusz,
  \protect\hyperlink{ref-csardi2006igraph}{2006}; Kahle and Wickham,
  \protect\hyperlink{ref-khale2013ggmap}{2013}; Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b};
  Paradis et al., \protect\hyperlink{ref-paradis2004ape}{2004}; Wickham,
  \protect\hyperlink{ref-wickham2009ggplot2}{2009}). Within-locus allelic
  diversity was analyzed across and within years and regions using the
  function \texttt{locus\_table()} from the R package \emph{poppr} (Table
  S1) (Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b}).
  To address the temporal and spatial aspects of the data, populations
  were analyzed both by year isolated and watershed region (Table
  \ref{tab:ramorum1}; Fig. \ref{fig:ramorum1}). Watershed regions were
  drawn with ArcMap version 10.2 (Environmental Systems Research
  Institute, Redlands, CA). The regions represent drainages or portions of
  drainages in which infected trees were discovered as the disease
  progressed over time. In most cases, ridgelines dividing drainages
  formed the boundary of a region. These regions were saved as shapefiles
  and imported into R with \emph{rgdal} (Bivand et al.,
  \protect\hyperlink{ref-bivand2014rgdal}{2014}).
  
  Genotypic diversity was analyzed within and across years and
  populations, with the Shannon-Wiener index (\emph{H}) and the Stoddard
  and Taylor's index (\emph{G}), (Shannon,
  \protect\hyperlink{ref-shannon2001mathematical}{2001}; Stoddart and
  Taylor, \protect\hyperlink{ref-stoddart1988genotypic}{1988}). Both
  \emph{G} and \emph{H} measure genotypic diversity, combining richness
  and evenness. If all genotypes are equally abundant, then the value of
  \emph{G} will be the number of MLGs and the value of \emph{H} will be
  the natural log of the number of MLGs. Both \emph{G} and \emph{H} are
  used as they weigh more or less abundant MLGs more heavily, respectively
  (Grünwald et al., \protect\hyperlink{ref-grunwald2003analysis}{2003}).
  Evenness was calculated as \emph{E\textsubscript{5}}, which is an
  estimator of evenness that utilizes both \emph{H} and \emph{G} that
  gives a ratio of the number of abundant genotypes to rare genotypes
  (Grünwald et al., \protect\hyperlink{ref-grunwald2003analysis}{2003};
  Ludwig and Reynolds,
  \protect\hyperlink{ref-ludwig1988statistical}{1988}; Pielou,
  \protect\hyperlink{ref-pielou1975ecological}{1975}). These were
  calculated with the R packages \emph{poppr} and \emph{vegan} (Kamvar et
  al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b};
  Oksanen et al., \protect\hyperlink{ref-oksanen2013vegan}{2013}).
  Confidence intervals were calculated using the R package \emph{boot}
  with 9,999 bootstrap resamplings (Canty and Ripley,
  \protect\hyperlink{ref-canty2015boot}{2015}). Richness, or the expected
  number of MLGs (\emph{eMLG}), was calculated using rarefaction from the
  R packages \emph{poppr} and \emph{vegan} (Heck et al.,
  \protect\hyperlink{ref-heck1975explicit}{1975}; Hurlbert,
  \protect\hyperlink{ref-hurlbert1971nonconcept}{1971}). Some statistics
  (AMOVA, genotypic diversity, index of association, allelic diversity,
  and Nei's distance) were also performed on clone-censored data where
  each MLG was represented once per population hierarchy.
  
  Because the analysis of genotypic diversity, richness and evenness is
  agnostic to specific alleles within MLGs, assessment of genetic
  relatedness between MLGs was performed using the function
  \texttt{bruvo.dist()} using \emph{poppr}, which calculates Bruvo's
  genetic distance, utilizing a stepwise mutation model for microsatellite
  loci (Bruvo et al., \protect\hyperlink{ref-bruvo2004simple}{2004};
  Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b}).
  This distance thus gives a more fine-scale picture of relationships
  between individuals than band-sharing models. These relationships were
  visualized with minimum spanning networks generated using the R packages
  \emph{igraph} and \emph{poppr} (Csardi and Nepusz,
  \protect\hyperlink{ref-csardi2006igraph}{2006}; Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b}).
  
  If the epidemic had a single origin, a correlation between genetic and
  geographic distance would be expected as populations acquire mutations
  over time and clonally diverge regardless of rates of spread. Divergence
  is affected by rates and distance of spread where long-distance
  dispersal or low rates of mutation would lead to less divergence and
  thus lower correlations between genetic and geographic distances. This
  was tested by performing Mantel tests across all hierarchical levels in
  the data set utilizing the function \texttt{mantel.randtest()} in the R
  package \emph{ade4} between Bruvo's distance as described above and
  Euclidean distances between geographic coordinates (Dray and Dufour,
  \protect\hyperlink{ref-dray2007ade4}{2007}; Mantel,
  \protect\hyperlink{ref-mantel1967detection}{1967}). \emph{P}-values were
  calculated using 99,999 bootstrap replicates.
  
  As the eradication efforts destroy the immediate habitat in an infected
  area, one question that we wanted to address was whether or not
  genotypes were clustering to specific regions or if they were evenly
  spread throughout Curry County (Prospero et al.,
  \protect\hyperlink{ref-prospero2007population}{2007}). This was tested
  using three methods: bootstrap analysis of Nei's genetic distance,
  Analysis of MOlecular VAriance (AMOVA), and Discriminant Analysis of
  Principal Components (DAPC) in the R packages \emph{poppr}, \emph{ade4},
  and \emph{adegenet} (Excoffier et al.,
  \protect\hyperlink{ref-excoffier1992analysis}{1992}; Jombart et al.,
  \protect\hyperlink{ref-jombart2010discriminant}{2010}; Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b}).
  The bootstrap analysis utilized 10,000 bootstrap replicates treating
  loci as independent units with the function \texttt{aboot()} in
  \emph{poppr} and was visualized as an unrooted neighbor-joining tree in
  figtree v. 1.4.2 (Figure \ref{fig:ramorum5}). AMOVA utilizes a distance
  matrix between genotypes for which hierarchical partitions are defined
  and attempts to analyze the variation within samples, between samples,
  between subpopulations within populations and finally between
  populations. In this case, we used both the hierarchies of samples
  within years within regions and samples within regions within years.
  DAPC is a multivariate, model-free approach to clustering based on prior
  population information (Jombart et al.,
  \protect\hyperlink{ref-jombart2010discriminant}{2010}). This allows us
  to analyze the population structure by assessing how well samples can be
  reassigned into previously defined populations. Both DAPC and AMOVA were
  run with and without Hunter Creek and Pistol River South Fork due to
  isolated genotypes and small sample size, respectively. For the DAPC
  analysis, these removed populations had their origins predicted from the
  DAPC object using the function predict.dapc in the R package
  \emph{adegenet} (Jombart et al.,
  \protect\hyperlink{ref-jombart2010discriminant}{2010}).
  
  Since DAPC is sensitive to the number of principal components used in
  analysis, the function \texttt{xvalDapc()} from the R package
  \emph{adegenet} was used to select the correct number of principal
  components with 1,000 replicates using a training set of 90\% of the
  data. The number of principal components was chosen based on the
  criteria that it had to produce the highest average percent of
  successful reassignment and lowest root mean squared error (Jombart et
  al., \protect\hyperlink{ref-jombart2010discriminant}{2010}). Significant
  deviations from random population structure was tested in AMOVA
  utilizing the function \texttt{randtest()} from the R package
  \emph{ade4} with 9,999 bootstrap replicates (Dray and Dufour,
  \protect\hyperlink{ref-dray2007ade4}{2007}).
  
  All data and R scripts to reproduce the analyses shown here are
  deposited publicly on github
  (\url{https://github.com/grunwaldlab/Sudden_Oak_Death_in_Oregon_Forests})
  and citable (DOI: 10.5281/zenodo.13007).
  
  \section{Results}\label{results}
  
  \subsection{Demographic pattern and genetic
  diversity}\label{demographic-pattern-and-genetic-diversity}
  
  The epidemic has expanded over time from the initial focus in Joe Hall
  Creek NE of Brookings, Oregon mostly north (first to N Fork Chetco High)
  and northwest (Coast, Pistol River South Fork), but also east (Chetco
  Main and Winchuck) (Fig. \ref{fig:ramorum1}; Table \ref{tab:ramorum1}).
  To date a total of 70 multilocus genotypes have been found in forest
  populations (Table \ref{tab:ramorum1}). MLG 22 is most abundant and the
  only MLG detected across the whole period (although it was not sampled
  in every year) (Fig. \ref{fig:ramorum2}). MLG 59, the second most
  abundant MLG, was only detected in 2011 and has a high frequency due to
  the sampling design applied: all 2011 strains were sampled in one
  concentrated area in the northwestern sampling range geographically
  distant from any other location (Fig. \ref{fig:ramorum2}). Given that
  sampling strategies for some years were not comprehensive, samples from
  some years have to be interpreted with caution (e.g., 2005-6, 2010-11).
  Samples from 2013 and 2014 are sampled from all regions and can be
  considered more representative.
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.4\linewidth]{figure/phytopathology/figure_2} 
  
  }
  
  \caption[Rank distribution of multilocus genotypes (MLGs) of \emph{P. ramorum}
  and recovery per year.]{Rank distribution of multilocus genotypes (MLGs) of \emph{P. ramorum}
  and recovery per year. The vertical axis denotes unique MLGs detected in
  the whole data set with decreasing abundance as indicated by the barplot
  on the right side. The horizontal axis indicates year of sampling. Each
  numbered circle represents the number of observations of each MLG with
  lines connecting genotypes found in multiple years.}\label{fig:ramorum2}
  \end{figure}
  
  \newpage
  
  Allelic and genotype diversity within loci revealed that PrMS43 had, on
  average, the highest number of alleles (n = 18). All other loci had 5 or
  fewer alleles with a moderate to high amount of diversity (Table S2).
  Nevertheless, the genotype accumulation curve showed a slight plateau,
  indicating that we have enough power in our data to describe a
  significant number of MLGs (Fig. S\ref{fig:ramS2}). Genotypic diversity
  (\emph{H} = 2.98, \emph{G} = 8.64), evenness (\emph{E\textsubscript{5}}
  = 0.41), and richness (\emph{eMLG} = 7) were low as expected for a
  clonal population slowly accumulating mutations over space and time
  (Table S3). A pattern of increasing diversity across years (with number
  of MLGs not fewer than 10) was also observed (Table S3). The minimum
  spanning network showed that MLGs 17, 22, and 28 clustered in the center
  of the network and had the highest number of connections to other
  genotypes in the forest populations (Fig. \ref{fig:ramorum3}). Most
  genotypes were connected to their immediate neighbors by a genetic
  distance of 0.05 or the equivalent of one mutational step across 5
  diploid loci.
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figure_3} 
  
  }
  
  \caption[Minimum spanning network based on Bruvo's genetic distance for
  microsatellite markers for \emph{P. ramorum} populations.]{Minimum spanning network based on Bruvo's genetic distance for
  microsatellite markers for \emph{P. ramorum} populations. Nodes
  (circles) represent individual multilocus genotypes. The 10 most
  abundant forest genotypes are labeled with their MLG designation. Node
  colors represent population membership proportional to the pie size.
  Node sizes are relatively scaled to \emph{log\textsubscript{1.75}n,}
  where \emph{n} is the number of samples in the nodes to avoid node
  overlap. Edges (lines) represent minimum genetic distance between
  individuals determined by Prim's algorithm. Nodes that are more closely
  related will have darker and thicker edges whereas nodes more distantly
  related will have lighter and thinner edges or no edge at all.
  Reticulation was introduced by finding exact ties in genetic distance
  after Prim's algorithm was run. Subgroups of \textgreater{}3 MLGs where
  all nodes are no more than one mutational step (d = 0.05) away from its
  neighbors, are highlighted in arbitrary colors.}\label{fig:ramorum3}
  \end{figure}
  
  \newpage
  
  \subsection{Spatial Correlation}\label{spatial-correlation}
  
  A Mantel test revealed significant correlations of genetic distance and
  geographic distance for most samples collected after 2003 (Table
  \ref{tab:ramorum3}). When partitioned by year, this correlation appears
  to increase and become more pronounced with the progression of the
  epidemic. When partitioned by region, those that are closer to the
  origin of the epidemic (Joe Hall Creek and N Fork Chetco River) show
  significant correlation. When the overall mantel test was run without
  Hunter Creek, the correlation coefficient was reduced (0.175), but was
  still significant (\emph{p} = 0.0001).
  
  \begin{sidewaystable}[ph!]
  \centering
  \begin{tabular}{@{}lllllllllllll@{}}
  \toprule
  & \textbf{2001} & \textbf{2002} & \textbf{2003} & \textbf{2004} &
  \textbf{2005} & \textbf{2006} & \textbf{2010} & \textbf{2011} &
  \textbf{2012} & \textbf{2013} & \textbf{2014} &
  \textbf{Pooled}\tabularnewline
  \midrule
  JHallCr & 0.06 & 0.24 & 0.14* & 0.28* & NaN & - & - & - & - & 0.18\~{}
  & NaN & 0.14*\tabularnewline
  NFChetHigh & - & - & NaN & - & - & - & - & - & 0.68 & 0.41* & -0.23 &
  0.35*\tabularnewline
  Coast & - & - & - & - & - & NaN & NaN & NaN & NaN & 0.55\^{} & -0.25 &
  0.13\tabularnewline
  HunterCr & - & - & - & - & - & - & - & 0.06 & - & - & - &
  0.06\tabularnewline
  Winchuck & - & - & - & - & - & - & - & - & 0.41\~{} & 0.03 & - &
  0.11\tabularnewline
  ChetcoMain & - & - & - & - & - & - & - & - & - & 0.53 & NaN &
  0.63*\tabularnewline
  PistolRSF & - & - & - & - & - & - & - & - & - & 0.94 & - &
  0.94\tabularnewline
  Pooled & 0.06 & 0.24 & 0.13* & 0.28* & NaN & NaN & NaN & 0.87* &
  0.59* & 0.15* & 0.14\^{} & \textbf{0.52*}\tabularnewline
  \bottomrule
  \end{tabular}
  \caption[Table of correlation coefficients generated across
  forest regions and years of \emph{P. ramorum} isolates using the Mantel
  test.]{Table of correlation coefficients generated across
  forest regions and years of \emph{P. ramorum} isolates using the Mantel
  test. Euclidean distances were calculated from geographic coordinates
  while genetic distance was based on Bruvo's distance. Significance of
  values are based on 99,999 Monte-Carlo permutations and marked as
  follows: \^{} $\leq$ 0.05, \~{} $\leq$ 0.01, * $\leq$ 0.001, - = no data, NaN =
  insufficient data for analysis.}
  \label{tab:ramorum3}
  \end{sidewaystable}
  
  \subsection{Population
  differentiation}\label{population-differentiation}
  
  Cluster analysis of populations with respect to year using Nei's genetic
  distance showed no significant (\textgreater{}70\%) bootstrap support
  for any clades, but does show that these tend to cluster by region as
  opposed to year (Fig. S\ref{fig:ramS3}). AMOVA analysis revealed
  significant population structure between regions on both clone-corrected
  (with respect to hierarchy) and uncorrected data sets (Table
  \ref{tab:ramtab4}). Significant structure was only found between years
  within regions on the uncorrected data set. Both patterns were observed
  without Hunter Creek and Pistol River South Fork isolates. DAPC
  clustering showed that the first discriminant component separated Hunter
  Creek from all other regions and the second discriminant component shows
  a gradient from Joe Hall Creek to the coast (Fig. \ref{fig:ramfig4}).
  This distinction was reflected in the percent of correct posterior
  assignment of isolates to their original populations. Over the whole
  data set there was an 81.5\% assignment-success rate. Hunter Creek
  received 100\% successful reassignment. Joe Hall Creek, Winchuck, and
  Coast all had \textgreater{}85\% successful reassignment whereas Chetco
  Main, North Fork of the Chetco, and Pistol River South Fork all had
  \textless{}69\% successful reassignment (Fig. S\ref{fig:ramS4}). The
  isolation of the Hunter Creek isolates in the DAPC analysis was found to
  be mainly driven by allele 493 at locus PrMS43 (Fig. S\ref{fig:ramS5}).
  The only other population to share this allele was Joe Hall Creek where
  it was present in a total of 4 isolates, and only isolates found in the
  coastal region or North Fork Chetco contained the allele 489, which is
  one mutational step away in a stepwise mutation model of a
  tetranucleotide repeat locus. When DAPC was run without Hunter Creek and
  Pistol River South Fork data, percent successful reassignment for all
  regions did not change significantly. Prediction of sources for the
  Hunter Creek data revealed that over 98\% of the genotypes were assigned
  to the Coast with a 99\% probability.
  
  \begin{sidewaystable}[ph!]
  \centering
  \begin{tabular}{lccccc}
   \textbf{Heirarchy} & \textbf{df} & \textbf{Sum of squares} & \textbf{Variation (\%)} & \textbf{\textit{P}} & \textbf{$\phi$ statistic} \\ 
    \midrule
  \textbf{Region by year} &  &  &  &  &  \\ 
    Between region & 10 (10) & 160 (21) & 11.6 (3) & 0.366 (0.175) & 0.448 (0.101) \\ 
    Between year within region & 12 (12) & 59.5 (19.1) & 33.3 (7.07) & 1e-04 (2e-04) & 0.376 (0.0729) \\ 
    Within year within region & 490 (129) & 281 (141) & 55.2 (89.9) & 1e-04 (1e-04) & 0.116 (0.03) \\ 
    \textbf{Year by region} &  &  &  &  &  \\ 
    Between year & 6 (6) & 197 (23) & 45 (12.3) & 1e-04 (1e-04) & 0.496 (0.12) \\ 
    Between region within year & 16 (16) & 22.5 (17.2) & 4.56 (-0.283) & 1e-04 (0.446) & 0.0829 (-0.00323) \\ 
    Within region within year & 490 (129) & 281 (141) & 50.4 (88) & 1e-04 (1e-04) & 0.45 (0.123) \\ 
     \bottomrule
  \end{tabular}
  \caption[AMOVA table generated comparing \textit{P. ramorum} isolates for
  two different hierarchies]{AMOVA table generated comparing \emph{P. ramorum} isolates for two
  different hierarchies, year within region and region within year,
  respectively. Results are rounded to three significant figures. Clone
  corrected results are provided in parentheses. P values are based on
  9,999 permutations.} 
  \label{tab:ramtab4}
  \end{sidewaystable}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figure_4} 
  
  }
  
  \caption[Scatterplot from DAPC of the first two principal components
  discriminating \emph{P. ramorum} populations by regions.]{Scatterplot from DAPC of the first two principal components
  discriminating \emph{P. ramorum} populations by regions. Points
  represent individual observations. Colors and lines represent population
  membership. Inertia ellipses represent an analog of a 67\% confidence
  interval based on a bivariate normal distribution.}\label{fig:ramfig4}
  \end{figure}
  
  \newpage
  
  \subsection{Clustering of forest with nursery
  populations}\label{clustering-of-forest-with-nursery-populations}
  
  We used previously published data to determine if nursery populations in
  California or Oregon could have been source populations for the Oregon
  forest epidemic (Goss et al.,
  \protect\hyperlink{ref-goss2009population}{2009},
  \protect\hyperlink{ref-goss2011phytophthora}{2011}; Grünwald et al.,
  \protect\hyperlink{ref-grunwald2009standardizing}{2009}; Prospero et
  al., \protect\hyperlink{ref-prospero2009migration}{2009},
  \protect\hyperlink{ref-prospero2007population}{2007}). Nursery data
  included 40 MLGs across 216 samples of NA1 clones. Of these 40, 12 MLGs
  matched the forest sample and 28 MLGs were unique to the nurseries. The
  only region that did not contain genotypes that matched those found in
  nurseries was Pistol River South Fork. When considering those 12
  genotypes that were present in both data sets, with the exception of Joe
  Hall Creek, all genotypes were first isolated from nurseries before
  discovery in the forest. At the most variable locus, PrMS43, both
  nursery populations had the allele 281 at frequencies of 4.5\% and 4.9\%
  for CA and OR, respectively. This allele was not observed in the forest
  population. Both populations contained allele 489 at \textgreater{}10\%
  frequency and the CA nursery population contained allele 493 at a
  frequency of 1.4\%.
  
  When nursery genotypes were added to the minimum spanning network, MLGs
  found at Hunter Creek, previously isolated in the network, connected by
  only a single MLG from the coast, gained more connections to nursery
  MLGs. Clustering with Nei's distance revealed the Nursery isolates from
  CA consistently clustering closest with Hunter Creek isolates in both
  clone-corrected and uncorrected data sets (Fig. \ref{fig:ramorum5}).
  DAPC clustering revealed a decrease in overall assignment-success rate
  at 78\%. The nursery isolates received 74\% and 83\% assignment success
  for CA and OR nurseries, respectively.
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figure_5} 
  
  }
  
  \caption[Unrooted, neighbor-joining tree with 10,000 bootstrap replicates of
  Nei's genetic distance for \emph{P. ramorum} populations defined by
  region.]{Unrooted, neighbor-joining tree with 10,000 bootstrap replicates of
  Nei's genetic distance for \emph{P. ramorum} populations defined by
  region. Tip labels are colored by region. Branches with bootstrap values
  greater than 50\% are shown in blue. Nursery populations are shown as
  originating from California (CA) or Oregon (OR).}\label{fig:ramorum5}
  \end{figure}
  
  \newpage
  
  The assignment successes for the regions before or after inclusion of
  nursery data changed less than 5\% for all regions except for the coast,
  which saw a decrease of 17.6\% when nursery populations were included
  (Fig. S\ref{fig:ramS4}). Prediction of sources for nursery genotypes
  against the forest data revealed that 48\% of these nursery isolates
  were predicted to share membership with the coast at \(\geq\) 95\%
  probability (Fig. S\ref{fig:ramS6}, S\ref{fig:ramS7}). A total of 3.2\%
  of the isolates were predicted to share membership with Hunter Creek at
  \(\geq\) 99.9\% membership probability. Furthermore, 21.75\% of the
  nursery data could not be assigned to any of the forest populations at
  \textgreater{}60\% probability.
  
  Since 3.2\% of the nursery isolates had a very strong signal for Hunter
  Creek, we predicted sources for Hunter Creek isolates when considering
  nursery isolates. This approach determines if Hunter Creek isolates
  cluster more readily with nursery or coast populations. Indeed, 92\% of
  the Hunter Creek isolates were predicted to share membership with
  California nurseries at \(\geq\) 99\% membership probability (Fig.
  S\ref{fig:ramS8}). No Hunter Creek isolate was predicted to share
  membership with a forest population at \textgreater{}0.45\% membership
  probability.
  
  \section{Discussion}\label{discussion-1}
  
  To date populations monitored from 2001-14 show presence of only the NA1
  clonal lineage observed previously (Prospero et al.,
  \protect\hyperlink{ref-prospero2009migration}{2009},
  \protect\hyperlink{ref-prospero2007population}{2007}). The fact that
  individuals belonging to the EU1 and NA2 clonal lineages have not been
  found in Oregon forests, despite their presence on the west coast from
  British Columbia to California is welcome news (Goss et al.,
  \protect\hyperlink{ref-goss2009population}{2009},
  \protect\hyperlink{ref-goss2011phytophthora}{2011}; Grünwald et al.,
  \protect\hyperlink{ref-grunwald2012emergence}{2012}). The lack of EU1 or
  NA2 isolates provides evidence that monitoring for \emph{P. ramorum} in
  nurseries by federal and state agencies is helping avoid emergence of
  new clones in Oregon's Forests.
  
  Our analysis provides support for a most parsimonious scenario of two
  introductions into Curry county from nurseries: one initial introduction
  into Curry County sometime before detection of the first infected
  tanoaks in 2001 from California (or less likely Oregon) nurseries
  followed by a second introduction into the Hunter Creek area again from
  nurseries. The relative position of the nursery populations in the
  minimum spanning network and DAPC scatter plot (Fig. \ref{fig:ramorum3},
  \ref{fig:ramfig4}) suggest that the introductions from nurseries were
  rare, though more even sampling and migration models could disprove this
  hypothesis. Since 2001, the epidemic has spread clonally throughout
  Southwestern Curry County mostly north, but also west, towards the
  coast, and southeast. This clonal spread of the pathogen from the Joe
  Hall area is supported partially by Mantel tests showing significant
  levels of isolation by distance in years following 2002 (Table
  \ref{tab:ramorum3}) along with significant AMOVA results across regions
  (Table \ref{tab:ramtab4}). The populations sampled in 2011 in Hunter
  Creek (Cape Sebastian) appear to have originated from a new source and
  cluster into a distinct group based on DAPC (Fig. \ref{fig:ramfig4}).
  Based on the minimum spanning network, this population would appear
  isolated in the epidemic if it were not for MLG 32, which is connected
  with MLG 33 (found on the coast in 2010) by one mutational step at locus
  PrMS43. This, in turn, is connected with the other MLGs from Hunter
  Creek by one mutational step at locus PrMS39. When considering
  clustering via Bruvo's distance in combination with data from nursery
  populations, however, these genotypes from Hunter Creek appear to be
  more similar to California nursery populations (Fig. \ref{fig:ramorum3})
  than Oregon forest populations. Predictions based on DAPC place samples
  from Hunter Creek as coming from California nurseries (Fig.
  S\ref{fig:ramS8}). This, in combination with the observation that purely
  forest genotypes (i.e.~those only found in the forest) are connected to
  Hunter Creek genotypes through nursery genotypes, indicates a possible
  contribution from nursery populations to the epidemic. This is supported
  by the observation that all population level clustering, with and
  without clone correction, places the Hunter Creek isolates adjacent to
  the nursery isolates from CA (Fig. \ref{fig:ramfig4},
  \ref{fig:ramorum5}). This appears to be driven by the high frequency of
  allele 246 at locus PrMS39, which interestingly appears to segregate in
  an east to west fashion and is increasing in frequency over time (Fig.
  S\ref{fig:ramS9}, S\ref{fig:ramS10}). This, along with the results from
  the DAPC clustering and subsequent prediction (Fig. S\ref{fig:ramS6},
  S\ref{fig:ramS7}) provide weak support for a potential third
  introduction into the coastal region from nurseries sometime after the
  Hunter Creek introduction event.
  
  An interesting aspect is the observation that there appeared to be more
  than one cluster of genotypes introduced into the Joe Hall area during
  the early stages of the epidemic. The two dominant clusters that
  appeared were the ones that contained MLG 22 and MLG 68. The former has
  been found in the most recent sampling year, whereas the latter has not
  been observed since 2005 or beyond the Joe Hall area. This latter group
  was also the most distantly related group overall, more distant than
  some nursery genotypes. While it is clear that the eradication effort
  has not been entirely successful, there is some evidence that it is
  having an effect as a major genotype cluster has effectively been
  eradicated, although disappearance of MLGs could also be explained by
  being less fit than lineages dominating now.
  
  The Curry County epidemic is in many ways different from the epidemic in
  California. When introduced into California in the mid 1990's, the
  causal agent of sudden oak death was unknown and thus gave it time to
  clonally expand and diversify as management strategies in natural forest
  systems were limited (Rizzo et al.,
  \protect\hyperlink{ref-rizzo2002phytophthora}{2002}). With the foresight
  of the epidemic in central California, the ODF was able to implement a
  quarantine effort against the import of hosts as soon as the causal
  agent was known (A Kanaskie, pers. comm.). This quarantine along with
  aggressive eradication efforts have affected the spread of \emph{P.
  ramorum} (Mascheretti et al.,
  \protect\hyperlink{ref-mascheretti2008reconstruction}{2008}). Drawing
  conclusions from previous population studies in California and applying
  them to the Oregon epidemic should be undertaken with great care given
  the drastically differing management scenarios (Mascheretti et al.,
  \protect\hyperlink{ref-mascheretti2009genetic}{2009},
  \protect\hyperlink{ref-mascheretti2008reconstruction}{2008}).
  
  Our work has some inherent drawbacks. Given the cost of aerial surveys
  and subsequent ground crew work, and the fact that trees are eradicated
  once found, populations are not hierarchically sampled across all years.
  The destructive nature of the management approach means that it was not
  possible to conduct controlled ecological experiments focusing on
  effects of climate and rainfall on the spread of disease as was possible
  in California trials (Eyre et al.,
  \protect\hyperlink{ref-eyre2013poulation}{2013}). In addition, most of
  our work only used 5 microsatellite loci for genotyping. Ideally, more
  loci should have been used as was done in other studies (Croucher et
  al., \protect\hyperlink{ref-croucher2013combining}{2013}). Although only
  5 loci were used, clear patterns of population dynamics in space and
  time emerged and the MLG accumulation curve supported the fact that loci
  are informative. Finally, the populations genotyped here are clonal and
  belong to the NA1 clonal lineage. Thus, much of the analytical power
  provided by population genetic theory does not apply given that basic
  assumptions would be violated (Grünwald and Goss,
  \protect\hyperlink{ref-grunwald2011evolution}{2011}). Our work uses
  appropriate methods to infer patterns that are model free, yet
  informative such as spatial clustering. Thus, we believe that this work
  provides novel and important insights into the \emph{P. ramorum}
  population biology in the Siskiyou forest. Our data indicates that there
  might have been at least two introductions into Oregon forests from
  nurseries. The nature of the data does not allow inference of
  directional migrations given the uneven sampling strategy and moderate
  number of loci used across all years. We are currently exploring
  genotyping-by-sequencing (GBS) as a method that could provide further
  detail on how these populations evolved over space and time (Elshire et
  al., \protect\hyperlink{ref-elshire2011robust}{2011}). GBS can provide
  richer detail by providing codominant SNP data across several thousand
  loci sampling the whole genome.
  
  \section{Acknowledgements}\label{acknowledgements-1}
  
  This work was supported in part by US Department of Agriculture (USDA)
  Agricultural Research Service Grant 5358-22000-039-00D, USDA APHIS, the
  USDA-ARS Floriculture Nursery Initiative, the Oregon Department of
  Agriculture/Oregon Association of Nurseries (ODA-OAN) and the
  USDA-Forest Service Forest Health Monitoring Program.
  
  \section{Supplementary Material}\label{supplementary-material}
  
  \subsection{Supplementary Text}\label{text:S1}
  
  For the years up to and including 2012, the multilocus genotype (MLG) of
  each \emph{P. ramorum} strain was determined based on microsatellite
  analysis of five loci, PrMS6, Pr9C3, PrMS39, PrMS45 and PrMS43, using
  previously published protocols (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2009standardizing}{2009},
  \protect\hyperlink{ref-grunwald2008susceptibility}{2008}\protect\hyperlink{ref-grunwald2008susceptibility}{b};
  Prospero et al., \protect\hyperlink{ref-prospero2004isolation}{2004},
  \protect\hyperlink{ref-prospero2007population}{2007}). Multilocus
  genotyping of \emph{P. ramorum} strains collected in 2013 and 2014
  included an extra nine loci, KI18, KI64, KI82a, KI82b, ILVOPrMS79,
  ILVOPrMS131, ILVOPrMS145a, ILVOPrMS145b, ILVOPrMS145c which are
  amplified by an additional six primer pairs (Ivors et al.,
  \protect\hyperlink{ref-ivors2006microsatellite}{2006}; Vercauteren et
  al., \protect\hyperlink{ref-vercauteren2010clonal}{2010},
  \protect\hyperlink{ref-vercauteren2011identification}{2011}). The locus
  ILVOPrMS79, amplifies up to three alleles, however two separate loci
  have yet to be described (Vercauteren et al.,
  \protect\hyperlink{ref-vercauteren2011identification}{2011}). The
  addition of nine loci to the genotyping assay coincided with the
  discovery by the Oregon Department of Agriculture of an EU1 \emph{P.
  ramorum} isolate in a Curry County nursery in 2012. Preceding 2012, only
  NA1 isolates had been found in Curry County. Because different loci are
  polymorphic for different clonal lineages, the entire panel of 14 loci
  was necessary to adequately describe the \emph{P. ramorum} population in
  the event that multiple lineages were discovered in the forest.
  
  Methods for genotyping the 2013 and 2014 \emph{P. ramorum} strains with
  all 14 loci use new multiplex protocol. Previously published primers
  were modified by the addition of a 5' PIG tail ``GTTT'' to reverse
  primers in an effort to reduced stutter peaks and hence to better
  facilitate allele scoring (Table \ref{tab:ramorum2}) (Brownstein et al.,
  \protect\hyperlink{ref-brownstein1996modulation}{1996}). In two cases
  (PrMS45, PrMS6), a PIG tail was not added to reverse primers to simplify
  scoring of overlapping alleles. Also, where a T residue was already
  present at the 5' end of a reverse primer, as in the case of KI18, a
  ``GTT'' was added instead of ``GTTT''. Forward primers were assigned
  fluorescent labels, 6-FAM, NED, VIC, or PET, to facilitate separation of
  overlapping markers (Table \ref{tab:ramorum2}). Primer concentrations
  were determined by visual inspection of electropherograms (Table
  \ref{tab:ramorum2}).
  
  Amplification of all 14 loci was separated into three reactions (8-plex,
  2-plex and simplex). The simplex reaction amplified the PrMS43 locus
  using methods described earlier (Grünwald et al.,
  \protect\hyperlink{ref-grunwald2009standardizing}{2009}; Prospero et
  al., \protect\hyperlink{ref-prospero2007population}{2007}). The 8-plex
  and 2-plex amplified the remaining loci and were performed under
  identical conditions with the exception of primers and primer
  concentrations (Table \ref{tab:ramorum2}). For the multiplex reactions,
  the QIAGEN Type-it Mutation Detect PCR Kit (QIAGEN, 206343, Valencia,
  CA) was used. Multiplex PCR reactions were performed in 5\(\mu\)l
  volumes with 10ng template DNA and1X final buffer concentration.
  Amplifications were run on a Veriti thermal cycler (Life Technologies,
  Grand Island, NY) with an initial denaturation at 95 \(^{\circ}\)C for 5
  min, followed by 33 cycles of 95 \(^{\circ}\)C for 30 s, 60
  \(^{\circ}\)C for 90 s, and 72 \(^{\circ}\)C for 20 s, and a final
  extension at 60 \(^{\circ}\)C for 30 min. Genotyping prior to 2012
  included three reference DNA lineages (EU1, NA1, and NA2). After 2012, a
  fourth lineage (EU2) was added as a reference (Poucke et al.,
  \protect\hyperlink{ref-vanpoucke2012discovery}{2012}).
  
  Electrophoresis and visualization of all microsatellites were performed
  on ABI3100, ABI3100 Avant, or ABI3130 genetic analyzers (Applied
  Biosystems). For evaluation of the loci, genotyped prior to 2013, the
  PCR products were diluted 10 times in ultrapure H\textsubscript{2}0 and
  1.5 \(\mu\)l of diluted product was added to both 8.5 \(\mu\)l of
  Hi-Di\textsuperscript{TM} Formamide (Applied Biosystems, 4311320) and
  0.25 \(\mu\)l of GeneScan\textsuperscript{TM} 500
  LIZ\textsuperscript{TM} size standard (Applied Biosystems, 4322682). The
  simplex reaction (PrMS43) was also diluted 10 times while the 8-plex and
  2-plex products were diluted 75 times. After dilution, 2.5 \(\mu\)l of
  the 8-plex, 2-plex and simplex products were added to 7.5 \(\mu\)l of
  Hi-Di\textsuperscript{TM} Formamide containing
  GeneScan\textsuperscript{TM} 500 LIZ\textsuperscript{TM} size standard
  at a ratio of 6 \(\mu\)l size standard to 1 ml Hi-Di\textsuperscript{TM}
  Formamide. Allele sizing was determined using
  GeneMapper\textregistered{} v3.7 and v5.0 software (Applied Biosystems).
  
  \subsection{Supplementary Figures}\label{supplementary-figures}
  
  \begin{figure}
  
  {\centering \includegraphics[width=1\linewidth]{figure/phytopathology/figureS1} 
  
  }
  
  \caption[Diagram of DNA extraction, genotyping, and sequencing protocols 
  utilized by two labs from 2001 to 2014.]{Diagram of DNA extraction, genotyping, and sequencing protocols 
  utilized by two labs from 2001 to 2014. See supplementary text for details.}\label{fig:ramS1}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS2} 
  
  }
  
  \caption[Genotype accumulation curve for OR forest \emph{P. ramorum} isolates.]{Genotype accumulation curve for OR forest \emph{P. ramorum} isolates.
  The vertical axis denotes the number of observed MLGs, from 0 to the
  observed number of MLG in the forest populations, for a number of loci,
  indicated on the horizontal axis, randomly sampled without replacement.
  Each boxplot contains 1,000 random samples representing different
  possible combinations of \emph{n} loci. The horizontal red dashed line
  represents 90\% of MLG resolution.}\label{fig:ramS2}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS3} 
  
  }
  
  \caption[Neighbor joining tree based on Nei's distance of the forest \emph{P.
  ramorum} isolates by region with respect to year.]{Neighbor joining tree based on Nei's distance of the forest \emph{P.
  ramorum} isolates by region with respect to year. Bootstrap values
  \textgreater{} 50\% of 10,000 replicates are shown in blue.}\label{fig:ramS3}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS4} 
  
  }
  
  \caption[Fractions of posterior population assignments from DAPC clustering of
  \emph{P. ramorum} isolates from forest populations.]{Fractions of posterior population assignments from DAPC clustering of
  \emph{P. ramorum} isolates from forest populations. The horizontal axis
  represents the fraction of samples whose posterior group membership
  matched their prior group membership on the vertical axis. Shape
  indicates presence or absence of nursery populations in the DAPC.}\label{fig:ramS4}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS5} 
  
  }
  
  \caption[Loading plot from DAPC of \emph{P. ramorum from} forest populations
  showing the contribution of alleles to the first DAPC eigenvalue
  separating Hunter Creek isolates from all other regions.]{Loading plot from DAPC of \emph{P. ramorum from} forest populations
  showing the contribution of alleles to the first DAPC eigenvalue
  separating Hunter Creek isolates from all other regions.}\label{fig:ramS5}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS6} 
  
  }
  
  \caption[Prediction of nursery genotypes of \emph{P. ramorum} into forest
  watershed regions.]{Prediction of nursery genotypes of \emph{P. ramorum} into forest
  watershed regions. The horizontal axis indicates the fraction of nursery
  genotypes to be predicted to be similar to the populations on the
  vertical axis with a 95, 99, and 99.9\% probability as indicated by the
  size of the points.}\label{fig:ramS6}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS7} 
  
  }
  
  \caption[Graphical representation of prediction of nursery isolates of \emph{P.
  ramorum} into forest watershed regions.]{Graphical representation of prediction of nursery isolates of \emph{P.
  ramorum} into forest watershed regions. Each column represents a
  different isolate. Colors within the columns represent membership
  probabilities from forest populations.}\label{fig:ramS7}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS8} 
  
  }
  
  \caption[Graphical representation of predicted membership of \emph{P. ramorum}
  isolates from Hunter Creek and Pistol River South Fork in forest and
  nursery populations.]{Graphical representation of predicted membership of \emph{P. ramorum}
  isolates from Hunter Creek and Pistol River South Fork in forest and
  nursery populations. Each column represents a different isolate. Colors
  within the columns represent membership probabilities from the
  populations indicated in the legend.}\label{fig:ramS8}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS9} 
  
  }
  
  \caption[Allele frequencies of locus PrMS39 of \emph{P. ramorum} across years of
  the forest populations.]{Allele frequencies of locus PrMS39 of \emph{P. ramorum} across years of
  the forest populations. Years 2005 through 2011 have been omitted due to
  small sample sizes and outlier genotypes.}\label{fig:ramS9}
  \end{figure}
  
  \begin{figure}
  
  {\centering \includegraphics[width=0.8\linewidth]{figure/phytopathology/figureS10} 
  
  }
  
  \caption[Map of the infected area in Curry county showing the \emph{P. ramorum}
  genotypes at locus PrMS39.]{Map of the infected area in Curry county showing the \emph{P. ramorum}
  genotypes at locus PrMS39. Each colored circle represents a different
  forest isolate while each yellow cross represents a sampled tree. Yellow
  borders denote different regions.}\label{fig:ramS10}
  \end{figure}
  
  \chapter{\texorpdfstring{{[}Tentative Title{]} Population Dynamics of
  the Plant Pathogen \emph{Phytophthora syringae} in Oregon
  Nurseries}{{[}Tentative Title{]} Population Dynamics of the Plant Pathogen Phytophthora syringae in Oregon Nurseries}}\label{tentative-title-population-dynamics-of-the-plant-pathogen-phytophthora-syringae-in-oregon-nurseries}
  
  \section{Abstract}\label{abstract-3}
  
  \section{Introduction}\label{introduction-3}
  
  \emph{Phytophthora syringae} is the most important species affecting
  ornamentals produced in the Pacific Northwest. Recent nursery sampling
  efforts, aimed at characterizing the diversity of \emph{Phytophthoras}
  within Oregon nurseries, have revealed the species \emph{P. syringae} to
  be among the most abundant taxa found in the nurseries surveyed (Parke
  et al., \protect\hyperlink{ref-parke2014phytophthora}{2014}). \emph{P.
  syringae} is adapted to cold weather and grows best in the cool, wet
  fall, winter and spring and is least active in summer (Erwin et al.,
  \protect\hyperlink{ref-erwin1996phytophthora}{1996}). Like \emph{P.
  ramorum}, it has a wide host range including \emph{Rhododendron},
  \emph{Camellia}, \emph{Malus}, and many other taxa. It has the
  capability for outcrossing, self-fertilizing, and reproducing clonally.
  This pathogen has been found globally since 1881 and is problematic on
  woody ornamentals such as crabapple (\emph{Malus spp.}), as it causes
  unsightly cankers that make the plant unsellable (Erwin et al.,
  \protect\hyperlink{ref-erwin1996phytophthora}{1996}). While the ecology
  of this pathogen has been studied to some degree, very little is known
  about the demographic history and population structure on a local and
  global scale.
  
  \chapter{{[}Tentative Title{]} The Effect of Population Dynamics, Sample
  Size, and Marker Choice on the Index of
  Association}\label{tentative-title-the-effect-of-population-dynamics-sample-size-and-marker-choice-on-the-index-of-association}
  
  \singlespacing
  
  \begin{center}
  
  
  Zhian N. Kamvar and Niklaus J. Grünwald
  
  
  
  \end{center}\vspace*{\fill}
  
  Target Journal: \textbf{Molecular Ecology}
  
  \doublespacing
  \newpage
  
  \section{Abstract}\label{abstract-4}
  
  TBD\ldots{}
  
  \section{Introduction}\label{introduction-4}
  
  Population genetics largely is based off of the neutral assumption that
  populations exist in Hardy-Weinberg Equilibrium (HWE) in that they are
  infinitely large, with no migration, no mutation, and randomly mating,
  where alleles across loci are I.I.D. This neutral model works well to
  describe populations that reproduce exclusively via sexual reproduction,
  but it becomes less useful when considering populations that undergo
  some form of clonal reproduction since the alleles for different samples
  are no longer I.I.D. This confounds the ability to accurately assess
  population demographics including effective population size (Orive,
  \protect\hyperlink{ref-orive1993effective}{1993}). Many clonal organisms
  are also pathogenic, and knowing basic population dynamics of these
  pathogens is important for management strategies (de Meeûs et al.,
  \protect\hyperlink{ref-de2006molecular}{2006}; Milgroom,
  \protect\hyperlink{ref-milgroom1996recombination}{1996}; Milgroom and
  Fry, \protect\hyperlink{ref-milgroom1997contributions}{1997};
  Nieuwenhuis and James,
  \protect\hyperlink{ref-nieuwenhuis2016frequency}{2016}; Smith et al.,
  \protect\hyperlink{ref-smith1993how}{1993}).
  
  The theory of population genetics for clonal organisms has been deeply
  investigated over time (Arnaud-Hanod et al.,
  \protect\hyperlink{ref-arnaud2007standardizing}{2007}; Halkett et al.,
  \protect\hyperlink{ref-halkett2005tackling}{2005}; Orive,
  \protect\hyperlink{ref-orive1993effective}{1993}). In particular, there
  has been a re-occurring question of how much sex can be detected in a
  population of organisms with (partially) clonal reproduction (Ali et
  al., \protect\hyperlink{ref-ali2016cloncase}{2016}; Balloux et al.,
  \protect\hyperlink{ref-balloux2003population}{2003}; de Meeûs and
  Balloux, \protect\hyperlink{ref-de2004clonal}{2004}; Nieuwenhuis and
  James, \protect\hyperlink{ref-nieuwenhuis2016frequency}{2016}; Smith et
  al., \protect\hyperlink{ref-smith1993how}{1993}). For populations with
  well-defined sexual and clonal phases occurring at separate times,
  methods like \emph{CloNcaSe} are effective for estimating the rate of
  sexual reproduction and effective population size (Ali et al.,
  \protect\hyperlink{ref-ali2016cloncase}{2016}). However, when sexual
  reproduction is not a cyclical phenomena, this estimator is useless.
  Currently, a method commonly used to assess clonal reproduction is the
  index of assocation (\(I_A\)), and its standardized version,
  \(\bar{r}_d\), which measure multilocus linkage disequilibrium (Agapow
  and Burt, \protect\hyperlink{ref-Agapowux5f2001}{2001}; Brown et al.,
  \protect\hyperlink{ref-brown1980multilocus}{1980}; de Meeûs and Balloux,
  \protect\hyperlink{ref-de2004clonal}{2004}; Haubold et al.,
  \protect\hyperlink{ref-haubold1998detecting}{1998}; Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b};
  Smith et al., \protect\hyperlink{ref-smith1993how}{1993}). The value of
  \(I_A\) is measured as the ratio of observed variance in genetic
  distance between samples (\(V_O\)) and expected variance (\(V_E\))
  (equation \eqref{eq:ia}) (Agapow and Burt,
  \protect\hyperlink{ref-Agapowux5f2001}{2001}; Smith et al.,
  \protect\hyperlink{ref-smith1993how}{1993}).
  
  \begin{equation}
  I_A = \frac{V_O}{V_E} - 1 \label{eq:ia}
  \end{equation}
  
  The expected variance is practically modeled as the sum of the variances
  over \emph{m} loci: \(V_E = \sum^m{var_j}\) (Agapow and Burt,
  \protect\hyperlink{ref-Agapowux5f2001}{2001}; Haubold et al.,
  \protect\hyperlink{ref-haubold1998detecting}{1998}). If the differences
  between samples are randomly distributed, we can expect the value of
  \(I_A\) to be zero (Agapow and Burt,
  \protect\hyperlink{ref-Agapowux5f2001}{2001}; Smith et al.,
  \protect\hyperlink{ref-smith1993how}{1993}). Under scenarios of
  non-random mating (e.g.~population structure or clonal reproduction),
  the observed variance would be greater than the expected due to a
  bimodal distribution of distances, and \(I_A\) would be greater than
  zero (Agapow and Burt, \protect\hyperlink{ref-Agapowux5f2001}{2001};
  Milgroom, \protect\hyperlink{ref-milgroom2015population}{2015}; Smith et
  al., \protect\hyperlink{ref-smith1993how}{1993}). Agapow and Burt
  (\protect\hyperlink{ref-Agapowux5f2001}{2001}) noted that this metric
  does not have an upper limit and increases with the number of loci. To
  correct this, they developed \(\bar{r}_d\) (equation \eqref{eq:rd}), which
  has a similar structure to a correlation coefficient and ranges from 0
  (no linkage) to 1 (complete linkage).
  
  \begin{align} % This mode aligns the equations to the '&=' signs
  \begin{split} % This mode groups the equations into one.
  \bar{r}_d &= \frac{\sum\sum{cov_{j,k}}}{
                     \sum\sum{\sqrt{var_{j} \cdot var_{k}}}} \\
            &= \frac{V_O - V_E}{2\sum\sum{\sqrt{var_{j} \cdot var_{k}}}}
  \end{split}
  \label{eq:rd}
  \end{align}
  
  Smith et al. (\protect\hyperlink{ref-smith1993how}{1993}) demonstrated
  that \(I_A\) has the ability to detect non-random mating due to
  population structure, strict clonal reproduction, or a recent epidemic,
  but at that time, no one had quantified how much clonal reproduction had
  to occur to produce a significant value of \(I_A\). This was of
  significant concern, considering that, as with any summary statistic, a
  lot of information is lost when evaluating a population based on \(I_A\)
  (Brown et al., \protect\hyperlink{ref-brown1980multilocus}{1980}). One
  way to assess the effect of different population genetic scenarios was
  via simulations.
  
  The effect of increasing levels of sexual reproduction on \(\bar{r}_d\)
  was investigated by de Meeûs and Balloux
  (\protect\hyperlink{ref-de2004clonal}{2004}) (noted in their publication
  as \(\bar{r}_D\)). It was found that the mean value of \(\bar{r}_d\)
  decreased rapidly with relatively low levels of sexual reproduction,
  with large variances at low levels of sexual reproduction. This
  indicates that \(\bar{r}_d\) alone might not be well suited as a measure
  of clonal reproduction. The effect of sampling design on \(\bar{r}_d\)
  was tested by Prugnolle and de Meeûs
  (\protect\hyperlink{ref-prugnolle2010apparent}{2010}), finding that the
  value of \(\bar{r}_d\) is drastically reduced when clones from multiple
  populations are sampled, which can lead to an over-estimation of the
  level of recombination.
  
  These studies laid the groundwork for understanding the behavior of
  \(\bar{r}_d\) under different scenarios of non-random mating in diploid
  organisms, but there were some limitations in available technology that
  prevented deep analysis from being performed. For both studies, the only
  software available to analyze \(\bar{r}_d\) for diploid organisms was
  \textsc{Multilocus}, which could only take one data set at a time
  (Agapow and Burt, \protect\hyperlink{ref-Agapowux5f2001}{2001}; de Meeûs
  and Balloux, \protect\hyperlink{ref-de2004clonal}{2004}; Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b};
  Prugnolle and de Meeûs,
  \protect\hyperlink{ref-prugnolle2010apparent}{2010}). This constrained
  the researchers to only analyze a minimal set of populations (20) per
  scenario.
  
  Since the distribution of \(I_A\) and \(\bar{r}_d\) are not known, the
  safest way to test for significance were random permutation tests that
  effectively created unlinked populations by shuffling individuals at
  each locus, independently and re-calculating \(I_A\) and \(\bar{r}_d\)
  (Agapow and Burt, \protect\hyperlink{ref-Agapowux5f2001}{2001}; Haubold
  et al., \protect\hyperlink{ref-haubold1998detecting}{1998}; Smith et
  al., \protect\hyperlink{ref-smith1993how}{1993}). A one-sided
  \emph{t}-test of significance was then used to see if the observed
  statistic was greater than the observed distribution.
  
  While significance testing was available in \textsc{Multilocus} in the
  form of random permutations, it was computationally expensive, and the
  analytical approach was only available in a software for haploid
  organisms (Agapow and Burt,
  \protect\hyperlink{ref-Agapowux5f2001}{2001}; Haubold and Hudson,
  \protect\hyperlink{ref-haubold2000lian}{2000}; Haubold et al.,
  \protect\hyperlink{ref-haubold1998detecting}{1998}; Kamvar et al.,
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b}).
  As a result, power analysis of \(\bar{r}_d\) to detect clonal
  reproduction is still yet to be performed (de Meeûs and Balloux,
  \protect\hyperlink{ref-de2004clonal}{2004}).
  
  Since these studies came out, reduced-representation, high-throughput
  sequencing methods such as Genotyping-By-Sequencing (GBS) and RAD-seq
  have rapidly become popular tools for population genetic analysis (Davey
  and Blaxter, \protect\hyperlink{ref-davey2010rad}{2010}; Davey et al.,
  \protect\hyperlink{ref-davey2011genome}{2011}; Elshire et al.,
  \protect\hyperlink{ref-elshire2011robust}{2011}). These methods have the
  capability to generate thousands of unlinked markers at a fraction of
  the cost and time necessary to develop high quality microsatellite
  markers. These marker systems are also prone to high error rates
  (Mastretta-Yanes et al.,
  \protect\hyperlink{ref-mastretta2015restriction}{2014}). The index of
  association was developed for multiple loci in a time when obtaining
  even 100 unlinked markers posed a significant challenge. With the advent
  of these current technologies, how does marker choice and genotyping
  error affect the index of association?
  
  Our current study aims to fill in the gaps left by the technical
  limitations of previous studies. In 2014, we developed the R package
  \emph{poppr} for analysis of clonal populations, removing the
  limitations of data input and computational expense of analyzing the
  index of association, and in 2015, we expanded this to analysis of
  genome-wide SNP data (Kamvar et al.,
  \protect\hyperlink{ref-kamvar2015novel}{2015}\protect\hyperlink{ref-kamvar2015novel}{b},
  \protect\hyperlink{ref-kamvar2014poppr}{2014}\protect\hyperlink{ref-kamvar2014poppr}{b};
  R Core Team, \protect\hyperlink{ref-R2016}{2016}). With these tools we
  set expand on previous studies by asking how sample size, marker choice,
  and clone-correction affect our ability to detect clonal reproduction in
  diploid populations. Our objectives to answer these questions are to (1)
  Re-analyze \(\bar{r}_d\) against increasing rates of sexual reproduction
  and different levels of population mixture in both microsatellite and
  SNP data sets. (2) Perform a power analysis of \(\bar{r}_d\). (3) Assess
  how genotypic and allelic evenness and diversity affects \(\bar{r}_d\).
  Because studies have observed significantly negative values of the
  \(I_A\) and \(\bar{r}_d\) (p \(\geq\) 0.95), we additionally seek to
  characterize these populations to find common patterns that would
  indicate clonal or sexual reproduction.
  
  \section{Methods}\label{methods}
  
  Initial sets of simulations were created for different levels of sexual
  reproduction for each marker type. All simulations were performed with
  the python package simuPOP version 1.1.7 in python version 3.4. For each
  scenario, 100 simulations with 10 replicates were created with a census
  size of 10,000 diploid individuals with equal mating type proportions
  evolved over 10,000 generations. From each replicate, 10, 25, 50, and
  100 individuals were sampled without replacement for downstream analysis
  in R version 3.2 with the package \emph{poppr} version 2.2.1 on full and
  clone-corrected data sets. All downstream analyses were run on the OSU
  CGRB Core Computing Facility (supplementary information).
  
  \subsection{Simulating Microsatellite
  Loci}\label{simulating-microsatellite-loci}
  
  Each population was simulated with 20 co-dominant, unlinked loci
  containing 6 to 10 alleles per locus with frequencies drawn from a
  uniform distribution and subsequently normalized. Before mating,
  mutations occurred at each locus at a rate of 1e-5 mutations/generation
  in a stepwise manner using the \texttt{StepwiseMutator()} operator.
  
  \subsection{GBS Simulations}\label{gbs-simulations}
  
  Simulations of 10,000 binary loci at spread evenly over 10 chromosomal
  fragments were simulated with a mutation rate of 1e-5 mutations per
  generation for forward and backward mutations using the
  \texttt{SNPMutator()} operator and and a recombination rate of 1e-5
  between adjacent loci using the \texttt{Recombinator()} operator.
  
  \subsection{Sexual Reproduction}\label{sexual-reproduction}
  
  Simulations of sexual reproduction were analyzed at 10 rates of sexual
  reproduction on a log scale (0.0, 1e-4, 5e-4, 1e-3, 5e-3, 1e-2, 5e-2,
  0.1, 0.5, 1.0) reflecting the fraction of individuals in generation
  \emph{t+1} produced via sexual reproduction. One to three offspring
  could be produced at each mating event. For sexual events, two parents
  were chosen randomly from the population with the
  \texttt{RandomSelection()} operator and offspring genotypes were created
  via the \texttt{MendelianGenoTransmitter()} operator. The clonal
  fraction was created by randomly sampling individuals from the
  population and duplicating their genotypes with the
  \texttt{CloneGenoTransmitter()} operator. If one mating type was lost
  before 10,000 generations, the simulation would continue to completion
  with only clonal reproduction.
  
  \subsection{Analysis of Microsatellite
  Data}\label{analysis-of-microsatellite-data}
  
  The standardized index of association (\(\bar{r}_d\)) was calculated for
  full and clone-corrected data using the \emph{poppr} function
  \texttt{ia()}. Tests for significance were performed by randomly
  permuting the alleles at each locus independently and then assessing
  \(\bar{r}_d\). This was done 999 times for each replicate population.
  Re-sampling was not performed with clone-corrected data. The p-values
  reflect the proportion of observations greater than the observed
  statistic. Estimates of genotypic diversity were assessed with the
  \emph{poppr} function \texttt{diversity\_boot()} with 999 bootstrap
  replicates, recording the estimate and variance.
  
  \subsection{Analysis of SNP Data}\label{analysis-of-snp-data}
  
  Because GBS data are associated with high error rates, we additionally
  wanted to assess the effect of error and missing data on analysis. To do
  this, we used scripts written for Kamvar et al.
  (\protect\hyperlink{ref-kamvar2015novel}{2015}\protect\hyperlink{ref-kamvar2015novel}{b})
  to randomly insert missing data (via the \texttt{pop\_NA()} function)
  and genotyping errors (via the \texttt{pop\_mutator()}) (Kamvar et al.,
  \protect\hyperlink{ref-kamvar2015poppr2supp}{2015}\protect\hyperlink{ref-kamvar2015poppr2supp}{a})
  at rates of 5\% and 10\% each. We then used the \emph{poppr} function
  \texttt{mlg.filter()} to with the farthest neighbor algorithm to
  collapse multilocus genotypes with 5\% and 10\% similarity and compared
  the results with the known genotypes. Genotypic diversity was assessed
  with the \emph{poppr} function \texttt{diversity\_boot()} with 999
  bootstrap replicates on the filtered data sets, recording the variance.
  
  The overall value of \(\bar{r}_d\) was calculated for each simulation
  with the \emph{poppr} function \texttt{bitwise.ia()}. To assess
  within-chromosome dynamics, values of \(\bar{r}_d\) were calculated
  along chromosomes using the \emph{poppr} function \texttt{win.ia()}
  along windows of 50 and 100 SNPs. Cross-chromosome dynamics of
  \(\bar{r}_d\) were assessed by utilizing random samples of 50, 100, and
  1000 SNPs replicated 999 times for each population sample.
  
  \subsection{Power Analysis}\label{power-analysis}
  
  The Receiver Operating Characteristic (ROC) curve is a method of
  assessing the explanatory power of a method by assessing the true
  positive fraction of tests to a false positive fraction (Metz,
  \protect\hyperlink{ref-metz1978basic}{1978}). Briefly, if a method has
  perfect explanatory power, the area under the ROC curve will be equal to
  1. If a method has no explanatory power, the area under the ROC curve
  will be equal to 0.5. ROC analysis was carried as described in Metz
  (\protect\hyperlink{ref-metz1978basic}{1978}) as a comparison of the
  false positive fraction to the true positive fraction as you increase
  the level of rejection, (\(\alpha\)). The null hypothesis for the ROC
  analysis was defined where the rate of sexual reproduction was 100\%.
  All other rates were considered to be separate alternative hypotheses.
  The curves were calculated by analyzing the data sets over increasing
  values of \(\alpha\) from 0 to 1 in increments of 0.01. The area under
  the ROC curves was calculated using the \texttt{trapz()} function in the
  R package \emph{pracma} version 1.9.5 (Borchers,
  \protect\hyperlink{ref-borchers2016pracma}{2016}). ANOVA was run with
  the following models:
  
  \begin{longtable}[c]{@{}c@{}}
  \toprule
  False Positive Rate \textasciitilde{} Sex Rate + Sample
  Size\tabularnewline
  Area Under ROC Curve \textasciitilde{} Sex Rate + Sample
  Size\tabularnewline
  \bottomrule
  \end{longtable}
  
  \backmatter
  
  \chapter{Bibliography}\label{bibliography}
  
  \noindent
  
  \setlength{\parindent}{-0.20in} \setlength{\leftskip}{0.20in}
  \setlength{\parskip}{8pt} \singlespacing
  
  \hypertarget{refs}{}
  \hypertarget{ref-Agapowux5f2001}{}
  Agapow, P.-M., and Burt, A. (2001). Indices of multilocus linkage
  disequilibrium. \emph{Molecular Ecology Notes} 1, 101--102.
  doi:\href{https://doi.org/10.1046/j.1471-8278.2000.00014.x}{10.1046/j.1471-8278.2000.00014.x}.
  
  \hypertarget{ref-ali2016cloncase}{}
  Ali, S., Soubeyrand, S., Gladieux, P., Giraud, T., Leconte, M., Gautier,
  A., et al. (2016). Cloncase: Estimation of sex frequency and effective
  population size by clonemate resampling in partially clonal organisms.
  \emph{Molecular Ecology Resources}.
  doi:\href{https://doi.org/10.1111/1755-0998.12511}{10.1111/1755-0998.12511}.
  
  \hypertarget{ref-anderson1995clonality}{}
  Anderson, J. B., and Kohn, L. M. (1995). Clonality in soilborne,
  plant-pathogenic fungi. \emph{Annual review of phytopathology} 33,
  369--391.
  
  \hypertarget{ref-arnaud2007standardizing}{}
  Arnaud-Hanod, S., Duarte, C. M., Alberto, F., and Serrão, E. A. (2007).
  Standardizing methods to address clonality in population studies.
  \emph{Molecular Ecology} 16, 5115--5139.
  doi:\href{https://doi.org/10.1111/j.1365-294X.2007.03535.x}{10.1111/j.1365-294X.2007.03535.x}.
  
  \hypertarget{ref-baldauf2003deep}{}
  Baldauf, S. (2003). The deep roots of eukaryotes. \emph{Science} 300,
  1703--1706.
  doi:\href{https://doi.org/10.1126/science.1085544}{10.1126/science.1085544}.
  
  \hypertarget{ref-balloux2003population}{}
  Balloux, F., Lehmann, L., and de Meeûs, T. (2003). The population
  genetics of clonal and partially clonal diploids. \emph{Genetics} 164,
  1635--1644.
  
  \hypertarget{ref-bivand2014rgdal}{}
  Bivand, R., Keitt, T., and Rowlingson, B. (2014). \emph{rgdal: Bindings
  for the Geospatial Data Abstraction Library}. Available at:
  \url{https://CRAN.R-project.org/package=rgdal}.
  
  \hypertarget{ref-borchers2016pracma}{}
  Borchers, H. W. (2016). \emph{Pracma: Practical numerical math
  functions}. Available at:
  \url{https://CRAN.R-project.org/package=pracma}.
  
  \hypertarget{ref-brown1980multilocus}{}
  Brown, A., Feldman, M., and Nevo, E. (1980). Multilocus structure of
  natural populations of \emph{Hordeum spontaneum}. \emph{Genetics} 96,
  523--536. Available at:
  \url{http://www.genetics.org/content/96/2/523.abstract}.
  
  \hypertarget{ref-brownstein1996modulation}{}
  Brownstein, M. J., Carpten, J. D., and Smith, J. R. (1996). Modulation
  of non-templated nucleotide addition by Taq DNA polymerase: primer
  modifications that facilitate genotyping. \emph{BioTechniques} 20,
  1004--6, 1008--10.
  
  \hypertarget{ref-bruvo2004simple}{}
  Bruvo, R., Michiels, N. K., D'Souza, T. G., and Schulenburg, H. (2004).
  A simple method for the calculation of microsatellite genotype distances
  irrespective of ploidy level. \emph{Molecular Ecology} 13, 2101--2106.
  
  \hypertarget{ref-butlin2000virgin}{}
  Butlin, R. K. (2000). Virgin rotifers. \emph{Trends in Ecology \&
  Evolution} 15, 389--390.
  doi:\href{https://doi.org/10.1016/S0169-5347(00)01937-6}{10.1016/S0169-5347(00)01937-6}.
  
  \hypertarget{ref-canty2015boot}{}
  Canty, A., and Ripley, B. D. (2015). \emph{Boot: Bootstrap r (s-plus)
  functions}. Available at: \url{https://CRAN.R-project.org/package=boot}.
  
  \hypertarget{ref-chakarov2015apparent}{}
  Chakarov, N., Linke, B., Boerner, M., Goesmann, A., Krüger, O., and
  Hoffman, J. I. (2015). Apparent vector-mediated parent-to-offspring
  transmission in an avian malaria-like parasite. \emph{Molecular ecology}
  24, 1355--1363.
  
  \hypertarget{ref-cooke2012genome}{}
  Cooke, D. E., Cano, L. M., Raffaele, S., Bain, R. A., Cooke, L. R.,
  Etherington, G. J., et al. (2012). Genome analyses of an aggressive and
  invasive lineage of the Irish potato famine pathogen. \emph{PLoS
  pathogens} 8, e1002940.
  
  \hypertarget{ref-croucher2013combining}{}
  Croucher, P. J. P., Mascheretti, S., and Garbelotto, M. (2013).
  Combining field epidemiological information and genetic data to
  comprehensively reconstruct the invasion history and the microevolution
  of the sudden oak death agent \emph{Phytophthora ramorum} (Stramenopila:
  Oomycetes) in California. \emph{Biological Invasions} 15, 2281--2297.
  doi:\href{https://doi.org/10.1007/s10530-013-0453-8}{10.1007/s10530-013-0453-8}.
  
  \hypertarget{ref-csardi2006igraph}{}
  Csardi, G., and Nepusz, T. (2006). The igraph software package for
  complex network research. \emph{InterJournal} Complex Systems, 1695.
  Available at: \url{http://igraph.org}.
  
  \hypertarget{ref-dagum1998openmp}{}
  Dagum, L., and Menon, R. (1998). OpenMP: An industry standard API for
  shared-memory programming. \emph{Computational Science \& Engineering,
  IEEE} 5, 46--55.
  
  \hypertarget{ref-davey2010rad}{}
  Davey, J. W., and Blaxter, M. L. (2010). RADSeq: next-generation
  population genetics. \emph{Briefings in Functional Genomics} 9,
  416--423.
  doi:\href{https://doi.org/10.1093/bfgp/elq031}{10.1093/bfgp/elq031}.
  
  \hypertarget{ref-davey2011genome}{}
  Davey, J. W., Hohenlohe, P. A., Etter, P. D., Boone, J. Q., Catchen, J.
  M., and Blaxter, M. L. (2011). Genome-wide genetic marker discovery and
  genotyping using next-generation sequencing. \emph{Nature Reviews
  Genetics} 12, 499--510.
  doi:\href{https://doi.org/10.1038/nrg3012}{10.1038/nrg3012}.
  
  \hypertarget{ref-de2004clonal}{}
  de Meeûs, T., and Balloux, F. (2004). Clonal reproduction and linkage
  disequilibrium in diploids: A simulation study. \emph{Infection,
  Genetics and Evolution} 4, 345--351.
  doi:\href{https://doi.org/10.1016/j.meegid.2004.05.002}{10.1016/j.meegid.2004.05.002}.
  
  \hypertarget{ref-de2006molecular}{}
  de Meeûs, T., Lehmann, L., and Balloux, F. (2006). Molecular
  epidemiology of clonal diploids: a quick overview and a short DIY (do it
  yourself) notice. \emph{Infection, Genetics and Evolution} 6, 163--170.
  doi:\href{https://doi.org/10.1016/j.meegid.2005.02.004}{10.1016/j.meegid.2005.02.004}.
  
  \hypertarget{ref-dobzhansky2013nothing}{}
  Dobzhansky, T. (1973). Nothing in biology makes sense except in the
  light of evolution. \emph{The American Biology Teacher} 75, 87--91.
  
  \hypertarget{ref-dray2007ade4}{}
  Dray, S., and Dufour, A.-B. (2007). The ade4 package: Implementing the
  duality diagram for ecologists. \emph{Journal of Statistical Software}
  22.
  doi:\href{https://doi.org/10.18637/jss.v022.i04}{10.18637/jss.v022.i04}.
  
  \hypertarget{ref-elshire2011robust}{}
  Elshire, R. J., Glaubitz, J. C., Sun, Q., Poland, J. A., Kawamoto, K.,
  Buckler, E. S., et al. (2011). A robust, simple genotyping-by-sequencing
  (GBS) approach for high diversity species. \emph{PLoS ONE} 6, e19379.
  doi:\href{https://doi.org/10.1371/journal.pone.0019379}{10.1371/journal.pone.0019379}.
  
  \hypertarget{ref-erwin1996phytophthora}{}
  Erwin, D. C., Ribeiro, O. K., and others (1996). \emph{Phytophthora
  diseases worldwide}. St. Paul, Minnesota, USA: American
  Phytopathological Society (APS Press).
  
  \hypertarget{ref-everhart2014fine}{}
  Everhart, S. E., and Scherm, H. (2015). Fine-scale genetic structure of
  \emph{Monilinia fructicola} during brown rot epidemics within individual
  peach tree canopies. \emph{Phytopathology} 105, 542--549.
  doi:\href{https://doi.org/10.1094/phyto-03-14-0088-r}{10.1094/phyto-03-14-0088-r}.
  
  \hypertarget{ref-everhart2014phytophthora}{}
  Everhart, S. E., Tabima, J. F., and Grünwald, N. J. (2014).
  ``\emph{Phytophthora ramorum},'' in \emph{Genomics of plant-associated
  fungi and oomycetes: Dicot pathogens} (Berlin, Heidelberg: Springer),
  159--174.
  doi:\href{https://doi.org/10.1007/978-3-662-44056-8_8}{10.1007/978-3-662-44056-8\_8}.
  
  \hypertarget{ref-excoffier1992analysis}{}
  Excoffier, L., Smouse, P. E., and Quattro, J. M. (1992). Analysis of
  molecular variance inferred from metric distances among DNA haplotypes:
  Application to human mitochondrial DNA restriction data. \emph{Genetics}
  131, 479--91.
  
  \hypertarget{ref-eyre2013poulation}{}
  Eyre, C. A., Kozanitas, M., and Garbelotto, M. (2013). Population
  Dynamics of Aerial and Terrestrial Populations of \emph{Phytophthora
  ramorum} in a California Forest Under Different Climatic Conditions.
  \emph{Phytopathology} 103, 1141--1152.
  doi:\href{https://doi.org/10.1094/phyto-11-12-0290-r}{10.1094/phyto-11-12-0290-r}.
  
  \hypertarget{ref-Falush01082003}{}
  Falush, D., Stephens, M., and Pritchard, J. K. (2003). Inference of
  population structure using multilocus genotype data: Linked loci and
  correlated allele frequencies. \emph{Genetics} 164, 1567--1587.
  Available at: \url{http://www.genetics.org/content/164/4/1567.abstract}.
  
  \hypertarget{ref-goss2009population}{}
  Goss, E. M., Larsen, M., Chastagner, G. A., Givens, D. R., and Grünwald,
  N. J. (2009). Population genetic analysis infers migration pathways of
  \emph{Phytophthora ramorum} in US nurseries. \emph{PLoS Pathog} 5,
  e1000583.
  doi:\href{https://doi.org/10.1371/journal.ppat.1000583}{10.1371/journal.ppat.1000583}.
  
  \hypertarget{ref-goss2011phytophthora}{}
  Goss, E. M., Larsen, M., Vercauteren, A., Werres, S., Heungens, K., and
  Grünwald, N. J. (2011). \emph{Phytophthora ramorum} in Canada: Evidence
  for Migration Within North America and from Europe.
  \emph{Phytopathology} 101, 166--171.
  doi:\href{https://doi.org/10.1094/phyto-05-10-0133}{10.1094/phyto-05-10-0133}.
  
  \hypertarget{ref-goss2014irish}{}
  Goss, E. M., Tabima, J. F., Cooke, D. E., Restrepo, S., Fry, W. E.,
  Forbes, G. A., et al. (2014). The Irish potato famine pathogen
  \emph{Phytophthora infestans} originated in central Mexico rather than
  the Andes. \emph{Proceedings of the National Academy of Sciences} 111,
  8791--8796.
  
  \hypertarget{ref-goudet2005hierfstat}{}
  Goudet, J. (2005). Hierfstat, a package for R to compute and test
  hierarchical F-statistics. \emph{Molecular Ecology Notes} 5, 184--186.
  
  \hypertarget{ref-gross2014population}{}
  Gross, A., Hosoya, T., and Queloz, V. (2014). Population structure of
  the invasive forest pathogen \emph{Hymenoscyphus pseudoalbidus}.
  \emph{Molecular ecology} 23, 2943--2960.
  
  \hypertarget{ref-grunwald2011evolution}{}
  Grünwald, N. J., and Goss, E. M. (2011). Evolution and Population
  Genetics of Exotic and Re-Emerging Pathogens: Novel Tools and
  Approaches. \emph{Annu. Rev. Phytopathol.} 49, 249--267.
  doi:\href{https://doi.org/10.1146/annurev-phyto-072910-095246}{10.1146/annurev-phyto-072910-095246}.
  
  \hypertarget{ref-grunwald2006hierarchical}{}
  Grünwald, N. J., and Hoheisel, G.-A. (2006). Hierarchical analysis of
  diversity, selfing, and genetic differentiation in populations of the
  oomycete \emph{Aphanomyces euteiches}. \emph{Phytopathology} 96,
  1134--1141.
  
  \hypertarget{ref-grunwald2012emergence}{}
  Grünwald, N. J., Garbelotto, M., Goss, E. M., Heungens, K., and
  Prospero, S. (2012). Emergence of the sudden oak death pathogen
  \emph{Phytophthora ramorum}. \emph{Trends in Microbiology} 20, 131--138.
  doi:\href{https://doi.org/10.1016/j.tim.2011.12.006}{10.1016/j.tim.2011.12.006}.
  
  \hypertarget{ref-grunwald2003analysis}{}
  Grünwald, N. J., Goodwin, S. B., Milgroom, M. G., and Fry, W. E. (2003).
  Analysis of Genotypic Diversity Data for Populations of Microorganisms.
  \emph{Phytopathology} 93, 738--746.
  doi:\href{https://doi.org/10.1094/phyto.2003.93.6.738}{10.1094/phyto.2003.93.6.738}.
  
  \hypertarget{ref-grunwald2008phytophthora}{}
  Grünwald, N. J., Goss, E. M., and Press, C. M. (2008a).
  \emph{Phytophthora ramorum}: a pathogen with a remarkably wide host
  range causing sudden oak death on oaks and ramorum blight on woody
  ornamentals. \emph{Molecular Plant Pathology} 9, 729--740.
  doi:\href{https://doi.org/10.1111/j.1364-3703.2008.00500.x}{10.1111/j.1364-3703.2008.00500.x}.
  
  \hypertarget{ref-grunwald2009standardizing}{}
  Grünwald, N. J., Goss, E. M., Ivors, K., Garbelotto, M., Martin, F. N.,
  Prospero, S., et al. (2009). Standardizing the Nomenclature for Clonal
  Lineages of the Sudden Oak Death Pathogen, \emph{Phytophthora ramorum}.
  \emph{Phytopathology} 99, 792--795.
  doi:\href{https://doi.org/10.1094/phyto-99-7-0792}{10.1094/phyto-99-7-0792}.
  
  \hypertarget{ref-grunwald2008susceptibility}{}
  Grünwald, N. J., Kitner, M., McDonald, V., and Goss, E. M. (2008b).
  Susceptibility in \emph{Viburnum} to \emph{Phytophthora ramorum}.
  \emph{Plant Disease} 92, 210--214.
  doi:\href{https://doi.org/10.1094/pdis-92-2-0210}{10.1094/pdis-92-2-0210}.
  
  \hypertarget{ref-grunwald2011phytophthora}{}
  Grünwald, N. J., Martin, F. N., Larsen, M. M., Sullivan, C. M., Press,
  C. M., Coffey, M. D., et al. (2011). Phytophthora-ID.org: a
  sequence-based \emph{Phytophthora} identification tool. \emph{Plant
  Disease} 95, 337--342.
  
  \hypertarget{ref-haas2009genome}{}
  Haas, B. J., Kamoun, S., Zody, M. C., Jiang, R. H., Handsaker, R. E.,
  Cano, L. M., et al. (2009). Genome sequence and analysis of the Irish
  potato famine pathogen \emph{Phytophthora infestans}. \emph{Nature} 461,
  393--398.
  doi:\href{https://doi.org/10.1038/nature08358}{10.1038/nature08358}.
  
  \hypertarget{ref-halkett2005tackling}{}
  Halkett, F., Simon, J., and Balloux, F. (2005). Tackling the population
  genetics of clonal and partially clonal organisms. \emph{Trends in
  Ecology \& Evolution} 20, 194--201.
  doi:\href{https://doi.org/10.1016/j.tree.2005.01.001}{10.1016/j.tree.2005.01.001}.
  
  \hypertarget{ref-hansen2008epidemiology}{}
  Hansen, E. M., Kanaskie, A., Prospero, S., McWilliams, M., Goheen, E.
  M., Osterbauer, N., et al. (2008). Epidemiology of \emph{Phytophthora
  ramorum} in Oregon tanoak forests. \emph{Canadian Journal of Forest
  Research} 38, 1133--1143.
  doi:\href{https://doi.org/10.1139/x07-217}{10.1139/x07-217}.
  
  \hypertarget{ref-haubold2000lian}{}
  Haubold, B., and Hudson, R. R. (2000). LIAN 3.0: Detecting linkage
  disequilibrium in multilocus data. \emph{Bioinformatics} 16, 847--849.
  doi:\href{https://doi.org/10.1093/bioinformatics/16.9.847}{10.1093/bioinformatics/16.9.847}.
  
  \hypertarget{ref-haubold1998detecting}{}
  Haubold, B., Travisano, M., Rainey, P. B., and Hudson, R. R. (1998).
  Detecting linkage disequilibrium in bacterial populations.
  \emph{Genetics} 150, 1341--8.
  
  \hypertarget{ref-heck1975explicit}{}
  Heck, K. L., van Belle, G., and Simberloff, D. (1975). Explicit
  calculation of the rarefaction diversity measurement and the
  determination of sufficient sample size. \emph{Ecology} 56, 1459--1461.
  doi:\href{https://doi.org/10.2307/1934716}{10.2307/1934716}.
  
  \hypertarget{ref-hurlbert1971nonconcept}{}
  Hurlbert, S. H. (1971). The nonconcept of species diversity: A critique
  and alternative parameters. \emph{Ecology} 52, 577--586.
  doi:\href{https://doi.org/10.2307/1934145}{10.2307/1934145}.
  
  \hypertarget{ref-ivors2006microsatellite}{}
  Ivors, K., Garbelotto, M., Vries, I. D. E., Ruyter-spira, C., Hekkert,
  B. T., Rosenzweig, N., et al. (2006). Microsatellite markers identify
  three lineages of \emph{Phytophthora ramorum} in US nurseries, yet
  single lineages in US forest and European nursery populations.
  \emph{Molecular Ecology} 15, 1493--1505.
  doi:\href{https://doi.org/10.1111/j.1365-294x.2006.02864.x}{10.1111/j.1365-294x.2006.02864.x}.
  
  \hypertarget{ref-Jombartux5f2008}{}
  Jombart, T. (2008). Adegenet: a R package for the multivariate analysis
  of genetic markers. \emph{Bioinformatics} 24, 1403--1405.
  doi:\href{https://doi.org/10.1093/bioinformatics/btn129}{10.1093/bioinformatics/btn129}.
  
  \hypertarget{ref-jombart2011adegenet}{}
  Jombart, T., and Ahmed, I. (2011). Adegenet 1.3-1: New tools for the
  analysis of genome-wide SNP data. \emph{Bioinformatics} 27, 3070--3071.
  
  \hypertarget{ref-jombart2010discriminant}{}
  Jombart, T., Devillard, S., and Balloux, F. (2010). Discriminant
  analysis of principal components: A new method for the analysis of
  genetically structured populations. \emph{BMC Genet} 11, 94.
  doi:\href{https://doi.org/10.1186/1471-2156-11-94}{10.1186/1471-2156-11-94}.
  
  \hypertarget{ref-khale2013ggmap}{}
  Kahle, D., and Wickham, H. (2013). ggmap: Spatial Visualization with
  ggplot2. \emph{The R Journal} 5, 144--161. Available at:
  \url{http://journal.r-project.org/archive/2013-1/kahle-wickham.pdf}.
  
  \hypertarget{ref-kamoun2014top}{}
  Kamoun, S., Furzer, O., Jones, J. D. G., Judelson, H. S., Ali, G. S.,
  Dalio, R. J. D., et al. (2014). The top 10 oomycete pathogens in
  molecular plant pathology. \emph{Molecular Plant Pathology} 16,
  413--434.
  doi:\href{https://doi.org/10.1111/mpp.12190}{10.1111/mpp.12190}.
  
  \hypertarget{ref-kamvar2015poppr2supp}{}
  Kamvar, Z. N., Brooks, J. C., and Grunwald, N. J. (2015a). Supplementary
  Material for Frontiers Plant Genetics and Genomics 'Novel R tools for
  analysis of genome-wide population genetic data with emphasis on
  clonality'.
  doi:\href{https://doi.org/10.5281/zenodo.17424}{10.5281/zenodo.17424}.
  
  \hypertarget{ref-kamvar2015novel}{}
  Kamvar, Z. N., Brooks, J. C., and Grünwald, N. J. (2015b). Novel R tools
  for analysis of genome-wide population genetic data with emphasis on
  clonality. \emph{Frontiers in Genetics} 6.
  doi:\href{https://doi.org/10.3389/fgene.2015.00208}{10.3389/fgene.2015.00208}.
  
  \hypertarget{ref-kamvar2014sudden}{}
  Kamvar, Z. N., Larsen, M. M., Kanaskie, A. M., Hansen, E. M., and
  Grünwald, N. J. (2014a). Sudden\_Oak\_Death\_in\_Oregon\_Forests:
  Spatial and temporal population dynamics of the sudden oak death
  epidemic in Oregon Forests.
  doi:\href{https://doi.org/10.5281/zenodo.13007}{10.5281/zenodo.13007}.
  
  \hypertarget{ref-kamvar2015spatial}{}
  Kamvar, Z. N., Larsen, M. M., Kanaskie, A. M., Hansen, E. M., and
  Grünwald, N. J. (2015c). Spatial and temporal analysis of populations of
  the sudden oak death pathogen in oregon forests. \emph{Phytopathology}
  105, 982--989.
  doi:\href{https://doi.org/10.1094/phyto-12-14-0350-fi}{10.1094/phyto-12-14-0350-fi}.
  
  \hypertarget{ref-kamvar2014poppr}{}
  Kamvar, Z. N., Tabima, J. F., and Grünwald, N. J. (2014b). Poppr : an R
  package for genetic analysis of populations with clonal, partially
  clonal, and/or sexual reproduction. \emph{PeerJ} 2, e281.
  doi:\href{https://doi.org/10.7717/peerj.281}{10.7717/peerj.281}.
  
  \hypertarget{ref-kroon2012genus}{}
  Kroon, L. P., Brouwer, H., Cock, A. W. de, and Govers, F. (2012). The
  Genus \emph{Phytophthora}. \emph{Phytopathology} 102, 348--364.
  doi:\href{https://doi.org/10.1094/PHYTO-01-11-0025}{10.1094/PHYTO-01-11-0025}.
  
  \hypertarget{ref-laloe2007consensus}{}
  Laloë, D., Jombart, T., Dufour, A.-B., and Moazami-Goudarzi, K. (2007).
  Consensus genetic structuring and typological value of markers using
  multiple co-inertia analysis. \emph{Genetics Selection Evolution} 39,
  1--23.
  
  \hypertarget{ref-lees2006novel}{}
  Lees, A., Wattier, R., Shaw, D., Sullivan, L., Williams, N., and Cooke,
  D. (2006). Novel microsatellite markers for the analysis of
  \emph{Phytophthora infestans} populations. \emph{Plant Pathology} 55,
  311--319.
  
  \hypertarget{ref-li2013efficient}{}
  Li, Y., Cooke, D. E., Jacobsen, E., and Lee, T. van der (2013).
  Efficient multiplex simple sequence repeat genotyping of the oomycete
  plant pathogen \emph{Phytophthora infestans}. \emph{Journal of
  microbiological methods} 92, 316--322.
  
  \hypertarget{ref-linde2002population}{}
  Linde, C., Zhan, J., and McDonald, B. (2002). Population structure of
  \emph{Mycosphaerella graminicola}: From lesions to continents.
  \emph{Phytopathology} 92, 946--955.
  
  \hypertarget{ref-ludwig1988statistical}{}
  Ludwig, J. A., and Reynolds, J. F. (1988). \emph{Statistical ecology: A
  primer in methods and computing}. John Wiley \& Sons.
  
  \hypertarget{ref-luikart2003power}{}
  Luikart, G., England, P. R., Tallmon, D., Jordan, S., and Taberlet, P.
  (2003). The power and promise of population genomics: From genotyping to
  genome typing. \emph{Nature Reviews Genetics} 4, 981--994.
  
  \hypertarget{ref-mantel1967detection}{}
  Mantel, N. (1967). The detection of disease clustering and a generalized
  regression approach. \emph{Cancer research} 27, 209--220.
  
  \hypertarget{ref-mascheretti2009genetic}{}
  Mascheretti, S., Croucher, P. J. P., Kozanitas, M., Baker, L., and
  Garbelotto, M. (2009). Genetic epidemiology of the Sudden Oak Death
  pathogen \emph{Phytophthora ramorum} in California. \emph{Molecular
  Ecology} 18, 4577--4590.
  doi:\href{https://doi.org/10.1111/j.1365-294x.2009.04379.x}{10.1111/j.1365-294x.2009.04379.x}.
  
  \hypertarget{ref-mascheretti2008reconstruction}{}
  Mascheretti, S., Croucher, P. J. P., Vettraino, A., Prospero, S., and
  Garbelotto, M. (2008). Reconstruction of the Sudden Oak Death epidemic
  in California through microsatellite analysis of the pathogen
  \emph{Phytophthora ramorum}. \emph{Molecular Ecology} 17, 2755--2768.
  doi:\href{https://doi.org/10.1111/j.1365-294x.2008.03773.x}{10.1111/j.1365-294x.2008.03773.x}.
  
  \hypertarget{ref-mastretta2015restriction}{}
  Mastretta-Yanes, A., Arrigo, N., Alvarez, N., Jorgensen, T. H., Piñero,
  D., and Emerson, B. C. (2014). Restriction site-associated DNA
  sequencing, genotyping error estimation and \emph{de novo} assembly
  optimization for population genetic inference. \emph{Molecular Ecology
  Resources} 15, 28--41.
  doi:\href{https://doi.org/10.1111/1755-0998.12291}{10.1111/1755-0998.12291}.
  
  \hypertarget{ref-Mcdonald2002}{}
  McDonald, B. A., and Linde, C. (2002). The population genetics of plant
  pathogens and breeding strategies for durable resistance.
  \emph{Euphytica} 124, 163--180.
  doi:\href{https://doi.org/10.1023/A:1015678432355}{10.1023/A:1015678432355}.
  
  \hypertarget{ref-meirmans2004genotype}{}
  Meirmans, P. G., and Van Tienderen, P. H. (2004). GENOTYPE and GENODIVE:
  Two programs for the analysis of genetic diversity of asexual organisms.
  \emph{Molecular Ecology Notes} 4, 792--794.
  
  \hypertarget{ref-metz1978basic}{}
  Metz, C. E. (1978). Basic principles of ROC analysis. in \emph{Seminars
  in nuclear medicine} (Elsevier), 283--298.
  doi:\href{https://doi.org/10.1016/S0001-2998(78)80014-2}{10.1016/S0001-2998(78)80014-2}.
  
  \hypertarget{ref-metzger2015horizontal}{}
  Metzger, M. J., Reinisch, C., Sherry, J., and Goff, S. P. (2015).
  Horizontal transmission of clonal cancer cells causes leukemia in
  soft-shell clams. \emph{Cell} 161, 255--263.
  
  \hypertarget{ref-michalakis1996generic}{}
  Michalakis, Y., and Excoffier, L. (1996). A generic estimation of
  population subdivision using distances between alleles with special
  reference for microsatellite loci. \emph{Genetics} 142, 1061--1064.
  
  \hypertarget{ref-milgroom1996recombination}{}
  Milgroom, M. G. (1996). Recombination and the multilocus structure of
  fungal populations. \emph{Annual review of phytopathology} 34, 457--477.
  
  \hypertarget{ref-milgroom2015population}{}
  Milgroom, M. G. (2015). \emph{Population biology of plant pathogens:
  Genetics, ecology, and evolution}. 3340 Pilot Knob Road, St. Paul, MN
  55121, USA: APS Press.
  
  \hypertarget{ref-milgroom1989population}{}
  Milgroom, M. G., Levin, S. A., and Fry, W. E. (1989). Population
  genetics theory and fungicide resistance. \emph{Plant disease
  epidemiology} 2, 340--367.
  
  \hypertarget{ref-milgroom1997contributions}{}
  Milgroom, M., and Fry, W. E. (1997). Contributions of population
  genetics to plant disease epidemiology and management. \emph{Advances in
  Botanical Research} 24, 1--30.
  doi:\href{https://doi.org/10.1016/S0065-2296(08)60069-5}{10.1016/S0065-2296(08)60069-5}.
  
  \hypertarget{ref-nei1973analysis}{}
  Nei, M. (1973). Analysis of gene diversity in subdivided populations.
  \emph{Proceedings of the National Academy of Sciences} 70, 3321--3323.
  
  \hypertarget{ref-nieuwenhuis2016frequency}{}
  Nieuwenhuis, B. P. S., and James, T. Y. (2016). The frequency of sex in
  fungi. \emph{Philosophical Transactions of the Royal Society B:
  Biological Sciences} 371, 20150540.
  doi:\href{https://doi.org/10.1098/rstb.2015.0540}{10.1098/rstb.2015.0540}.
  
  \hypertarget{ref-oksanen2013vegan}{}
  Oksanen, J., Blanchet, F. G., Kindt, R., Legendre, P., Minchin, P. R.,
  O'Hara, R. B., et al. (2013). \emph{Vegan: Community ecology package}.
  Available at: \url{https://CRAN.R-project.org/package=vegan}.
  
  \hypertarget{ref-oksanen2015vegan}{}
  Oksanen, J., Blanchet, F. G., Kindt, R., Legendre, P., Minchin, P. R.,
  O'Hara, R. B., et al. (2015). \emph{Vegan: Community ecology package}.
  Available at: \url{http://CRAN.R-project.org/package=vegan}.
  
  \hypertarget{ref-orive1993effective}{}
  Orive, M. E. (1993). Effective population size in organisms with complex
  life-histories. \emph{Theoretical Population Biology} 44, 316--340.
  doi:\href{https://doi.org/10.1006/tpbi.1993.1031}{10.1006/tpbi.1993.1031}.
  
  \hypertarget{ref-paradis2004ape}{}
  Paradis, E., Claude, J., and Strimmer, K. (2004). APE: Analyses of
  phylogenetics and evolution in r language. \emph{Bioinformatics} 20,
  289--290.
  doi:\href{https://doi.org/10.1093/bioinformatics/btg412}{10.1093/bioinformatics/btg412}.
  
  \hypertarget{ref-parke2014phytophthora}{}
  Parke, J. L., Knaus, B. J., Fieland, V. J., Lewis, C., and Grünwald, N.
  J. (2014). Phytophthora community structure analyses in Oregon nurseries
  inform systems approaches to disease management. \emph{Phytopathology}
  104, 1052--1062.
  doi:\href{https://doi.org/10.1094/PHYTO-01-14-0014-R}{10.1094/PHYTO-01-14-0014-R}.
  
  \hypertarget{ref-pielou1975ecological}{}
  Pielou, E. (1975). Ecological diversity.
  
  \hypertarget{ref-vanpoucke2012discovery}{}
  Poucke, K. V., Franceschini, S., Webber, J. F., Vercauteren, A., Turner,
  J. A., McCracken, A. R., et al. (2012). Discovery of a fourth
  evolutionary lineage of \emph{Phytophthora ramorum}: EU2. \emph{Fungal
  Biology} 116, 1178--1191.
  doi:\href{https://doi.org/10.1016/j.funbio.2012.09.003}{10.1016/j.funbio.2012.09.003}.
  
  \hypertarget{ref-prevosti1975distances}{}
  Prevosti, A., Ocaña, J., and Alonso, G. (1975). Distances between
  populations of \emph{Drosophila subobscura}, based on chromosome
  arrangement frequencies. \emph{Theoretical and Applied Genetics} 45,
  231--241.
  
  \hypertarget{ref-prospero2004isolation}{}
  Prospero, S., Black, J. A., and Winton, L. M. (2004). Isolation and
  characterization of microsatellite markers in \emph{Phytophthora
  ramorum}, the causal agent of sudden oak death. \emph{Molecular Ecology
  Notes} 4, 672--674.
  doi:\href{https://doi.org/10.1111/j.1471-8286.2004.00778.x}{10.1111/j.1471-8286.2004.00778.x}.
  
  \hypertarget{ref-prospero2009migration}{}
  Prospero, S., Grünwald, N. J., Winton, L. M., and Hansen, E. M. (2009).
  Migration Patterns of the Emerging Plant Pathogen \emph{Phytophthora
  ramorum} on the West Coast of the United States of America.
  \emph{Phytopathology} 99, 739--749.
  doi:\href{https://doi.org/10.1094/phyto-99-6-0739}{10.1094/phyto-99-6-0739}.
  
  \hypertarget{ref-prospero2007population}{}
  Prospero, S., Hansen, E. M., Grünwald, N. J., and Winton, L. M. (2007).
  Population dynamics of the sudden oak death pathogen \emph{Phytophthora
  ramorum} in Oregon from 2001 to 2004. \emph{Molecular Ecology} 16,
  2958--2973.
  doi:\href{https://doi.org/10.1111/j.1365-294x.2007.03343.x}{10.1111/j.1365-294x.2007.03343.x}.
  
  \hypertarget{ref-prugnolle2010apparent}{}
  Prugnolle, F., and de Meeûs, T. (2010). Apparent high recombination
  rates in clonal parasitic organisms due to inappropriate sampling
  design. \emph{Heredity} 104, 135--140.
  doi:\href{https://doi.org/10.1038/hdy.2009.128}{10.1038/hdy.2009.128}.
  
  \hypertarget{ref-R2014}{}
  R Core Team (2014). \emph{R: A language and environment for statistical
  computing}. Vienna, Austria: R Foundation for Statistical Computing
  Available at: \url{https://www.R-project.org/}.
  
  \hypertarget{ref-R}{}
  R Core Team (2015). \emph{R: A language and environment for statistical
  computing}. Vienna, Austria: R Foundation for Statistical Computing
  Available at: \url{http://www.R-project.org/}.
  
  \hypertarget{ref-R2016}{}
  R Core Team (2016). \emph{R: A language and environment for statistical
  computing}. Vienna, Austria: R Foundation for Statistical Computing
  Available at: \url{https://www.R-project.org/}.
  
  \hypertarget{ref-rizzo2005phytophthora}{}
  Rizzo, D. M., Garbelotto, M., and Hansen, E. M. (2005).
  \emph{Phytophthora ramorum}: Integrative Research and Management of an
  Emerging Pathogen in California and Oregon Forests. \emph{Annu. Rev.
  Phytopathol.} 43, 309--335.
  doi:\href{https://doi.org/10.1146/annurev.phyto.42.040803.140418}{10.1146/annurev.phyto.42.040803.140418}.
  
  \hypertarget{ref-rizzo2002phytophthora}{}
  Rizzo, D. M., Garbelotto, M., Davidson, J. M., Slaughter, G. W., and
  Koike, S. T. (2002). \emph{Phytophthora ramorum} as the Cause of
  Extensive Mortality of Quercus spp. and Lithocarpus densiflorus in
  California. \emph{Plant Disease} 86, 205--214.
  doi:\href{https://doi.org/10.1094/pdis.2002.86.3.205}{10.1094/pdis.2002.86.3.205}.
  
  \hypertarget{ref-rosvall2008maps}{}
  Rosvall, M., and Bergstrom, C. T. (2008). Maps of random walks on
  complex networks reveal community structure. \emph{Proceedings of the
  National Academy of Sciences} 105, 1118--1123.
  
  \hypertarget{ref-shannon2001mathematical}{}
  Shannon, C. (2001). A mathematical theory of communication. \emph{ACM
  SIGMOBILE Mobile Computing and Communications Review} 5, 3--55.
  Available at:
  \url{http://cm.bell-labs.com/cm/ms/what/shannonday/shannon1948.pdf}.
  
  \hypertarget{ref-smith1993how}{}
  Smith, J. M., Smith, N. H., O'Rourke, M., and Spratt, B. G. (1993). How
  clonal are bacteria? \emph{Proceedings of the National Academy of
  Sciences} 90, 4384--4388.
  doi:\href{https://doi.org/10.1073/pnas.90.10.4384}{10.1073/pnas.90.10.4384}.
  
  \hypertarget{ref-sokal1958statistical}{}
  Sokal, R. R. (1958). A statistical method for evaluating systematic
  relationships. \emph{Univ Kans Sci Bull} 38, 1409--1438.
  
  \hypertarget{ref-stoddart1988genotypic}{}
  Stoddart, J. A., and Taylor, J. F. (1988). Genotypic diversity:
  Estimation and prediction in samples. \emph{Genetics} 118, 705--11.
  
  \hypertarget{ref-taylor2003fungal}{}
  Taylor, J. W., and Fisher, M. C. (2003). Fungal multilocus sequence
  typing -- it's not just for bacteria. \emph{Current opinion in
  microbiology} 6, 351--356.
  
  \hypertarget{ref-tyler2007phytophthora}{}
  Tyler, B. M. (2007). \emph{Phytophthora sojae}: root rot pathogen of
  soybean and model oomycete. \emph{Molecular Plant Pathology} 8, 1--8.
  doi:\href{https://doi.org/10.1111/j.1364-3703.2006.00373.x}{10.1111/j.1364-3703.2006.00373.x}.
  
  \hypertarget{ref-vercauteren2010clonal}{}
  Vercauteren, A., Dobbelaere, I. D., Grünwald, N. J., Bonants, P.,
  Bockstaele, E. V., Maes, M., et al. (2010). Clonal expansion of the
  Belgian \emph{Phytophthora ramorum} populations based on new
  microsatellite markers. \emph{Molecular Ecology} 19, 92--107.
  doi:\href{https://doi.org/10.1111/j.1365-294x.2009.04443.x}{10.1111/j.1365-294x.2009.04443.x}.
  
  \hypertarget{ref-vercauteren2011identification}{}
  Vercauteren, A., Larsen, M., Goss, E., Grunwald, N. J., Maes, M., and
  Heungens, K. (2011). Identification of new polymorphic microsatellite
  markers in the NA1 and NA2 lineages of \emph{Phytophthora ramorum}.
  \emph{Mycologia} 103, 1245--1249.
  doi:\href{https://doi.org/10.3852/10-420}{10.3852/10-420}.
  
  \hypertarget{ref-welch2000evidence}{}
  Welch, D. B. M., and Meselson, M. (2000). Evidence for the evolution of
  bdelloid rotifers without sexual reproduction or genetic exchange.
  \emph{Science} 288, 1211--1215.
  doi:\href{https://doi.org/10.1126/science.288.5469.1211}{10.1126/science.288.5469.1211}.
  
  \hypertarget{ref-welch2001rates}{}
  Welch, D. B. M., and Meselson, M. S. (2001). Rates of nucleotide
  substitution in sexual and anciently asexual rotifers. \emph{Proceedings
  of the National Academy of Sciences} 98, 6720--6724.
  doi:\href{https://doi.org/10.1073/pnas.111144598}{10.1073/pnas.111144598}.
  
  \hypertarget{ref-werres2001phytophthora}{}
  Werres, S., Marwitz, R., veld, W. A. M. I., Cock, A. W. D., Bonants, P.
  J., Weerdt, M. D., et al. (2001). \emph{Phytophthora ramorum sp. nov.},
  a new pathogen on Rhododendron and Viburnum. \emph{Mycological Research}
  105, 1155--1165.
  doi:\href{https://doi.org/10.1016/s0953-7562(08)61986-3}{10.1016/s0953-7562(08)61986-3}.
  
  \hypertarget{ref-wickham2009ggplot2}{}
  Wickham, H. (2009). \emph{ggplot2: Elegant Graphics for Data Analysis}.
  Springer-Verlag New York Available at: \url{http://ggplot2.org}.
  
  \hypertarget{ref-wickham2015devtools}{}
  Wickham, H., and Chang, W. (2015). \emph{Devtools: Tools to make
  developing R packages easier}. Available at:
  \url{http://CRAN.R-project.org/package=devtools}.
  
  \hypertarget{ref-winton2001molecular}{}
  Winton, L. M., and Hansen, E. M. (2001). Molecular diagnosis of
  \emph{Phytophthora lateralis} in trees, water, and foliage baits using
  multiplex polymerase chain reaction. \emph{Forest Pathol} 31, 275--283.
  doi:\href{https://doi.org/10.1046/j.1439-0329.2001.00251.x}{10.1046/j.1439-0329.2001.00251.x}.
  
  \hypertarget{ref-yoshida2013rise}{}
  Yoshida, K., Schuenemann, V. J., Cano, L. M., Pais, M., Mishra, B.,
  Sharma, R., et al. (2013). The rise and fall of the \emph{Phytophthora
  infestans} lineage that triggered the Irish potato famine. \emph{eLife}
  2, e00731.
  doi:\href{https://doi.org/10.7554/eLife.00731.001}{10.7554/eLife.00731.001}.


\end{document}

