% This is the Oregon State University LaTeX template. To the best of my
% knowledge, most of the work was done by those acknowledged in beavtex.cls.

%%
%% Preamble
%%
% \documentclass{<something>} must begin each LaTeX document
\documentclass[double,12pt]{beavtex}
% Added by CII
\usepackage{graphicx,latexsym}
\usepackage{amsmath}
\usepackage{amssymb,amsthm}
\usepackage{longtable,booktabs,setspace}
\usepackage[hyphens]{url}
\usepackage[colorlinks = true, 
			urlcolor = blue,
			linkcolor = black,
			citecolor = black,
			anchorcolor = black]{hyperref}
\usepackage{lmodern}
\usepackage{float}
\floatplacement{figure}{H}
% End of CII addition
\usepackage{rotating} % Package added to allow the rotation of figures and chart
                      % on a page, {sidewaysfigure} command
\usepackage{tablefootnote} % Packaged added to allow footnotes in the tabular
                           % environment, use \tablefootnote command

% This has to do with a default pandoc thing
% http://tex.stackexchange.com/a/258486/77699
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

% Added by CII (Thanks, Hadley!)
% Use ref for internal links
\renewcommand{\hyperref}[2][???]{\autoref{#1}}
\def\chapterautorefname{Chapter}
\def\sectionautorefname{Section}
\def\subsectionautorefname{Subsection}
% End of CII addition

% Added by CII
\usepackage{caption}
\captionsetup{width=5in}
% End of CII addition

\title{An Analysis of Something} % {An Analysis of Something}
\author{Zhian N. Kamvar} % {Joseph A. Student}
\degree{Doctor of Philosophy} % {Master of Science}
\doctype{Dissertation}
\submitdate{December 6, 2016} % {January 1, 2013}
\commencementyear{2017} % {2013}

\department{Botany and Plant Pathology} % {Nuclear Engineering and Radiation Health Physics}

\depttype{Department} % {School}

\depthead{Head} % {Director}

\major{Plant Pathology} % {Radiation Health Physics}

\advisor{Niklaus J. Grünwald} % {Jane R. Professor}

\abstract{This is a \LaTeX template derived from the \beavtex template found on
overleaf
(\url{https://www.overleaf.com/latex/templates/oregon-state-university-thesis-and-dissertation/wnvzcdhqshxf})
and cloned using
\texttt{git\ clone\ https://git.overleaf.com/5973847rnpkdf} I have made
the following changes

\begin{itemize}
\tightlist
\item
  modified \texttt{beavtex.cls} from v1.1 to include support for
  verbatim font styles in R code
\item
  stripped the template of its original content to create a pandoc
  template
\item
  placed this into a bookdown (\url{https://bookdown.org/}) framework
\item
  added packages latexsym, amsmath amssymb,amsthm, longtable, booktabs,
  setspace, url, hyperref, lmodern, caption, and float
\end{itemize}

This is still a WIP, but it seems to work for the moment. Zhian N.
Kamvar 2016-08-25}
\acknowledgements{I would like to acknowledge\ldots{}Lorem ipsum dolor sit amet,
consectetur adipiscing elit. Maecenas vel eros sed mauris porttitor
semper nec a orci. Nullam vestibulum mi nec condimentum posuere.
Pellentesque eget diam id sapien aliquet ullamcorper. Pellentesque
blandit nec lectus ut mollis. Praesent in facilisis justo. Vestibulum
ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia
Curae; Sed eget congue leo, sed consequat libero. In rutrum malesuada
nisi. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices
posuere cubilia Curae; Morbi sollicitudin tortor ut sem facilisis
mollis.}


	\contributors{The following people contributed to this dissertation:

\subsubsection*{Chapter 1}\label{chapter-1}
\addcontentsline{toc}{subsubsection}{Chapter 1}

Jane R. Professor assisted in the design, analysis, and editing of the
manuscript.

\subsubsection*{Chapter 2}\label{chapter-2}
\addcontentsline{toc}{subsubsection}{Chapter 2}

Lisa Simpson developed the initial concept and experimental design for
the study. Ellen Ripley advised and assisted with statistical analysis.
Jane R. Professor assisted in the design, analysis, and editing of the
manuscript.

\subsubsection*{Chapter 3}\label{chapter-3}
\addcontentsline{toc}{subsubsection}{Chapter 3}

Jane R. Professor assisted in the design, analysis, and editing of the
manuscript.

\subsubsection*{Chapter 4}\label{chapter-4}
\addcontentsline{toc}{subsubsection}{Chapter 4}

Jane R. Professor assisted in the design, analysis, and editing of the
manuscript.}


	\dedication{This is for my mother who paved the way.}




\begin{document}

\maketitle
\mainmatter


  \chapter{Introduction}\label{introduction}
  
  The objectives of my work were two-fold and included
  
  \begin{enumerate}
  \def\labelenumi{\arabic{enumi}.}
  \tightlist
  \item
    development of computational tools to characterize populations and
  \item
    application of these tools to populations of the plant pathogen genus
    \emph{Phytophthora}.
  \end{enumerate}
  
  \section{Population genetics of clonal
  organisms}\label{population-genetics-of-clonal-organisms}
  
  Clonal populations are a special case \ldots{}
  
  \section{\texorpdfstring{The Genus
  \emph{Phytophthora}}{The Genus Phytophthora}}\label{the-genus-phytophthora}
  
  The genus \emph{Phytophthora}, translating to ``plant destroyer'' in
  Greek, contains over 100 species (Kroon \emph{et al.} 2012), many of
  which have significant impact on US agriculture. \emph{P. sojae} is a
  major problem on soybean, causing \$1-2 billion in losses each year
  (Tyler 2007). \emph{P. ramorum} is changing the landscape of the North
  American West due to its wide host range (Grünwald \emph{et al.} 2008),
  which results in devastating losses for the US forestry and nursery
  industries. And \emph{P. infestans}, which was a root cause of over a
  million deaths during the Irish Potato Famine and continues to be a
  problem on tomato and potato crops, resulting in losses exceeding \$6
  billion, annually (Haas \emph{et al.} 2009). \emph{Phytophthora spp.}
  are water molds characterized by production of oospores and biflagellate
  zoospores that place them into the Stramenopiles (Baldauf 2003). They
  are most closely related to golden brown algae and quite diverged from
  fungi. One of the distinguishing molecular characteristics of
  \emph{Phytophthora spp.} are the presence of effector proteins with an
  RxLR motif. These proteins are necessary to confer virulence against the
  host and they are under extreme diversifying selection within species
  (Haas \emph{et al.} 2009; Yoshida \emph{et al.} 2013).
  
  \subsection{life cycle}\label{life-cycle}
  
  \subsection{Sex and mating types}\label{sex-and-mating-types}
  
  \subsection{\texorpdfstring{Heterothallic, clonal: \emph{P.
  ramorum}}{Heterothallic, clonal: P. ramorum}}\label{heterothallic-clonal-p.-ramorum}
  
  \begin{itemize}
  \tightlist
  \item
    Sudden Oak Death
  \item
    Population genetics in US nurseries (Goss \emph{et al.} 2009)
  \end{itemize}
  
  \subsection{\texorpdfstring{Homothallic, partially clonal: \emph{P.
  syringae}}{Homothallic, partially clonal: P. syringae}}\label{homothallic-partially-clonal-p.-syringae}
  
  \begin{itemize}
  \tightlist
  \item
    Abundant in OR Nurseries (found in foliar isolates) (Parke \emph{et
    al.} 2014)
  \item
    Genetic structure uncharacterized
  \end{itemize}
  
  \section{Tools for analysis of clonal population
  genetics}\label{tools-for-analysis-of-clonal-population-genetics}
  
  Recommendations have been made for analysis (Arnaud-Hanod \emph{et al.}
  2007)
  
  \begin{itemize}
  \tightlist
  \item
    MLG diversity
  \item
    Genotype Accumulation Curve
  \item
    \(P_{sex}\) and \(P_{gen}\)
  \end{itemize}
  
  \subsection{Index of Association}\label{index-of-association}
  
  \begin{itemize}
  \tightlist
  \item
    Standardized Version (Agapow \& Burt 2001)
  \item
    Previous Simulation Analyses (de Meeûs \& Balloux 2004)
  \item
    What's missing
  
    \begin{itemize}
    \tightlist
    \item
      Sympatric clonal lineages
    \item
      Analysis of significance testing
    \item
      HTS markers
    \end{itemize}
  \end{itemize}
  
  \subsection{Software Limitations}\label{software-limitations}
  
  Plethora of tools, most designed for sexual populations except:
  
  \begin{itemize}
  \tightlist
  \item
    GenClone
  \item
    GenoDive
  \end{itemize}
  
  Problems with file formatting, time, and reproducibility.
  
  \subsection{poppr}\label{poppr}
  
  \begin{itemize}
  \tightlist
  \item
    R
  \item
    poppr
  \end{itemize}
  
  \section{Applications of novel tools for analysis of partially-clonal
  populations}\label{applications-of-novel-tools-for-analysis-of-partially-clonal-populations}
  
  \begin{itemize}
  \tightlist
  \item
    Simulation analysis of \(\bar{r}_d\)
  \item
    SSR analysis of \emph{P. ramorum}
  \item
    GBS analysis of \emph{P. syringae}
  \end{itemize}
  
  \section{Conclusion}\label{conclusion}
  
  \begin{itemize}
  \tightlist
  \item
    Open Source Scientific Software Development of Poppr
  
    \begin{itemize}
    \tightlist
    \item
      Related tools
    \end{itemize}
  \item
    Simulation Analysis
  \item
    Pop gen info for two \emph{Phytophthoras}
  \item
    Major results of your work in 2-4 sentences \ldots{}
  \end{itemize}
  
  \chapter{\texorpdfstring{\emph{Poppr}: an R Package For Genetic Analysis
  of Populations With Clonal, Partially Clonal, and/or Sexual
  Reproduction}{Poppr: an R Package For Genetic Analysis of Populations With Clonal, Partially Clonal, and/or Sexual Reproduction}}\label{poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction}
  
  \section{Abstract}\label{abstract}
  
  Many microbial, fungal, or oomcyete populations violate assumptions for
  population genetic analysis because these populations are clonal,
  admixed, partially clonal, and/or sexual. Furthermore, few tools exist
  that are specifically designed for analyzing data from clonal
  populations, making analysis difficult and haphazard. We developed the R
  package poppr providing unique tools for analysis of data from admixed,
  clonal, mixed, and/or sexual populations. Currently, poppr can be used
  for dominant/codominant and haploid/diploid genetic data. Data can be
  imported from several formats including GenAlEx formatted text files and
  can be analyzed on a user-defined hierarchy that includes unlimited
  levels of subpopulation structure and clone censoring. New functions
  include calculation of Bruvo's distance for microsatellites,
  batch-analysis of the index of association with several indices of
  genotypic diversity, and graphing including dendrograms with bootstrap
  support and minimum spanning networks. While functions for genotypic
  diversity and clone censoring are specific for clonal populations,
  several functions found in poppr are also valuable to analysis of any
  populations. A manual with documentation and examples is provided. Poppr
  is open source and major releases are available on CRAN:
  \url{http://cran.r-project.org/package=poppr}. More supporting
  documentation and tutorials can be found under `resources' at:
  \url{http://grunwaldlab.cgrb.oregonstate.edu/}.
  
  \chapter{Spatial and Temporal Analysis of Populations of the Sudden Oak
  Death Pathogen in Oregon
  Forests}\label{spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests}
  
  \section{Abstract}\label{abstract-1}
  
  Sudden oak death caused by the oomycete \emph{Phytophthora ramorum} was
  first discovered in California toward the end of the 20th century and
  subsequently emerged on tanoak forests in Oregon before its first
  detection in 2001 by aerial surveys. The Oregon Department of Forestry
  has since monitored the epidemic and sampled symptomatic tanoak trees
  from 2001 to the present. Populations sampled over this period were
  genotyped using microsatellites and studied to infer the population
  genetic history. To date, only the NA1 clonal lineage is established in
  this region, although three lineages exist on the North American west
  coast. The original introduction into the Joe Hall area eventually
  spread to several regions: mostly north but also east and southwest. A
  new introduction into Hunter Creek appears to correspond to a second
  introduction not clustering with the early introduction. Our data are
  best explained by both introductions originating from nursery
  populations in California or Oregon and resulting from two distinct
  introduction events. Continued vigilance and eradication of nursery
  populations of \emph{P. ramorum} are important to avoid further
  emergence and potential introduction of other clonal lineages.
  
  \chapter{Novel R Tools For Analysis of Genome-Wide Population Genetic
  Data With Emphasis on
  Clonality}\label{novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality}
  
  \section{Abstract}\label{abstract-2}
  
  To gain a detailed understanding of how plant microbes evolve and adapt
  to hosts, pesticides, and other factors, knowledge of the population
  dynamics and evolutionary history of populations is crucial. Plant
  pathogen populations are often clonal or partially clonal which requires
  different analytical tools. With the advent of high throughput
  sequencing technologies, obtaining genome-wide population genetic data
  has become easier than ever before. We previously contributed the R
  package \emph{poppr} specifically addressing issues with analysis of
  clonal populations. In this paper we provide several significant
  extensions to \emph{poppr} with a focus on large, genome-wide SNP data.
  Specifically, we provide several new functionalities including the new
  function \texttt{mlg.filter} to define clone boundaries allowing for
  inspection and definition of what is a clonal lineage, minimum spanning
  networks with reticulation, a sliding-window analysis of the index of
  association, modular bootstrapping of any genetic distance, and analyses
  across any level of hierarchies.
  
  \chapter{\texorpdfstring{{[}Tentative Title{]} Population Dynamics of
  the Plant Pathogen \emph{Phytophthora syringae} in Oregon
  Nurseries}{{[}Tentative Title{]} Population Dynamics of the Plant Pathogen Phytophthora syringae in Oregon Nurseries}}\label{tentative-title-population-dynamics-of-the-plant-pathogen-phytophthora-syringae-in-oregon-nurseries}
  
  \section{Abstract}\label{abstract-3}
  
  \section{Introduction}\label{introduction-1}
  
  \emph{Phytophthora syringae} is the most important species affecting
  ornamentals produced in the Pacific Northwest. Recent nursery sampling
  efforts, aimed at characterizing the diversity of \emph{Phytophthoras}
  within Oregon nurseries, have revealed the species \emph{P. syringae} to
  be among the most abundant taxa found in the nurseries surveyed (Parke
  \emph{et al.} 2014). \emph{P. syringae} is adapted to cold weather and
  grows best in the cool, wet fall, winter and spring and is least active
  in summer (Erwin \emph{et al.} 1996). Like \emph{P. ramorum}, it has a
  wide host range including \emph{Rhododendron}, \emph{Camellia},
  \emph{Malus}, and many other taxa. It has the capability for
  outcrossing, self-fertilizing, and reproducing clonally. This pathogen
  has been found globally since 1881 and is problematic on woody
  ornamentals such as crabapple (\emph{Malus spp.}), as it causes
  unsightly cankers that make the plant unsellable (Erwin \emph{et al.}
  1996). While the ecology of this pathogen has been studied to some
  degree, very little is known about the demographic history and
  population structure on a local and global scale.
  
  \chapter{{[}Tentative Title{]} The Effect of Population Dynamics, Sample
  Size, and Marker Choice on the Index of
  Association}\label{tentative-title-the-effect-of-population-dynamics-sample-size-and-marker-choice-on-the-index-of-association}
  
  \section{Abstract}\label{abstract-4}
  
  TBD\ldots{}
  
  \section{Introduction}\label{introduction-2}
  
  \begin{itemize}
  \tightlist
  \item
    Population Genetics of partially clonal organisms
  
    \begin{itemize}
    \tightlist
    \item
      This has been studied in the past (Orive 1993; Smith \emph{et al.}
      1993; Balloux \emph{et al.} 2003; de Meeûs \& Balloux 2004)
    \item
      The index of association (Brown \emph{et al.} 1980; Smith \emph{et
      al.} 1993; Agapow \& Burt 2001)
    \item
      In de Meeûs \& Balloux (2004), it was shown that \(\bar{r}_d\) has a
      high variance in clonal populations and Smith \emph{et al.} (1993)
      showed that it's affected by population structure.
    \end{itemize}
  \item
    Methods for assessing level of clonal reproduction (CloNcaSe) (Ali
    \emph{et al.} 2016)
  
    \begin{itemize}
    \tightlist
    \item
      This only works for populations with discrete generations with an
      observable sexual stage.
    \end{itemize}
  \item
    Limitations of previous studies
  
    \begin{itemize}
    \tightlist
    \item
      No HTS markers
    \item
      Significance tests are routinely performed via permutation analysis,
      but could not be performed due to software limitations (Burt
      \emph{et al.} 1996; de Meeûs \& Balloux 2004)
    \end{itemize}
  \item
    Objectives
  
    \begin{enumerate}
    \def\labelenumi{\arabic{enumi}.}
    \tightlist
    \item
      Analyze mixtures of clonal populations to assess effect of sampling
      multiple clonal populations
    \item
      Re-analyze rates of sexual reproduction to confirm previous study
    \item
      Assess significance tests for \(\bar{r}_d\)
    \end{enumerate}
  \end{itemize}
  
  \section{Methods}\label{methods}
  
  All simulations were performed with the python package simuPOP version
  1.1.7 in python version 3.4. For each scenario, 100 simulations with 10
  replicates were created with a census size of 10,000 individuals evolved
  over 10,000 generations. From each replicate, 10, 25, 50, and 100
  individuals were sampled without replacement for downstream analysis.
  
  \subsection{Microsatellite Simulation}\label{microsatellite-simulation}
  
  Each population was simulated with 20 co-dominant loci containing 6 to
  10 alleles with frequencies drawn from a uniform distribution and
  normalized. Before mating, mutations occurred at each locus at a rate of
  1e-5 mutations/generation.
  
  \subsection{Microsatellite Analysis}\label{microsatellite-analysis}
  
  The standardized index of association (\(\bar{r}_d\)) was calculated in
  R version 3.2 with the package \emph{poppr} version 2.2.1 using the
  function \texttt{ia()} within custom scripts (supplementary
  information). Tests for significance were performed by randomly
  permuting the alleles at each locus independently and then assessing
  \(\bar{r}_d\). This was done 999 times for each replicate population.
  The p-values reflect the proportion of observations greater than the
  observed statistic.
  
  \subsection{GBS Simulations}\label{gbs-simulations}
  
  Simulations of 10,000 binary loci at intervals of 1 Mbp over 100
  chromosomal fragments were simulated with a mutation rate of 1e-5
  mutations per generation and a recombination rate of 1e-5 for sexually
  recombining populations.
  
  \backmatter
  
  \chapter{References}\label{references}
  
  \noindent
  
  \setlength{\parindent}{-0.20in} \setlength{\leftskip}{0.20in}
  \setlength{\parskip}{8pt}
  
  \hypertarget{refs}{}
  \hypertarget{ref-Agapow_2001}{}
  Agapow P-M, Burt A (2001) Indices of multilocus linkage disequilibrium.
  \emph{Molecular Ecology Notes}, \textbf{1}, 101--102.
  
  \hypertarget{ref-ali2016cloncase}{}
  Ali S, Soubeyrand S, Gladieux P \emph{et al.} (2016) Cloncase:
  Estimation of sex frequency and effective population size by clonemate
  resampling in partially clonal organisms. \emph{Molecular ecology
  resources}.
  
  \hypertarget{ref-arnaud2007standardizing}{}
  Arnaud-Hanod S, Duarte CM, Alberto F, Serrão EA (2007) Standardizing
  methods to address clonality in population studies. \emph{Molecular
  Ecology}, \textbf{16}, 5115--5139.
  
  \hypertarget{ref-baldauf2003deep}{}
  Baldauf S (2003) The deep roots of eukaryotes. \emph{Science},
  \textbf{300}, 1703--1706.
  
  \hypertarget{ref-balloux2003population}{}
  Balloux F, Lehmann L, de Meeûs T (2003) The population genetics of
  clonal and partially clonal diploids. \emph{Genetics}, \textbf{164},
  1635--1644.
  
  \hypertarget{ref-brown1980multilocus}{}
  Brown A, Feldman M, Nevo E (1980) Multilocus structure of natural
  populations of \emph{Hordeum spontaneum}. \emph{Genetics}, \textbf{96},
  523--536.
  
  \hypertarget{ref-burt1996molecular}{}
  Burt A, Carter DA, Koenig GL, White TJ, Taylor JW (1996) Molecular
  markers reveal cryptic sex in the human pathogen \emph{Coccidioides
  immitis}. \emph{Proceedings of the National Academy of Sciences},
  \textbf{93}, 770--773.
  
  \hypertarget{ref-de2004clonal}{}
  de Meeûs T, Balloux F (2004) Clonal reproduction and linkage
  disequilibrium in diploids: A simulation study. \emph{Infection,
  genetics and evolution}, \textbf{4}, 345--351.
  
  \hypertarget{ref-erwin1996phytophthora}{}
  Erwin DC, Ribeiro OK, others (1996) \emph{Phytophthora diseases
  worldwide}. American Phytopathological Society (APS Press), St. Paul,
  Minnesota, USA.
  
  \hypertarget{ref-goss2009population}{}
  Goss EM, Larsen M, Chastagner GA, Givens DR, Grünwald NJ (2009)
  Population genetic analysis infers migration pathways of
  \emph{Phytophthora ramorum} in US nurseries. \emph{PLoS Pathog},
  \textbf{5}, e1000583.
  
  \hypertarget{ref-grunwald2008phytophthora}{}
  Grünwald NJ, Goss EM, Press CM (2008) \emph{Phytophthora ramorum}: a
  pathogen with a remarkably wide host range causing sudden oak death on
  oaks and ramorum blight on woody ornamentals. \emph{Molecular Plant
  Pathology}, \textbf{9}, 729--740.
  
  \hypertarget{ref-grunwald2011phytophthora}{}
  Grünwald NJ, Martin FN, Larsen MM \emph{et al.} (2011)
  Phytophthora-ID.org: a sequence-based \emph{Phytophthora} identification
  tool. \emph{Plant Disease}, \textbf{95}, 337--342.
  
  \hypertarget{ref-haas2009genome}{}
  Haas BJ, Kamoun S, Zody MC \emph{et al.} (2009) Genome sequence and
  analysis of the Irish potato famine pathogen \emph{Phytophthora
  infestans}. \emph{Nature}, \textbf{461}, 393--398.
  
  \hypertarget{ref-kroon2012genus}{}
  Kroon LP, Brouwer H, Cock AW de, Govers F (2012) The Genus
  \emph{Phytophthora}. \emph{Phytopathology}, \textbf{102}, 348--364.
  
  \hypertarget{ref-orive1993effective}{}
  Orive ME (1993) Effective population size in organisms with complex
  life-histories. \emph{Theoretical population biology}, \textbf{44},
  316--340.
  
  \hypertarget{ref-parke2014phytophthora}{}
  Parke JL, Knaus BJ, Fieland VJ, Lewis C, Grünwald NJ (2014) Phytophthora
  community structure analyses in Oregon nurseries inform systems
  approaches to disease management. \emph{Phytopathology}, \textbf{104},
  1052--1062.
  
  \hypertarget{ref-smith1993how}{}
  Smith JM, Smith NH, O'Rourke M, Spratt BG (1993) How clonal are
  bacteria? \emph{Proceedings of the National Academy of Sciences},
  \textbf{90}, 4384--4388.
  
  \hypertarget{ref-tyler2007phytophthora}{}
  Tyler BM (2007) \emph{Phytophthora sojae}: root rot pathogen of soybean
  and model oomycete. \emph{Molecular plant pathology}, \textbf{8}, 1--8.
  
  \hypertarget{ref-yoshida2013rise}{}
  Yoshida K, Schuenemann VJ, Cano LM \emph{et al.} (2013) The rise and
  fall of the \emph{Phytophthora infestans} lineage that triggered the
  Irish potato famine. \emph{Elife}, \textbf{2}, e00731.


\end{document}

