% This is the Oregon State University LaTeX template. To the best of my
% knowledge, most of the work was done by those acknowledged in beavtex.cls.

%%
%% Preamble
%%
% \documentclass{<something>} must begin each LaTeX document
\documentclass[double,12pt]{beavtex}
% Added by CII
\usepackage{graphicx,latexsym}
\usepackage{amsmath}
\usepackage{amssymb,amsthm}
\usepackage{longtable,booktabs,setspace}
\usepackage[hyphens]{url}
\usepackage[colorlinks = true, 
			urlcolor = blue,
			linkcolor = black,
			citecolor = black,
			anchorcolor = black]{hyperref}
\usepackage{lmodern}
\usepackage{float}
\floatplacement{figure}{H}
% End of CII addition
\usepackage{rotating} % Package added to allow the rotation of figures and chart
                      % on a page, {sidewaysfigure} command
\usepackage{tablefootnote} % Packaged added to allow footnotes in the tabular
                           % environment, use \tablefootnote command

% This has to do with a default pandoc thing
% http://tex.stackexchange.com/a/258486/77699
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

% Added by CII (Thanks, Hadley!)
% Use ref for internal links
\renewcommand{\hyperref}[2][???]{\autoref{#1}}
\def\chapterautorefname{Chapter}
\def\sectionautorefname{Section}
\def\subsectionautorefname{Subsection}
% End of CII addition

% Added by CII
\usepackage{caption}
\captionsetup{width=5in}
% End of CII addition

\title{An Analysis of Something} % {An Analysis of Something}
\author{Zhian N. Kamvar} % {Joseph A. Student}
\degree{Doctor of Philosophy} % {Master of Science}
\doctype{Dissertation}
\submitdate{January 1, 2013} % {January 1, 2013}
\commencementyear{2017} % {2013}

\department{Botany and Plant Pathology} % {Nuclear Engineering and Radiation Health Physics}

\depttype{Department} % {School}

\depthead{Head} % {Director}

\major{Plant Pathology} % {Radiation Health Physics}

\advisor{Niklaus J. Grünwald} % {Jane R. Professor}

\abstract{This is a \LaTeX template derived from the \beavtex template found on
overleaf
(\url{https://www.overleaf.com/latex/templates/oregon-state-university-thesis-and-dissertation/wnvzcdhqshxf})
and cloned using
\texttt{git\ clone\ https://git.overleaf.com/5973847rnpkdf} I have made
the following changes

\begin{itemize}
\tightlist
\item
  modified \texttt{beavtex.cls} from v1.1 to include support for
  verbatim font styles in R code
\item
  stripped the template of its original content to create a pandoc
  template
\item
  placed this into a bookdown (\url{https://bookdown.org/}) framework
\item
  added packages latexsym, amsmath amssymb,amsthm, longtable, booktabs,
  setspace, url, hyperref, lmodern, caption, and float
\end{itemize}

This is still a WIP, but it seems to work for the moment. Zhian N.
Kamvar 2016-08-25}
\acknowledgements{I would like to acknowledge\ldots{}Lorem ipsum dolor sit amet,
consectetur adipiscing elit. Maecenas vel eros sed mauris porttitor
semper nec a orci. Nullam vestibulum mi nec condimentum posuere.
Pellentesque eget diam id sapien aliquet ullamcorper. Pellentesque
blandit nec lectus ut mollis. Praesent in facilisis justo. Vestibulum
ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia
Curae; Sed eget congue leo, sed consequat libero. In rutrum malesuada
nisi. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices
posuere cubilia Curae; Morbi sollicitudin tortor ut sem facilisis
mollis.}


	\contributors{The following people contributed to this dissertation:

\subsubsection*{Chapter 1}\label{chapter-1}
\addcontentsline{toc}{subsubsection}{Chapter 1}

Jane R. Professor assisted in the design, analysis, and editing of the
manuscript.

\subsubsection*{Chapter 2}\label{chapter-2}
\addcontentsline{toc}{subsubsection}{Chapter 2}

Lisa Simpson developed the initial concept and experimental design for
the study. Ellen Ripley advised and assisted with statistical analysis.
Jane R. Professor assisted in the design, analysis, and editing of the
manuscript.

\subsubsection*{Chapter 3}\label{chapter-3}
\addcontentsline{toc}{subsubsection}{Chapter 3}

Jane R. Professor assisted in the design, analysis, and editing of the
manuscript.

\subsubsection*{Chapter 4}\label{chapter-4}
\addcontentsline{toc}{subsubsection}{Chapter 4}

Jane R. Professor assisted in the design, analysis, and editing of the
manuscript.}


	\dedication{This is for my mother who paved the way.}




\begin{document}

\maketitle
\mainmatter


  \chapter{Introduction}\label{introduction}
  
  The objectives of my work were two-fold and included 1. development of
  computational tools to characterize populations and 2. application of
  these tools to populations of the plant pathogen \emph{Phytophthora}.
  
  \section{What is population genetics}\label{what-is-population-genetics}
  
  \section{Need for tools for population
  genetics}\label{need-for-tools-for-population-genetics}
  
  Clonal pathogens are a special case \ldots{}
  
  In the past need to reformat files for each program and use of a
  plethora of programs
  
  The R plafrorm
  
  The need for poppr
  
  Ia
  
  Educational tools
  
  \section{\texorpdfstring{Introduction to
  \emph{Phytophthora}}{Introduction to Phytophthora}}\label{introduction-to-phytophthora}
  
  What is the genus \emph{Phytophthora}.
  
  \emph{P. ramorum}
  
  \emph{P. syringae}
  
  \section{Conclusions}\label{conclusions}
  
  \section{Literature cited}\label{literature-cited}
  
  \chapter{\texorpdfstring{\emph{Poppr}: an R Package For Genetic Analysis
  of Populations With Clonal, Partially Clonal, and/or Sexual
  Reproduction}{Poppr: an R Package For Genetic Analysis of Populations With Clonal, Partially Clonal, and/or Sexual Reproduction}}\label{poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction}
  
  \section{Abstract}\label{abstract}
  
  Many microbial, fungal, or oomcyete populations violate assumptions for
  population genetic analysis because these populations are clonal,
  admixed, partially clonal, and/or sexual. Furthermore, few tools exist
  that are specifically designed for analyzing data from clonal
  populations, making analysis difficult and haphazard. We developed the R
  package poppr providing unique tools for analysis of data from admixed,
  clonal, mixed, and/or sexual populations. Currently, poppr can be used
  for dominant/codominant and haploid/diploid genetic data. Data can be
  imported from several formats including GenAlEx formatted text files and
  can be analyzed on a user-defined hierarchy that includes unlimited
  levels of subpopulation structure and clone censoring. New functions
  include calculation of Bruvo's distance for microsatellites,
  batch-analysis of the index of association with several indices of
  genotypic diversity, and graphing including dendrograms with bootstrap
  support and minimum spanning networks. While functions for genotypic
  diversity and clone censoring are specific for clonal populations,
  several functions found in poppr are also valuable to analysis of any
  populations. A manual with documentation and examples is provided. Poppr
  is open source and major releases are available on CRAN:
  \url{http://cran.r-project.org/package=poppr}. More supporting
  documentation and tutorials can be found under `resources' at:
  \url{http://grunwaldlab.cgrb.oregonstate.edu/}.
  
  \chapter{Spatial and Temporal Analysis of Populations of the Sudden Oak
  Death Pathogen in Oregon
  Forests}\label{spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests}
  
  \section{Abstract}\label{abstract-1}
  
  Sudden oak death caused by the oomycete \emph{Phytophthora ramorum} was
  first discovered in California toward the end of the 20th century and
  subsequently emerged on tanoak forests in Oregon before its first
  detection in 2001 by aerial surveys. The Oregon Department of Forestry
  has since monitored the epidemic and sampled symptomatic tanoak trees
  from 2001 to the present. Populations sampled over this period were
  genotyped using microsatellites and studied to infer the population
  genetic history. To date, only the NA1 clonal lineage is established in
  this region, although three lineages exist on the North American west
  coast. The original introduction into the Joe Hall area eventually
  spread to several regions: mostly north but also east and southwest. A
  new introduction into Hunter Creek appears to correspond to a second
  introduction not clustering with the early introduction. Our data are
  best explained by both introductions originating from nursery
  populations in California or Oregon and resulting from two distinct
  introduction events. Continued vigilance and eradication of nursery
  populations of \emph{P. ramorum} are important to avoid further
  emergence and potential introduction of other clonal lineages.
  
  \chapter{Novel R Tools For Analysis of Genome-Wide Population Genetic
  Data With Emphasis on
  Clonality}\label{novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality}
  
  \section{Abstract}\label{abstract-2}
  
  To gain a detailed understanding of how plant microbes evolve and adapt
  to hosts, pesticides, and other factors, knowledge of the population
  dynamics and evolutionary history of populations is crucial. Plant
  pathogen populations are often clonal or partially clonal which requires
  different analytical tools. With the advent of high throughput
  sequencing technologies, obtaining genome-wide population genetic data
  has become easier than ever before. We previously contributed the R
  package \emph{poppr} specifically addressing issues with analysis of
  clonal populations. In this paper we provide several significant
  extensions to \emph{poppr} with a focus on large, genome-wide SNP data.
  Specifically, we provide several new functionalities including the new
  function \texttt{mlg.filter} to define clone boundaries allowing for
  inspection and definition of what is a clonal lineage, minimum spanning
  networks with reticulation, a sliding-window analysis of the index of
  association, modular bootstrapping of any genetic distance, and analyses
  across any level of hierarchies.
  
  \chapter{\texorpdfstring{{[}Tentative Title{]} Population Dynamics of
  the Plant Pathogen \emph{Phytophthora syringae} in Oregon
  Nurseries}{{[}Tentative Title{]} Population Dynamics of the Plant Pathogen Phytophthora syringae in Oregon Nurseries}}\label{tentative-title-population-dynamics-of-the-plant-pathogen-phytophthora-syringae-in-oregon-nurseries}
  
  \section{Abstract}\label{abstract-3}
  
  TBD\ldots{}
  
  \chapter{{[}Tentative Title{]} The Effect of Population Dynamics, Sample
  Size, and Marker Choice on the Index of
  Association}\label{tentative-title-the-effect-of-population-dynamics-sample-size-and-marker-choice-on-the-index-of-association}
  
  \section{Abstract}\label{abstract-4}
  
  TBD\ldots{}
  
  \backmatter
  
  \chapter{References}\label{references}
  
  \noindent
  
  \setlength{\parindent}{-0.20in} \setlength{\leftskip}{0.20in}
  \setlength{\parskip}{8pt}
  
  \hypertarget{refs}{}
  \hypertarget{ref-grunwald2011phytophthora}{}
  Grünwald, N. J., Martin, F. N., Larsen, M. M., Sullivan, C. M., Press,
  C. M., Coffey, M. D., \ldots{} Parke, J. L. (2011). Phytophthora-ID.
  org: a sequence-based \emph{Phytophthora} identification tool.
  \emph{Plant Disease}, \emph{95}(3), 337--342.


\end{document}

