# Introduction


The work presented here presents tools designed to answer questions related to
clonal population genetics and plant pathology. The term 'tool' in this context
is not in reference to new derivations of mathematical models or theory for
population genetics of clonal organisms; rather, 'tool' refers to software code 
used to apply mathematical models or theory to population genetic data. The 
merit of this work lies within the context of reproducible science 
[@buckheit1995wavelab]. It provides the tools and demonstrates the usefulness
and flexibility by applying these tools to the outbreak of sudden oak death from
2001 to 2014 in Curry County, OR, USA and to simulation data, assessing the
power of the index of association to detect clonal reproduction.

## Plant Pathology and Clonal Population Genetics

Sex exists in all major groups of organisms [@hietman2012evolution]. 

Pathogenic species can use reproductive mode to adapt to different environmental
factors.

Knowing the reproductive mode of a pathogen can help you understand how to
develop management strategies e.g. if you have a clonal population, you want to
make sure you capture the diversity of genotypes before developing 
anti-microbial treatments [@taylor1999evolutionary].

## Scientific Software and Reproducible Research

[kamoun2015pubpeer]: https://pubpeer.com/topics/1/877A42B12A198A44306771E0D80AB8

@buckheit1995wavelab didn't invent the concept of reproducible research in
scientific computing, but they did show computational research community that
such a concept was possible. While the original goal of providing the environment used to generate figures seems excessive, it indeed has real world implications even in plant pathology; case in point, Sophien Kamoun's paper from 2012 was targeted on the post-publication review site http://pubpeer.com: [kamoun2015pubpeer]. The paper had a single duplicated image in a figure. In response, Kamoun's team found the original file and realized that the image had accidentally been pasted on top of the original. 

## Part 1: Tools

### Summary of Chapter 2

To address the lack of tools for reproducible research in clonal population
genetics, we present the software package *poppr* in the R computing language
[@R2016; @kamvar2014poppr]. Previously, tools necessary for analysis of clonal
populations were available in several stand-alone software programs, each
requiring different data input formats. Moreover, each program had different
levels of documentation and limited support for all computing platforms. The
novelty of *poppr* was to introduce indices of multilocus genotype diversity,
the index of association, and a fast implementation of Bruvo's genetic distance
over unlimited levels of a user-specified population hierarchy
[@arnaud2007standardizing; @Agapow_2001; @bruvo2004simple]. Because this was
implemented in R these analyses could be performed in a reproducible manner on
all computing platforms.

### Summary of Chapter 3

The initial implementation of *poppr* contained basic tools for analysis of
clonal populations [@kamvar2014poppr], but lacked tools for custom definitions
of multilocus genotypes and performed poorly with genomic-scale data. Chapter 3
introduces an updated and improved *poppr* version 2.0. With high throughput
sequencing (HTS) data, the amount of missing data and genotyping error
increases, and the definition of a multilocus genotype becomes unclear.
Moreover, the calculation of the index of association scales poorly with an
increase in the number of loci. To address these limitations, we improved poppr
with new functionalities to define multilocus genotypes based on genetic
distance and calculate the index of association over random samples or windows
of SNP loci.

## Part 2: Applications



### Summary of Chapter 4

