# Introduction

```{r, introduction1, echo = FALSE, fig.cap = figcap1, fig.scap = figscap1, out.width = "50%"}
knitr::include_graphics("figure/introduction/dissertation.png")

figcap1 <- "A graphical representation of the three themes governing the work
presented in this dissertation." -> figscap1
```

The work presented here presents tools designed to answer questions related to
clonal population genetics and plant pathology. In this context, the term 'tool'
is not in reference to new derivations of mathematical models or theory for
population genetics of clonal organisms; rather, 'tool' refers to software code 
used to apply mathematical models or theory to population genetic data. The 
merit of this work lies within the context of reproducible science 
[@buckheit1995wavelab]. We demonstrate the usefulness and flexibility of these
tools by using them to show evidence for multiple introductions in the Curry
County, OR outbreak of sudden oak death, and by assessing the power of
multilocus linkage disequilibrium measure $\bar{r}_d$ to detect clonal
reproduction.

## Clonal Population Genetics and Plant Pathology

### Population Genetics 

Dobzhansky's infamous quote "Nothing in biology makes sense except in the light
of evolution" is clichÃ©, but it's hard to find a statement that more elegantly 
captures the essence of biological study [-@dobzhansky2013nothing]. In a more
reductionist sense, it is perhaps appropriate to state that nothing in evolution
makes sense except in the light of mutation; that is to say that mutation in an
organism's genetic code is the seed upon which evolution acts
[@lynch2016genetic]. Of course, a seed left alone will not grow; it needs the
right soil and environmental conditions. In order to thrive, a mutation needs
a population. A population can be defined as a group of interbreeding 
individuals, sharing the same space and time (though we will see later that this
is not always the best definition) [@milgroom1997contributions]. In a
population, if the conditions are right, a mutation can grow in frequency and
become an established allele. The study of population genetics seeks to
understand how evolution acts upon these alleles to shape populations as they
grow, shrink, split, and adapt to their environment. Ultimately, these 
populations are what give rise to species. 

### Microbial Pathogen Genetics

Before molecular techniques were widely available, the variation within most
pathogenic microorganisms was described using phenotypic traits describing their
pathogenicity (ability to cause disease) and virulence (severity of disease), 
placing them into different pathotypes [@milgroom1997contributions;
@levin1999population]. The problem was, with only pathotype information, it was
not possible to ask questions about inheritance, which is of the upmost
importance if one is attempting to find a source of disease 
[@tibayrenc1995population].



### The Question of Sex

Sex exists in all major groups of organisms [@heitman2012evolution]. 

Pathogenic species can use reproductive mode to adapt to different environmental
factors.

> ...uniparental reproduction is ... predominant enough in natural populations
> to generate clones that are stable in space and time, even on an evolutionary
> time scale.
> 
> ...the hypothesis of clonality does not rule out the possibility of occasional
> genetic recombination, but rather it indicates that such recombination is not
> important enough for altering significantly the prevailing pattern of clonal
> population structure.
>
> -- [@tibayrenc1990clonal]

Knowing the reproductive mode of a pathogen can help you understand how to
develop management strategies e.g. if you have a clonal population, you want to
make sure you capture the diversity of genotypes before developing 
anti-microbial treatments [@taylor1999evolutionary].

### Population Genetics Under the Lens of Plant Pathology

Agricultural ecosystems behave very differently than natural ecosystems
[@milgroom1997contributions; @stukenbrock2008origins]. They can serve as
incubator for new variants of plant pathogens due to selective pressures.

Life cycle concerns: overwintering oospores, sexual structures (image of *P.
infestans* life cycle).

Sexual reproduction risk: *Puccinia* pathway.

These are both examples of partially clonal organisms.

Examine entirely clonal *Phytophthora ramorum*. 

## The need for reproducible research

Replicate the standards we have for wet lab research to computational analyses.



## Goals

## Part 1: Tools

### Summary of Chapter 2

To address the lack of tools for reproducible research in clonal population
genetics, we present the software package *poppr* in the R computing language
[@R2016; @kamvar2014poppr]. Previously, tools necessary for analysis of clonal
populations were available in several stand-alone software programs, each
requiring different data input formats. Moreover, each program had different
levels of documentation and limited support for all computing platforms. The
novelty of *poppr* was to introduce indices of multilocus genotype diversity,
the index of association, and a fast implementation of Bruvo's genetic distance
over unlimited levels of a user-specified population hierarchy
[@arnaud2007standardizing; @Agapow_2001; @bruvo2004simple]. Because this was
implemented in R these analyses could be performed in a reproducible manner on
all computing platforms.

### Summary of Chapter 3

The initial implementation of *poppr* contained basic tools for analysis of
clonal populations [@kamvar2014poppr], but lacked tools for custom definitions
of multilocus genotypes and performed poorly with genomic-scale data. Chapter 3
introduces an updated and improved *poppr* version 2.0. With high throughput
sequencing (HTS) data, the amount of missing data and genotyping error
increases, and the definition of a multilocus genotype becomes unclear.
Moreover, the calculation of the index of association scales poorly with an
increase in the number of loci. To address these limitations, we improved poppr
with new functionalities to define multilocus genotypes based on genetic
distance and calculate the index of association over random samples or windows
of SNP loci.

## Part 2: Applications

The end goal of part one was not simply the development of the software, but
rather, we developed the software for the goal of analyzing our own data.
Serendipitously, by analyzing our own data with this software, we are able to
not only demonstrate that it can be used, but also demonstrate that an entire
analysis can be conducted in an open and reproducible manner. The next two
chapters focus on answering practical and theoretical questions related to plant
pathology and clonal population genetics, respectively.

### Summary of Chapter 4

A newly-emerged disease of oak---called Sudden Oak Death---spread from
California to the Southwest corner of Oregon in 2001. Because of intense
management strategies, the epidemic was largely contained to Curry County for
the next 15 years. In 2011, an isolated patch of disease appeared in Cape
Sebastian, 12 miles from the nearest infected site. With microsatellite
genotyping performed across 2 labs and 15 years, we sought to describe the
spread of the epidemic in a population genetic context and ask the question of
whether or not there was evidence for more than one introduction event. All the
analyses were performed in an open-source and reproducible manner using R.

### Summary of Chapter 5

The index of association is a measure of multilocus linkage disequilibrium, that
is, a correlation coefficient across multiple loci. In sexual populations, loci
are randomly assorting due to recombination, resulting in a near-zero value of
the index of association. In clonal populations, recombination is non-existent,
meaning that loci are passed from parent to offspring in a non-independent
fashion, resulting in a significantly non-zero value of the index of
association. @de2004clonal showed that this index shows high variance with low
levels of sexual reproduction, but due to limitations in software were not able
to perform power analyses. We use *poppr* to investigate the power of the index
of association to detect sexual reproduction in simulated data sets generated
with microsatellite and genomic markers. 