# Introduction

```{r, introduction1, echo = FALSE, fig.cap = figcap1, fig.scap = figscap1, out.width = "50%"}
knitr::include_graphics("figure/introduction/dissertation.png")

figcap1 <- "A graphical representation of the three themes governing the work
presented in this dissertation." -> figscap1
```

The work presented here presents tools designed to answer questions related to
clonal population genetics and plant pathology (Fig. \@ref(fig:introduction1)). 
In this context, the term 'tool' is not in reference to new derivations of
mathematical models or theory for population genetics of clonal organisms;
rather, 'tool' refers to software code used to apply mathematical models or
theory to population genetic data. The merit of this work lies within the
context of reproducible science [@buckheit1995wavelab]. We demonstrate the
usefulness and flexibility of these tools by using them to show evidence for
multiple introductions in the Curry County, OR outbreak of sudden oak death, and
by assessing the power of multilocus linkage disequilibrium measure $\bar{r}_d$
to detect clonal reproduction.

## Population Genetics and Clonality

Dobzhansky's infamous quote "Nothing in biology makes sense except in the light
of evolution" is clich√©, but it's hard to find a statement that more elegantly 
captures the essence of biological study [-@dobzhansky2013nothing]. In a more
reductionist sense, it is perhaps appropriate to state that nothing in evolution
makes sense except in the light of mutation; that is to say that mutation in an
organism's genetic code is the seed upon which evolution acts
[@lynch2016genetic]. Of course, a seed left alone will not grow; it needs the
right soil and environmental conditions. In turn, a mutation needs a population
in order to thrive. A population can be defined as a group of interbreeding 
individuals, coexisting temporally and spatially (though we will see later that
this is not always the best definition) [@milgroom1997contributions]. In a
population, if the conditions are right, a mutation can grow in frequency and
become an established allele. The study of population genetics seeks to
understand how evolution acts upon these alleles to shape populations as they
grow, shrink, split, and adapt to their environment. Ultimately, these
populations are what give rise to species.

The above definition of a population is ultimately reliant upon the assumption
that the individuals within the populations are panmictic, that is that they are
randomly mating so that the individuals in the next generation will have a
random assortment of alleles as compared to their parents [@rice2002evolution;
@nielsen2013introduction; @hartl1997principles]. This assumption is a relatively
safe one considering that sex occurs in all major organismal groups
[@rice2002evolution; @heitman2012evolution]. However, it is erroneous to say
that sex occurs in all organisms as several plants, fungi, protists, bacteria,
animals, and viruses are known to exclusively reproduce clonally
[@rice2002evolution; @heitman2012evolution; @arnaud2012implications; 
@orive1993effective; @arnaud2007standardizing]. Clonal reproduction here means
reproduction that results in offspring that are genetically identical to the
parents (note that this can include organisms derived from selfing in a haploid
organism). Thus, the definition of a population is perhaps best rephrased to
simply indicate a group of individuals of the same species^[This depends on your
definition of a species, which is out of the scope of this dissertation and
therefore best debated over copious quantities of alcohol] coexisting in the
spatially and temporally [@milgroom1997contributions]. There are several costs
and benefits associated with both clonal and sexual reproductive strategies in
an evolutionary perspective.

For organisms whose main mode of reproduction is sexual, there is the two-fold
cost of outcrossing: 1) only half of the available genetic material is inherited
by the offspring and 2) the loss of advantageous gene combinations due to
recombination [@rice2002evolution; @heitman2012evolution;
@nieuwenhuis2016frequency]. These costs are balanced by the benefits that come
with recombining genetic material, which allows these organisms to purge
deleterious mutations. As clonally reproducing organisms do not undergo regular
periods of recombination, they explicitly do not benefit from the previously
mentioned advantage of recombination [@heitman2012evolution;
@nieuwenhuis2016frequency]. This leads to a theory known as Muller's ratchet,
which indicates that the overall fitness of small populations of clonally
reproducing organisms will decrease over time as the number of deleterious
mutations accumulate, eventually leading to extinction
[@felsenstein1974evolutionary; @loewe2006quantifying; @lynch1993mutational; 
@lynch1990mutation]. Despite this, clonal organisms still have the advantage of
passing on 100% of their genetic material on to the next generation.

It's easy to see the advantage of both of these strategies in the context of 
genetic adaptation to environment. Clonal populations that are well adapted to
their environment get to keep the combination of genes that allow them to be so
well adapted [@heitman2012evolution]. This strategy provides an advantage when
resources are scarce and competition is high, but does not lend itself well to
adaptation against changing environments due to low genotypic diversity 
[@milgroom1996recombination]. Conversely, while sexual populations lose the 
competitive advantage of conserved genotypic structure, they make up for it with
high genotypic diversity, increasing the chances of surviving drastic change in
the environment. As we shall see in the following sections, many microbial
pathogens have the ability to reproduce both sexually and asexually
[@tibayrenc1995population; @anderson1995clonality; @milgroom1996recombination].

## Microbial Pathogens and the Question of Sex

The question of sexual reproduction and population structure in microbial 
pathogens is important for management strategies [@tibayrenc1990clonal;
@tibayrenc1995population; @milgroom1997contributions; @taylor1999evolutionary].
If one were developing an antimicrobial agent on a known set of strains, the 
effectiveness of this agent will be diminished if these strains represent only
a fraction of the genotypic diversity due to sexual recombination
[@taylor1999evolutionary; @tibayrenc1990clonal; @smith1993how]. Since most
inference of population structure is based on neutral models that assume sexual
reproduction, it's important to identify whether or not the population in 
question is reproducing sexually or clonally before one attempts to assess 
population structure and infer evolutionary processes [@tibayrenc1990clonal]. 
The question then becomes how do you detect the presence of clonal or sexual
reproduction?

### A Word on Molecular Markers

Before molecular techniques were widely available, the variation within most
pathogenic microorganisms was described using phenotypic traits that reflected 
their pathogenicity (ability to cause disease) and virulence (severity of 
disease), placing them into different pathotypes [@milgroom1997contributions;
@levin1999population]. When considering markers for population genetic analysis,
one of the most important criteria is that these markers are heritable to a 
degree.

The problem was, unlike the eye color in Thomas Hunt Morgan's *Drosophila* or
the shape of the Mendel's peas, pathotype wasn't a heritable trait
[@milgroom1997contributions; @tibayrenc1995population]. This meant that, while
it was possible to define a panel of pathotypes upon which to develop 
anti-microbial treatments or resistant host cultivars, there was no way of 
assessing the evolutionary potential of the pathogens to overcome host 
resistance or develop drug resistance. If pathologists wanted to use population
genetic techniques, they needed to embrace molecular markers
[@milgroom1997contributions; @tibayrenc1995population; @mcdonald1997population;
@taylor1999evolutionary].

Under the central dogma of genetics, all heritable traits originate in the DNA.
A wide variety of molecular markers exist from protein mobility assays to 
randomly amplified DNA, to length polymorphisms to direct sequencing of the
genetic code that all reflect this principle [@tibayrenc1995population;
@halkett2005tackling]. It's not necessary to expound here on the variety of
molecular markers available to us, but it's useful to take note of the common
features. Genetic markers come in two major flavors, selectively neutral and
selected markers [@milgroom1997contributions; @mcdonald1997population; 
@tibayrenc1995population]. Selected markers, based on protein coding genes that
confer and evolutionary advantage to the organism, can be used to infer the
evolutionary potential of a population. Selectively neutral markers, on the
other hand, are used as independent variables that can reflect the degree of
population size, subdivision, and migration. For selectively neutral markers, it
is important that they be unlinked within the genome [@mcdonald1997population].

### Detecting Sex

Because neutral genetic markers vary independently within a population, these
are more appropriate than selected markers to detect sexual reproduction. The
theory behind this is simple; sexual reproduction breaks up associations between
unlinked markers due to recombination and assortment while clonal reproduction
transfers the entire genome in tact to the next generation, preserving these 
associations [@orive1993effective; @heitman2012evolution; 
@nieuwenhuis2016frequency; @mcdonald1997population; @milgroom1997contributions;
@tibayrenc1995population]. This means that not only will there be significant
associations between alleles in a clonal population, but because of their
inability to purge mutations, we also expect to see excess heterozygosity
[@tibayrenc1990clonal; @tibayrenc1996towards].

By using these basic principles, core sets of statistical analyses have been
proposed to detect sexual reproduction by analyzing diversity within loci (for
diploid organisms), linkage among loci, and diversity of multi-locus genotypes
within the population [@tibayrenc1996towards; @arnaud2007standardizing; 
@parks1993study; @smith1993how; @mcdonald1997population; @grunwald2003analysis].
None of these tests alone can give a definitive answer as to the nature of 
reproduction in a given population [@tibayrenc1995population; @de2004clonal; 
@nieuwenhuis2016frequency]. Researchers still need to use biological evidence
such as sexual reproductive structures to support the presence of sex.

## Population Genetics Under the Lens of Plant Pathology

In plant pathology, where many of the diseases are caused by fungi or oomycetes,
sexual reproduction often confers another advantage to pathogen populations:
survival. For several species of plant pathogenic fungi and oomycetes, sexual
reproduction produces spores that have the ability to survive in the soil for
several years [@mcdonald2002pathogen]. This, in combination with the increase in
genotypic diversity makes understanding the reproductive strategy of plant
pathogens ever more important. Here, I will review how agricultural ecosystems
promote pathogen evolution and then give two examples of how the use of 
population genetic theory and detection of sexual reproduction has helped 
researchers manage black stem rust of wheat and potato late blight. 

One of the over-arching themes in both ecology and evolution is that diversity
is important for stable population and community structure 
[@magurran1988diversity; @mcdonald2002pathogen]. Agricultural systems, however,
behave very differently to natural ecosystems [@milgroom1997contributions; 
@stukenbrock2008origins]. The plants and the landscape are homogenized, 
providing a steady source of food for people, but this homogeneity combined with
intensive management practices also provides fertile ground for the evolution of
plant pathogens [@stukenbrock2008origins]. McDonald identified five major 
factors influencing the evolution of plant pathogens in agricultural systems, 1)
mutation 2) genetic drift 3) gene/genotype flow 4) mating system 5) selection
due to host resistance [-@mcdonald2002pathogen]. Understanding these factors 
have helped plant pathologists developed salient strategies for combating
rapidly evolving pathogens.





```{r pinflife, echo = FALSE, fig.cap = figcap2, fig.scap = figscap2}
knitr::include_graphics("figure/introduction/Pinfestans_life.png")

figcap2 <- "Life cycle of the oomycete pathogen, *Phytophthora infestans* on 
potato by M. Piepenbring. M! = meiosis, P! = plasmogamy, K! = karyogamy"
figcap2 <- beaverdown::render_caption(caption = figcap2, figname = "fig2", to = TO) -> figscap2
```

Life cycle concerns: overwintering oospores, sexual structures (image of *P.
infestans* life cycle).

Sexual reproduction risk: *Puccinia* pathway.

These are both examples of partially clonal organisms.

Examine entirely clonal *Phytophthora ramorum*. 

## The need for reproducible research

Replicate the standards we have for wet lab research to computational analyses.

There have been a call for standards in analyzing clonal organisms for the past
twenty years [@tibayrenc1996towards; @arnaud2007standardizing].


## Goals

## Part 1: Tools

### Summary of Chapter 2

To address the lack of tools for reproducible research in clonal population
genetics, we present the software package *poppr* in the R computing language
[@R2016; @kamvar2014poppr]. Previously, tools necessary for analysis of clonal
populations were available in several stand-alone software programs, each
requiring different data input formats. Moreover, each program had different
levels of documentation and limited support for all computing platforms. The
novelty of *poppr* was to introduce indices of multilocus genotype diversity,
the index of association, and a fast implementation of Bruvo's genetic distance
over unlimited levels of a user-specified population hierarchy
[@arnaud2007standardizing; @Agapow_2001; @bruvo2004simple]. Because this was
implemented in R these analyses could be performed in a reproducible manner on
all computing platforms.

### Summary of Chapter 3

The initial implementation of *poppr* contained basic tools for analysis of
clonal populations [@kamvar2014poppr], but lacked tools for custom definitions
of multilocus genotypes and performed poorly with genomic-scale data. Chapter 3
introduces an updated and improved *poppr* version 2.0. With high throughput
sequencing (HTS) data, the amount of missing data and genotyping error
increases, and the definition of a multilocus genotype becomes unclear.
Moreover, the calculation of the index of association scales poorly with an
increase in the number of loci. To address these limitations, we improved poppr
with new functionalities to define multilocus genotypes based on genetic
distance and calculate the index of association over random samples or windows
of SNP loci.

## Part 2: Applications

The end goal of part one was not simply the development of the software, but
rather, we developed the software for the goal of analyzing our own data.
Serendipitously, by analyzing our own data with this software, we are able to
not only demonstrate that it can be used, but also demonstrate that an entire
analysis can be conducted in an open and reproducible manner. The next two
chapters focus on answering practical and theoretical questions related to plant
pathology and clonal population genetics, respectively.

### Summary of Chapter 4

A newly-emerged disease of oak---called Sudden Oak Death---spread from
California to the Southwest corner of Oregon in 2001. Because of intense
management strategies, the epidemic was largely contained to Curry County for
the next 15 years. In 2011, an isolated patch of disease appeared in Cape
Sebastian, 12 miles from the nearest infected site. With microsatellite
genotyping performed across 2 labs and 15 years, we sought to describe the
spread of the epidemic in a population genetic context and ask the question of
whether or not there was evidence for more than one introduction event. All the
analyses were performed in an open-source and reproducible manner using R.

### Summary of Chapter 5

The index of association is a measure of multilocus linkage disequilibrium, that
is, a correlation coefficient across multiple loci. In sexual populations, loci
are randomly assorting due to recombination, resulting in a near-zero value of
the index of association. In clonal populations, recombination is non-existent,
meaning that loci are passed from parent to offspring in a non-independent
fashion, resulting in a significantly non-zero value of the index of
association. @de2004clonal showed that this index shows high variance with low
levels of sexual reproduction, but due to limitations in software were not able
to perform power analyses. We use *poppr* to investigate the power of the index
of association to detect sexual reproduction in simulated data sets generated
with microsatellite and genomic markers. 