# Introduction

<!-- Development of Tools for Genetic Analysis of Clonal Populations and Applications -->


```{r, introduction1, echo = FALSE, fig.cap = figcap1, fig.scap = figscap1, out.width = "50%"}
knitr::include_graphics("figure/introduction/dissertation.png")

figcap1 <- "A graphical representation of the three themes governing the work
presented in this dissertation." -> figscap1
```



The work presented here presents tools designed to answer questions related to
clonal population genetics and plant pathology. The term 'tool' in this context
is not in reference to new derivations of mathematical models or theory for
population genetics of clonal organisms; rather, 'tool' refers to software code 
used to apply mathematical models or theory to population genetic data. The 
merit of this work lies within the context of reproducible science 
[@buckheit1995wavelab]. It provides the tools and demonstrates the usefulness
and flexibility by applying these tools to the outbreak of sudden oak death from
2001 to 2014 in Curry County, OR, USA and to simulation data, assessing the
power of the index of association to detect clonal reproduction.


## Plant Pathology and Clonal Population Genetics

Sex exists in all major groups of organisms [@heitman2012evolution]. 

Pathogenic species can use reproductive mode to adapt to different environmental
factors.

Knowing the reproductive mode of a pathogen can help you understand how to
develop management strategies e.g. if you have a clonal population, you want to
make sure you capture the diversity of genotypes before developing 
anti-microbial treatments [@taylor1999evolutionary].

## Reproducible Research and Scientific Software


> *Clearly, a discipline is defined by the questions asked, not the tools used.*
> 
> -- @milgroom1997contributions [p. 4]

The above quote by Michael Milgroom and William Fry in 1997 was in reference to
the use of molecular markers in molecular biology as compared to population
genetics. While it is undisputed that questions shape a field of inquiry, the
notion that tools are not influential in disciplines is misleading. Tools are
necessary for answering the questions proposed, but the answers they give are
always in the context of the assumptions of the field at the time the tools were
produced. A simple example in which assumptions can be violated is in 
'Anscombe's quartet' (Fig. \@ref(fig:introduction2)) [-@anscombe1973graphs].
Here we see four data sets, only one of which has a linear relationship between
the dependent and independent variable (case 1); the others violate the 
assumptions of linearity or normality. Based on linear regression, however, all
have identical variance, correlation, slope, and intercept. The fact that these
clearly different data give the same result is not due to an error in the linear
regression analysis, it was a failure to meet the assumptions of the tool
provided.

```{r, introduction2, echo = FALSE, fig.cap = anscomb, fig.scap = ansc, out.width = "50%"}
library('tidyverse')
ans <- gather(anscombe, case, y, y1, y2, y3, y4) %>% 
  gather(xcase, x, x1, x2, x3, x4) %>% 
  filter(as.numeric(substr(case, 2, 2)) == as.numeric(substr(xcase, 2, 2))) %>%
  mutate(case = factor(case, labels = c("case 1", "case 2", "case 3", "case 4")))
ggplot(ans, aes(x, y)) +
  geom_point(size = 3) +
  geom_abline(intercept = 3, slope = 0.5, lty = 2, color = "grey40") +
  facet_wrap(~case) +
  theme_bw() +
  theme(text = element_text(size = 20, face = "bold")) +
  theme(title = element_blank()) +
  theme(strip.background = element_blank()) +
  theme(strip.text = element_text(size = 20, face = "bold"))

cap <- "A reproduction of 'Anscombe's quartet' [@anscombe1973graphs] 
demonstrating different situations in which linear regression would give the 
same answer."

anscomb <- beaverdown::render_caption(caption = cap, figname = "fig1", to = TO)
ansc    <- "A reproduction of 'Anscombe's quartet'"
```

Science moves forward by asking questions of natural phenomena and then
investigating further the results of these questions, narrowing the scope of the
succeeding questions to pin down an detailed mechanism that can explain the
initial phenomena. Yes, a discipline is defined by the questions asked, but the
tools provide the results that set the context for the follow up questions.

----


[kamoun2015pubpeer]: https://pubpeer.com/topics/1/877A42B12A198A44306771E0D80AB8

@buckheit1995wavelab didn't invent the concept of reproducible research in
scientific computing, but they did show computational research community that
such a concept was possible. While the original goal of providing the
environment used to generate figures seems excessive, it indeed has real world
implications even in plant pathology; case in point, Sophien Kamoun's paper from
2012 was targeted on the post-publication review site http://pubpeer.com:
[kamoun2015pubpeer]. The paper had a single duplicated image in a figure. In
response, Kamoun's team found the original file and realized that the image had
accidentally been pasted on top of the original.

## Goals

## Part 1: Tools

### Summary of Chapter 2

To address the lack of tools for reproducible research in clonal population
genetics, we present the software package *poppr* in the R computing language
[@R2016; @kamvar2014poppr]. Previously, tools necessary for analysis of clonal
populations were available in several stand-alone software programs, each
requiring different data input formats. Moreover, each program had different
levels of documentation and limited support for all computing platforms. The
novelty of *poppr* was to introduce indices of multilocus genotype diversity,
the index of association, and a fast implementation of Bruvo's genetic distance
over unlimited levels of a user-specified population hierarchy
[@arnaud2007standardizing; @Agapow_2001; @bruvo2004simple]. Because this was
implemented in R these analyses could be performed in a reproducible manner on
all computing platforms.

### Summary of Chapter 3

The initial implementation of *poppr* contained basic tools for analysis of
clonal populations [@kamvar2014poppr], but lacked tools for custom definitions
of multilocus genotypes and performed poorly with genomic-scale data. Chapter 3
introduces an updated and improved *poppr* version 2.0. With high throughput
sequencing (HTS) data, the amount of missing data and genotyping error
increases, and the definition of a multilocus genotype becomes unclear.
Moreover, the calculation of the index of association scales poorly with an
increase in the number of loci. To address these limitations, we improved poppr
with new functionalities to define multilocus genotypes based on genetic
distance and calculate the index of association over random samples or windows
of SNP loci.

## Part 2: Applications

The end goal of part one was not simply the development of the software, but
rather, we developed the software for the goal of analyzing our own data.
Serendipitously, by analyzing our own data with this software, we are able to
not only demonstrate that it can be used, but also demonstrate that an entire
analysis can be conducted in an open and reproducible manner. The next two
chapters focus on answering practical and theoretical questions related to plant
pathology and clonal population genetics, respectively.

### Summary of Chapter 4

A newly-emerged disease of oak---called Sudden Oak Death---spread from
California to the Southwest corner of Oregon in 2001. Because of intense
management strategies, the epidemic was largely contained to Curry County for
the next 15 years. In 2011, an isolated patch of disease appeared in Cape
Sebastian, 12 miles from the nearest infected site. With microsatellite
genotyping performed across 2 labs and 15 years, we sought to describe the
spread of the epidemic in a population genetic context and ask the question of
whether or not there was evidence for more than one introduction event. All the
analyses were performed in an open-source and reproducible manner using R.

### Summary of Chapter 5

The index of association is a measure of multilocus linkage disequilibrium, that
is, a correlation coefficient across multiple loci. In sexual populations, loci
are randomly assorting due to recombination, resulting in a near-zero value of
the index of association. In clonal populations, recombination is non-existent,
meaning that loci are passed from parent to offspring in a non-independent
fashion, resulting in a significantly non-zero value of the index of
association. @de2004clonal showed that this index shows high variance with low
levels of sexual reproduction, but due to limitations in software were not able
to perform power analyses. We use *poppr* to investigate the power of the index
of association to detect sexual reproduction in simulated data sets generated
with microsatellite and genomic markers. 