# Factors Influencing Recombination Inference in Diploid Populations

```{r, results = "asis", echo = FALSE}
out <- "
\\singlespacing
\\begin{center}
"
cat(beaverdown::iflatex(out))
```


Zhian N. Kamvar and Niklaus J. GrÃ¼nwald


```{r, results = "asis", echo = FALSE}
out <- "
\\end{center}
\\vspace*{\\fill}"
cat(beaverdown::iflatex(out))
```

Target Journal: **Molecular Ecology**

```{r, results = "asis", echo = FALSE}
out <- "
\\doublespacing
\\newpage
"
cat(beaverdown::iflatex(out))
```


## Abstract

TBD...

## Introduction

Population genetic theory is largely based on model populations such as the neutral Wright-Fisher model 
in which populations are infinitely large, with discrete generations, randomly assorting alleles,
no migration, and no mutation [@hartl1997principles;
@nielsen2013introduction]. By using such models, population geneticists are able to
ask fundamental questions and test hypotheses about evolutionary processes that
could lead to structured populations.

These neutral models, however, cannot be applied to populations whose life history
violates the fundamental assumption of random assortment of alleles, such as
populations that undergo clonal reproduction [@orive1993effective; 
@milgroom1996recombination]. For many clonal populations, the contribution of
genetic variation from mutation is greater than that of recombination. While 
this increases the risk of the accumulation of deleterious mutations, the 
two-fold cost of sex is drastically reduced, meaning that selectively
advantageous combinations of alleles are maintained [@heitman2012evolution]. 
Detecting recombination in populations of pathogenic microorganisms is therefore
important for management strategies as a prevalence of sexual reproduction could
create resistant genotypes [@smith1993how; @milgroom1996recombination; 
@goss2014irish; @de2006molecular; @nieuwenhuis2016frequency].

<!-- The study of population genetics of clonal organisms is not a new field 
[@orive1993effective; @halkett2005tackling;
@arnaud2007standardizing]. In particular, there has been a re-occurring question
of how much sex can be detected in a population of organisms with (partially)
clonal reproduction [@smith1993how; @balloux2003population; @de2004clonal;
@ali2016cloncase; @nieuwenhuis2016frequency].  -->


Several studies have been conducted in an attempt to quantify the amount of sex
in populations that undergo clonal reproduction [@smith1993how;
@balloux2003population; @de2004clonal; @ali2016cloncase; 
@nieuwenhuis2016frequency]. For populations with well-defined sexual and clonal
phases occurring at separate times, such as rust type fungi, methods like
*CloNcaSe* are effective for estimating the rate of sexual reproduction and
effective population size [@ali2016cloncase]. However, this method cannot be
applied to populations where the reproductive cycle is not partitioned into
discrete generations.

Simply detecting the presence of clonal reproduction, however can be useful in
and of itself [@milgroom1996recombination]. A method commonly used to assess 
this is the index of association ($I_A$), and its standardized version,
$\bar{r}_d$, which measure multilocus linkage disequilibrium
[@brown1980multilocus; @smith1993how; @haubold1998detecting; @Agapow_2001; 
@de2004clonal; @kamvar2014poppr]. The value of $I_A$, as shown in equation
\@ref(eq:ia), is measured as the ratio of observed variance ($V_O$) and expected
variance ($V_E$) in genetic distance between samples [@smith1993how;
@Agapow_2001].

\begin{equation}
I_A = \frac{V_O}{V_E} - 1 (\#eq:ia)
\end{equation}

The expected variance is practically modeled as the sum of the variances over 
*m* loci: $V_E = \sum^m{var_j}$ [@Agapow_2001; @haubold1998detecting]. If the 
differences between samples are randomly distributed (linkage equilibrium), we
can expect the value of $I_A$ to be zero [@smith1993how; @Agapow_2001]. Under
scenarios of non-random mating (e.g. population structure or clonal
reproduction), the observed variance would be greater than the expected due to a
multi-modal distribution of distances, and $I_A$ would be greater than zero
[@smith1993how; @Agapow_2001; @milgroom2015population]. @Agapow_2001 noted that 
this metric does not have an upper limit and increases with the number of loci.
To correct this, they developed $\bar{r}_d$ (equation \@ref(eq:rd)), which has a
similar structure to a correlation coefficient and ranges from 0 (no linkage) to
1 (complete linkage).

\begin{align} % This mode aligns the equations to the '&=' signs
\begin{split} % This mode groups the equations into one.
\bar{r}_d &= \frac{\sum\sum{cov_{j,k}}}{
                   \sum\sum{\sqrt{var_{j} \cdot var_{k}}}} \\
          &= \frac{V_O - V_E}{2\sum\sum{\sqrt{var_{j} \cdot var_{k}}}}
\end{split}
(\#eq:rd)
\end{align}

<!-- @smith1993how demonstrated that $I_A$ has the ability to detect non-random
mating due to population structure, strict clonal reproduction, or a recent
epidemic, but at that time, no one had quantified how much clonal reproduction
had to occur to produce a significant value of $I_A$. This was of significant
concern, considering that, as with any summary statistic, a lot of information
is lost when evaluating a population based on $I_A$ [@brown1980multilocus]. One
way to assess the effect of different population genetic scenarios was via
simulations.
 -->

@de2004clonal investigated the effect of increasing levels of sexual
reproduction on $\bar{r}_d$ (noted in their publication as $\bar{r}_D$). They
found that very little (1%) sexual reproduction is required to produce a value
of $\bar{r}_d$ close to zero. This indicates that $\bar{r}_d$ alone might not be
well suited as a measure of clonal reproduction. @prugnolle2010apparent tested
the effect of sampling design on $\bar{r}_d$, finding that the its value is
drastically reduced when clones from multiple populations are sampled, leading
to an over-estimation of the level of recombination.

These studies laid the groundwork for understanding the behavior of $\bar{r}_d$
under different scenarios of non-random mating in diploid organisms, but there
were some limitations in available technology that prevented deep analysis from
being performed. For both studies, the only software available for calculation
of $\bar{r}_d$ for diploid organisms was 
<span style="font-variant:small-caps;">Multilocus</span>, which could only take
one data set at a time [@Agapow_2001; @de2004clonal; @prugnolle2010apparent;
@kamvar2014poppr]. This constrained the researchers to only analyze a minimal
set of populations (20) per scenario. 

Since the distribution of $I_A$ and $\bar{r}_d$ are not known, the safest way to
test for significance are random permutation tests. These tests effectively
create unlinked populations by shuffling individuals at each locus,
independently and re-calculating $I_A$ and $\bar{r}_d$ [@smith1993how;
@Agapow_2001; @haubold1998detecting]. A one-sided *t*-test of significance was 
then used to see if the observed statistic was greater than the observed 
distribution. 

While significance testing is available in 
<span style="font-variant:small-caps;">Multilocus</span> in the form of
random permutations, it's computationally expensive, and can only take one data
set at a time [@Agapow_2001; @kamvar2014poppr]. As a result, power analysis of 
$\bar{r}_d$ to detect clonal reproduction has not yet been performed 
[@de2004clonal]. Our current study aims to evaluate the power and specificity of
assessing $\bar{r}_d$ via permutation analysis to detect non-random mating.

In the years since the studies conducted by @de2004clonal and
@prugnolle2010apparent, reduced-representation, high-throughput sequencing
methods such as Genotyping-By-Sequencing (GBS) and RAD-seq have rapidly become
popular tools for population genetic analysis [@davey2010rad; @davey2011genome;
@elshire2011robust]. These methods have the capability to generate thousands of
unlinked markers at a fraction of the cost and time necessary to develop high
quality microsatellite markers. These marker systems are also prone to missing
data and high error rates [@mastretta2015restriction]. The index of association
was developed for multiple loci in a time when obtaining even 100 unlinked
markers posed a significant challenge. With the advent of these current
technologies, how does marker choice and genotyping error affect the index of
association?

In 2014, we developed the R package *poppr* for analysis of clonal populations,
removing the limitations of data input and computational expense of analyzing
the index of association, and in 2015, we expanded this to analysis of genome-
wide SNP data [@R2016; @kamvar2014poppr; @kamvar2015novel]. With these tools we
expand on previous studies by asking how sample size, marker choice,
clone-correction, and the assumption of homogeneous mutation rates affect our
ability to detect clonal reproduction in diploid populations. Our objectives to
answer these questions are to (1) re-analyze $\bar{r}_d$ against increasing
rates of sexual reproduction and different levels of population mixture in both
microsatellite and SNP data sets, (2) perform a power analysis of $\bar{r}_d$
to assess sensitivity and specificity, (3) assess how genotypic and allelic
evenness and diversity affects $\bar{r}_d$. Because studies have observed
significantly negative values of the $I_A$ and $\bar{r}_d$ (p $\geq$ 0.95), we
additionally seek to determine what factors result in negative $\bar{r}_d$ 
values.


<!--  - Population Genetics of partially clonal organisms
 	- This has been studied in the past [@smith1993how; @balloux2003population;
 	  @de2004clonal; @orive1993effective]
 	- The index of association [@brown1980multilocus; @smith1993how;
 	  @Agapow_2001]
 	- In @de2004clonal, it was shown that $\bar{r}_d$ has a high variance in
 	  clonal populations and @smith1993how showed that it's affected by
 	  population structure.
 - Methods for assessing level of clonal reproduction (CloNcaSe)
   [@ali2016cloncase]
 	- This only works for populations with discrete generations with an
 	  observable sexual stage.
 - Limitations of previous studies
 	- No HTS markers
 	- Significance tests are routinely performed via permutation analysis, but
 	  could not be performed due to software limitations [@burt1996molecular;
 	  @de2004clonal]
 - Questions
 	1. With the advent of GBS, does marker choice affect inference of clonality?
 	
 	2. Does sample size significantly affect our ability to detect clonal
 	   reproduction?
 	3. Can a combination of genotypic diversity estimators and allelic diversity
 	   estimator provide support for significantly negative values of
 	   $\bar{r}_d$?
 - Objectives
 	1. Analyze mixtures of clonal populations to assess effect of sampling
 	   multiple clonal populations.
 	1. Assess the effect of a single hypervariable locus on $\bar{r}_d$
 	2. Re-analyze rates of sexual reproduction to confirm previous study
 	   with and without clone-correction [@de2004clonal].
 	3. Power analysis of $\bar{r}_d$.
 	4. Investigate dynamics of populations with significantly negative (p = 1)
 	   values of $\bar{r}_d$.
	5. Assess $E_5$, *H*, $eMLG_{10}$, and unbiased Simpson's diversity.

 -->

## Methods

Initial sets of simulations were created for different levels of sexual
reproduction for each marker type. All simulations were performed with the
python package simuPOP version 1.1.7 in python version 3.4 [@peng2008forward]. For each scenario,
100 simulations with 10 replicates were created with a census size of 10,000
diploid individuals with equal mating type proportions evolved over 10,000
generations. The simulated populations were first stored in the native simuPOP format and then transferred to feather format using the python and R packages *feather* version 0.3.0 for downstream analyses. Microsatellite simulations were performed on Ubuntu Linux version 14.04; SNP simulations were performed on CentOS Linux version 6.8. During downstream analysis, 10, 25, 50, and 100 individuals were sampled
without replacement for each replicate in R version 3.2 with the package
*poppr* version 2.2.1 [@R2016; @kamvar2014poppr; @kamvar2015novel]. Analyses (described below) were performed on both full and clone-corrected data sets. All downstream
analyses were run on the OSU CGRB Core Computing Facility (supplementary
information).

### Simulating Microsatellite Loci

Each population was simulated with 21 co-dominant, unlinked loci containing 6 to
10 alleles per locus with frequencies drawn from a uniform distribution and
subsequently normalized. Before mating, mutations occurred at each locus at a
rate of 1e-5 mutations/generation with the exception of the first locus, at which the mutation rate was set to 1e-3. All mutations were applied in a stepwise manner using the
`StepwiseMutator()` operator in simuPOP.

### GBS Simulations

Simulations of 10,000 binary loci spread evenly over 10 chromosomal fragments
were simulated with a mutation rate of 1e-5 mutations per generation for forward
and backward mutations using the `SNPMutator()` operator and and a recombination
rate of 1e-5 between adjacent loci using the `Recombinator()` operator in
simuPOP. 

### Mating

Simulations of sexual reproduction were conducted at 10 rates of sexual
reproduction on a log scale (0.0, 1e-4, 5e-4, 1e-3, 5e-3, 1e-2, 5e-2, 0.1, 0.5,
1.0) reflecting the fraction of individuals in generation *t+1* produced via
sexual reproduction. One to three offspring could be produced at each mating
event. For sexual events, two parents were chosen randomly from the population
with the `RandomSelection()` operator and offspring genotypes were created via
the `MendelianGenoTransmitter()` operator. The clonal fraction was created by
randomly sampling individuals from the population and duplicating their
genotypes with the `CloneGenoTransmitter()` operator. If one mating type was
lost before 10,000 generations, the simulation would continue to completion with
only clonal reproduction.

### Analysis of Microsatellite Data

The standardized index of association ($\bar{r}_d$) was calculated for full and
clone-corrected data using the *poppr* function `ia()`. Tests for significance
were performed by randomly permuting the alleles at each locus independently and
then assessing $\bar{r}_d$. This was done 999 times for each replicate
population. This was done for both full and clone-corrected data. The p-values
reflect the proportion of observations greater than the observed statistic.
Estimates of genotypic diversity were assessed with the *poppr* function
`diversity_boot()` with 999 bootstrap replicates, recording the estimate and
variance. The genotypic diversity statistics we calculated were Shannon's Index,
$H = -\sum p_i ln p_i$ [@shannon2001mathematical], Stoddart and Taylor's Index,
$G = 1/\sum p_i^2$ [@stoddart1988genotypic], and Evenness, 
$E_5 = (G - 1)/(e^H - 1)$ [@pielou1975ecological] where $p_i$ is the frequency 
of the *i*th genotype.

We additionally Nei's 1978 expected heterozygosity, also known as gene diversity
[@Nei:1978], calculated mean allelic evenness $E_{5A} = (1/m) \sum E_{5l}$ 
where *m* is the number of loci and $E_{5l}$ is the evenness of the alleles at 
locus *l*.


### Analysis of SNP Data

Because GBS data are associated with high error rates, we additionally wanted to
assess the effect of missing data on analysis. To do this, we used
scripts written for @kamvar2015novel to randomly insert missing data via the
`pop_NA()` function [@kamvar2015poppr2supp] at rates of 1%, 5%, and 10% each. 

The overall value of $\bar{r}_d$ was calculated for each simulation with the
*poppr* function `bitwise.ia()`. Significance was first assessed by randomly
shuffling genotypes at each locus independently and then, to preserve existing 
background linkage structure, at each chromosome independently. This was done
999 times for each replicate population. P-values represent the proportion of 
random samples greater than or equal to the observed statistic.

### Power Analysis

Permutation analysis of the index of association is commonly used as a
diagnostic for detecting non-random mating. If the observed value of $\bar{r}_d$
is greater than 95% of the results from the permutations, then the population is
declared to be clonal. Standard practice for analyzing microbial populations is to
perform this test on both the whole data set (wd) and clone-corrected data (cc),
where each multilocus genotype is represented only once per population to avoid
signatures of linkage that arise from re-sampling the same individual. 
[@goss2014irish; @milgroom1996recombination; @mcdonald1997population]. If the 
p-value for $\bar{r}_d$ is significant after clone-correction, then the 
population is considered clonal. Because knowing a pathogen's mode of 
reproduction can affect management strategies, it's important to know how sample
size, rate of sexual reproduction, mutation rate homogeneity, and 
clone-correction affect this method's sensitivity and specificity.

The Receiver Operating Characteristic (ROC) curve is a method of assessing the
the balance between sensitivity and specificity of a diagnostic method [@metz1978basic]. This is done by simultaneously assessing the true positive fraction of tests
to a false positive fraction along a gradient of thresholds of increasing leniency. 
Briefly, if a method has perfect
explanatory power, the area under the ROC curve will be equal to 1. If a method
has no explanatory power, the area under the ROC curve will be equal to 0.5. We used this method to assess the efficacy of $\bar{r}_d$ to detect non-random mating. 

To calculate the ROC curve, we first have to define defined what a positive
value is. We define it as a significant value of $\bar{r}_d$ based on a
threshold, $\alpha$, where $\alpha$ is a value in [0, 1].
Therefore, any randomly mating population (sexual reproduction is equal to one)
with a significant value of $\bar{r}_d$ is considered a false positive, and
subsequently, any non-randomly mating population with a significant value of
$\bar{r}_d$ is considered a true positive (Table \@ref(tab:simtab1)).
 
 Reproductive Mode   $p \leq \alpha$   $p > \alpha$
------------------- ----------------- ---------------
 Non-Random Mating    True Positive    False Negative
   Random Mating     False Positive     True Negative

Table: (\#tab:simtab1) Definitions of false positive and true positive values
for ROC analysis of simulations. *p* is the p-value for $\bar{r}_d$, $\alpha$ is
a threshold value in [0, 1] Random Mating populations are simulated with a sex
rate of 1. Non-Random Mating populations are simulated with a sex rate less than
one.

We constructed each curve by assessing the ROC over values of $\alpha$ in
increments of 0.01 from 0 to 1 and plotting the true positive fraction on the y
axis and false positive fraction on the x axis. Curves were calculated
hierarchically by rate of sexual reproduction (< 1) and unique seed used to
generate the populations. For each hierarchical level, separate curves were
calculated for sample size, mutation rate, and clone-correction. The areas under
the ROC curves (AURC) were calculated using the `auc()` function in the R
package *flux* version 0.3-0 [@jurasinski2014flux].

Because simulations across rates of sexual reproduction were generated from
unique seeds, we additionally constructed ROC curves for each seed with respect
to clone-correction, sample size, and mutation rate. To test for significant 
effects of clone-correction, sample size, and mutation rate on the inference of 
non-random mating, we then separated the data by rate of sexual reproduction and
performed a three way (type III) ANOVA on each rate separately with formula
\@ref(eq:anova) (for microsatellite data).

\begin{equation}
AURC \sim Clone Correction \times Sample Size \times Mutation Rate (\#eq:anova)
\end{equation}

## Results

For both microsatellite and SNP data, as found in @de2004clonal, large variances
were observed with extremely low levels of sexual reproduction (Figs.
\@ref(fig:sim1), \@ref(fig:sim5)). The variances and means of $\bar{r}_d$ 
decrease with increasing levels of sexual reproduction. The most striking
feature observed, however, was the bimodal nature of the distributions at these
low levels of sexual reproduction. As this feature appears in both the
microsatellite and SNP data, it appears not to be an artifact. Additionally,
with larger sample sizes, the variance also decreases, suggesting that this
index is influenced by sample size.

### Microsatellite Data

The range of the values of $\bar{r}_d$ for completely clonal and nearly clonal
populations were extreme. They ranged from -0.577 to 1 in completely clonal
data sets (Table \@ref(tab:simtab2)).  

mutation rate   clone-correction    n = 10   n = 25   n = 50   n = 100
--------------  -----------------  -------  -------  -------  --------
even            uncorrected         -0.192   -0.158   -0.064    -0.115
even            clone-corrected       -Inf   -0.500   -0.333    -0.333
uneven          uncorrected         -0.268   -0.029   -0.023    -0.007
uneven          clone-corrected     -0.577   -0.115   -0.084    -0.059

Table: (\#tab:simtab2) Minimum values for $\bar{r}_d$ for clonal microsatellite
data separated by mutation rate, clone correction, and sample size.

```{r sim1, echo = FALSE, fig.cap = figcap1, fig.scap = figscap1}
knitr::include_graphics("figure/simulations/rd_sexrate.png")
cap <- "Violin plots showing the decay of the Index of Association measured with
declining rates of sexual reproduction for data sets simulated with even and
uneven mutation rates over 20 and 21 loci, respectively (columns) and 
sub-sampled in four different population sizes (rows). $\\bar{r}_d$ was
calculated for both whole and clone corrected data sets. Color indicates whether
or not the calculations were performed on whole or clone-corrected data sets.
Each violin plot contains 1000 unique data sets. Black lines mark the 25, 50,
and 75th percentile."

scap <- "Violin plots showing the decay of the Index of Association measured
with declining rates of sexual reproduction."

figcap1  <- beaverdown::render_caption(caption = cap, figname = "fig1", to = TO)
if (!LATEX) figcap1 <- gsub("@ref", "\\\\@ref", figcap1)
figscap1 <- beaverdown::render_caption(caption = scap, figname = "fig1s", to = TO)
```

We found that the power (p <= 0.01) to detect non-random mating decreases
significantly with very little sexual reproduction (Fig. \@ref(fig:sim4)). This
is further affected by both mutation rate, sample size, and clone correction.
Clone correction, however, appears to affect even mutation rates more strongly
than uneven mutation rates. For all scenarios the power to detect non-random
mating crashes to below 0.25 with rates of sexual reproduction greater than 1%.
This trend is only moderately improved when the threshold is raised to p <=
0.05. Strikingly, there is less power to detect non-random mating at 0% and
0.01% sex as compared to 0.05% and 0.1% sex. This appears to be due to
dramatically negative values of $\bar{r}_d$ which result in a p-value of 1. At p
<= 0.01, there additionally appears to be a false positive rate of 1.5% in the
worst case scenario.

```{r sim4, echo = FALSE, fig.cap = figcap3, fig.scap = figscap3, out.width = "100%"}
knitr::include_graphics("figure/simulations/ssr_power.png")
figcap3 <- "Power (true positive rate) to detect non-random mating decreasing
with increasing rates of sexual reproduction in SSR data. Plots are arranged in
a grid with clone-correction in rows and mutation rate in columns. Values are
shown for $\\alpha$ = 0.01"
figcap3  <- beaverdown::render_caption(caption = figcap3, figname = "fig1", to = TO)
figscap3 <- "Power (true positive rate) to detect non-random mating decreasing
with increasing rates of sexual reproduction in SSR data."
```

The AURC analysis showed a steady decrease in the ability to detect non-random
mating with increasing levels of sexual reproduction, mainly due to a loss in
sensitivity with increasing levels of $\alpha$ (Fig. \@ref(fig:sim2)). As seen
in the power analysis (Fig. \@ref(fig:sim4)), clone correction consistently
appears to negatively affect the AURC. For all scenarios, the diagnostic power
of $\bar{r}_d$ to detect non-random mating is no better than random at 10% or
greater rates of sexual reproduction. 

```{r sim2, echo = FALSE, fig.cap = figcap2, fig.scap = figscap2}
knitr::include_graphics("figure/simulations/AURC_box_plot.png")
figcap2 <- "Box plots showing the distribution of the area under the ROC Curve
for each independent seed. boxes span the interquartile range (IQR) and the
whiskers extend to 1.5 * IQR. An AURC of 1 indicates a perfect classifier, while
an AURC of 0.5 indicates the classifier is no better than a random guess. Any
values below 0.5 (shaded in grey) is considered worse than random."

figscap2 <- "Box plots showing the distribution of the area under the ROC Curve for 
each independent seed."
```

A three-way ANOVA adds support for the hypothesis that clone correction
significantly affects the diagnostic power of $\bar{r}_d$ (Fig. \@ref(fig:sim3),
Table \@ref(tab:sim3)). The results show that, while there are no significant
effects at 50% sex, clone correction, mutation rate, and sample size all have a
very real effect on $\bar{r}_d$'s diagnostic properties. At low levels of sex (<
1%), the interaction of clone correction and mutation rate shows a significant
effect, which is not seen in the interaction of clone correction and sample
size. Only at sex rates of 0.01% and 0.05% do we see significant effects across
all interactions.

```{r sim3, echo = FALSE, fig.cap = figcap3, fig.scap = figscap3, out.width = "100%"}
knitr::include_graphics("figure/simulations/AURC_ANOVA.png")
figcap3 <- "Heatmap representing p-values from ANOVA analysis on AURC
distributions with the model $AURC \\sim Clone Correction \\times Sample Size 
\\times Mutation Rate$ for each rate of sexual reproduction separately."
figcap3  <- beaverdown::render_caption(caption = figcap3, figname = "fig1", to = TO)
figscap3 <- "Heatmap representing p-values from ANOVA analysis on AURC distributions."
```


sexrate   term             sumsq     df    statistic   p.value
--------  -------------  -------  -----  -----------  --------
0.0000    (Intercept)     63.936      1    10569.674     0.000
          CC               0.855      1      141.309     0.000
          SS               0.060      3        3.289     0.020
          MR               0.556      1       91.913     0.000
          CC X SS          0.001      3        0.049     0.985
          CC X MR          0.174      1       28.747     0.000
          SS X MR          0.002      3        0.100     0.960
          CC X SS X MR     0.000      3        0.006     0.999
          Residuals        9.582   1584           NA        NA
0.0001    (Intercept)     70.972      1    14652.094     0.000
          CC               0.496      1      102.400     0.000
          SS               0.027      3        1.886     0.130
          MR               0.340      1       70.257     0.000
          CC X SS          0.009      3        0.596     0.617
          CC X MR          0.076      1       15.698     0.000
          SS X MR          0.001      3        0.077     0.972
          CC X SS X MR     0.003      3        0.241     0.868
          Residuals        7.673   1584           NA        NA
0.0005    (Intercept)     91.939      1   245382.751     0.000
          CC               0.072      1      191.181     0.000
          SS               0.089      3       78.753     0.000
          MR               0.050      1      132.414     0.000
          CC X SS          0.047      3       42.205     0.000
          CC X MR          0.027      1       72.663     0.000
          SS X MR          0.035      3       31.105     0.000
          CC X SS X MR     0.018      3       15.818     0.000
          Residuals        0.593   1584           NA        NA
0.0010    (Intercept)     86.202      1   157278.033     0.000
          CC               0.224      1      408.905     0.000
          SS               0.269      3      163.853     0.000
          MR               0.113      1      206.697     0.000
          CC X SS          0.135      3       82.053     0.000
          CC X MR          0.060      1      109.071     0.000
          SS X MR          0.067      3       40.994     0.000
          CC X SS X MR     0.034      3       20.426     0.000
          Residuals        0.868   1584           NA        NA
0.0050    (Intercept)     52.266      1    14127.572     0.000
          CC               0.936      1      253.111     0.000
          SS               2.592      3      233.531     0.000
          MR               0.138      1       37.393     0.000
          CC X SS          0.234      3       21.126     0.000
          CC X MR          0.028      1        7.675     0.006
          SS X MR          0.026      3        2.385     0.068
          CC X SS X MR     0.022      3        1.974     0.116
          Residuals        5.860   1584           NA        NA
0.0100    (Intercept)     39.376      1     5724.212     0.000
          CC               0.686      1       99.672     0.000
          SS               2.455      3      118.955     0.000
          MR               0.068      1        9.817     0.002
          CC X SS          0.173      3        8.380     0.000
          CC X MR          0.003      1        0.378     0.539
          SS X MR          0.051      3        2.488     0.059
          CC X SS X MR     0.056      3        2.720     0.043
          Residuals       10.896   1584           NA        NA
0.0500    (Intercept)     28.671      1     1903.303     0.000
          CC               0.072      1        4.806     0.029
          SS               0.239      3        5.293     0.001
          MR               0.001      1        0.080     0.778
          CC X SS          0.391      3        8.646     0.000
          CC X MR          0.000      1        0.032     0.858
          SS X MR          0.037      3        0.809     0.489
          CC X SS X MR     0.001      3        0.028     0.994
          Residuals       23.861   1584           NA        NA
0.1000    (Intercept)     26.010      1     1441.495     0.000
          CC               0.009      1        0.524     0.469
          SS               0.131      3        2.419     0.065
          MR               0.000      1        0.001     0.979
          CC X SS          0.162      3        2.984     0.030
          CC X MR          0.000      1        0.013     0.908
          SS X MR          0.014      3        0.256     0.857
          CC X SS X MR     0.001      3        0.015     0.998
          Residuals       28.581   1584           NA        NA
0.5000    (Intercept)     25.669      1     1489.342     0.000
          CC               0.001      1        0.033     0.857
          SS               0.042      3        0.806     0.491
          MR               0.002      1        0.099     0.753
          CC X SS          0.001      3        0.025     0.995
          CC X MR          0.000      1        0.006     0.939
          SS X MR          0.004      3        0.076     0.973
          CC X SS X MR     0.001      3        0.011     0.998
          Residuals       27.301   1584           NA        NA

Table: (\#tab:sim3) ANOVA table comparing the model $AURC \sim Clone Correction 
\times Sample Size \times Mutation Rate$ for each rate of sexual reproduction
separately. CC = Clone Correction, SS = Sample Size, MR = Mutation Rate.


### SNP Data

Because of the computational intensity of the simulations we were only
able to run 24 unique seeds over all rates of sexual reproduction with 10
replicates per seed. Because some replicates failed to save, we randomly sampled
five replicates per seed per rate of sexual reproduction, giving us a total of 
1200 total populations for analysis.

```{r sim5, echo = FALSE, fig.cap = figcap5, fig.scap = figscap5}
knitr::include_graphics("figure/simulations/genomic_rd.png")
cap <- "Violin plots showing the decay of the Index of Association measured with
declining rates of sexual reproduction for genomic-scale data sets sub-sampled
in four different population sizes. $\\bar{r}_d$ was calculated for both whole
and clone corrected data sets. points represent the observed values. Black lines
mark the 25, 50, and 75th percentile."

scap <- "Violin plots showing the decay of the Index of Association measured
with declining rates of sexual reproduction for genomic-scale data."

figcap5  <- beaverdown::render_caption(caption = cap, figname = "fig2", to = TO)
if (!LATEX) figcap1 <- gsub("@ref", "\\\\@ref", figcap1)
figscap5 <- beaverdown::render_caption(caption = scap, figname = "fig2s", to = TO)
```


```{r sim6, echo = FALSE, fig.cap = figcap6, fig.scap = figscap6}
knitr::include_graphics("figure/simulations/AURC_genomic.png")
figcap6 <- "Plot of AURC for genomic data."

figscap6 <- "Plot of AURC for genomic data."
```

```{r sim7, echo = FALSE, fig.cap = figcap7, fig.scap = figscap7}
knitr::include_graphics("figure/simulations/genomic_power.png")
figcap7 <- "Power (true positive rate) to detect non-random mating decreasing with
increasing rates of sexual reproduction in genomic-scale data sub-sampled in
four different population sizes. False positive rate is shown as increasing
transparency."

figscap7 <- "Power to detect non-random mating decreasing with increasing rates of
sexual reproduction in genomic data."
```

## Discussion

### Not using multiple tests correction

We did not use multiple tests correction on the p-values derived from tests of
significance on $\bar{r}_d$ because each p-value represents that derived from
an independent study on an independent population as opposed to independent
variables within a population. Much like we wouldn't perform a multiple tests
correction on the results of 1,000 NIH studies, we wouldn't perform one on the
results of 1,000 independently evolved populations. 