# [Tentative Title] The Effect of Population Dynamics, Sample Size, and Marker Choice on the Index of Association

## Abstract

TBD...

## Introduction

<!-- ZHIAN

 Thinking about different topics to address here. 

 -->

 - Population Genetics of partially clonal organisms
 	- This has been studied in the past [@smith1993how; @balloux2003population;
 	  @de2004clonal; @orive1993effective]
 	- The index of association [@brown1980multilocus; @smith1993how;
 	  @Agapow_2001]
 	- In @de2004clonal, it was shown that $\bar{r}_d$ has a high variance in
 	  clonal populations and @smith1993how showed that it's affected by
 	  population structure.
 - Methods for assessing level of clonal reproduction (CloNcaSe)
   [@ali2016cloncase]
 	- This only works for populations with discrete generations with an
 	  observable sexual stage.
 - Limitations of previous studies
 	- No HTS markers
 	- Significance tests are routinely performed via permutation analysis, but
 	  could not be performed due to software limitations [@burt1996molecular;
 	  @de2004clonal]
 - Objectives
 	1. Analyze mixtures of clonal populations to assess effect of sampling
 	   multiple clonal populations
 	2. Re-analyze rates of sexual reproduction to confirm previous study
 	3. Assess significance tests for $\bar{r}_d$


## Methods

All simulations were performed with the python package simuPOP version 1.1.7 in
python version 3.4. For each scenario, 100 simulations with 10 replicates were
created with a census size of 10,000 individuals with equal mating type
proportions evolved over 10,000 generations. From each replicate, 10, 25, 50,
and 100 individuals were sampled without replacement for downstream analysis in
R version 3.2 with the package *poppr* version 2.2.1 on full and clone-corrected
data sets. All downstream analyses were run via custom scripts on the OSU CGRB
Core Computing Facility (supplementary information).

### Sexual Reproduction

Simulations of sexual reproduction were analyzed at 10 rates of sexual
reproduction on a log scale (0.0, 1e-4, 5e-4, 1e-3, 5e-3, 1e-2, 5e-2, 0.1, 0.5,
1.0) reflecting the fraction of individuals in generation *t+1* produced via
sexual reproduction. One to three offspring could be produced at each mating
event. For sexual events, two parents were chosen randomly from the population
with the `RandomSelection()` operator and offspring genotypes were created via the
`MendelianGenoTransmitter()` operator. The clonal fraction was created by randomly
sampling individuals from the population and duplicating their genotypes with
the `CloneGenoTransmitter()` method. If one mating type was lost before 10,000
generations, the simulation would continue to completion with only clonal
reproduction.

### Microsatellite Simulation

Each population was simulated with 20 co-dominant loci containing 6 to 10
alleles with frequencies drawn from a uniform distribution and subsequently
normalized. Before mating, mutations occurred at each locus at a rate of 1e-5
mutations/generation in a stepwise manner using the `StepwiseMutator()`
operator.


### Microsatellite Analysis


The standardized index of association ($\bar{r}_d$) was calculated using the
*poppr* function `ia()`.  Tests for significance were performed by randomly
permuting the alleles at each locus independently and then assessing
$\bar{r}_d$. This was done 999 times for each replicate population. The p-values
reflect the proportion of observations greater than the observed statistic.
Estimates of genotypic diversity were assessed with the *poppr* function
`diversity_boot()` with 999 bootstrap replicates, recording the variance.


### GBS Simulations

Simulations of 10,000 binary loci at intervals of 1 Mbp over 100 chromosomal
fragments were simulated with a mutation rate of 1e-5 mutations per generation
and a recombination rate of 1e-5 for sexually recombining populations.