# [Tentative Title] The Effect of Population Dynamics, Sample Size, and Marker Choice on the Index of Association

## Abstract

TBD...

## Introduction

<!-- ZHIAN

 Thinking about different topics to address here. 

 -->

Population genetics largely is based off of the neutral assumption that
populations exist in Hardy-Weinberg Equilibrium (HWE) in that they are
infinitely large, with no migration, no mutation, and randomly mating, where
alleles across loci are I.I.D. This neutral model works well to describe
populations that reproduce exclusively via sexual reproduction, but it becomes
less useful when considering populations that undergo some form of clonal
reproduction since the alleles for different samples are no longer I.I.D. This
confounds the ability to accurately assess population demographics including
effective population size [@orive1993effective]. Many clonal organisms are also
pathogenic, and knowing basic population dynamics of these pathogens is 
important for management strategies [@smith1993how; @milgroom1996recombination;
@milgroom1997contributions; @de2006molecular; @nieuwenhuis2016frequency]. 

The theory of population genetics for clonal organisms has been deeply
investigated over time [@orive1993effective; @halkett2005tackling;
@arnaud2007standardizing]. In particular, there has been a re-occurring question
of how much sex can be detected in a population of organisms with (partially)
clonal reproduction [@smith1993how; @balloux2003population; @de2004clonal;
@ali2016cloncase; @nieuwenhuis2016frequency]. For populations with well-defined
sexual and clonal phases occurring at separate times, methods like *CloNcaSe*
are effective for estimating the rate of sexual reproduction and effective
population size [@ali2016cloncase]. However, when sexual reproduction is not a
cyclical phenomena, this estimator is useless. Currently, a method commonly used
to assess clonal reproduction is the index of assocation ($I_A$), and its
standardized version, $\bar{r}_d$, which measure multilocus linkage
disequilibrium [@brown1980multilocus; @smith1993how; @haubold1998detecting;
@Agapow_2001; @de2004clonal; @kamvar2014poppr]. The value of $I_A$ is measured
as the ratio of observed variance in genetic distance between samples ($V_O$)
and expected variance ($V_E$) (equation \@ref(eq:ia)) [@smith1993how;
@Agapow_2001].

\begin{equation}
I_A = \frac{V_O}{V_E} - 1 (\#eq:ia)
\end{equation}

The expected variance is practically modeled as the sum of the variances over 
*m* loci: $V_E = \sum^m{var_j}$ [@Agapow_2001; @haubold1998detecting]. If the 
differences between samples are randomly distributed, we can expect the value of
$I_A$ to be zero [@smith1993how; @Agapow_2001]. Under scenarios of non-random
mating (e.g. population structure or clonal reproduction), the observed variance
would be greater than the expected due to a bimodal distribution of distances,
and $I_A$ would be greater than zero [@smith1993how; @Agapow_2001;
@milgroom2015population]. @Agapow_2001 noted that this metric does not have an
upper limit and increases with the number of loci. To correct this, they 
developed $\bar{r}_d$ (equation \@ref(eq:rd)), which has a similar structure to
a correlation coefficient and ranges from 0 (no linkage) to 1 (complete
linkage).

\begin{align} % This mode aligns the equations to the '&=' signs
\begin{split} % This mode groups the equations into one.
\bar{r}_d &= \frac{\sum\sum{cov_{j,k}}}{
                   \sum\sum{\sqrt{var_{j} \cdot var_{k}}}} \\
          &= \frac{V_O - V_E}{2\sum\sum{\sqrt{var_{j} \cdot var_{k}}}}
\end{split}
(\#eq:rd)
\end{align}

@smith1993how demonstrated that $I_A$ has the ability to detect non-random
mating due to population structure, strict clonal reproduction, or a recent
epidemic, but at that time, no one had quantified how much clonal reproduction
had to occur to produce a significant value of $I_A$. This was of significant
concern, considering that, as with any summary statistic, a lot of information
is lost when evaluating a population based on $I_A$ [@brown1980multilocus]. One
way to assess the effect of different population genetic scenarios was via
simulations.

The effect of increasing levels of sexual reproduction on $\bar{r}_d$ was
investigated by @de2004clonal (noted in their publication as $\bar{r}_D$). It
was found that the mean value of $\bar{r}_d$ decreased rapidly with relatively
low levels of sexual reproduction, with large variances at low levels of sexual
reproduction. This indicates that $\bar{r}_d$ alone might not be well suited as 
a measure of clonal reproduction. The effect of sampling design was on
$\bar{r}_d$ was tested by @prugnolle2010apparent, finding that the value of
$\bar{r}_d$ is drastically reduced when clones from multiple populations are
sampled, which can lead to an over-estimation of the level of recombination.

These studies laid the groundwork for understanding the behavior of $\bar{r}_d$
under different scenarios of non-random mating, but there were some limitations
to their studies. For both studies, the only software available to analyze
$\bar{r}_d$ was <span style="font-variant:small-caps;">Multilocus</span>, which
could only take one data set at a time [@Agapow_2001; @de2004clonal;
@prugnolle2010apparent; @kamvar2014poppr], meaning that the researchers could
only analyze a minimal set of populations (20) per scenario. Furthermore, while
a permutation test of significance existed in
<span style="font-variant:small-caps;">Multilocus</span>, it was computationally
expensive and significance testing was not performed.

Our current study aims to fill in the gaps left by the limitations of the
naughties. In 2014, we developed the R package *poppr* for analysis of clonal
populations, removing the limitations of data input and computational expense
[@R2016; @kamvar2014poppr]. These analyses can thus be performed with an
increased number of samples and, utilizing significance testing, assess the
power of $\bar{r}_d$ to detect clonal reproduction [@de2004clonal]. Furthermore,
with the advent of high-throughput sequencing methods such as 
Genotyping-By-Sequencing (GBS), we had previously extended the index of 
association to genome-scale SNP data [@kamvar2015novel]. 

 - Population Genetics of partially clonal organisms
 	- This has been studied in the past [@smith1993how; @balloux2003population;
 	  @de2004clonal; @orive1993effective]
 	- The index of association [@brown1980multilocus; @smith1993how;
 	  @Agapow_2001]
 	- In @de2004clonal, it was shown that $\bar{r}_d$ has a high variance in
 	  clonal populations and @smith1993how showed that it's affected by
 	  population structure.
 - Methods for assessing level of clonal reproduction (CloNcaSe)
   [@ali2016cloncase]
 	- This only works for populations with discrete generations with an
 	  observable sexual stage.
 - Limitations of previous studies
 	- No HTS markers
 	- Significance tests are routinely performed via permutation analysis, but
 	  could not be performed due to software limitations [@burt1996molecular;
 	  @de2004clonal]
 - Questions
 	1. With the advent of GBS, does marker choice affect inference of clonality?
 	2. Does sample size significantly affect our ability to detect clonal
 	   reproduction?
 	3. Can a combination of genotypic diversity estimators and allelic diversity
 	   estimator provide support for significantly negative values of
 	   $\bar{r}_d$?
 - Objectives
 	1. Analyze mixtures of clonal populations to assess effect of sampling
 	   multiple clonal populations.
 	1. Assess the effect of a single hypervariable locus on $\bar{r}_d$
 	2. Re-analyze rates of sexual reproduction to confirm previous study
 	   with and without clone-correction [@de2004clonal].
 	3. Power analysis of $\bar{r}_d$.
 	4. Investigate dynamics of populations with significantly negative (p = 1)
 	   values of $\bar{r}_d$.
	5. Assess $E_5$, *H*, $eMLG_{10}$, and unbiased Simpson's diversity.



## Methods

Initial sets of simulations were created for different levels of sexual
reproduction for each marker type. All simulations were performed with the
python package simuPOP version 1.1.7 in python version 3.4. For each scenario,
100 simulations with 10 replicates were created with a census size of 10,000
diploid individuals with equal mating type proportions evolved over 10,000
generations. From each replicate, 10, 25, 50, and 100 individuals were sampled
without replacement for downstream analysis in R version 3.2 with the package
*poppr* version 2.2.1 on full and clone-corrected data sets. All downstream
analyses were run on the OSU CGRB Core Computing Facility (supplementary
information).

### Simulating Microsatellite Loci

Each population was simulated with 20 co-dominant, unlinked loci containing 6 to
10 alleles per locus with frequencies drawn from a uniform distribution and
subsequently normalized. Before mating, mutations occurred at each locus at a
rate of 1e-5 mutations/generation in a stepwise manner using the
`StepwiseMutator()` operator.

### GBS Simulations

Simulations of 10,000 binary loci at spread evenly over 10 chromosomal fragments
were simulated with a mutation rate of 1e-5 mutations per generation for forward
and backward mutations using the `SNPMutator()` operator and and a recombination
rate of 1e-5 between adjacent loci using the `Recombinator()` operator.

### Sexual Reproduction

Simulations of sexual reproduction were analyzed at 10 rates of sexual
reproduction on a log scale (0.0, 1e-4, 5e-4, 1e-3, 5e-3, 1e-2, 5e-2, 0.1, 0.5,
1.0) reflecting the fraction of individuals in generation *t+1* produced via
sexual reproduction. One to three offspring could be produced at each mating
event. For sexual events, two parents were chosen randomly from the population
with the `RandomSelection()` operator and offspring genotypes were created via
the `MendelianGenoTransmitter()` operator. The clonal fraction was created by
randomly sampling individuals from the population and duplicating their
genotypes with the `CloneGenoTransmitter()` operator. If one mating type was
lost before 10,000 generations, the simulation would continue to completion with
only clonal reproduction.

### Analysis of Microsatellite Data

The standardized index of association ($\bar{r}_d$) was calculated for full and
clone-corrected data using the *poppr* function `ia()`. Tests for significance
were performed by randomly permuting the alleles at each locus independently and
then assessing $\bar{r}_d$. This was done 999 times for each replicate
population. Re-sampling was not performed with clone-corrected data. The p-values
reflect the proportion of observations greater than the observed statistic.
Estimates of genotypic diversity were assessed with the *poppr* function
`diversity_boot()` with 999 bootstrap replicates, recording the estimate and
variance.

### Analysis of SNP Data

Because GBS data are associated with high error rates, we additionally wanted to
assess the effect of error and missing data on analysis. To do this, we used
scripts written for @kamvar2015novel to randomly insert missing data (via the
`pop_NA()` function) and genotyping errors (via the `pop_mutator()`)
[@kamvar2015poppr2supp] at rates of 5% and 10% each. We then used the *poppr*
function `mlg.filter()` to with the farthest neighbor algorithm to collapse
multilocus genotypes with 5% and 10% similarity and compared the results with
the known genotypes. Genotypic diversity was assessed with the *poppr* function
`diversity_boot()` with 999 bootstrap replicates on the filtered data sets,
recording the variance.

The overall value of $\bar{r}_d$ was calculated for each simulation with the
*poppr* function `bitwise.ia()`. To assess within-chromosome dynamics, values of
$\bar{r}_d$ were calculated along chromosomes using the *poppr* function
`win.ia()` along windows of 50 and 100 SNPs. Cross-chromosome dynamics of
$\bar{r}_d$ were assessed by utilizing random samples of 50, 100, and 1000 SNPs
replicated 999 times for each population sample. 

### Power Analysis

The Receiver Operating Characteristic (ROC) curve is a method of assessing the
explanatory power of a method by assessing the true positive fraction of tests
to a false positive fraction [@metz1978basic]. Briefly, if a method has perfect
explanatory power, the area under the ROC curve will be equal to 1. If a method
has no explanatory power, the area under the ROC curve will be equal to 0.5.
ROC analysis was carried as described in @metz1978basic as a comparison of the
false positive fraction to the true positive fraction as you increase the level
of rejection, ($\alpha$). The null hypothesis for the ROC analysis was defined
where the rate of sexual reproduction was 100%. All other rates were considered
to be separate alternative hypotheses. The curves were calculated by analyzing
the data sets over increasing values of $\alpha$ from 0 to 1 in increments of
0.01. The area under the ROC curves was calculated using the `trapz()` function
in the R package *pracma* version 1.9.5 [@borchers2016pracma]. ANOVA was run
with the following models:

|                                               |
|:---------------------------------------------:|
| False Positive Rate ~ Sex Rate + Sample Size  |
| Area Under ROC Curve ~ Sex Rate + Sample Size |

