# Factors Influencing Recombination Inference in Diploid Populations

```{r, results = "asis", echo = FALSE}
out <- "
\\singlespacing
\\begin{center}
"
cat(beaverdown::iflatex(out))
```


Zhian N. Kamvar and Niklaus J. GrÃ¼nwald


```{r, results = "asis", echo = FALSE}
out <- "
\\end{center}
\\vspace*{\\fill}"
cat(beaverdown::iflatex(out))
```

Target Journal: **Molecular Ecology**

```{r, results = "asis", echo = FALSE}
out <- "
\\doublespacing
\\newpage
"
cat(beaverdown::iflatex(out))
```


## Abstract

TBD...

## Introduction

<!-- ZHIAN

 Thinking about different topics to address here. 

 -->

Population genetics largely is based off of the neutral assumption that
populations exist in Hardy-Weinberg Equilibrium (HWE) in that they are
infinitely large, with no migration, no mutation, and randomly mating, where
alleles across loci are I.I.D. This neutral model works well to describe
populations that reproduce exclusively via sexual reproduction, but it becomes
less useful when considering populations that undergo some form of clonal
reproduction since the alleles for different samples are no longer I.I.D. This
confounds the ability to accurately assess population demographics including
effective population size [@orive1993effective]. 


<!-- ZHIAN

Expand on this point here... yes, many populations are pathogenic,
but why do we care? How is this fact important for management strategies? It
will be useful to go further into the biology here.
-->
Many clonal organisms are also
pathogenic, and knowing basic population dynamics of these pathogens is 
important for management strategies [@smith1993how; @milgroom1996recombination;
@milgroom1997contributions; @de2006molecular; @nieuwenhuis2016frequency]. 

The theory of population genetics for clonal organisms has been deeply
investigated over time [@orive1993effective; @halkett2005tackling;
@arnaud2007standardizing]. In particular, there has been a re-occurring question
of how much sex can be detected in a population of organisms with (partially)
clonal reproduction [@smith1993how; @balloux2003population; @de2004clonal;
@ali2016cloncase; @nieuwenhuis2016frequency]. For populations with well-defined
sexual and clonal phases occurring at separate times, methods like *CloNcaSe*
are effective for estimating the rate of sexual reproduction and effective
population size [@ali2016cloncase]. However, when sexual reproduction is not a
cyclical phenomena, this estimator is useless. Currently, a method commonly used
to assess clonal reproduction is the index of assocation ($I_A$), and its
standardized version, $\bar{r}_d$, which measure multilocus linkage
disequilibrium [@brown1980multilocus; @smith1993how; @haubold1998detecting;
@Agapow_2001; @de2004clonal; @kamvar2014poppr]. The value of $I_A$ is measured
as the ratio of observed variance in genetic distance between samples ($V_O$)
and expected variance ($V_E$) (equation \@ref(eq:ia)) [@smith1993how;
@Agapow_2001].

\begin{equation}
I_A = \frac{V_O}{V_E} - 1 (\#eq:ia)
\end{equation}

The expected variance is practically modeled as the sum of the variances over 
*m* loci: $V_E = \sum^m{var_j}$ [@Agapow_2001; @haubold1998detecting]. If the 
differences between samples are randomly distributed, we can expect the value of
$I_A$ to be zero [@smith1993how; @Agapow_2001]. Under scenarios of non-random
mating (e.g. population structure or clonal reproduction), the observed variance
would be greater than the expected due to a bimodal distribution of distances,
and $I_A$ would be greater than zero [@smith1993how; @Agapow_2001;
@milgroom2015population]. @Agapow_2001 noted that this metric does not have an
upper limit and increases with the number of loci. To correct this, they 
developed $\bar{r}_d$ (equation \@ref(eq:rd)), which has a similar structure to
a correlation coefficient and ranges from 0 (no linkage) to 1 (complete
linkage).

\begin{align} % This mode aligns the equations to the '&=' signs
\begin{split} % This mode groups the equations into one.
\bar{r}_d &= \frac{\sum\sum{cov_{j,k}}}{
                   \sum\sum{\sqrt{var_{j} \cdot var_{k}}}} \\
          &= \frac{V_O - V_E}{2\sum\sum{\sqrt{var_{j} \cdot var_{k}}}}
\end{split}
(\#eq:rd)
\end{align}

@smith1993how demonstrated that $I_A$ has the ability to detect non-random
mating due to population structure, strict clonal reproduction, or a recent
epidemic, but at that time, no one had quantified how much clonal reproduction
had to occur to produce a significant value of $I_A$. This was of significant
concern, considering that, as with any summary statistic, a lot of information
is lost when evaluating a population based on $I_A$ [@brown1980multilocus]. One
way to assess the effect of different population genetic scenarios was via
simulations.

<!-- ZHIAN

This would be a good place to set up the need for simulation analyses and what
questions needed to be addressed.
 -->

The effect of increasing levels of sexual reproduction on $\bar{r}_d$ was
investigated by @de2004clonal (noted in their publication as $\bar{r}_D$). It
was found that the mean value of $\bar{r}_d$ decreased rapidly with relatively
low levels of sexual reproduction, with large variances at low levels of sexual
reproduction. This indicates that $\bar{r}_d$ alone might not be well suited as 
a measure of clonal reproduction. The effect of sampling design on $\bar{r}_d$ 
was tested by @prugnolle2010apparent, finding that the value of $\bar{r}_d$ 
is drastically reduced when clones from multiple populations are sampled, which 
can lead to an over-estimation of the level of recombination.

These studies laid the groundwork for understanding the behavior of $\bar{r}_d$
under different scenarios of non-random mating in diploid organisms, but there
were some limitations in available technology that prevented deep analysis from
being performed. For both studies, the only software available to analyze
$\bar{r}_d$ for diploid organisms was 
<span style="font-variant:small-caps;">Multilocus</span>, which could only take
one data set at a time [@Agapow_2001; @de2004clonal; @prugnolle2010apparent;
@kamvar2014poppr]. This constrained the researchers to only analyze a minimal
set of populations (20) per scenario. 

Since the distribution of $I_A$ and $\bar{r}_d$ are not known, the safest way to
test for significance were random permutation tests that effectively created 
unlinked populations by shuffling individuals at each locus, independently and
re-calculating $I_A$ and $\bar{r}_d$ [@smith1993how; @Agapow_2001; 
@haubold1998detecting]. A one-sided *t*-test of significance was then used to
see if the observed statistic was greater than the observed distribution. 

While significance testing was available in 
<span style="font-variant:small-caps;">Multilocus</span> in the form of
random permutations, it was computationally expensive, and the analytical
approach was only available in a software for haploid organisms [@Agapow_2001;
@haubold1998detecting; @haubold2000lian; @kamvar2014poppr]. As a result, power
analysis of $\bar{r}_d$ to detect clonal reproduction is still yet to be
performed [@de2004clonal]. 

Since these studies came out, reduced-representation, high-throughput sequencing
methods such as Genotyping-By-Sequencing (GBS) and RAD-seq have rapidly become
popular tools for population genetic analysis [@davey2010rad; @davey2011genome; 
@elshire2011robust]. These methods have the capability to generate thousands of
unlinked markers at a fraction of the cost and time necessary to develop high
quality microsatellite markers. These marker systems are also prone to high 
error rates [@mastretta2015restriction]. The index of association was developed
for multiple loci in a time when obtaining even 100 unlinked markers posed a
significant challenge. With the advent of these current technologies, how does
marker choice and genotyping error affect the index of association?

Our current study aims to fill in the gaps left by the technical limitations of 
previous studies. In 2014, we developed the R package *poppr* for analysis of
clonal populations, removing the limitations of data input and computational
expense of analyzing the index of association, and in 2015, we expanded this to
analysis of genome-wide SNP data [@R2016; @kamvar2014poppr; @kamvar2015novel].
With these tools we set expand on previous studies by asking how sample size,
marker choice, and clone-correction affect our ability to detect clonal
reproduction in diploid populations. Our objectives to answer these questions
are to (1) Re-analyze $\bar{r}_d$ against increasing rates of sexual
reproduction and different levels of population mixture in both microsatellite
and SNP data sets. (2) Perform a power analysis of $\bar{r}_d$. (3) Assess how
genotypic and allelic evenness and diversity affects $\bar{r}_d$. Because
studies have observed significantly negative values of the $I_A$ and $\bar{r}_d$
(p $\geq$ 0.95), we additionally seek to characterize these populations to find
common patterns that would indicate clonal or sexual reproduction.


<!--  - Population Genetics of partially clonal organisms
 	- This has been studied in the past [@smith1993how; @balloux2003population;
 	  @de2004clonal; @orive1993effective]
 	- The index of association [@brown1980multilocus; @smith1993how;
 	  @Agapow_2001]
 	- In @de2004clonal, it was shown that $\bar{r}_d$ has a high variance in
 	  clonal populations and @smith1993how showed that it's affected by
 	  population structure.
 - Methods for assessing level of clonal reproduction (CloNcaSe)
   [@ali2016cloncase]
 	- This only works for populations with discrete generations with an
 	  observable sexual stage.
 - Limitations of previous studies
 	- No HTS markers
 	- Significance tests are routinely performed via permutation analysis, but
 	  could not be performed due to software limitations [@burt1996molecular;
 	  @de2004clonal]
 - Questions
 	1. With the advent of GBS, does marker choice affect inference of clonality?
 	
 	2. Does sample size significantly affect our ability to detect clonal
 	   reproduction?
 	3. Can a combination of genotypic diversity estimators and allelic diversity
 	   estimator provide support for significantly negative values of
 	   $\bar{r}_d$?
 - Objectives
 	1. Analyze mixtures of clonal populations to assess effect of sampling
 	   multiple clonal populations.
 	1. Assess the effect of a single hypervariable locus on $\bar{r}_d$
 	2. Re-analyze rates of sexual reproduction to confirm previous study
 	   with and without clone-correction [@de2004clonal].
 	3. Power analysis of $\bar{r}_d$.
 	4. Investigate dynamics of populations with significantly negative (p = 1)
 	   values of $\bar{r}_d$.
	5. Assess $E_5$, *H*, $eMLG_{10}$, and unbiased Simpson's diversity.

 -->

## Methods

Initial sets of simulations were created for different levels of sexual
reproduction for each marker type. All simulations were performed with the
python package simuPOP version 1.1.7 in python version 3.4[@peng2008forward]. For each scenario,
100 simulations with 10 replicates were created with a census size of 10,000
diploid individuals with equal mating type proportions evolved over 10,000
generations. The simulated populations were first stored in the native simuPOP format and then transferred to feather format using the python and R packages *feather* version 0.3.0 for downstream analyses. Micorsatellite simulations were performed on Ubuntu Linux version 14.04; SNP simulations were performed on CentOS Linux version 6.8. During downstream analysis, 10, 25, 50, and 100 individuals were sampled
without replacement for each replicate in R version 3.2 with the package
*poppr* version 2.2.1 [@R2016; @kamvar2014poppr; @kamvar2015novel]. Analyses (described below) were performed on both full and clone-corrected data sets. All downstream
analyses were run on the OSU CGRB Core Computing Facility (supplementary
information).

### Simulating Microsatellite Loci

Each population was simulated with 21 co-dominant, unlinked loci containing 6 to
10 alleles per locus with frequencies drawn from a uniform distribution and
subsequently normalized. Before mating, mutations occurred at each locus at a
rate of 1e-5 mutations/generation with the exception of the first locus, at which the mutation rate was set to 1e-3. All mutations were applied in a stepwise manner using the
`StepwiseMutator()` operator in simuPOP.

### GBS Simulations

Simulations of 10,000 binary loci spread evenly over 10 chromosomal fragments
were simulated with a mutation rate of 1e-5 mutations per generation for forward
and backward mutations using the `SNPMutator()` operator and and a recombination
rate of 1e-5 between adjacent loci using the `Recombinator()` operator in
simuPOP. 

### Mating

Simulations of sexual reproduction were conducted at 10 rates of sexual
reproduction on a log scale (0.0, 1e-4, 5e-4, 1e-3, 5e-3, 1e-2, 5e-2, 0.1, 0.5,
1.0) reflecting the fraction of individuals in generation *t+1* produced via
sexual reproduction. One to three offspring could be produced at each mating
event. For sexual events, two parents were chosen randomly from the population
with the `RandomSelection()` operator and offspring genotypes were created via
the `MendelianGenoTransmitter()` operator. The clonal fraction was created by
randomly sampling individuals from the population and duplicating their
genotypes with the `CloneGenoTransmitter()` operator. If one mating type was
lost before 10,000 generations, the simulation would continue to completion with
only clonal reproduction.

### Analysis of Microsatellite Data

The standardized index of association ($\bar{r}_d$) was calculated for full and
clone-corrected data using the *poppr* function `ia()`. Tests for significance
were performed by randomly permuting the alleles at each locus independently and
then assessing $\bar{r}_d$. This was done 999 times for each replicate
population. This was done for both full and clone-corrected data. The p-values
reflect the proportion of observations greater than the observed statistic.
Estimates of genotypic diversity were assessed with the *poppr* function
`diversity_boot()` with 999 bootstrap replicates, recording the estimate and
variance. The genotypic diversity statistics we calculated were Shannon's Index,
$H = -\sum p_i ln p_i$ [@shannon2001mathematical], Stoddart and Taylor's Index,
$G = 1/\sum p_i^2$ [@stoddart1988genotypic], and Evenness, 
$E_5 = (G - 1)/(e^H - 1)$ [@pielou1975ecological] where $p_i$ is the frequency 
of the *i*th genotype.

We additionally Nei's 1978 expected heterozygosity, also known as gene diversity
[@Nei:1978], calculated mean allelic evenness $E_{5A} = (1/m) \sum E_{5l}$ 
where *m* is the number of loci and $E_{5l}$ is the evenness of the alleles at 
locus *l*.


### Analysis of SNP Data

Because GBS data are associated with high error rates, we additionally wanted to
assess the effect of missing data on analysis. To do this, we used
scripts written for @kamvar2015novel to randomly insert missing data via the
`pop_NA()` function [@kamvar2015poppr2supp] at rates of 1%, 5%, and 10% each. 

The overall value of $\bar{r}_d$ was calculated for each simulation with the
*poppr* function `bitwise.ia()`. Significance was first assessed by randomly
shuffling genotypes at each locus independently and then, to preserve existing 
background linkage structure, at each chromosome independently. This was done
999 times for each replicate population. P-values represent the proportion of 
random samples greater than or equal to the observed statistic.

### Power Analysis

Permutation analysis of the index of association is commonly used as a
diagnostic for detecting non-random mating. If the observed value of $\bar{r}_d$
is greater than 95% of the results from the permutations, then the population is
declared to be clonal. Because knowing a pathogen's mode of reproduction can
affect management strategies, it's important to know how sample size, rate of
sexual reproduction, mutation rate homogeneity, and clone-correction affect this
method's sensitivity and specificity.

The Receiver Operating Characteristic (ROC) curve is a method of assessing the
the balance between sensitivity and specificity of a diagnostic method [@metz1978basic]. This is done by simultaneously assessing the true positive fraction of tests
to a false positive fraction along a gradient of thresholds of increasing leniency. 
Briefly, if a method has perfect
explanatory power, the area under the ROC curve will be equal to 1. If a method
has no explanatory power, the area under the ROC curve will be equal to 0.5. We used this method to assess the efficacy of $\bar{r}_d$ to detect non-random mating. 

To calculate the ROC curve, we first have to define defined what a positive
value is. We define it as a significant value of $\bar{r}_d$ based on a
threshold, $\alpha$, where $\alpha$ is a value in [0, 1].
Therefore, any randomly mating population (sexual reproduction is equal to one)
with a significant value of $\bar{r}_d$ is considered a false positive, and
subsequently, any non-randomly mating population with a significant value of
$\bar{r}_d$ is considered a true positive (Table \@ref(tab:simtab1)).
 
 Reproductive Mode   $p \leq \alpha$   $p > \alpha$
------------------- ----------------- ---------------
 Non-Random Mating    True Positive    False Negative
   Random Mating     False Positive     True Negative

Table: (\#tab:simtab1) Definitions of false positive and true positive values
for ROC analysis of simulations. *p* is the p-value for $\bar{r}_d$, $\alpha$ is
a threshold value in [0, 1] Random Mating populations are simulated with a sex
rate of 1. Non-Random Mating populations are simulated with a sex rate less than
one.

We constructed each curve by assessing the ROC over values of $\alpha$ in
increments of 0.01 from 0 to 1 and plotting the true positive fraction on the y
axis and false positive fraction on the x axis. The area under the ROC curves
was calculated using the `auc()` function in the R package *flux* version 0.3-0
[@jurasinski2014flux]. These curves were calculated for the data separately for
sample size, mutation rate, and clone correction. In addition to the full data
set, we calculated separate curve for each rate of sexual reproduction less than
one and each seed. 

## Results

### Microsatellite Data

### SNP Data

Because of the computational intensity of the simulations we were only
able to run 24 unique seeds over all rates of sexual reproduction with 10
replicates per seed. Because some replicates failed to save, we randomly sampled
five replicates per seed per rate of sexual reproduction, giving us a total of 
1200 total populations for analysis.