<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Development and Application of Tools for Genetic Analysis of Clonal Populations</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Development and Application of Tools for Genetic Analysis of Clonal Populations">
  <meta name="generator" content="bookdown 0.1.18 and GitBook 2.6.7">

  <meta property="og:title" content="Development and Application of Tools for Genetic Analysis of Clonal Populations" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Development and Application of Tools for Genetic Analysis of Clonal Populations" />
  
  
  

<meta name="author" content="Zhian N. Kamvar">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="1-introduction.html">
<link rel="next" href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>


  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#population-genetics-and-clonality"><i class="fa fa-check"></i><b>1.1</b> Population Genetics and Clonality</a></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#population-genetics-under-the-lens-of-plant-pathology"><i class="fa fa-check"></i><b>1.2</b> Population Genetics Under the Lens of Plant Pathology</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-need-for-reproducible-research"><i class="fa fa-check"></i><b>1.3</b> The Need For Reproducible Research</a></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#overview"><i class="fa fa-check"></i><b>1.4</b> Overview</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-2"><i class="fa fa-check"></i><b>1.4.1</b> Summary of Chapter 2</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-3"><i class="fa fa-check"></i><b>1.4.2</b> Summary of Chapter 3</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-4"><i class="fa fa-check"></i><b>1.4.3</b> Summary of Chapter 4</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-5"><i class="fa fa-check"></i><b>1.4.4</b> Summary of Chapter 5</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><i class="fa fa-check"></i><b>2</b> <em>Poppr</em>: an R Package For Genetic Analysis of Populations With Clonal, Partially Clonal, and/or Sexual Reproduction</a><ul>
<li class="chapter" data-level="2.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#abstract"><i class="fa fa-check"></i><b>2.1</b> Abstract</a></li>
<li class="chapter" data-level="2.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#introduction-1"><i class="fa fa-check"></i><b>2.2</b> Introduction</a></li>
<li class="chapter" data-level="2.3" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#materials-and-methods"><i class="fa fa-check"></i><b>2.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#data-import"><i class="fa fa-check"></i><b>2.3.1</b> Data import</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#data-analysis"><i class="fa fa-check"></i><b>2.3.2</b> Data analysis</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#visualizations"><i class="fa fa-check"></i><b>2.3.3</b> Visualizations</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#performance"><i class="fa fa-check"></i><b>2.3.4</b> Performance</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#citation-of-methods-implemented-in-poppr"><i class="fa fa-check"></i><b>2.4</b> Citation of methods implemented in <em>poppr</em></a></li>
<li class="chapter" data-level="2.5" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#results-and-discussion"><i class="fa fa-check"></i><b>2.5</b> Results and Discussion</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#new-functionalities"><i class="fa fa-check"></i><b>2.5.1</b> New functionalities</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#performance-1"><i class="fa fa-check"></i><b>2.5.2</b> Performance</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
<li class="chapter" data-level="2.7" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#acknowledgements"><i class="fa fa-check"></i><b>2.7</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><i class="fa fa-check"></i><b>3</b> Novel R Tools For Analysis of Genome-Wide Population Genetic Data With Emphasis on Clonality</a><ul>
<li class="chapter" data-level="3.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#implementations-and-examples"><i class="fa fa-check"></i><b>3.3</b> Implementations and Examples</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#clonal-identification"><i class="fa fa-check"></i><b>3.3.1</b> Clonal identification</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#minimum-spanning-networks-with-reticulation"><i class="fa fa-check"></i><b>3.3.2</b> Minimum Spanning Networks with Reticulation</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#bootstrapping"><i class="fa fa-check"></i><b>3.3.3</b> Bootstrapping</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#genotype-accumulation-curve"><i class="fa fa-check"></i><b>3.3.4</b> Genotype Accumulation Curve</a></li>
<li class="chapter" data-level="3.3.5" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#index-of-association"><i class="fa fa-check"></i><b>3.3.5</b> Index of association</a></li>
<li class="chapter" data-level="3.3.6" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#data-format-updates-population-strata-and-hierarchies"><i class="fa fa-check"></i><b>3.3.6</b> Data format updates: population strata and hierarchies</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#availability"><i class="fa fa-check"></i><b>3.4</b> Availability</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#requirements"><i class="fa fa-check"></i><b>3.4.1</b> Requirements</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#installation"><i class="fa fa-check"></i><b>3.4.2</b> Installation</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
<li class="chapter" data-level="3.6" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#acknowledgements-1"><i class="fa fa-check"></i><b>3.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><i class="fa fa-check"></i><b>4</b> Spatial and Temporal Analysis of Populations of the Sudden Oak Death Pathogen in Oregon Forests</a><ul>
<li class="chapter" data-level="4.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#introduction-3"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#materials-and-methods-1"><i class="fa fa-check"></i><b>4.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#location"><i class="fa fa-check"></i><b>4.3.1</b> Location</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#sampling"><i class="fa fa-check"></i><b>4.3.2</b> Sampling</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#isolation-identification-and-dna-extraction"><i class="fa fa-check"></i><b>4.3.3</b> Isolation, identification and DNA extraction</a></li>
<li class="chapter" data-level="4.3.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#genotyping-data-validation-and-harmonization"><i class="fa fa-check"></i><b>4.3.4</b> Genotyping, data validation, and harmonization</a></li>
<li class="chapter" data-level="4.3.5" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#nursery-populations"><i class="fa fa-check"></i><b>4.3.5</b> Nursery populations</a></li>
<li class="chapter" data-level="4.3.6" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#data-analysis-1"><i class="fa fa-check"></i><b>4.3.6</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#results"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#demographic-pattern-and-genetic-diversity"><i class="fa fa-check"></i><b>4.4.1</b> Demographic pattern and genetic diversity</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#spatial-correlation"><i class="fa fa-check"></i><b>4.4.2</b> Spatial Correlation</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#population-differentiation"><i class="fa fa-check"></i><b>4.4.3</b> Population differentiation</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#clustering-of-forest-with-nursery-populations"><i class="fa fa-check"></i><b>4.4.4</b> Clustering of forest with nursery populations</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#discussion-1"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#acknowledgements-2"><i class="fa fa-check"></i><b>4.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.7" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-material"><i class="fa fa-check"></i><b>4.7</b> Supplementary Material</a><ul>
<li class="chapter" data-level="4.7.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#text:S1"><i class="fa fa-check"></i><b>4.7.1</b> Supplementary Text</a></li>
<li class="chapter" data-level="4.7.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-figures"><i class="fa fa-check"></i><b>4.7.2</b> Supplementary Figures</a></li>
<li class="chapter" data-level="4.7.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-tables"><i class="fa fa-check"></i><b>4.7.3</b> Supplementary Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><i class="fa fa-check"></i><b>5</b> Factors Influencing Inference of Clonality in Diploid Populations</a><ul>
<li class="chapter" data-level="5.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#introduction-4"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a><ul>
<li class="chapter" data-level="5.3.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#simulating-microsatellite-loci"><i class="fa fa-check"></i><b>5.3.1</b> Simulating Microsatellite Loci</a></li>
<li class="chapter" data-level="5.3.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#simulating-snp-loci"><i class="fa fa-check"></i><b>5.3.2</b> Simulating SNP Loci</a></li>
<li class="chapter" data-level="5.3.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#mating"><i class="fa fa-check"></i><b>5.3.3</b> Mating</a></li>
<li class="chapter" data-level="5.3.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#analysis-of-microsatellite-data"><i class="fa fa-check"></i><b>5.3.4</b> Analysis of Microsatellite Data</a></li>
<li class="chapter" data-level="5.3.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#analysis-of-snp-data"><i class="fa fa-check"></i><b>5.3.5</b> Analysis of SNP Data</a></li>
<li class="chapter" data-level="5.3.6" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#power-analysis"><i class="fa fa-check"></i><b>5.3.6</b> Power Analysis</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#clone-correction-negatively-impacts-clonal-inference"><i class="fa fa-check"></i><b>5.4.1</b> Clone Correction Negatively Impacts Clonal Inference</a></li>
<li class="chapter" data-level="5.4.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#negative-values-of-barr_d-in-clonal-populations-correlated-with-low-allelic-diversity"><i class="fa fa-check"></i><b>5.4.2</b> Negative Values of <span class="math inline">\(\bar{r}_d\)</span> in Clonal Populations Correlated With Low Allelic Diversity</a></li>
<li class="chapter" data-level="5.4.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#power-to-detect-clonal-reproduction-in-snp-data"><i class="fa fa-check"></i><b>5.4.3</b> Power To Detect Clonal Reproduction in SNP Data</a></li>
<li class="chapter" data-level="5.4.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#common-features-of-microsatellite-and-snp-markers"><i class="fa fa-check"></i><b>5.4.4</b> Common Features of Microsatellite and SNP Markers</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#discussion-2"><i class="fa fa-check"></i><b>5.5</b> Discussion</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#clone-correction-increases-false-negatives"><i class="fa fa-check"></i><b>5.5.1</b> Clone Correction Increases False Negatives</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#mutation-rates-and-barr_d"><i class="fa fa-check"></i><b>5.5.2</b> Mutation Rates and <span class="math inline">\(\bar{r}_d\)</span></a></li>
<li class="chapter" data-level="5.5.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#allelic-diversity-influences-the-value-of-barr_d-in-clonal-populations"><i class="fa fa-check"></i><b>5.5.3</b> Allelic Diversity Influences the value of <span class="math inline">\(\bar{r}_d\)</span> in Clonal Populations</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#linkage-in-snp-data-increases-barr_d-unilaterally"><i class="fa fa-check"></i><b>5.5.4</b> Linkage in SNP Data Increases <span class="math inline">\(\bar{r}_d\)</span> Unilaterally</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#limitations-of-this-study"><i class="fa fa-check"></i><b>5.5.5</b> Limitations of This Study</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#conclusions-1"><i class="fa fa-check"></i><b>5.5.6</b> Conclusions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-conclusions-2.html"><a href="6-conclusions-2.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a></li>
<li class="chapter" data-level="7" data-path="7-bibliography.html"><a href="7-bibliography.html"><i class="fa fa-check"></i><b>7</b> Bibliography</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Development and Application of Tools for Genetic Analysis of Clonal Populations</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> <em>Poppr</em>: an R Package For Genetic Analysis of Populations With Clonal, Partially Clonal, and/or Sexual Reproduction</h1>
<p>Zhian N. Kamvar, Javier F. Tabima, and Niklaus J. Grünwald</p>
<p>Journal: <strong>PeerJ</strong></p>
<p>PO Box 614, Corte Madera, CA 94976, USA</p>
<p>Published 2014-03-04, Issue: <strong>2</strong>:e281, DOI: <a href="https://dx.doi.org/10.7717/peerj.281">10.7717/peerj.281</a></p>
<div id="abstract" class="section level2">
<h2><span class="header-section-number">2.1</span> Abstract</h2>
<p>Many microbial, fungal, or oomcyete populations violate assumptions for population genetic analysis because these populations are clonal, admixed, partially clonal, and/or sexual. Furthermore, few tools exist that are specifically designed for analyzing data from clonal populations, making analysis difficult and haphazard. We developed the R package poppr providing unique tools for analysis of data from admixed, clonal, mixed, and/or sexual populations. Currently, poppr can be used for dominant/codominant and haploid/diploid genetic data. Data can be imported from several formats including GenAlEx formatted text files and can be analyzed on a user-defined hierarchy that includes unlimited levels of subpopulation structure and clone censoring. New functions include calculation of Bruvo’s distance for microsatellites, batch-analysis of the index of association with several indices of genotypic diversity, and graphing including dendrograms with bootstrap support and minimum spanning networks. While functions for genotypic diversity and clone censoring are specific for clonal populations, several functions found in poppr are also valuable to analysis of any populations. A manual with documentation and examples is provided. Poppr is open source and major releases are available on CRAN: <a href="http://cran.r-project.org/package=poppr" class="uri">http://cran.r-project.org/package=poppr</a>. More supporting documentation and tutorials can be found under ‘resources’ at: <a href="http://grunwaldlab.cgrb.oregonstate.edu/" class="uri">http://grunwaldlab.cgrb.oregonstate.edu/</a>.</p>
</div>
<div id="introduction-1" class="section level2">
<h2><span class="header-section-number">2.2</span> Introduction</h2>
<p>The Wright-Fisher model of populations is one of the oldest models utilized in population genetic theory. Populations in this model are characterized as having non-overlapping generations with a constant size free from any selective pressures <span class="citation">(Hartl &amp; Clark, <a href="#ref-hartl1997principles">2007</a>; Nielsen &amp; Slatkin, <a href="#ref-nielsen2013introduction">2013</a>; Weir &amp; Cockerham, <a href="#ref-weir1996genetic">1996</a>)</span>. Conceptually, these populations are represented as pools of alleles that are independently assorting where random mating is approximated by randomly sampling alleles with replacement from one generation to the next. Assumptions of this model, or related models, are implicitly assumed for common population genetic analysis tools. In clonal populations, however, alleles are not independently passed on from one generation to the next, and these assumptions are violated. Classical textbooks on population genetics do not provide much guidance on how to analyze clonal or mixed clonal and sexual populations. In reality, many populations are not strictly clonal or sexual, but can range from completely sexual to completely clonal and this is commonly observed for fungal, oomycete, or microbial populations <span class="citation">(Anderson &amp; Kohn, <a href="#ref-anderson1995clonality">1995</a>; Milgroom, <a href="#ref-milgroom1996recombination">1996</a>)</span>. Currently, analysis of these populations is not straightforward as we lack the sophisticated tools and methods developed for model populations that are typically either haploid or diploid <span class="citation">(Grünwald &amp; Goss, <a href="#ref-grunwald2011evolution">2011</a>)</span>.</p>
<p>Inferring population structure with many commonly used model-based clustering approaches such as the program <span style="font-variant: small-caps;"> Structure</span> <span class="citation">(Pritchard et al., <a href="#ref-pritchard2000inference">2000</a>)</span> is inherently problematic for clonal populations. These approaches cannot be used as clonal populations violate basic assumptions of panmixia and Hardy-Weinberg equilibrium. Thus, model free methods such as those relying on k-means clustering, dendrograms including bootstrap support for clades, or minimum spanning networks are more appropriate <span class="citation">(Cooke et al., <a href="#ref-cooke2012genome">2012</a>; Goss et al., <a href="#ref-goss2009population">2009</a>; Mascheretti et al., <a href="#ref-mascheretti2008reconstruction">2008</a>)</span>. Furthermore, analysis of mixed or clonal populations traditionally relies on calculation of diversity of genotypes observed and analysis of clone-censored versus non-censored populations <span class="citation">(Grünwald &amp; Hoheisel, <a href="#ref-grunwald2006hierarchical">2006</a>; McDonald, <a href="#ref-mcdonald1997population">1997</a>; Milgroom, <a href="#ref-milgroom1996recombination">1996</a>)</span>. Clone censoring involves reduction of any population sample to a single observation for each multilocus genotype (MLG) in a population thereby approximating panmictic populations and removing the effect of genetic linkage <span class="citation">(Milgroom, <a href="#ref-milgroom1996recombination">1996</a>)</span>. Analysis of diversity, in turn, involves calculation of the number of genotypes observed (richness), diversity, and evenness <span class="citation">(Grünwald et al., <a href="#ref-grunwald2003analysis">2003</a>)</span>. Typical measures of genotypic diversity are borrowed from ecology and use either the Shannon-Wiener or Stoddart and Taylor index <span class="citation">(Grünwald et al., <a href="#ref-grunwald2003analysis">2003</a>; Shannon, <a href="#ref-shannon2001mathematical">1948</a>; Stoddart &amp; Taylor, <a href="#ref-stoddart1988genotypic">1988</a>)</span>.</p>
<p>A critical aspect of analyzing clonal or mixed populations is testing a null hypothesis of panmixia <span class="citation">(Milgroom, <a href="#ref-milgroom1996recombination">1996</a>)</span>. Testing of this hypothesis for potentially clonal populations typically relies on assessment of linkage disequilibrium among loci <span class="citation">(Milgroom, <a href="#ref-milgroom1996recombination">1996</a>)</span>. This is achieved via calculation of the index of association or related indices in combination with resampling of the data to obtain a null distribution for the expectation of random mating <span class="citation">(Brown et al., <a href="#ref-brown1980multilocus">1980</a>; Burt et al., <a href="#ref-burt1996molecular">1996</a>; Milgroom, <a href="#ref-milgroom1996recombination">1996</a>; Smith et al., <a href="#ref-smith1993how">1993</a>)</span>. These approaches have, for example, been applied to <em>Pyrenophora teres</em> <span class="citation">(Peever &amp; Milgroom, <a href="#ref-peever1994genetic">1994</a>)</span> and <em>Aphanomyces euteiches</em> <span class="citation">(Grünwald &amp; Hoheisel, <a href="#ref-grunwald2006hierarchical">2006</a>)</span> and are routinely used in the analyses of clonal populations although they are not easily calculated given available software including <span style="font-variant: small-caps;">Multilocus</span>, which is no longer supported, and <span style="font-variant: small-caps;">LIAN</span>, which only works for haploids <span class="citation">(Agapow &amp; Burt, <a href="#ref-Agapow_2001">2001</a>; Haubold &amp; Hudson, <a href="#ref-Haubold:2000">2000</a>)</span>.</p>
<p>Hierarchical sampling adds another layer of complexity to analysis of clonal populations. With microbial populations, the geographic structure of each population is not entirely clear, and it is often important to sample temporally to see if clones persist over time <span class="citation">(Grünwald &amp; Hoheisel, <a href="#ref-grunwald2006hierarchical">2006</a>)</span>. A common approach when faced with multiple levels of sampling is to create a separate data set for each level or combination of levels and to analyze them separately. However, the number of data sets undergo a factorial increase with each hierarchical level, therefore increasing the chances of human error in data reformatting or analysis. Thus, tools are needed for analysis of population data across hierarchies or subsets of data.</p>
<p>Here, we introduce the R package <em>poppr</em> that is specifically designed for analysis of populations that are clonal, admixed and/or sexual. <em>Poppr</em> complements and builds on previously existing R packages including <em>adegenet</em> and <em>vegan</em> <span class="citation">(Jombart, <a href="#ref-Jombart_2008">2008</a>; Jombart &amp; Ahmed, <a href="#ref-jombart2011adegenet">2011</a>; Oksanen et al., <a href="#ref-oksanen2013vegan">2013</a>)</span> while implementing tools novel to R significantly facilitating data import, population genetic analyses, and graphing of clonal or partially clonal populations. These tools include among others: analysis across hierarchies of populations, subsetting of populations, clone-censoring, Bruvo’s genetic distance <span class="citation">Bruvo et al. (<a href="#ref-bruvo2004simple">2004</a>)</span>, the index of association and related statistics <span class="citation">(Brown et al., <a href="#ref-brown1980multilocus">1980</a>; Smith et al., <a href="#ref-smith1993how">1993</a>)</span>, and bootstrap support for trees based on Bruvo’s distance. By providing a centralized suite of tools appropriate for many data types, this package represents a novel and useful resource specifically tailored for analysis of clonal populations.</p>
</div>
<div id="materials-and-methods" class="section level2">
<h2><span class="header-section-number">2.3</span> Materials and Methods</h2>
<div id="data-import" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Data import</h3>
<p><em>Poppr</em> allows import of data in several formats for dominant/codominant, haploid/diploid and geographic data. The R package <em>adegenet</em>, that defines the <code>genind</code> data structure that <em>poppr</em> utilizes, allows support for importing data natively from <span style="font-variant: small-caps;">Structure, Genetix, Genepop</span>, and <span style="font-variant: small-caps;">Fstat</span>. While these formats are very common and widely supported, these do not allow for import of geographic and/or regional data. Furthermore, <em>adegenet</em> will only handle diploids with this format, though manual import is possible. To aid in importing data, <em>poppr</em> has newly added the function <code>read.genalex()</code>, to read data from <em>GenAlEx</em> formatted text files into the <code>genind</code> data object of the package <em>adegenet</em> <span class="citation">(Jombart, <a href="#ref-Jombart_2008">2008</a>; Jombart &amp; Ahmed, <a href="#ref-jombart2011adegenet">2011</a>; Peakall &amp; Smouse, <a href="#ref-Peakall:2006">2006</a>)</span>. <em>GenAlEx</em> is a popular add-in for <span style="font-variant: small-caps;">Microsoft Excel</span> that can handle data including codominant/dominant and haploid/diploid markers as well as geographic and regional data. This function further facilitates the import of haploid, geographic, and regional data.</p>
<p>Transferring data to new formats and manipulating data by hand, such as collapsing data into clones or subsetting data into different hierarchical levels, is tedious, creates redundancy, and can result in lost or misrepresented data. <em>Poppr</em> includes tools to automate such repetitive tasks. Many currently available data formats and software implementations allow analysis of only one or two levels of a population hierarchy. With <em>poppr</em> the user can import a single data set with an unlimited number of hierarchical levels. This is achieved by having the user combine the levels using a common delimiter (e.g. “Year_Country_City”). These combined levels are then used as the defining population factor in the input file and can easily be manipulated within R.</p>
</div>
<div id="data-analysis" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Data analysis</h3>
<p>Once data is imported into R, the user can dynamically access and manipulate the population hierarchy with the function <code>splitcombine()</code>, subset the data set by population with <code>popsub()</code>, and check for cloned multilocus genotypes using <code>mlg()</code>. For data sets that include clones, the <em>poppr</em> function <code>clonecorrect()</code> will censor exact clones with respect to any level of a population hierarchy by creating a new data set that includes only unique multilocus genotypes (MLGs) per population. A full list of functions available in <em>poppr</em> is provided in table <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#tab:poppr1">2.1</a>.</p>
<table>
<caption><span id="tab:poppr1">Table 2.1: </span> List of functions found in <em>poppr</em> and short descriptions.</caption>
<thead>
<tr class="header">
<th align="left"><strong>Function</strong></th>
<th align="left"><strong>Description</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><strong>Import/Export</strong></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left"><code>getfile</code></td>
<td align="left">Provides a quick GUI to grab files for import</td>
</tr>
<tr class="odd">
<td align="left"><code>read.genalex</code></td>
<td align="left">Read <em>GenAlEx</em> formatted csv files to a genind object</td>
</tr>
<tr class="even">
<td align="left"><code>genind2genalex</code></td>
<td align="left">Converts genind objects to <em>GenAlEx</em> formatted csv files</td>
</tr>
<tr class="odd">
<td align="left"><strong>Manipulation</strong></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left"><code>missingno</code></td>
<td align="left">Handles missing data</td>
</tr>
<tr class="odd">
<td align="left"><code>clonecorrect</code></td>
<td align="left">Clone censors at a specified population hierarchy</td>
</tr>
<tr class="even">
<td align="left"><code>informloci</code></td>
<td align="left">Detects and removes phylogenetically uninformative loci.</td>
</tr>
<tr class="odd">
<td align="left"><code>popsub</code></td>
<td align="left">Subsets genind objects by population</td>
</tr>
<tr class="even">
<td align="left"><code>shufflepop</code></td>
<td align="left">Shuffles genotypes at each locus using four different shuffling algorithms (details in table <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#tab:poppr3">2.3</a>)</td>
</tr>
<tr class="odd">
<td align="left"><code>splitcombine</code></td>
<td align="left">Manipulates population hierarchy</td>
</tr>
<tr class="even">
<td align="left"><strong>Analysis</strong></td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left"><code>bruvo.boot</code></td>
<td align="left">Produces dendrograms with bootstrap support based on Bruvo’s distance</td>
</tr>
<tr class="even">
<td align="left"><code>bruvo.dist</code></td>
<td align="left">Calculates Bruvo’s distance</td>
</tr>
<tr class="odd">
<td align="left"><code>diss.dist</code></td>
<td align="left">Calculates the percent allelic dissimilarity</td>
</tr>
<tr class="even">
<td align="left"><code>ia</code></td>
<td align="left">Calculates the index of association</td>
</tr>
<tr class="odd">
<td align="left"><code>mlg</code></td>
<td align="left">Calculates the number of multilocus genotypes</td>
</tr>
<tr class="even">
<td align="left"><code>mlg.crosspop</code></td>
<td align="left">Finds all multilocus genotypes that cross populations</td>
</tr>
<tr class="odd">
<td align="left"><code>mlg.table</code></td>
<td align="left">Returns a table of populations by multilocus genotypes</td>
</tr>
<tr class="even">
<td align="left"><code>mlg.vector</code></td>
<td align="left">Returns a vector of a numeric multilocus genotype assignment for each individual</td>
</tr>
<tr class="odd">
<td align="left"><code>poppr</code></td>
<td align="left">Returns a diversity table by population</td>
</tr>
<tr class="even">
<td align="left"><code>poppr.all</code></td>
<td align="left">Returns a diversity table by population for all compatible files specified</td>
</tr>
<tr class="odd">
<td align="left"><strong>Visualization</strong></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left"><code>greycurve</code></td>
<td align="left">Helper to determine the appropriate parameters for adjusting the grey level for msn functions</td>
</tr>
<tr class="odd">
<td align="left"><code>bruvo.msn</code></td>
<td align="left">Produces minimum spanning networks based off Bruvo’s distance colored by population</td>
</tr>
<tr class="even">
<td align="left"><code>poppr.msn</code></td>
<td align="left">Produces a minimum spanning network for any pairwise distance matrix related to the data</td>
</tr>
</tbody>
</table>
<p>Typical analyses in <em>poppr</em> start with summary statistics for diversity, rarefaction, evenness, MLG counts, and calculation of distance measures such as Bruvo’s distance, providing a suitable stepwise mutation model appropriate for microsatellite markers <span class="citation">(Bruvo et al., <a href="#ref-bruvo2004simple">2004</a>)</span>. <em>Poppr</em> will define MLGs in your data set, show where they cross populations, and can produce graphs and tables of MLGs by population that can be used for further analysis with the R package <em>vegan</em> <span class="citation">Oksanen et al. (<a href="#ref-oksanen2013vegan">2013</a>)</span>. Many of the diversity indices calculated by the <em>vegan</em> function <code>diversity()</code> are useful in analyzing the diversity of partially clonal populations. For this reason, <em>poppr</em> features a quick summary table (Table <a href="#tab:poppr2"><strong>??</strong></a>) that incorporates these indices along with the index of association, <span class="math inline">\(I_A\)</span> <span class="citation">(Brown et al., <a href="#ref-brown1980multilocus">1980</a>; Smith et al., <a href="#ref-smith1993how">1993</a>)</span>, and its standardized form, <span class="math inline">\(\bar{r}_d\)</span>, which accounts for the number of loci sampled <span class="citation">(Agapow &amp; Burt, <a href="#ref-Agapow_2001">2001</a>)</span>. Both measures of association can detect signatures of multilocus linkage and values significantly departing from the null model of no linkage among markers are detected via permutation analysis utilizing one of four algorithms described in table <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#tab:poppr3">2.3</a> <span class="citation">(Agapow &amp; Burt, <a href="#ref-Agapow_2001">2001</a>)</span>. The user can specify the number of samples taken from the observed data set to obtain the null distribution expected for a randomly mating population. Detailed examples of these analyses can be found in the <em>poppr</em> manual.</p>
<table>
<caption><span id="tab:poptab2">Table 2.2: </span>Summary table shown as it would appear in the R console produced by the <code>poppr()</code> function with 999 permutations to calculate <span class="math inline">\(I_A\)</span> and <span class="math inline">\(\bar{r}_d\)</span> p-values from the <code>Aeut</code> data set in  from <span class="citation">(Grünwald et al., <a href="#ref-grunwald2003analysis">2003</a>)</span>. <code>N</code>: census size, <code>MLG</code>: multilocus genotypes, <code>eMLG</code>: expected MLG based on rarefaction, <code>SE</code>: standard error from rarefaction, <code>H</code>: Shannon-Wiener Index, <code>G</code>: Stoddart and Taylor’s Index, <code>Hexp</code>: Nei 1978 Expected Heterozygosity, <code>E.5</code>: Evenness (<span class="math inline">\(E_5\)</span>), <code>Ia</code>: <span class="math inline">\(I_A\)</span>, <code>p.Ia</code>: p-value for <span class="math inline">\(I_A\)</span>, <code>rbarD</code>: <span class="math inline">\(\bar{r}_d\)</span>, <code>p.rD</code>: p-value for <span class="math inline">\(\bar{r}_d\)</span>. Table was obtained with the following code: <code>library(poppr); data(Aeut); poppr(Aeut, sample = 999)</code>.</caption>
<thead>
<tr class="header">
<th align="right">Pop</th>
<th align="right">N</th>
<th align="right">MLG</th>
<th align="right">eMLG</th>
<th align="right">SE</th>
<th align="right">H</th>
<th align="right">G</th>
<th align="right">Hexp</th>
<th align="right">E.5</th>
<th align="right">Ia</th>
<th align="right">p.Ia</th>
<th align="right">rbarD</th>
<th align="right">p.rD</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">Athena</td>
<td align="right">97</td>
<td align="right">70</td>
<td align="right">65.981</td>
<td align="right">1.246</td>
<td align="right">4.063</td>
<td align="right">42.193</td>
<td align="right">0.986</td>
<td align="right">0.721</td>
<td align="right">2.906</td>
<td align="right">0.001</td>
<td align="right">0.072</td>
<td align="right">0.001</td>
</tr>
<tr class="even">
<td align="right">Mt. Vernon</td>
<td align="right">90</td>
<td align="right">50</td>
<td align="right">50.000</td>
<td align="right">0.000</td>
<td align="right">3.668</td>
<td align="right">28.723</td>
<td align="right">0.976</td>
<td align="right">0.726</td>
<td align="right">13.302</td>
<td align="right">0.001</td>
<td align="right">0.282</td>
<td align="right">0.001</td>
</tr>
<tr class="odd">
<td align="right">Total</td>
<td align="right">187</td>
<td align="right">119</td>
<td align="right">68.453</td>
<td align="right">2.989</td>
<td align="right">4.558</td>
<td align="right">68.972</td>
<td align="right">0.991</td>
<td align="right">0.720</td>
<td align="right">14.371</td>
<td align="right">0.001</td>
<td align="right">0.271</td>
<td align="right">0.001</td>
</tr>
</tbody>
</table>
<table>
<caption><span id="tab:poppr3">Table 2.3: </span> Permutation algorithms in <em>poppr</em> implemented in the calculation of <span class="math inline">\(I_A\)</span> and <span class="math inline">\(\bar{r}_d\)</span> p-values, iterated over all loci independently.</caption>
<thead>
<tr class="header">
<th align="center"><strong>Method</strong></th>
<th align="left"><strong>Name</strong></th>
<th align="center"><strong>Units Sampled</strong></th>
<th align="center"><strong>With Replacement</strong></th>
<th align="center"><strong>Weight</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">1</td>
<td align="left"><code>permutation</code></td>
<td align="center">alleles</td>
<td align="center">No</td>
<td align="center">-</td>
</tr>
<tr class="even">
<td align="center">2</td>
<td align="left"><code>parametric bootstrap</code></td>
<td align="center">alleles</td>
<td align="center">Yes</td>
<td align="center">allele frequencies</td>
</tr>
<tr class="odd">
<td align="center">3</td>
<td align="left"><code>non-parametric bootstrap</code></td>
<td align="center">alleles</td>
<td align="center">Yes</td>
<td align="center">equal</td>
</tr>
<tr class="even">
<td align="center">4</td>
<td align="left"><code>multilocus</code></td>
<td align="center">genotypes</td>
<td align="center">No</td>
<td align="center">-</td>
</tr>
</tbody>
</table>
</div>
<div id="visualizations" class="section level3">
<h3><span class="header-section-number">2.3.3</span> Visualizations</h3>
<p><em>Poppr</em> generates bar charts of MLG counts found within each population of your data set (Fig. <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#fig:poppr1">2.1</a>). Histograms with rug plots for <span class="math inline">\(I_A\)</span> and <span class="math inline">\(\bar{r}_d\)</span> allow visual assessment of the quality of the distribution derived from resampling to see if a higher number of replications are necessary (Fig. <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#fig:poppr2">2.2</a>). <em>Poppr</em> automatically produces custom minimum spanning networks for Bruvo’s or other distances using Prim’s algorithm, as implemented in the package <em>igraph</em> <span class="citation">Csardi &amp; Nepusz (<a href="#ref-csardi2006igraph">2006</a>)</span>, with the functions <code>bruvo.msn()</code> for Bruvo’s distance (Fig. <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#fig:poppr3">2.3</a>) and <code>poppr.msn()</code> for any distance matrix. The combination of data structures from <em>adegenet</em> and <em>igraph</em> allow graphing that is color coded by population with vertices grouped by MLG <span class="citation">(Csardi &amp; Nepusz, <a href="#ref-csardi2006igraph">2006</a>; Jombart, <a href="#ref-Jombart_2008">2008</a>; Jombart &amp; Ahmed, <a href="#ref-jombart2011adegenet">2011</a>)</span>. <em>Poppr</em> also includes visualization of dendrograms using UPGMA <span class="citation">Schliep (<a href="#ref-phangorn">2011</a>)</span> and Neighbor-Joining <span class="citation">Paradis et al. (<a href="#ref-paradis2004ape">2004</a>)</span> algorithms with bootstrap support for Bruvo’s distance using the function <code>bruvo.boot()</code> (Fig. <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#fig:poppr4">2.4</a>). Neither graphing of minimum spanning networks or dendrograms with bootstrap support are currently possible for populations in any other R packages.</p>
<div class="figure" style="text-align: center"><span id="fig:poppr1"></span>
<img src="figure/poppr/Finland.png" alt="Distribution of 12 multilocus genotypes from the `Finland` population of
the `H3N2` SNP data set [@Jombart_2008]" width="80%" />
<p class="caption">
Figure 2.1: Distribution of 12 multilocus genotypes from the <code>Finland</code> population of the <code>H3N2</code> SNP data set <span class="citation">(Jombart, <a href="#ref-Jombart_2008">2008</a>)</span>
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:poppr2"></span>
<img src="figure/poppr/nancy_5.png" alt="Visualizations of tests for linkage disequilibrium, where observed
values (blue dashed lines) of $I_A$ and $\bar{r}_d$ are compared to
histograms showing results of 999 permutations using method 1 in table
\@ref(tab:poppr1). Results are shown for the sexual population `5` of the
`nancycats` data set [@Jombart_2008] (a) and for the clonal `Athena`
population of the `Aeut` data set [@grunwald2003analysis] (b)." width="70%" />
<p class="caption">
Figure 2.2: Visualizations of tests for linkage disequilibrium, where observed values (blue dashed lines) of <span class="math inline">\(I_A\)</span> and <span class="math inline">\(\bar{r}_d\)</span> are compared to histograms showing results of 999 permutations using method 1 in table <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#tab:poppr1">2.1</a>. Results are shown for the sexual population <code>5</code> of the <code>nancycats</code> data set <span class="citation">(Jombart, <a href="#ref-Jombart_2008">2008</a>)</span> (a) and for the clonal <code>Athena</code> population of the <code>Aeut</code> data set <span class="citation">(Grünwald et al., <a href="#ref-grunwald2003analysis">2003</a>)</span> (b).
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:poppr3"></span>
<img src="figure/poppr/bruvo_color.png" alt="Example minimum spanning network using Bruvo's distance on a simulated
partially clonal data set with 50 individuals genotyped over 10
microsatellite loci produced with the software SimuPOP v.1.0.8
[@peng2008forward]. Each node represents a unique multilocus genotype.
Node shading (colors) represent population membership, while edge widths
and shading represent relatedness. Edge length is arbitrary." width="80%" />
<p class="caption">
Figure 2.3: Example minimum spanning network using Bruvo’s distance on a simulated partially clonal data set with 50 individuals genotyped over 10 microsatellite loci produced with the software SimuPOP v.1.0.8 <span class="citation">(Peng &amp; Amos, <a href="#ref-peng2008forward">2008</a>)</span>. Each node represents a unique multilocus genotype. Node shading (colors) represent population membership, while edge widths and shading represent relatedness. Edge length is arbitrary.
</p>
</div>
<div class="figure" style="text-align: center"><span id="fig:poppr4"></span>
<img src="figure/poppr/nancy9-boot-redo.png" alt="UPGMA tree produced from Bruvo's distance with 1000 bootstrap replicates
(node values greater than 50% are shown). Data from population `9` of
the `nancycats` data set [@Jombart_2008]" width="80%" />
<p class="caption">
Figure 2.4: UPGMA tree produced from Bruvo’s distance with 1000 bootstrap replicates (node values greater than 50% are shown). Data from population <code>9</code> of the <code>nancycats</code> data set <span class="citation">(Jombart, <a href="#ref-Jombart_2008">2008</a>)</span>
</p>
</div>
</div>
<div id="performance" class="section level3">
<h3><span class="header-section-number">2.3.4</span> Performance</h3>
<p>Most of the functions in <em>Poppr</em> were written and optimized for performance in R and are available for inspection and/or download at <a href="https://github.com/grunwaldlab/poppr" class="uri">https://github.com/grunwaldlab/poppr</a>. Algorithms of <span class="math inline">\(\geq O(n^2)\)</span> complexity were written in the byte-compiled C language to optimize runtime performance.</p>
<p>For comparisons of <span class="math inline">\(I_A\)</span> and Bruvo’s distance, we utilized the data set ‘nancycats’ (237 diploid individuals genotyped at nine microsatellite loci) from the <em>adegenet</em> package. Calculations were run independently 10 times and then averaged. Bruvo’s distance was calculated on a machine with OSX 10.8.4 and a 2.9 GHz intel processor. The <span class="math inline">\(I_A\)</span> and <span class="math inline">\(\bar{r}_d\)</span> calculations were performed on a machine with OSX 10.5.8 and a 2.4 GHz intel processor due to the inability of the software <span style="font-variant: small-caps;">multilocus</span> to work on any later version of OSX.</p>
</div>
</div>
<div id="citation-of-methods-implemented-in-poppr" class="section level2">
<h2><span class="header-section-number">2.4</span> Citation of methods implemented in <em>poppr</em></h2>
<p>Several of the methods implemented in <em>poppr</em> are described elsewhere. Users should refer to the original publications for interpretations and citation. See table <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#tab:poppr4">2.4</a> for a full list of citations. As with any R package, users should always cite the R Core Team <span class="citation">(R Core Team, <a href="#ref-R2013">2013</a>)</span>.</p>
<table>
<caption><span id="tab:poppr4">Table 2.4: </span> Citation of methods and indices implemented in <em>poppr</em></caption>
<thead>
<tr class="header">
<th align="left">Method/Index</th>
<th align="center">Citation</th>
<th align="center">Function(s) in <em>poppr</em></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Expected MLG (rarefaction)</td>
<td align="center"><span class="citation">(Hurlbert, <a href="#ref-hurlbert1971nonconcept">1971</a>)</span>, <span class="citation">(Heck et al., <a href="#ref-heck1975explicit">1975</a>)</span> (for std. err.)</td>
<td align="center"><code>poppr()</code></td>
</tr>
<tr class="even">
<td align="left"><span class="math inline">\(H\)</span></td>
<td align="center"><span class="citation">(Shannon, <a href="#ref-shannon2001mathematical">1948</a>)</span></td>
<td align="center"><code>poppr()</code></td>
</tr>
<tr class="odd">
<td align="left"><span class="math inline">\(G\)</span></td>
<td align="center"><span class="citation">(Stoddart &amp; Taylor, <a href="#ref-stoddart1988genotypic">1988</a>)</span></td>
<td align="center"><code>poppr()</code></td>
</tr>
<tr class="even">
<td align="left"><span class="math inline">\(H_{exp}\)</span></td>
<td align="center"><span class="citation">(Nei, <a href="#ref-Nei:1978">1978</a>)</span></td>
<td align="center"><code>poppr()</code></td>
</tr>
<tr class="odd">
<td align="left"><span class="math inline">\(E_{5}\)</span></td>
<td align="center"><span class="citation">(Grünwald &amp; Hoheisel, <a href="#ref-grunwald2006hierarchical">2006</a>; Ludwig &amp; Reynolds, <a href="#ref-ludwig1988statistical">1988</a>; Pielou, <a href="#ref-pielou1975ecological">1975</a>)</span></td>
<td align="center"><code>poppr()</code></td>
</tr>
<tr class="even">
<td align="left"><span class="math inline">\(I_A\)</span>/<span class="math inline">\(\bar{r}_d\)</span></td>
<td align="center"><span class="citation">(Brown et al., <a href="#ref-brown1980multilocus">1980</a>; Smith et al., <a href="#ref-smith1993how">1993</a>)</span> (<span class="math inline">\(I_A\)</span>), <span class="citation">(Agapow &amp; Burt, <a href="#ref-Agapow_2001">2001</a>)</span> (<span class="math inline">\(\bar{r}_d\)</span>)</td>
<td align="center"><code>ia()</code></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="center"></td>
<td align="center"><code>poppr()</code></td>
</tr>
<tr class="even">
<td align="left">Clone correction</td>
<td align="center"><span class="citation">(Grünwald &amp; Hoheisel, <a href="#ref-grunwald2006hierarchical">2006</a>; Grünwald et al., <a href="#ref-grunwald2003analysis">2003</a>; Milgroom, <a href="#ref-milgroom1996recombination">1996</a>)</span></td>
<td align="center"><code>clonecorrect()</code></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="center"></td>
<td align="center"><code>poppr()</code></td>
</tr>
<tr class="even">
<td align="left">Minimum Spanning Networks</td>
<td align="center"><span class="citation">(Csardi &amp; Nepusz, <a href="#ref-csardi2006igraph">2006</a>)</span></td>
<td align="center"><code>poppr.msn()</code></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="center"></td>
<td align="center"><code>bruvo.msn()</code></td>
</tr>
<tr class="even">
<td align="left">Bruvo’s Distance</td>
<td align="center"><span class="citation">(Bruvo et al., <a href="#ref-bruvo2004simple">2004</a>)</span></td>
<td align="center"><code>bruvo.dist()</code></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="center"></td>
<td align="center"><code>bruvo.msn()</code></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="center"></td>
<td align="center"><code>bruvo.boot()</code></td>
</tr>
<tr class="odd">
<td align="left">Bootstrapping</td>
<td align="center"><span class="citation">(Paradis et al., <a href="#ref-paradis2004ape">2004</a>)</span></td>
<td align="center"><code>bruvo.boot()</code></td>
</tr>
<tr class="even">
<td align="left">Neighbor Joining</td>
<td align="center"><span class="citation">(Paradis et al., <a href="#ref-paradis2004ape">2004</a>)</span></td>
<td align="center"><code>bruvo.boot()</code></td>
</tr>
<tr class="odd">
<td align="left">UPGMA</td>
<td align="center"><span class="citation">(Schliep, <a href="#ref-phangorn">2011</a>)</span></td>
<td align="center"><code>bruvo.boot()</code></td>
</tr>
</tbody>
</table>
</div>
<div id="results-and-discussion" class="section level2">
<h2><span class="header-section-number">2.5</span> Results and Discussion</h2>
<p><em>Poppr</em> provides significant, convenient tools for analysis of clonal and partially clonal populations available in one environment on all major operating systems. The ability to analyze data for multiple populations across a user-defined hierarchy and clone-censoring provide novel functionality in R. Combined with R’s graphing abilities, publication-ready figures are thus obtained conveniently.</p>
<div id="new-functionalities" class="section level3">
<h3><span class="header-section-number">2.5.1</span> New functionalities</h3>
<table>
<caption><span id="tab:poppr5">Table 2.5: </span> Comparison of programs that calculate <span class="math inline">\(I_A\)</span></caption>
<thead>
<tr class="header">
<th></th>
<th align="center"><strong>Haploids</strong></th>
<th align="center"><strong>Diploids</strong></th>
<th align="center"><span class="math inline">\(\bar{r}_d\)</span></th>
<th align="center"><strong>All Platforms</strong></th>
<th align="center"><strong>Batch Analysis</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><em>poppr</em></td>
<td align="center">Yes</td>
<td align="center">Yes</td>
<td align="center">Yes</td>
<td align="center">Yes</td>
<td align="center">Yes</td>
</tr>
<tr class="even">
<td><em>LIAN</em></td>
<td align="center">Yes</td>
<td align="center">No</td>
<td align="center">No</td>
<td align="center">Yes</td>
<td align="center">Yes</td>
</tr>
<tr class="odd">
<td><em>multilocus</em></td>
<td align="center">Yes</td>
<td align="center">Yes</td>
<td align="center">Yes</td>
<td align="center">No</td>
<td align="center">No</td>
</tr>
</tbody>
</table>
<p><em>Poppr</em> implements several new functionalities. As of this writing, aside from <em>poppr</em>, there exist two programs that calculate <span class="math inline">\(I_A\)</span>: <span style="font-variant: small-caps;">LIAN</span> <span class="citation">(Haubold &amp; Hudson, <a href="#ref-Haubold:2000">2000</a>)</span> and <span style="font-variant: small-caps;">multilocus</span> <span class="citation">(Agapow &amp; Burt, <a href="#ref-Agapow_2001">2001</a>)</span>. <span style="font-variant: small-caps;">LIAN</span> can calculate <span class="math inline">\(I_A\)</span> for haploid data and is only available online or for *nix systems with a C compiler such as OSX and Linux <span class="citation">(Haubold &amp; Hudson, <a href="#ref-Haubold:2000">2000</a>)</span>. <span style="font-variant: small-caps;">Multilocus</span> implemented <span class="math inline">\(\bar{r}_d\)</span>, a novel correction for <span class="math inline">\(I_A\)</span>, but is no longer supported <span class="citation">(Agapow &amp; Burt, <a href="#ref-Agapow_2001">2001</a>)</span>. <span style="font-variant: small-caps;">Multilocus</span> will only calculate index values for one data set at a time and <span style="font-variant: small-caps;">LIAN</span> requires the user to structure the data set with populations in contiguous blocks to analyze multiple populations within a single file. Thus <em>poppr</em> provides significant improvements for calculation of linkage disequilibrium, and handles both haploid and diploid data, works on all major operating systems, and is capable of batch analysis of multiple files and multiple populations defined within a file including the possibility of clone correction and sub-setting. A comparison of the capabilities of these programs are summarized in Table <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#tab:poppr5">2.5</a>.</p>
<p>To test significance for <span class="math inline">\(I_A\)</span> and <span class="math inline">\(\bar{r}_d\)</span>, <em>poppr</em> offers four permutation algorithms. Each one will randomly shuffle data at each locus, effectively unlinking the loci. The algorithm previously utilized by <span style="font-variant: small-caps;">mutlilocus</span> is included. The <span style="font-variant: small-caps;">multilocus</span>-style algorithm shuffles genotypes, maintaining the associations between alleles at each locus <span class="citation">(Agapow &amp; Burt, <a href="#ref-Agapow_2001">2001</a>)</span>. More appropriately, alleles are expected to assort independently in panmictic populations. <em>Poppr</em> thus provides three new algorithms for permutation that allow for independent allele assortment at each locus. The default algorithm permutes the alleles at each locus and the remaining two will randomly sample alleles from a multinomial distribution parametrically and non-parametrically <span class="citation">(Weir &amp; Cockerham, <a href="#ref-weir1996genetic">1996</a>)</span>. Details of these algorithms are presented in table <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#tab:poppr3">2.3</a>. Because the index of association is calculated using a binary measure of dissimilarity, we have also made this available as a distance measure called <code>diss.dist()</code>. This pairwise distance is based on the percent allelic differences.</p>
<p><em>Poppr</em> also newly implements Bruvo’s genetic distance that utilizes a stepwise mutation model appropriate for microsatellite data <span class="citation">(Bruvo et al., <a href="#ref-bruvo2004simple">2004</a>)</span>. While this distance is implemented in the program <span style="font-variant: small-caps;"> GenoDive</span> <span class="citation">Meirmans &amp; Van Tienderen (<a href="#ref-meirmans2004genotype">2004</a>)</span> and the R package <em>polysat</em> <span class="citation">Clark &amp; Jasieniuk (<a href="#ref-polysat">2011</a>)</span>, there are a few caveats with these two implementations. <span style="font-variant: small-caps;">GenoDive</span> is closed-source, and only implemented in OSX. Both <em>poppr</em> and <em>polysat</em> are open-source and available on all platforms, but <em>polysat</em>, being optimized for polyploid individuals with ambiguous allelic dosage, is inappropriate for analyzing diploids. <em>Polysat</em> will collapse homozygous individuals into a single allele and attempt to infer the second allelic state in comparison with heterozygous individuals. Since haploid and diploid individuals show clear allelic dosage, this procedure creates a bias misrepresenting the true distance. Not only is <em>poppr</em> not subject to this bias, but it also newly introduces bootstrap support for this distance as shown in Figure <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#fig:poppr4">2.4</a>.</p>
</div>
<div id="performance-1" class="section level3">
<h3><span class="header-section-number">2.5.2</span> Performance</h3>
<table>
<caption><span id="tab:poppr6">Table 2.6: </span> Comparison of performance on one data set of 237 individuals over nine loci. Each time point represents an average of 10 independent runs. Calculations of <span class="math inline">\(I_A\)</span> are based on 100 permutations.</caption>
<thead>
<tr class="header">
<th></th>
<th align="center"><strong><span class="math inline">\(I_A\)</span> (seconds)</strong></th>
<th align="center"><strong>Bruvo’s distance (seconds)</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><em>poppr</em></td>
<td align="center">13.4</td>
<td align="center">0.3</td>
</tr>
<tr class="even">
<td><em>polysat</em></td>
<td align="center">-</td>
<td align="center">58.3</td>
</tr>
<tr class="odd">
<td><em>multilocus</em></td>
<td align="center">547.2</td>
<td align="center">-</td>
</tr>
</tbody>
</table>
<p><em>Poppr</em> reduces the amount of intermediate files and repetitive tasks needed for basic population genetic analyses and implements computationally intensive functions, such as Bruvo’s distance and the index of association in C to improve performance. The <em>polysat</em> package calculation of Bruvo’s distance took 58.3 seconds on average whereas <em>poppr</em>’s calculation was over 190 times faster, averaging 0.3 seconds (Table <a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#tab:poppr6">2.6</a>). For calculation of <span class="math inline">\(I_A\)</span> and <span class="math inline">\(\bar{r}_d\)</span> with 100 permutations and Nei’s genotypic diversity <span class="citation">(Nei, <a href="#ref-Nei:1978">1978</a>)</span>, <span style="font-variant: small-caps;">multilocus</span> required around 9.12 minutes on average, as compared to 13.4 seconds with <em>poppr</em>.</p>
</div>
</div>
<div id="conclusions" class="section level2">
<h2><span class="header-section-number">2.6</span> Conclusions</h2>
<p>The R package <em>poppr</em> provides new functions and tools specifically tailored for analysis of data from clonal or partially clonal populations. No software currently available provides this set of tools. Novel capabilities include analysis across multiple populations at multiple levels of hierarchies, clone-censoring, and subsetting. These in combination with R’s command line interface and scripting capabilities makes analyses of these populations more streamlined and tractable. By implementing computationally expensive algorithms such as Bruvo’s distance and <span class="math inline">\(I_A\)</span> in C, analyses of multiple populations that would normally take hours to complete can now be finished in a matter of minutes. This allowed us to expand the utility of these measures to convenient new graphing abilities such as automatically creating dendrograms with bootstrap support for Bruvo’s distance and minimum spanning networks. While major releases of <em>poppr</em> are available on CRAN, we are continuing to develop this package to be able to efficiently handle genome-sized SNP data. Development versions are available on github at <a href="https://github.com/grunwaldlab/poppr" class="uri">https://github.com/grunwaldlab/poppr</a>.</p>
</div>
<div id="acknowledgements" class="section level2">
<h2><span class="header-section-number">2.7</span> Acknowledgements</h2>
<p>The authors would like to thank Sydney Everhart and Corine Schoebel for invaluable alpha testing and Paul-Michael Agapow for providing the <em>multilocus</em> C++ source code for reference. We thank Sydney Everhart and Brian Knaus for comments that significantly improved this manuscript. Mention of trade names or commercial products in this manuscript are solely for the purpose of providing specific information and do not imply recommendation or endorsement.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-hartl1997principles">
<p>Hartl, D. L., &amp; Clark, A. G. (2007). <em>Principles of population genetics</em>. Sunderland, MA, USA: Sinauer Associates.</p>
</div>
<div id="ref-nielsen2013introduction">
<p>Nielsen, R., &amp; Slatkin, M. (2013). <em>An introduction to population genetics: Theory and applications</em>. Sinauer Associates, Incorporated. Retrieved from <a href="http://books.google.com/books?id=Iy08kgEACAAJ" class="uri">http://books.google.com/books?id=Iy08kgEACAAJ</a></p>
</div>
<div id="ref-weir1996genetic">
<p>Weir, B. S., &amp; Cockerham, C. (1996). Genetic data analysis II: Methods for discrete population genetic data.</p>
</div>
<div id="ref-anderson1995clonality">
<p>Anderson, J. B., &amp; Kohn, L. M. (1995). Clonality in soilborne, plant-pathogenic fungi. <em>Annual Review of Phytopathology</em>, <em>33</em>(1), 369–391. <a href="https://doi.org/10.1146/annurev.py.33.090195.002101" class="uri">https://doi.org/10.1146/annurev.py.33.090195.002101</a></p>
</div>
<div id="ref-milgroom1996recombination">
<p>Milgroom, M. G. (1996). Recombination and the multilocus structure of fungal populations. <em>Annual Review of Phytopathology</em>, <em>34</em>(1), 457–477.</p>
</div>
<div id="ref-grunwald2011evolution">
<p>Grünwald, N. J., &amp; Goss, E. M. (2011). Evolution and population genetics of exotic and re-emerging pathogens: Novel tools and approaches. <em>Annual Review of Phytopathology</em>, <em>49</em>(1), 249–267. <a href="https://doi.org/10.1146/annurev-phyto-072910-095246" class="uri">https://doi.org/10.1146/annurev-phyto-072910-095246</a></p>
</div>
<div id="ref-pritchard2000inference">
<p>Pritchard, J. K., Stephens, M., &amp; Donnelly, P. (2000). Inference of population structure using multilocus genotype data. <em>Genetics</em>, <em>155</em>(2), 945–959.</p>
</div>
<div id="ref-cooke2012genome">
<p>Cooke, D. E. L., Cano, L. M., Raffaele, S., Bain, R. A., Cooke, L. R., Etherington, G. J., Deahl, K. L., Farrer, R. A., Gilroy, E. M., Goss, E. M., Grünwald, N. J., Hein, I., MacLean, D., McNicol, J. W., Randall, E., Oliva, R. F., Pel, M. A., Shaw, D. S., Squires, J. N., Taylor, M. C., Vleeshouwers, V. G. A. A., Birch, P. R. J., Lees, A. K., &amp; Kamoun, S. (2012). Genome analyses of an aggressive and invasive lineage of the irish potato famine pathogen. <em>PLoS Pathogens</em>, <em>8</em>(10), e1002940. <a href="https://doi.org/10.1371/journal.ppat.1002940" class="uri">https://doi.org/10.1371/journal.ppat.1002940</a></p>
</div>
<div id="ref-goss2009population">
<p>Goss, E. M., Larsen, M., Chastagner, G. A., Givens, D. R., &amp; Grünwald, N. J. (2009). Population genetic analysis infers migration pathways of <em>Phytophthora ramorum</em> in US nurseries. <em>PLoS Pathogens</em>, <em>5</em>(9), e1000583. <a href="https://doi.org/10.1371/journal.ppat.1000583" class="uri">https://doi.org/10.1371/journal.ppat.1000583</a></p>
</div>
<div id="ref-mascheretti2008reconstruction">
<p>Mascheretti, S., Croucher, P. J. P., Vettraino, A., Prospero, S., &amp; Garbelotto, M. (2008).  Reconstruction of the Sudden Oak Death epidemic in California through microsatellite analysis of the pathogen <em>Phytophthora ramorum</em>. <em>Molecular Ecology</em>, <em>17</em>(11), 2755–2768. <a href="https://doi.org/10.1111/j.1365-294x.2008.03773.x" class="uri">https://doi.org/10.1111/j.1365-294x.2008.03773.x</a></p>
</div>
<div id="ref-grunwald2006hierarchical">
<p>Grünwald, N. J., &amp; Hoheisel, G.-A. (2006). Hierarchical analysis of diversity, selfing, and genetic differentiation in populations of the oomycete <em>Aphanomyces euteiches</em>. <em>Phytopathology</em>, <em>96</em>(10), 1134–1141.</p>
</div>
<div id="ref-mcdonald1997population">
<p>McDonald, B. A. (1997). The population genetics of fungi: tools and techniques. <em>Phytopathology</em>, <em>87</em>(4), 448–453.</p>
</div>
<div id="ref-grunwald2003analysis">
<p>Grünwald, N. J., Goodwin, S. B., Milgroom, M. G., &amp; Fry, W. E. (2003). Analysis of genotypic diversity data for populations of microorganisms. <em>Phytopathology</em>, <em>93</em>(6), 738–746. <a href="https://doi.org/10.1094/phyto.2003.93.6.738" class="uri">https://doi.org/10.1094/phyto.2003.93.6.738</a></p>
</div>
<div id="ref-shannon2001mathematical">
<p>Shannon, C. E. (1948). A mathematical theory of communication. <em>ACM SIGMOBILE Mobile Computing and Communications Review</em>, <em>5</em>(1), 3–55.</p>
</div>
<div id="ref-stoddart1988genotypic">
<p>Stoddart, J. A., &amp; Taylor, J. F. (1988). Genotypic diversity: Estimation and prediction in samples. <em>Genetics</em>, <em>118</em>(4), 705–11.</p>
</div>
<div id="ref-brown1980multilocus">
<p>Brown, A., Feldman, M., &amp; Nevo, E. (1980). Multilocus structure of natural populations of <em>Hordeum spontaneum</em>. <em>Genetics</em>, <em>96</em>(2), 523–536. Retrieved from <a href="http://www.genetics.org/content/96/2/523.abstract" class="uri">http://www.genetics.org/content/96/2/523.abstract</a></p>
</div>
<div id="ref-burt1996molecular">
<p>Burt, A., Carter, D. A., Koenig, G. L., White, T. J., &amp; Taylor, J. W. (1996). Molecular markers reveal cryptic sex in the human pathogen <em>Coccidioides immitis</em>. <em>Proceedings of the National Academy of Sciences</em>, <em>93</em>(2), 770–773.</p>
</div>
<div id="ref-smith1993how">
<p>Smith, J. M., Smith, N. H., O’Rourke, M., &amp; Spratt, B. G. (1993). How clonal are bacteria? <em>Proceedings of the National Academy of Sciences</em>, <em>90</em>(10), 4384–4388. <a href="https://doi.org/10.1073/pnas.90.10.4384" class="uri">https://doi.org/10.1073/pnas.90.10.4384</a></p>
</div>
<div id="ref-peever1994genetic">
<p>Peever, T. L., &amp; Milgroom, M. G. (1994). Genetic structure of <em>Pyrenophora teres</em> populations determined with random amplified polymorphic DNA markers. <em>Canadian Journal of Botany</em>, <em>72</em>(7), 915–923.</p>
</div>
<div id="ref-Agapow_2001">
<p>Agapow, P.-M., &amp; Burt, A. (2001). Indices of multilocus linkage disequilibrium. <em>Molecular Ecology Notes</em>, <em>1</em>(1-2), 101–102. <a href="https://doi.org/10.1046/j.1471-8278.2000.00014.x" class="uri">https://doi.org/10.1046/j.1471-8278.2000.00014.x</a></p>
</div>
<div id="ref-Haubold:2000">
<p>Haubold, B., &amp; Hudson, R. R. (2000). LIAN 3.0: Detecting linkage disequilibrium in multilocus data. <em>Bioinformatics</em>, <em>16</em>(9), 847–849. <a href="https://doi.org/10.1093/bioinformatics/16.9.847" class="uri">https://doi.org/10.1093/bioinformatics/16.9.847</a></p>
</div>
<div id="ref-Jombart_2008">
<p>Jombart, T. (2008). Adegenet: a R package for the multivariate analysis of genetic markers. <em>Bioinformatics</em>, <em>24</em>(11), 1403–1405. <a href="https://doi.org/10.1093/bioinformatics/btn129" class="uri">https://doi.org/10.1093/bioinformatics/btn129</a></p>
</div>
<div id="ref-jombart2011adegenet">
<p>Jombart, T., &amp; Ahmed, I. (2011). Adegenet 1.3-1: New tools for the analysis of genome-wide SNP data. <em>Bioinformatics</em>, <em>27</em>(21), 3070–3071.</p>
</div>
<div id="ref-oksanen2013vegan">
<p>Oksanen, J., Blanchet, F. G., Kindt, R., Legendre, P., Minchin, P. R., O’Hara, R. B., Simpson, G. L., Solymos, P., Stevens, M. H. H., &amp; Wagner, H. (2013). <em>Vegan: Community ecology package</em>. Retrieved from <a href="https://CRAN.R-project.org/package=vegan" class="uri">https://CRAN.R-project.org/package=vegan</a></p>
</div>
<div id="ref-bruvo2004simple">
<p>Bruvo, R., Michiels, N. K., D’Souza, T. G., &amp; Schulenburg, H. (2004). A simple method for the calculation of microsatellite genotype distances irrespective of ploidy level. <em>Molecular Ecology</em>, <em>13</em>(7), 2101–2106.</p>
</div>
<div id="ref-Peakall:2006">
<p>Peakall, R., &amp; Smouse, P. E. (2006). GenAlEx 6: Genetic analysis in excel. population genetic software for teaching and research. <em>Molecular Ecology Notes</em>, <em>6</em>(1), 288–295.</p>
</div>
<div id="ref-csardi2006igraph">
<p>Csardi, G., &amp; Nepusz, T. (2006). The igraph software package for complex network research. <em>InterJournal</em>, <em>Complex Systems</em>, 1695. Retrieved from <a href="http://igraph.org" class="uri">http://igraph.org</a></p>
</div>
<div id="ref-phangorn">
<p>Schliep, K. (2011). Phangorn: phylogenetic analysis in R. <em>Bioinformatics</em>, <em>27</em>(4), 592–593.</p>
</div>
<div id="ref-paradis2004ape">
<p>Paradis, E., Claude, J., &amp; Strimmer, K. (2004). APE: Analyses of phylogenetics and evolution in R language. <em>Bioinformatics</em>, <em>20</em>(2), 289–290. <a href="https://doi.org/10.1093/bioinformatics/btg412" class="uri">https://doi.org/10.1093/bioinformatics/btg412</a></p>
</div>
<div id="ref-peng2008forward">
<p>Peng, B., &amp; Amos, C. I. (2008). Forward-time simulations of non-random mating populations using simuPOP. <em>Bioinformatics</em>, <em>24</em>(11), 1408–1409. <a href="https://doi.org/10.1093/bioinformatics/btn179" class="uri">https://doi.org/10.1093/bioinformatics/btn179</a></p>
</div>
<div id="ref-R2013">
<p>R Core Team. (2013). <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="http://www.R-project.org/" class="uri">http://www.R-project.org/</a></p>
</div>
<div id="ref-hurlbert1971nonconcept">
<p>Hurlbert, S. H. (1971). The nonconcept of species diversity: A critique and alternative parameters. <em>Ecology</em>, <em>52</em>(4), 577–586. <a href="https://doi.org/10.2307/1934145" class="uri">https://doi.org/10.2307/1934145</a></p>
</div>
<div id="ref-heck1975explicit">
<p>Heck, K. L., van Belle, G., &amp; Simberloff, D. (1975). Explicit calculation of the rarefaction diversity measurement and the determination of sufficient sample size. <em>Ecology</em>, <em>56</em>(6), 1459–1461. <a href="https://doi.org/10.2307/1934716" class="uri">https://doi.org/10.2307/1934716</a></p>
</div>
<div id="ref-Nei:1978">
<p>Nei, M. (1978). Estimation of average heterozygosity and genetic distance from a small number of individuals. <em>Genetics</em>, <em>89</em>(3), 583–590. Retrieved from <a href="http://www.genetics.org/content/89/3/583.abstract" class="uri">http://www.genetics.org/content/89/3/583.abstract</a></p>
</div>
<div id="ref-ludwig1988statistical">
<p>Ludwig, J. A., &amp; Reynolds, J. F. (1988). <em>Statistical ecology: A primer in methods and computing</em> (Vol. 1). John Wiley &amp; Sons.</p>
</div>
<div id="ref-pielou1975ecological">
<p>Pielou, E. (1975). <em>Ecological diversity</em>. New York: Wiley &amp; Sons.</p>
</div>
<div id="ref-meirmans2004genotype">
<p>Meirmans, P. G., &amp; Van Tienderen, P. H. (2004). GENOTYPE and GENODIVE: Two programs for the analysis of genetic diversity of asexual organisms. <em>Molecular Ecology Notes</em>, <em>4</em>(4), 792–794. <a href="https://doi.org/10.1111/j.1471-8286.2004.00770.x" class="uri">https://doi.org/10.1111/j.1471-8286.2004.00770.x</a></p>
</div>
<div id="ref-polysat">
<p>Clark, L., &amp; Jasieniuk, M. (2011). Polysat: an R package for polyploid microsatellite analysis. <em>Molecular Ecology Resources</em>, <em>11</em>(3), 562–566. <a href="https://doi.org/10.1111/j.1755-0998.2011.02985.x" class="uri">https://doi.org/10.1111/j.1755-0998.2011.02985.x</a></p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html" class="navigation navigation-next " aria-label="Next page""><i class="fa fa-angle-right"></i></a>

<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["http://hdl.handle.net/1957/60042", "PDF"], ["dissertation.tex", "TEX"], ["https://github.com/zkamvar/dissertation", "SOURCE CODE"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
