<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Development and Application of Tools for Genetic Analysis of Clonal Populations</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Development and Application of Tools for Genetic Analysis of Clonal Populations">
  <meta name="generator" content="bookdown 0.1.18 and GitBook 2.6.7">

  <meta property="og:title" content="Development and Application of Tools for Genetic Analysis of Clonal Populations" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Development and Application of Tools for Genetic Analysis of Clonal Populations" />
  
  
  

<meta name="author" content="Zhian N. Kamvar">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="6-conclusions-2.html">


<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>


  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#population-genetics-and-clonality"><i class="fa fa-check"></i><b>1.1</b> Population Genetics and Clonality</a></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#population-genetics-under-the-lens-of-plant-pathology"><i class="fa fa-check"></i><b>1.2</b> Population Genetics Under the Lens of Plant Pathology</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-need-for-reproducible-research"><i class="fa fa-check"></i><b>1.3</b> The Need For Reproducible Research</a></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#overview"><i class="fa fa-check"></i><b>1.4</b> Overview</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-2"><i class="fa fa-check"></i><b>1.4.1</b> Summary of Chapter 2</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-3"><i class="fa fa-check"></i><b>1.4.2</b> Summary of Chapter 3</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-4"><i class="fa fa-check"></i><b>1.4.3</b> Summary of Chapter 4</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-5"><i class="fa fa-check"></i><b>1.4.4</b> Summary of Chapter 5</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><i class="fa fa-check"></i><b>2</b> <em>Poppr</em>: an R Package For Genetic Analysis of Populations With Clonal, Partially Clonal, and/or Sexual Reproduction</a><ul>
<li class="chapter" data-level="2.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#abstract"><i class="fa fa-check"></i><b>2.1</b> Abstract</a></li>
<li class="chapter" data-level="2.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#introduction-1"><i class="fa fa-check"></i><b>2.2</b> Introduction</a></li>
<li class="chapter" data-level="2.3" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#materials-and-methods"><i class="fa fa-check"></i><b>2.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#data-import"><i class="fa fa-check"></i><b>2.3.1</b> Data import</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#data-analysis"><i class="fa fa-check"></i><b>2.3.2</b> Data analysis</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#visualizations"><i class="fa fa-check"></i><b>2.3.3</b> Visualizations</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#performance"><i class="fa fa-check"></i><b>2.3.4</b> Performance</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#citation-of-methods-implemented-in-poppr"><i class="fa fa-check"></i><b>2.4</b> Citation of methods implemented in <em>poppr</em></a></li>
<li class="chapter" data-level="2.5" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#results-and-discussion"><i class="fa fa-check"></i><b>2.5</b> Results and Discussion</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#new-functionalities"><i class="fa fa-check"></i><b>2.5.1</b> New functionalities</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#performance-1"><i class="fa fa-check"></i><b>2.5.2</b> Performance</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
<li class="chapter" data-level="2.7" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#acknowledgements"><i class="fa fa-check"></i><b>2.7</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><i class="fa fa-check"></i><b>3</b> Novel R Tools For Analysis of Genome-Wide Population Genetic Data With Emphasis on Clonality</a><ul>
<li class="chapter" data-level="3.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#implementations-and-examples"><i class="fa fa-check"></i><b>3.3</b> Implementations and Examples</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#clonal-identification"><i class="fa fa-check"></i><b>3.3.1</b> Clonal identification</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#minimum-spanning-networks-with-reticulation"><i class="fa fa-check"></i><b>3.3.2</b> Minimum Spanning Networks with Reticulation</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#bootstrapping"><i class="fa fa-check"></i><b>3.3.3</b> Bootstrapping</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#genotype-accumulation-curve"><i class="fa fa-check"></i><b>3.3.4</b> Genotype Accumulation Curve</a></li>
<li class="chapter" data-level="3.3.5" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#index-of-association"><i class="fa fa-check"></i><b>3.3.5</b> Index of association</a></li>
<li class="chapter" data-level="3.3.6" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#data-format-updates-population-strata-and-hierarchies"><i class="fa fa-check"></i><b>3.3.6</b> Data format updates: population strata and hierarchies</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#availability"><i class="fa fa-check"></i><b>3.4</b> Availability</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#requirements"><i class="fa fa-check"></i><b>3.4.1</b> Requirements</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#installation"><i class="fa fa-check"></i><b>3.4.2</b> Installation</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
<li class="chapter" data-level="3.6" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#acknowledgements-1"><i class="fa fa-check"></i><b>3.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><i class="fa fa-check"></i><b>4</b> Spatial and Temporal Analysis of Populations of the Sudden Oak Death Pathogen in Oregon Forests</a><ul>
<li class="chapter" data-level="4.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#introduction-3"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#materials-and-methods-1"><i class="fa fa-check"></i><b>4.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#location"><i class="fa fa-check"></i><b>4.3.1</b> Location</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#sampling"><i class="fa fa-check"></i><b>4.3.2</b> Sampling</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#isolation-identification-and-dna-extraction"><i class="fa fa-check"></i><b>4.3.3</b> Isolation, identification and DNA extraction</a></li>
<li class="chapter" data-level="4.3.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#genotyping-data-validation-and-harmonization"><i class="fa fa-check"></i><b>4.3.4</b> Genotyping, data validation, and harmonization</a></li>
<li class="chapter" data-level="4.3.5" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#nursery-populations"><i class="fa fa-check"></i><b>4.3.5</b> Nursery populations</a></li>
<li class="chapter" data-level="4.3.6" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#data-analysis-1"><i class="fa fa-check"></i><b>4.3.6</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#results"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#demographic-pattern-and-genetic-diversity"><i class="fa fa-check"></i><b>4.4.1</b> Demographic pattern and genetic diversity</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#spatial-correlation"><i class="fa fa-check"></i><b>4.4.2</b> Spatial Correlation</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#population-differentiation"><i class="fa fa-check"></i><b>4.4.3</b> Population differentiation</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#clustering-of-forest-with-nursery-populations"><i class="fa fa-check"></i><b>4.4.4</b> Clustering of forest with nursery populations</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#discussion-1"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#acknowledgements-2"><i class="fa fa-check"></i><b>4.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.7" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-material"><i class="fa fa-check"></i><b>4.7</b> Supplementary Material</a><ul>
<li class="chapter" data-level="4.7.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#text:S1"><i class="fa fa-check"></i><b>4.7.1</b> Supplementary Text</a></li>
<li class="chapter" data-level="4.7.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-figures"><i class="fa fa-check"></i><b>4.7.2</b> Supplementary Figures</a></li>
<li class="chapter" data-level="4.7.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-tables"><i class="fa fa-check"></i><b>4.7.3</b> Supplementary Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><i class="fa fa-check"></i><b>5</b> Factors Influencing Inference of Clonality in Diploid Populations</a><ul>
<li class="chapter" data-level="5.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#introduction-4"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a><ul>
<li class="chapter" data-level="5.3.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#simulating-microsatellite-loci"><i class="fa fa-check"></i><b>5.3.1</b> Simulating Microsatellite Loci</a></li>
<li class="chapter" data-level="5.3.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#simulating-snp-loci"><i class="fa fa-check"></i><b>5.3.2</b> Simulating SNP Loci</a></li>
<li class="chapter" data-level="5.3.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#mating"><i class="fa fa-check"></i><b>5.3.3</b> Mating</a></li>
<li class="chapter" data-level="5.3.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#analysis-of-microsatellite-data"><i class="fa fa-check"></i><b>5.3.4</b> Analysis of Microsatellite Data</a></li>
<li class="chapter" data-level="5.3.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#analysis-of-snp-data"><i class="fa fa-check"></i><b>5.3.5</b> Analysis of SNP Data</a></li>
<li class="chapter" data-level="5.3.6" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#power-analysis"><i class="fa fa-check"></i><b>5.3.6</b> Power Analysis</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#clone-correction-negatively-impacts-clonal-inference"><i class="fa fa-check"></i><b>5.4.1</b> Clone Correction Negatively Impacts Clonal Inference</a></li>
<li class="chapter" data-level="5.4.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#negative-values-of-barr_d-in-clonal-populations-correlated-with-low-allelic-diversity"><i class="fa fa-check"></i><b>5.4.2</b> Negative Values of <span class="math inline">\(\bar{r}_d\)</span> in Clonal Populations Correlated With Low Allelic Diversity</a></li>
<li class="chapter" data-level="5.4.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#power-to-detect-clonal-reproduction-in-snp-data"><i class="fa fa-check"></i><b>5.4.3</b> Power To Detect Clonal Reproduction in SNP Data</a></li>
<li class="chapter" data-level="5.4.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#common-features-of-microsatellite-and-snp-markers"><i class="fa fa-check"></i><b>5.4.4</b> Common Features of Microsatellite and SNP Markers</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#discussion-2"><i class="fa fa-check"></i><b>5.5</b> Discussion</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#clone-correction-increases-false-negatives"><i class="fa fa-check"></i><b>5.5.1</b> Clone Correction Increases False Negatives</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#mutation-rates-and-barr_d"><i class="fa fa-check"></i><b>5.5.2</b> Mutation Rates and <span class="math inline">\(\bar{r}_d\)</span></a></li>
<li class="chapter" data-level="5.5.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#allelic-diversity-influences-the-value-of-barr_d-in-clonal-populations"><i class="fa fa-check"></i><b>5.5.3</b> Allelic Diversity Influences the value of <span class="math inline">\(\bar{r}_d\)</span> in Clonal Populations</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#linkage-in-snp-data-increases-barr_d-unilaterally"><i class="fa fa-check"></i><b>5.5.4</b> Linkage in SNP Data Increases <span class="math inline">\(\bar{r}_d\)</span> Unilaterally</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#limitations-of-this-study"><i class="fa fa-check"></i><b>5.5.5</b> Limitations of This Study</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#conclusions-1"><i class="fa fa-check"></i><b>5.5.6</b> Conclusions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-conclusions-2.html"><a href="6-conclusions-2.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a></li>
<li class="chapter" data-level="7" data-path="7-bibliography.html"><a href="7-bibliography.html"><i class="fa fa-check"></i><b>7</b> Bibliography</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Development and Application of Tools for Genetic Analysis of Clonal Populations</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="bibliography" class="section level1">
<h1><span class="header-section-number">Chapter 7</span> Bibliography</h1>
<!--
To remove the indentation of the first entry.
-->
<p></p>
<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->
<p>   </p>
<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.


---
nocite: | 
  @angel2000, @angel2001, @angel2002a @grunwald2011phytophthora
...
-->

<div id="refs" class="references">
<div>
<p>Adamack, A. T., &amp; Gruber, B. (2014). PopGenReport: Simplifying basic population genetic analyses in R. <em>Methods in Ecology and Evolution</em>, <em>5</em>(4), 384–387. <a href="https://doi.org/10.1111/2041-210x.12158" class="uri">https://doi.org/10.1111/2041-210x.12158</a></p>
</div>
<div>
<p>Agapow, P.-M., &amp; Burt, A. (2001). Indices of multilocus linkage disequilibrium. <em>Molecular Ecology Notes</em>, <em>1</em>(1-2), 101–102. <a href="https://doi.org/10.1046/j.1471-8278.2000.00014.x" class="uri">https://doi.org/10.1046/j.1471-8278.2000.00014.x</a></p>
</div>
<div>
<p>Ali, S., Soubeyrand, S., Gladieux, P., Giraud, T., Leconte, M., Gautier, A., Mboup, M., Chen, W., Vallavieille-Pope, C., &amp; Enjalbert, J. (2016). Cloncase: Estimation of sex frequency and effective population size by clonemate resampling in partially clonal organisms. <em>Molecular Ecology Resources</em>. <a href="https://doi.org/10.1111/1755-0998.12511" class="uri">https://doi.org/10.1111/1755-0998.12511</a></p>
</div>
<div>
<p>Anderson, J. B., &amp; Kohn, L. M. (1995). Clonality in soilborne, plant-pathogenic fungi. <em>Annual Review of Phytopathology</em>, <em>33</em>(1), 369–391. <a href="https://doi.org/10.1146/annurev.py.33.090195.002101" class="uri">https://doi.org/10.1146/annurev.py.33.090195.002101</a></p>
</div>
<div>
<p>Anscombe, F. J. (1973). Graphs in statistical analysis. <em>The American Statistician</em>, <em>27</em>(1), 17. <a href="https://doi.org/10.2307/2682899" class="uri">https://doi.org/10.2307/2682899</a></p>
</div>
<div>
<p>Arnaud-Hanod, S., Duarte, C. M., Alberto, F., &amp; Serrão, E. A. (2007). Standardizing methods to address clonality in population studies. <em>Molecular Ecology</em>, <em>16</em>(24), 5115–5139. <a href="https://doi.org/10.1111/j.1365-294X.2007.03535.x" class="uri">https://doi.org/10.1111/j.1365-294X.2007.03535.x</a></p>
</div>
<div>
<p>Arnaud-Haond, S., &amp; Belkhir, K. (2006). Genclone: A computer program to analyse genotypic data, test for clonality and describe spatial clonal organization. <em>Molecular Ecology Notes</em>, <em>7</em>(1), 15–17. <a href="https://doi.org/10.1111/j.1471-8286.2006.01522.x" class="uri">https://doi.org/10.1111/j.1471-8286.2006.01522.x</a></p>
</div>
<div>
<p>Arnaud-Haond, S., Duarte, C. M., Diaz-Almela, E., Marbà, N., Sintes, T., &amp; Serrão, E. A. (2012). Implications of Extreme Life Span in Clonal Organisms: Millenary Clones in Meadows of the Threatened Seagrass <em>Posidonia oceanica</em>. <em>PLoS ONE</em>, <em>7</em>(2), e30454. <a href="https://doi.org/10.1371/journal.pone.0030454" class="uri">https://doi.org/10.1371/journal.pone.0030454</a></p>
</div>
<div>
<p>Bailleul, D., Stoeckel, S., &amp; Arnaud-Haond, S. (2016). RClone: A package to identify MultiLocus clonal lineages and handle clonal data sets inr. <em>Methods in Ecology and Evolution</em>, <em>7</em>(8), 966–970. <a href="https://doi.org/10.1111/2041-210x.12550" class="uri">https://doi.org/10.1111/2041-210x.12550</a></p>
</div>
<div>
<p>Balloux, F., Lehmann, L., &amp; de Meeûs, T. (2003). The population genetics of clonal and partially clonal diploids. <em>Genetics</em>, <em>164</em>(4), 1635–1644.</p>
</div>
<div>
<p>Baxter, S. M., Day, S. W., Fetrow, J. S., &amp; Reisinger, S. J. (2006). Scientific software development is not an oxymoron. <em>PLoS Computational Biology</em>, <em>2</em>(9), e87. <a href="https://doi.org/10.1371/journal.pcbi.0020087" class="uri">https://doi.org/10.1371/journal.pcbi.0020087</a></p>
</div>
<div>
<p>Bivand, R., Keitt, T., &amp; Rowlingson, B. (2014). <em>rgdal: Bindings for the Geospatial Data Abstraction Library</em>. Retrieved from <a href="https://CRAN.R-project.org/package=rgdal" class="uri">https://CRAN.R-project.org/package=rgdal</a></p>
</div>
<div>
<p>Brown, A., Feldman, M., &amp; Nevo, E. (1980). Multilocus structure of natural populations of <em>Hordeum spontaneum</em>. <em>Genetics</em>, <em>96</em>(2), 523–536. Retrieved from <a href="http://www.genetics.org/content/96/2/523.abstract" class="uri">http://www.genetics.org/content/96/2/523.abstract</a></p>
</div>
<div>
<p>Brownstein, M. J., Carpten, J. D., &amp; Smith, J. R. (1996). Modulation of non-templated nucleotide addition by Taq DNA polymerase: Primer modifications that facilitate genotyping. <em>BioTechniques</em>, <em>20</em>(6), 1004–6, 1008–10.</p>
</div>
<div>
<p>Bruvo, R., Michiels, N. K., D’Souza, T. G., &amp; Schulenburg, H. (2004). A simple method for the calculation of microsatellite genotype distances irrespective of ploidy level. <em>Molecular Ecology</em>, <em>13</em>(7), 2101–2106.</p>
</div>
<div>
<p>Buckheit, J. B., &amp; Donoho, D. L. (1995). WaveLab and reproducible research. In <em>Wavelets and statistics</em> (pp. 55–81). Springer. <a href="https://doi.org/10.1007/978-1-4612-2544-7_5" class="uri">https://doi.org/10.1007/978-1-4612-2544-7_5</a></p>
</div>
<div>
<p>Burt, A., Carter, D. A., Koenig, G. L., White, T. J., &amp; Taylor, J. W. (1996). Molecular markers reveal cryptic sex in the human pathogen <em>Coccidioides immitis</em>. <em>Proceedings of the National Academy of Sciences</em>, <em>93</em>(2), 770–773.</p>
</div>
<div>
<p>Canty, A., &amp; Ripley, B. D. (2015). <em>Boot: Bootstrap r (s-plus) functions</em>. Retrieved from <a href="https://CRAN.R-project.org/package=boot" class="uri">https://CRAN.R-project.org/package=boot</a></p>
</div>
<div>
<p>Chakarov, N., Linke, B., Boerner, M., Goesmann, A., Krüger, O., &amp; Hoffman, J. I. (2015). Apparent vector-mediated parent-to-offspring transmission in an avian malaria-like parasite. <em>Molecular Ecology</em>, <em>24</em>(6), 1355–1363.</p>
</div>
<div>
<p>Clark, L., &amp; Jasieniuk, M. (2011). Polysat: an R package for polyploid microsatellite analysis. <em>Molecular Ecology Resources</em>, <em>11</em>(3), 562–566. <a href="https://doi.org/10.1111/j.1755-0998.2011.02985.x" class="uri">https://doi.org/10.1111/j.1755-0998.2011.02985.x</a></p>
</div>
<div>
<p>Cooke, D. E. L., Cano, L. M., Raffaele, S., Bain, R. A., Cooke, L. R., Etherington, G. J., Deahl, K. L., Farrer, R. A., Gilroy, E. M., Goss, E. M., Grünwald, N. J., Hein, I., MacLean, D., McNicol, J. W., Randall, E., Oliva, R. F., Pel, M. A., Shaw, D. S., Squires, J. N., Taylor, M. C., Vleeshouwers, V. G. A. A., Birch, P. R. J., Lees, A. K., &amp; Kamoun, S. (2012). Genome analyses of an aggressive and invasive lineage of the irish potato famine pathogen. <em>PLoS Pathogens</em>, <em>8</em>(10), e1002940. <a href="https://doi.org/10.1371/journal.ppat.1002940" class="uri">https://doi.org/10.1371/journal.ppat.1002940</a></p>
</div>
<div>
<p>Croucher, P. J. P., Mascheretti, S., &amp; Garbelotto, M. (2013). Combining field epidemiological information and genetic data to comprehensively reconstruct the invasion history and the microevolution of the Sudden Oak Death agent <em>Phytophthora ramorum</em> (Stramenopila: Oomycetes) in California. <em>Biological Invasions</em>, <em>15</em>(10), 2281–2297. <a href="https://doi.org/10.1007/s10530-013-0453-8" class="uri">https://doi.org/10.1007/s10530-013-0453-8</a></p>
</div>
<div>
<p>Csardi, G., &amp; Nepusz, T. (2006). The igraph software package for complex network research. <em>InterJournal</em>, <em>Complex Systems</em>, 1695. Retrieved from <a href="http://igraph.org" class="uri">http://igraph.org</a></p>
</div>
<div>
<p>Dagum, L., &amp; Menon, R. (1998). OpenMP: An industry standard API for shared-memory programming. <em>Computational Science &amp; Engineering, IEEE</em>, <em>5</em>(1), 46–55.</p>
</div>
<div>
<p>Davey, J. W., &amp; Blaxter, M. L. (2010). RADSeq: next-generation population genetics. <em>Briefings in Functional Genomics</em>, <em>9</em>(5-6), 416–423. <a href="https://doi.org/10.1093/bfgp/elq031" class="uri">https://doi.org/10.1093/bfgp/elq031</a></p>
</div>
<div>
<p>Davey, J. W., Hohenlohe, P. A., Etter, P. D., Boone, J. Q., Catchen, J. M., &amp; Blaxter, M. L. (2011). Genome-wide genetic marker discovery and genotyping using next-generation sequencing. <em>Nature Reviews Genetics</em>, <em>12</em>(7), 499–510. <a href="https://doi.org/10.1038/nrg3012" class="uri">https://doi.org/10.1038/nrg3012</a></p>
</div>
<div>
<p>de Meeûs, T., &amp; Balloux, F. (2004). Clonal reproduction and linkage disequilibrium in diploids: A simulation study. <em>Infection, Genetics and Evolution</em>, <em>4</em>(4), 345–351. <a href="https://doi.org/10.1016/j.meegid.2004.05.002" class="uri">https://doi.org/10.1016/j.meegid.2004.05.002</a></p>
</div>
<div>
<p>de Meeûs, T., Lehmann, L., &amp; Balloux, F. (2006). Molecular epidemiology of clonal diploids: A quick overview and a short DIY (do it yourself) notice. <em>Infection, Genetics and Evolution</em>, <em>6</em>(2), 163–170. <a href="https://doi.org/10.1016/j.meegid.2005.02.004" class="uri">https://doi.org/10.1016/j.meegid.2005.02.004</a></p>
</div>
<div>
<p>Dobzhansky, T. (1973). Nothing in biology makes sense except in the light of evolution. <em>The American Biology Teacher</em>, <em>75</em>(2), 87–91.</p>
</div>
<div>
<p>Dray, S., &amp; Dufour, A.-B. (2007). The ade4 package: Implementing the duality diagram for ecologists. <em>Journal of Statistical Software</em>, <em>22</em>(4). <a href="https://doi.org/10.18637/jss.v022.i04" class="uri">https://doi.org/10.18637/jss.v022.i04</a></p>
</div>
<div>
<p>Elshire, R. J., Glaubitz, J. C., Sun, Q., Poland, J. A., Kawamoto, K., Buckler, E. S., &amp; Mitchell, S. E. (2011). A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species. <em>PLoS ONE</em>, <em>6</em>(5), e19379. <a href="https://doi.org/10.1371/journal.pone.0019379" class="uri">https://doi.org/10.1371/journal.pone.0019379</a></p>
</div>
<div>
<p>Erwin, D. C., Ribeiro, O. K., &amp; others. (1996). <em>Phytophthora diseases worldwide</em>. St. Paul, Minnesota, USA: American Phytopathological Society (APS Press).</p>
</div>
<div>
<p>Everhart, S. E., &amp; Scherm, H. (2015). Fine-scale genetic structure of <em>Monilinia fructicola</em> during brown rot epidemics within individual peach tree canopies. <em>Phytopathology</em>, <em>105</em>(4), 542–549. <a href="https://doi.org/10.1094/phyto-03-14-0088-r" class="uri">https://doi.org/10.1094/phyto-03-14-0088-r</a></p>
</div>
<div>
<p>Everhart, S. E., Tabima, J. F., &amp; Grünwald, N. J. (2014). <em>Phytophthora ramorum</em>. In <em>Genomics of plant-associated fungi and oomycetes: Dicot pathogens</em> (pp. 159–174). Berlin, Heidelberg: Springer. <a href="https://doi.org/10.1007/978-3-662-44056-8_8" class="uri">https://doi.org/10.1007/978-3-662-44056-8_8</a></p>
</div>
<div>
<p>Excoffier, L., &amp; Heckel, G. (2006). Computer programs for population genetics data analysis: A survival guide. <em>Nature Reviews Genetics</em>, <em>7</em>(10), 745–758. <a href="https://doi.org/10.1038/nrg1904" class="uri">https://doi.org/10.1038/nrg1904</a></p>
</div>
<div>
<p>Excoffier, L., Smouse, P. E., &amp; Quattro, J. M. (1992). Analysis of molecular variance inferred from metric distances among DNA haplotypes: Application to human mitochondrial DNA restriction data. <em>Genetics</em>, <em>131</em>(2), 479–91.</p>
</div>
<div>
<p>Eyre, C. A., Kozanitas, M., &amp; Garbelotto, M. (2013). Population dynamics of aerial and terrestrial populations of <em>Phytophthora ramorum</em> in a California forest under different climatic conditions. <em>Phytopathology</em>, <em>103</em>(11), 1141–1152. <a href="https://doi.org/10.1094/phyto-11-12-0290-r" class="uri">https://doi.org/10.1094/phyto-11-12-0290-r</a></p>
</div>
<div>
<p>Falush, D., Stephens, M., &amp; Pritchard, J. K. (2003). Inference of population structure using multilocus genotype data: Linked loci and correlated allele frequencies. <em>Genetics</em>, <em>164</em>(4), 1567–1587. Retrieved from <a href="http://www.genetics.org/content/164/4/1567.abstract" class="uri">http://www.genetics.org/content/164/4/1567.abstract</a></p>
</div>
<div>
<p>Felsenstein, J. (1974). The evolutionary advantage of recombination. <em>Genetics</em>, <em>78</em>(2), 737–756. Retrieved from <a href="http://www.genetics.org/content/78/2/737" class="uri">http://www.genetics.org/content/78/2/737</a></p>
</div>
<div>
<p>Felsenstein, J. (1989). PHYLIP-phylogeny inference package (version 3.2). <em>Cladistics</em>, <em>5</em>, 163–166.</p>
</div>
<div>
<p>Galindo, A., &amp; Gallegly, M. (1960). The nature of sexuality in <em>Phytophthora infestans</em>. <em>Phytopathology</em>, <em>50</em>, 123–28.</p>
</div>
<div>
<p>Gigerenzer, G. (1991). From tools to theories: A heuristic of discovery in cognitive psychology. <em>Psychological Review</em>, <em>98</em>(2), 254–267. <a href="https://doi.org/10.1037/0033-295x.98.2.254" class="uri">https://doi.org/10.1037/0033-295x.98.2.254</a></p>
</div>
<div>
<p>Goecks, J., Nekrutenko, A., Taylor, J., &amp; Team, T. G. (2010). Galaxy: A comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences. <em>Genome Biology</em>, <em>11</em>(8), R86. <a href="https://doi.org/10.1186/gb-2010-11-8-r86" class="uri">https://doi.org/10.1186/gb-2010-11-8-r86</a></p>
</div>
<div>
<p>Goodwin, S. B., Cohen, B. A., &amp; Fry, W. E. (1994). Panglobal distribution of a single clonal lineage of the Irish potato famine fungus. <em>Proceedings of the National Academy of Sciences</em>, <em>91</em>(24), 11591–11595.</p>
</div>
<div>
<p>Goss, E. M., Larsen, M., Chastagner, G. A., Givens, D. R., &amp; Grünwald, N. J. (2009). Population genetic analysis infers migration pathways of <em>Phytophthora ramorum</em> in US nurseries. <em>PLoS Pathogens</em>, <em>5</em>(9), e1000583. <a href="https://doi.org/10.1371/journal.ppat.1000583" class="uri">https://doi.org/10.1371/journal.ppat.1000583</a></p>
</div>
<div>
<p>Goss, E. M., Larsen, M., Vercauteren, A., Werres, S., Heungens, K., &amp; Grünwald, N. J. (2011). <em>Phytophthora ramorum</em> in Canada: Evidence for migration within North America and from Europe. <em>Phytopathology</em>, <em>101</em>(1), 166–171. <a href="https://doi.org/10.1094/phyto-05-10-0133" class="uri">https://doi.org/10.1094/phyto-05-10-0133</a></p>
</div>
<div>
<p>Goss, E. M., Tabima, J. F., Cooke, D. E., Restrepo, S., Fry, W. E., Forbes, G. A., Fieland, V. J., Cardenas, M., &amp; Grünwald, N. J. (2014). The Irish potato famine pathogen <em>Phytophthora infestans</em> originated in central Mexico rather than the Andes. <em>Proceedings of the National Academy of Sciences</em>, <em>111</em>(24), 8791–8796.</p>
</div>
<div>
<p>Goudet, J. (1995). FSTAT (version 1.2): A computer program to calculate F-statistics. <em>Journal of Heredity</em>, <em>86</em>(6), 485–486. Retrieved from <a href="http://jhered.oxfordjournals.org/content/86/6/485" class="uri">http://jhered.oxfordjournals.org/content/86/6/485</a></p>
</div>
<div>
<p>Goudet, J. (2005). Hierfstat, a package for R to compute and test hierarchical F-statistics. <em>Molecular Ecology Notes</em>, <em>5</em>(1), 184–186.</p>
</div>
<div>
<p>Gross, A., Hosoya, T., &amp; Queloz, V. (2014). Population structure of the invasive forest pathogen <em>Hymenoscyphus pseudoalbidus</em>. <em>Molecular Ecology</em>, <em>23</em>(12), 2943–2960.</p>
</div>
<div>
<p>Grünwald, N. J., &amp; Goss, E. M. (2011). Evolution and population genetics of exotic and re-emerging pathogens: Novel tools and approaches. <em>Annual Review of Phytopathology</em>, <em>49</em>(1), 249–267. <a href="https://doi.org/10.1146/annurev-phyto-072910-095246" class="uri">https://doi.org/10.1146/annurev-phyto-072910-095246</a></p>
</div>
<div>
<p>Grünwald, N. J., &amp; Hoheisel, G.-A. (2006). Hierarchical analysis of diversity, selfing, and genetic differentiation in populations of the oomycete <em>Aphanomyces euteiches</em>. <em>Phytopathology</em>, <em>96</em>(10), 1134–1141.</p>
</div>
<div>
<p>Grünwald, N. J., Garbelotto, M., Goss, E. M., Heungens, K., &amp; Prospero, S. (2012). Emergence of the Sudden Oak Death pathogen <em>Phytophthora ramorum</em>. <em>Trends in Microbiology</em>, <em>20</em>(3), 131–138. <a href="https://doi.org/10.1016/j.tim.2011.12.006" class="uri">https://doi.org/10.1016/j.tim.2011.12.006</a></p>
</div>
<div>
<p>Grünwald, N. J., Goodwin, S. B., Milgroom, M. G., &amp; Fry, W. E. (2003). Analysis of genotypic diversity data for populations of microorganisms. <em>Phytopathology</em>, <em>93</em>(6), 738–746. <a href="https://doi.org/10.1094/phyto.2003.93.6.738" class="uri">https://doi.org/10.1094/phyto.2003.93.6.738</a></p>
</div>
<div>
<p>Grünwald, N. J., Goss, E. M., &amp; Press, C. M. (2008a). <em>Phytophthora ramorum</em>: a pathogen with a remarkably wide host range causing Sudden Oak Death on oaks and ramorum blight on woody ornamentals. <em>Molecular Plant Pathology</em>, <em>9</em>(6), 729–740. <a href="https://doi.org/10.1111/j.1364-3703.2008.00500.x" class="uri">https://doi.org/10.1111/j.1364-3703.2008.00500.x</a></p>
</div>
<div>
<p>Grünwald, N. J., Goss, E. M., Ivors, K., Garbelotto, M., Martin, F. N., Prospero, S., Hansen, E., Bonants, P. J. M., Hamelin, R. C., Chastagner, G., Werres, S., Rizzo, D. M., Abad, G., Beales, P., Bilodeau, G. J., Blomquist, C. L., Brasier, C., Bri‘ere, S. C., Chandelier, A., Davidson, J. M., Denman, S., Elliott, M., Frankel, S. J., Goheen, E. M., de Gruyter, H., Heungens, K., James, D., Kanaskie, A., McWilliams, M. G., in ‘t Veld, W. M., Moralejo, E., Osterbauer, N. K., Palm, M. E., Parke, J. L., Sierra, A. M. P., Shamoun, S. F., Shishkoff, N., Tooley, P. W., Vettraino, A. M., Webber, J., &amp; Widmer, T. L. (2009). Standardizing the nomenclature for clonal lineages of the sudden oak death pathogen, <em>Phytophthora ramorum</em>. <em>Phytopathology</em>, <em>99</em>(7), 792–795. <a href="https://doi.org/10.1094/phyto-99-7-0792" class="uri">https://doi.org/10.1094/phyto-99-7-0792</a></p>
</div>
<div>
<p>Grünwald, N. J., Kitner, M., McDonald, V., &amp; Goss, E. M. (2008b).  Susceptibility in <em>Viburnum</em> to <em>Phytophthora ramorum</em>. <em>Plant Disease</em>, <em>92</em>(2), 210–214. <a href="https://doi.org/10.1094/pdis-92-2-0210" class="uri">https://doi.org/10.1094/pdis-92-2-0210</a></p>
</div>
<div>
<p>Grünwald, N. J., Martin, F. N., Larsen, M. M., Sullivan, C. M., Press, C. M., Coffey, M. D., Hansen, E. M., &amp; Parke, J. L. (2011). Phytophthora-ID.org: a sequence-based <em>Phytophthora</em> identification tool. <em>Plant Disease</em>, <em>95</em>(3), 337–342.</p>
</div>
<div>
<p>Halkett, F., Simon, J., &amp; Balloux, F. (2005). Tackling the population genetics of clonal and partially clonal organisms. <em>Trends in Ecology &amp; Evolution</em>, <em>20</em>(4), 194–201. <a href="https://doi.org/10.1016/j.tree.2005.01.001" class="uri">https://doi.org/10.1016/j.tree.2005.01.001</a></p>
</div>
<div>
<p>Hansen, E. M., Kanaskie, A., Prospero, S., McWilliams, M., Goheen, E. M., Osterbauer, N., Reeser, P., &amp; Sutton, W. (2008).  Epidemiology of <em>Phytophthora ramorum</em> in Oregon tanoak forests. <em>Canadian Journal of Forest Research</em>, <em>38</em>(5), 1133–1143. <a href="https://doi.org/10.1139/x07-217" class="uri">https://doi.org/10.1139/x07-217</a></p>
</div>
<div>
<p>Hartl, D. L., &amp; Clark, A. G. (2007). <em>Principles of population genetics</em>. Sunderland, MA, USA: Sinauer Associates.</p>
</div>
<div>
<p>Haubold, B., &amp; Hudson, R. R. (2000). LIAN 3.0: Detecting linkage disequilibrium in multilocus data. <em>Bioinformatics</em>, <em>16</em>(9), 847–849. <a href="https://doi.org/10.1093/bioinformatics/16.9.847" class="uri">https://doi.org/10.1093/bioinformatics/16.9.847</a></p>
</div>
<div>
<p>Haubold, B., Travisano, M., Rainey, P. B., &amp; Hudson, R. R. (1998). Detecting linkage disequilibrium in bacterial populations. <em>Genetics</em>, <em>150</em>(4), 1341–8.</p>
</div>
<div>
<p>Heck, K. L., van Belle, G., &amp; Simberloff, D. (1975). Explicit calculation of the rarefaction diversity measurement and the determination of sufficient sample size. <em>Ecology</em>, <em>56</em>(6), 1459–1461. <a href="https://doi.org/10.2307/1934716" class="uri">https://doi.org/10.2307/1934716</a></p>
</div>
<div>
<p>Heitman, J., Sun, S., &amp; James, T. Y. (2012). Evolution of fungal sexual reproduction. <em>Mycologia</em>, <em>105</em>(1), 1–27. <a href="https://doi.org/10.3852/12-253" class="uri">https://doi.org/10.3852/12-253</a></p>
</div>
<div>
<p>Hurlbert, S. H. (1971). The nonconcept of species diversity: A critique and alternative parameters. <em>Ecology</em>, <em>52</em>(4), 577–586. <a href="https://doi.org/10.2307/1934145" class="uri">https://doi.org/10.2307/1934145</a></p>
</div>
<div>
<p>Ioannidis, J. P. A., Allison, D. B., Ball, C. A., Coulibaly, I., Cui, X., Culhane, A., Falchi, M., Furlanello, C., Game, L., Jurman, G., Mangion, J., Mehta, T., Nitzberg, M., Page, G. P., Petretto, E., &amp; van Noort, V. (2008). Repeatability of published microarray gene expression analyses. <em>Nature Genetics</em>, <em>41</em>(2), 149–155. <a href="https://doi.org/10.1038/ng.295" class="uri">https://doi.org/10.1038/ng.295</a></p>
</div>
<div>
<p>Ivors, K., Garbelotto, M., Vries, I. D. E., Ruyter-spira, C., Hekkert, B. T., Rosenzweig, N., &amp; Bonants, P. (2006). Microsatellite markers identify three lineages of <em>Phytophthora ramorum</em> in US nurseries, yet single lineages in US forest and European nursery populations. <em>Molecular Ecology</em>, <em>15</em>(6), 1493–1505. <a href="https://doi.org/10.1111/j.1365-294x.2006.02864.x" class="uri">https://doi.org/10.1111/j.1365-294x.2006.02864.x</a></p>
</div>
<div>
<p>Jombart, T. (2008). Adegenet: a R package for the multivariate analysis of genetic markers. <em>Bioinformatics</em>, <em>24</em>(11), 1403–1405. <a href="https://doi.org/10.1093/bioinformatics/btn129" class="uri">https://doi.org/10.1093/bioinformatics/btn129</a></p>
</div>
<div>
<p>Jombart, T., &amp; Ahmed, I. (2011). Adegenet 1.3-1: New tools for the analysis of genome-wide SNP data. <em>Bioinformatics</em>, <em>27</em>(21), 3070–3071.</p>
</div>
<div>
<p>Jombart, T., Devillard, S., &amp; Balloux, F. (2010). Discriminant analysis of principal components: A new method for the analysis of genetically structured populations. <em>BMC Genetics</em>, <em>11</em>(1), 94. <a href="https://doi.org/10.1186/1471-2156-11-94" class="uri">https://doi.org/10.1186/1471-2156-11-94</a></p>
</div>
<div>
<p>Jurasinski, G., Koebsch, F., Guenther, A., &amp; Beetz, S. (2014). <em>flux: Flux rate calculation from dynamic closed chamber measurements</em>. Retrieved from <a href="https://CRAN.R-project.org/package=flux" class="uri">https://CRAN.R-project.org/package=flux</a></p>
</div>
<div>
<p>Kahle, D., &amp; Wickham, H. (2013). ggmap: Spatial Visualization with ggplot2. <em>The R Journal</em>, <em>5</em>(1), 144–161. Retrieved from <a href="http://journal.r-project.org/archive/2013-1/kahle-wickham.pdf" class="uri">http://journal.r-project.org/archive/2013-1/kahle-wickham.pdf</a></p>
</div>
<div>
<p>Kamoun, S., Furzer, O., Jones, J. D. G., Judelson, H. S., Ali, G. S., Dalio, R. J. D., Roy, S. G., Schena, L., Zambounis, A., Panabières, F., Cahill, D., Ruocco, M., Figueiredo, A., Chen, X.-R., Hulvey, J., Stam, R., Lamour, K., Gijzen, M., Tyler, B. M., Grünwald, N. J., Mukhtar, M. S., Toé, D. F. A., Tör, M., Ackerveken, G. V. D., McDowell, J., Daayf, F., Fry, W. E., Lindqvist-Kreuze, H., Meijer, H. J. G., Petre, B., Ristaino, J., Yoshida, K., Birch, P. R. J., &amp; Govers, F. (2014). The top 10 oomycete pathogens in molecular plant pathology. <em>Molecular Plant Pathology</em>, <em>16</em>(4), 413–434. <a href="https://doi.org/10.1111/mpp.12190" class="uri">https://doi.org/10.1111/mpp.12190</a></p>
</div>
<div>
<p>Kamvar, Z. N., Brooks, J. C., &amp; Grünwald, N. J. (2015a). Novel R tools for analysis of genome-wide population genetic data with emphasis on clonality. <em>Frontiers in Genetics</em>, <em>6</em>. <a href="https://doi.org/10.3389/fgene.2015.00208" class="uri">https://doi.org/10.3389/fgene.2015.00208</a></p>
</div>
<div>
<p>Kamvar, Z. N., Brooks, J. C., &amp; Grünwald, N. J. (2015b, May). Supplementary Material for Frontiers Plant Genetics and Genomics “Novel R tools for analysis of genome-wide population genetic data with emphasis on clonality”. ZENODO. <a href="https://doi.org/10.5281/zenodo.17424" class="uri">https://doi.org/10.5281/zenodo.17424</a></p>
</div>
<div>
<p>Kamvar, Z. N., Larsen, M. M., Kanaskie, A. M., Hansen, E. M., &amp; Grünwald, N. J. (2014a, December). Sudden_Oak_Death_in_Oregon_Forests: Spatial and temporal population dynamics of the sudden oak death epidemic in Oregon Forests. ZENODO. <a href="https://doi.org/10.5281/zenodo.13007" class="uri">https://doi.org/10.5281/zenodo.13007</a></p>
</div>
<div>
<p>Kamvar, Z. N., Larsen, M. M., Kanaskie, A. M., Hansen, E. M., &amp; Grünwald, N. J. (2015c). Spatial and temporal analysis of populations of the sudden oak death pathogen in oregon forests. <em>Phytopathology</em>, <em>105</em>(7), 982–989. <a href="https://doi.org/10.1094/phyto-12-14-0350-fi" class="uri">https://doi.org/10.1094/phyto-12-14-0350-fi</a></p>
</div>
<div>
<p>Kamvar, Z. N., López-Uribe, M. M., Coughlan, S., Grünwald, N. J., Lapp, H., &amp; Manel, S. (2016). Developing educational resources for population genetics in R: An open and collaborative approach. <em>Molecular Ecology Resources</em>. <a href="https://doi.org/10.1111/1755-0998.12558" class="uri">https://doi.org/10.1111/1755-0998.12558</a></p>
</div>
<div>
<p>Kamvar, Z. N., Tabima, J. F., &amp; Grünwald, N. J. (2014b).  Poppr : an R package for genetic analysis of populations with clonal, partially clonal, and/or sexual reproduction. <em>PeerJ</em>, <em>2</em>, e281. <a href="https://doi.org/10.7717/peerj.281" class="uri">https://doi.org/10.7717/peerj.281</a></p>
</div>
<div>
<p>Kuhn, T. S. (1996). <em>The structure of scientific revolutions</em>. University of Chicago Press. <a href="https://doi.org/10.7208/chicago/9780226458106.001.0001" class="uri">https://doi.org/10.7208/chicago/9780226458106.001.0001</a></p>
</div>
<div>
<p>Laloë, D., Jombart, T., Dufour, A.-B., &amp; Moazami-Goudarzi, K. (2007). Consensus genetic structuring and typological value of markers using multiple co-inertia analysis. <em>Genetics Selection Evolution</em>, <em>39</em>(5), 1–23.</p>
</div>
<div>
<p>Lees, A., Wattier, R., Shaw, D., Sullivan, L., Williams, N., &amp; Cooke, D. (2006). Novel microsatellite markers for the analysis of <em>Phytophthora infestans</em> populations. <em>Plant Pathology</em>, <em>55</em>(3), 311–319.</p>
</div>
<div>
<p>Levin, B. R. (1999). Population biology, evolution, and infectious disease: Convergence and synthesis. <em>Science</em>, <em>283</em>(5403), 806–809. <a href="https://doi.org/10.1126/science.283.5403.806" class="uri">https://doi.org/10.1126/science.283.5403.806</a></p>
</div>
<div>
<p>Li, Y., Cooke, D. E., Jacobsen, E., &amp; Lee, T. van der. (2013). Efficient multiplex simple sequence repeat genotyping of the oomycete plant pathogen <em>Phytophthora infestans</em>. <em>Journal of Microbiological Methods</em>, <em>92</em>(3), 316–322.</p>
</div>
<div>
<p>Linde, C., Zhan, J., &amp; McDonald, B. (2002). Population structure of <em>Mycosphaerella graminicola</em>: From lesions to continents. <em>Phytopathology</em>, <em>92</em>(9), 946–955.</p>
</div>
<div>
<p>Loewe, L. (2006). Quantifying the genomic decay paradox due to Muller’s ratchet in human mitochondrial DNA. <em>Genetical Research</em>, <em>87</em>(02), 133. <a href="https://doi.org/10.1017/s0016672306008123" class="uri">https://doi.org/10.1017/s0016672306008123</a></p>
</div>
<div>
<p>Ludwig, J. A., &amp; Reynolds, J. F. (1988). <em>Statistical ecology: A primer in methods and computing</em> (Vol. 1). John Wiley &amp; Sons.</p>
</div>
<div>
<p>Luikart, G., England, P. R., Tallmon, D., Jordan, S., &amp; Taberlet, P. (2003). The power and promise of population genomics: From genotyping to genome typing. <em>Nature Reviews Genetics</em>, <em>4</em>(12), 981–994.</p>
</div>
<div>
<p>Lynch, M., &amp; Gabriel, W. (1990). Mutation load and the survival of small populations. <em>Evolution</em>, 1725–1737.</p>
</div>
<div>
<p>Lynch, M., Ackerman, M. S., Gout, J.-F., Long, H., Sung, W., Thomas, W. K., &amp; Foster, P. L. (2016). Genetic drift, selection and the evolution of the mutation rate. <em>Nature Reviews Genetics</em>, <em>17</em>(11), 704–714. <a href="https://doi.org/10.1038/nrg.2016.104" class="uri">https://doi.org/10.1038/nrg.2016.104</a></p>
</div>
<div>
<p>Lynch, M., Bürger, R., Butcher, D., &amp; Gabriel, W. (1993). The mutational meltdown in asexual populations. <em>Journal of Heredity</em>, <em>84</em>(5), 339–344. Retrieved from <a href="http://jhered.oxfordjournals.org/content/84/5/339.abstract" class="uri">http://jhered.oxfordjournals.org/content/84/5/339.abstract</a></p>
</div>
<div>
<p>Lynch, M., Sung, W., Morris, K., Coffey, N., Landry, C. R., Dopman, E. B., Dickinson, W. J., Okamoto, K., Kulkarni, S., Hartl, D. L., &amp; Thomas, W. K. (2008). A genome-wide view of the spectrum of spontaneous mutations in yeast. <em>Proceedings of the National Academy of Sciences</em>, <em>105</em>(27), 9272–9277. <a href="https://doi.org/10.1073/pnas.0803466105" class="uri">https://doi.org/10.1073/pnas.0803466105</a></p>
</div>
<div>
<p>Magurran, A. E. (1988). Why diversity? In <em>Ecological diversity and its measurement</em> (pp. 1–5). Dordrecht: Springer Netherlands. <a href="https://doi.org/10.1007/978-94-015-7358-0_1" class="uri">https://doi.org/10.1007/978-94-015-7358-0_1</a></p>
</div>
<div>
<p>Mantel, N. (1967). The detection of disease clustering and a generalized regression approach. <em>Cancer Research</em>, <em>27</em>(2 Part 1), 209–220.</p>
</div>
<div>
<p>Mascheretti, S., Croucher, P. J. P., Kozanitas, M., Baker, L., &amp; Garbelotto, M. (2009).  Genetic epidemiology of the Sudden Oak Death pathogen <em>Phytophthora ramorum</em> in California. <em>Molecular Ecology</em>, <em>18</em>(22), 4577–4590. <a href="https://doi.org/10.1111/j.1365-294x.2009.04379.x" class="uri">https://doi.org/10.1111/j.1365-294x.2009.04379.x</a></p>
</div>
<div>
<p>Mascheretti, S., Croucher, P. J. P., Vettraino, A., Prospero, S., &amp; Garbelotto, M. (2008).  Reconstruction of the Sudden Oak Death epidemic in California through microsatellite analysis of the pathogen <em>Phytophthora ramorum</em>. <em>Molecular Ecology</em>, <em>17</em>(11), 2755–2768. <a href="https://doi.org/10.1111/j.1365-294x.2008.03773.x" class="uri">https://doi.org/10.1111/j.1365-294x.2008.03773.x</a></p>
</div>
<div>
<p>Mastretta-Yanes, A., Arrigo, N., Alvarez, N., Jorgensen, T. H., Piñero, D., &amp; Emerson, B. C. (2014). Restriction site-associated DNA sequencing, genotyping error estimation and <em>de novo</em> assembly optimization for population genetic inference. <em>Molecular Ecology Resources</em>, <em>15</em>(1), 28–41. <a href="https://doi.org/10.1111/1755-0998.12291" class="uri">https://doi.org/10.1111/1755-0998.12291</a></p>
</div>
<div>
<p>McDonald, B. A. (1997). The population genetics of fungi: tools and techniques. <em>Phytopathology</em>, <em>87</em>(4), 448–453.</p>
</div>
<div>
<p>McDonald, B. A., &amp; Linde, C. (2002a). Pathogen population genetics, evolutionary potential, and durable resistance. <em>Annual Review of Phytopathology</em>, <em>40</em>(1), 349–379. <a href="https://doi.org/10.1146/annurev.phyto.40.120501.101443" class="uri">https://doi.org/10.1146/annurev.phyto.40.120501.101443</a></p>
</div>
<div>
<p>McDonald, B. A., &amp; Linde, C. (2002b). The population genetics of plant pathogens and breeding strategies for durable resistance. <em>Euphytica</em>, <em>124</em>(2), 163–180. <a href="https://doi.org/10.1023/A:1015678432355" class="uri">https://doi.org/10.1023/A:1015678432355</a></p>
</div>
<div>
<p>McKiernan, E. C., Bourne, P. E., Brown, C. T., Buck, S., Kenall, A., Lin, J., McDougall, D., Nosek, B. A., Ram, K., Soderberg, C. K., Spies, J. R., Thaney, K., Updegrove, A., Woo, K. H., &amp; Yarkoni, T. (2016). How open science helps researchers succeed. <em>eLife</em>, <em>5</em>. <a href="https://doi.org/10.7554/elife.16800" class="uri">https://doi.org/10.7554/elife.16800</a></p>
</div>
<div>
<p>Meirmans, P. G., &amp; Van Tienderen, P. H. (2004). GENOTYPE and GENODIVE: Two programs for the analysis of genetic diversity of asexual organisms. <em>Molecular Ecology Notes</em>, <em>4</em>(4), 792–794. <a href="https://doi.org/10.1111/j.1471-8286.2004.00770.x" class="uri">https://doi.org/10.1111/j.1471-8286.2004.00770.x</a></p>
</div>
<div>
<p>Metz, C. E. (1978). Basic principles of ROC analysis. In <em>Seminars in nuclear medicine</em> (Vol. 8, pp. 283–298). Elsevier. <a href="https://doi.org/10.1016/S0001-2998(78)80014-2" class="uri">https://doi.org/10.1016/S0001-2998(78)80014-2</a></p>
</div>
<div>
<p>Metzger, M. J., Reinisch, C., Sherry, J., &amp; Goff, S. P. (2015). Horizontal transmission of clonal cancer cells causes leukemia in soft-shell clams. <em>Cell</em>, <em>161</em>(2), 255–263.</p>
</div>
<div>
<p>Michalakis, Y., &amp; Excoffier, L. (1996). A generic estimation of population subdivision using distances between alleles with special reference for microsatellite loci. <em>Genetics</em>, <em>142</em>(3), 1061–1064.</p>
</div>
<div>
<p>Milgroom, M. G. (1996). Recombination and the multilocus structure of fungal populations. <em>Annual Review of Phytopathology</em>, <em>34</em>(1), 457–477.</p>
</div>
<div>
<p>Milgroom, M. G. (2015). <em>Population biology of plant pathogens: Genetics, ecology, and evolution</em>. 3340 Pilot Knob Road, St. Paul, MN 55121, USA: APS Press.</p>
</div>
<div>
<p>Milgroom, M. G., Levin, S. A., &amp; Fry, W. E. (1989). Population genetics theory and fungicide resistance. <em>Plant Disease Epidemiology</em>, <em>2</em>, 340–367.</p>
</div>
<div>
<p>Milgroom, M., &amp; Fry, W. (1997). Contributions of population genetics to plant disease epidemiology and management. In <em>Advances in botanical research</em> (pp. 1–30). Elsevier BV. <a href="https://doi.org/10.1016/s0065-2296(08)60069-5" class="uri">https://doi.org/10.1016/s0065-2296(08)60069-5</a></p>
</div>
<div>
<p>Nei, M. (1973). Analysis of gene diversity in subdivided populations. <em>Proceedings of the National Academy of Sciences</em>, <em>70</em>(12), 3321–3323.</p>
</div>
<div>
<p>Nei, M. (1978). Estimation of average heterozygosity and genetic distance from a small number of individuals. <em>Genetics</em>, <em>89</em>(3), 583–590. Retrieved from <a href="http://www.genetics.org/content/89/3/583.abstract" class="uri">http://www.genetics.org/content/89/3/583.abstract</a></p>
</div>
<div>
<p>Nielsen, R., &amp; Slatkin, M. (2013). <em>An introduction to population genetics: Theory and applications</em>. Sinauer Associates, Incorporated. Retrieved from <a href="http://books.google.com/books?id=Iy08kgEACAAJ" class="uri">http://books.google.com/books?id=Iy08kgEACAAJ</a></p>
</div>
<div>
<p>Nieuwenhuis, B. P. S., &amp; James, T. Y. (2016). The frequency of sex in fungi. <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em>, <em>371</em>(1706), 20150540. <a href="https://doi.org/10.1098/rstb.2015.0540" class="uri">https://doi.org/10.1098/rstb.2015.0540</a></p>
</div>
<div>
<p>Oksanen, J., Blanchet, F. G., Kindt, R., Legendre, P., Minchin, P. R., O’Hara, R. B., Simpson, G. L., Solymos, P., Stevens, M. H. H., &amp; Wagner, H. (2013). <em>Vegan: Community ecology package</em>. Retrieved from <a href="https://CRAN.R-project.org/package=vegan" class="uri">https://CRAN.R-project.org/package=vegan</a></p>
</div>
<div>
<p>Oksanen, J., Blanchet, F. G., Kindt, R., Legendre, P., Minchin, P. R., O’Hara, R. B., Simpson, G. L., Solymos, P., Stevens, M. H. H., &amp; Wagner, H. (2015). <em>Vegan: Community ecology package</em>. Retrieved from <a href="http://CRAN.R-project.org/package=vegan" class="uri">http://CRAN.R-project.org/package=vegan</a></p>
</div>
<div>
<p>Orive, M. E. (1993). Effective population size in organisms with complex life-histories. <em>Theoretical Population Biology</em>, <em>44</em>(3), 316–340. <a href="https://doi.org/10.1006/tpbi.1993.1031" class="uri">https://doi.org/10.1006/tpbi.1993.1031</a></p>
</div>
<div>
<p>Ouzounis, C. A., &amp; Valencia, A. (2003). Early bioinformatics: The birth of a discipline–a personal view. <em>Bioinformatics</em>, <em>19</em>(17), 2176–2190. <a href="https://doi.org/10.1093/bioinformatics/btg309" class="uri">https://doi.org/10.1093/bioinformatics/btg309</a></p>
</div>
<div>
<p>Paradis, E., Claude, J., &amp; Strimmer, K. (2004). APE: Analyses of phylogenetics and evolution in R language. <em>Bioinformatics</em>, <em>20</em>(2), 289–290. <a href="https://doi.org/10.1093/bioinformatics/btg412" class="uri">https://doi.org/10.1093/bioinformatics/btg412</a></p>
</div>
<div>
<p>Paradis, E., Gosselin, T., Grünwald, N. J., Jombart, T., Manel, S., &amp; Lapp, H. (2016). Towards an integrated ecosystem of R packages for the analysis of population genetic data. <em>Molecular Ecology Resources</em>. <a href="https://doi.org/10.1111/1755-0998.12636" class="uri">https://doi.org/10.1111/1755-0998.12636</a></p>
</div>
<div>
<p>Parks, J. C., &amp; Werth, C. R. (1993). A Study of Spatial Features of Clones in a Population of Bracken Fern, <em>Pteridium aquilinum</em> (Dennstaedtiaceae). <em>American Journal of Botany</em>, <em>80</em>(5), 537. <a href="https://doi.org/10.2307/2445369" class="uri">https://doi.org/10.2307/2445369</a></p>
</div>
<div>
<p>Partridge, D., Lopez, P. D., &amp; Johnston, V. S. (1984). Computer programs as theories in biology. <em>Journal of Theoretical Biology</em>, <em>108</em>(4), 539–564. <a href="https://doi.org/10.1016/s0022-5193(84)80079-x" class="uri">https://doi.org/10.1016/s0022-5193(84)80079-x</a></p>
</div>
<div>
<p>Peakall, R., &amp; Smouse, P. E. (2006). GenAlEx 6: Genetic analysis in excel. population genetic software for teaching and research. <em>Molecular Ecology Notes</em>, <em>6</em>(1), 288–295.</p>
</div>
<div>
<p>Peever, T. L., &amp; Milgroom, M. G. (1994). Genetic structure of <em>Pyrenophora teres</em> populations determined with random amplified polymorphic DNA markers. <em>Canadian Journal of Botany</em>, <em>72</em>(7), 915–923.</p>
</div>
<div>
<p>Peng, B., &amp; Amos, C. I. (2008). Forward-time simulations of non-random mating populations using simuPOP. <em>Bioinformatics</em>, <em>24</em>(11), 1408–1409. <a href="https://doi.org/10.1093/bioinformatics/btn179" class="uri">https://doi.org/10.1093/bioinformatics/btn179</a></p>
</div>
<div>
<p>Pielou, E. (1975). <em>Ecological diversity</em>. New York: Wiley &amp; Sons.</p>
</div>
<div>
<p>Piepenbring, M. (2015). Biologische Schemata, gezeichnet und freigegeben von M. Piepenbring. Retrieved from <a href="https://goo.gl/XO7TmS" class="uri">https://goo.gl/XO7TmS</a></p>
</div>
<div>
<p>Poucke, K. V., Franceschini, S., Webber, J. F., Vercauteren, A., Turner, J. A., McCracken, A. R., Heungens, K., &amp; Brasier, C. M. (2012). Discovery of a fourth evolutionary lineage of <em>Phytophthora ramorum</em>: EU2. <em>Fungal Biology</em>, <em>116</em>(11), 1178–1191. <a href="https://doi.org/10.1016/j.funbio.2012.09.003" class="uri">https://doi.org/10.1016/j.funbio.2012.09.003</a></p>
</div>
<div>
<p>Prevosti, A., Ocaña, J., &amp; Alonso, G. (1975). Distances between populations of <em>Drosophila subobscura</em>, based on chromosome arrangement frequencies. <em>Theoretical and Applied Genetics</em>, <em>45</em>(6), 231–241.</p>
</div>
<div>
<p>Pritchard, J. K., Stephens, M., &amp; Donnelly, P. (2000). Inference of population structure using multilocus genotype data. <em>Genetics</em>, <em>155</em>(2), 945–959.</p>
</div>
<div>
<p>Prlić, A., &amp; Procter, J. B. (2012). Ten simple rules for the open development of scientific software. <em>PLoS Computational Biology</em>, <em>8</em>(12), e1002802. <a href="https://doi.org/10.1371/journal.pcbi.1002802" class="uri">https://doi.org/10.1371/journal.pcbi.1002802</a></p>
</div>
<div>
<p>Prospero, S., Black, J. A., &amp; Winton, L. M. (2004). Isolation and characterization of microsatellite markers in <em>Phytophthora ramorum</em>, the causal agent of Sudden Oak Death. <em>Molecular Ecology Notes</em>, <em>4</em>(4), 672–674. <a href="https://doi.org/10.1111/j.1471-8286.2004.00778.x" class="uri">https://doi.org/10.1111/j.1471-8286.2004.00778.x</a></p>
</div>
<div>
<p>Prospero, S., Grünwald, N. J., Winton, L. M., &amp; Hansen, E. M. (2009).  Migration Patterns of the Emerging Plant Pathogen <em>Phytophthora ramorum</em> on the West Coast of the United States of America. <em>Phytopathology</em>, <em>99</em>(6), 739–749. <a href="https://doi.org/10.1094/phyto-99-6-0739" class="uri">https://doi.org/10.1094/phyto-99-6-0739</a></p>
</div>
<div>
<p>Prospero, S., Hansen, E. M., Grünwald, N. J., &amp; Winton, L. M. (2007). Population dynamics of the Sudden Oak Death pathogen <em>Phytophthora ramorum</em> in Oregon from 2001 to 2004. <em>Molecular Ecology</em>, <em>16</em>(14), 2958–2973. <a href="https://doi.org/10.1111/j.1365-294x.2007.03343.x" class="uri">https://doi.org/10.1111/j.1365-294x.2007.03343.x</a></p>
</div>
<div>
<p>Prugnolle, F., &amp; de Meeûs, T. (2010). Apparent high recombination rates in clonal parasitic organisms due to inappropriate sampling design. <em>Heredity</em>, <em>104</em>(2), 135–140. <a href="https://doi.org/10.1038/hdy.2009.128" class="uri">https://doi.org/10.1038/hdy.2009.128</a></p>
</div>
<div>
<p>R Core Team. (2013). <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="http://www.R-project.org/" class="uri">http://www.R-project.org/</a></p>
</div>
<div>
<p>R Core Team. (2014). <em>R: A language and environment for statistical computing</em>. Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="https://www.R-project.org/" class="uri">https://www.R-project.org/</a></p>
</div>
<div>
<p>R Core Team. (2015). <em>R: A language and environment for statistical computing</em>. Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="http://www.R-project.org/" class="uri">http://www.R-project.org/</a></p>
</div>
<div>
<p>R Core Team. (2016). <em>R: A language and environment for statistical computing</em>. Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="https://www.R-project.org/" class="uri">https://www.R-project.org/</a></p>
</div>
<div>
<p>Rice, W. R. (2002). Evolution of sex: Experimental tests of the adaptive significance of sexual recombination. <em>Nature Reviews Genetics</em>, <em>3</em>(4), 241–251. <a href="https://doi.org/10.1038/nrg760" class="uri">https://doi.org/10.1038/nrg760</a></p>
</div>
<div>
<p>Rizzo, D. M., Garbelotto, M., &amp; Hansen, E. M. (2005). <em>Phytophthora ramorum</em>: Integrative research and management of an emerging pathogen in California and Oregon forests. <em>Annual Review of Phytopathology</em>, <em>43</em>(1), 309–335. <a href="https://doi.org/10.1146/annurev.phyto.42.040803.140418" class="uri">https://doi.org/10.1146/annurev.phyto.42.040803.140418</a></p>
</div>
<div>
<p>Rizzo, D. M., Garbelotto, M., Davidson, J. M., Slaughter, G. W., &amp; Koike, S. T. (2002). <em>Phytophthora ramorum</em> as the cause of extensive mortality of <em>Quercus</em> spp. and <em>Lithocarpus densiflorus</em> in California. <em>Plant Disease</em>, <em>86</em>(3), 205–214. <a href="https://doi.org/10.1094/pdis.2002.86.3.205" class="uri">https://doi.org/10.1094/pdis.2002.86.3.205</a></p>
</div>
<div>
<p>Rosvall, M., &amp; Bergstrom, C. T. (2008). Maps of random walks on complex networks reveal community structure. <em>Proceedings of the National Academy of Sciences</em>, <em>105</em>(4), 1118–1123.</p>
</div>
<div>
<p>Schliep, K. (2011). Phangorn: phylogenetic analysis in R. <em>Bioinformatics</em>, <em>27</em>(4), 592–593.</p>
</div>
<div>
<p>Searls, D. B. (2010). The roots of bioinformatics. <em>PLoS Computational Biology</em>, <em>6</em>(6), e1000809. <a href="https://doi.org/10.1371/journal.pcbi.1000809" class="uri">https://doi.org/10.1371/journal.pcbi.1000809</a></p>
</div>
<div>
<p>Shannon, C. E. (1948). A mathematical theory of communication. <em>ACM SIGMOBILE Mobile Computing and Communications Review</em>, <em>5</em>(1), 3–55.</p>
</div>
<div>
<p>Smith, J. M., Smith, N. H., O’Rourke, M., &amp; Spratt, B. G. (1993). How clonal are bacteria? <em>Proceedings of the National Academy of Sciences</em>, <em>90</em>(10), 4384–4388. <a href="https://doi.org/10.1073/pnas.90.10.4384" class="uri">https://doi.org/10.1073/pnas.90.10.4384</a></p>
</div>
<div>
<p>Sokal, R. R. (1958). A statistical method for evaluating systematic relationships. <em>Univ Kans Sci Bull</em>, <em>38</em>, 1409–1438.</p>
</div>
<div>
<p>Stoddart, J. A., &amp; Taylor, J. F. (1988). Genotypic diversity: Estimation and prediction in samples. <em>Genetics</em>, <em>118</em>(4), 705–11.</p>
</div>
<div>
<p>Stukenbrock, E. H., &amp; McDonald, B. A. (2008). The origins of plant pathogens in agro-ecosystems. <em>Annual Review of Phytopathology</em>, <em>46</em>(1), 75–100. <a href="https://doi.org/10.1146/annurev.phyto.010708.154114" class="uri">https://doi.org/10.1146/annurev.phyto.010708.154114</a></p>
</div>
<div>
<p>Sujkowski, L. S. (1994). Increased genotypic diversity via migration and possible occurrence of sexual reproduction of <em>Phytophthora infestans</em> in Poland. <em>Phytopathology</em>, <em>84</em>(2), 201. <a href="https://doi.org/10.1094/phyto-84-201" class="uri">https://doi.org/10.1094/phyto-84-201</a></p>
</div>
<div>
<p>Taylor, J. W., &amp; Fisher, M. C. (2003). Fungal multilocus sequence typing – it’s not just for bacteria. <em>Current Opinion in Microbiology</em>, <em>6</em>(4), 351–356.</p>
</div>
<div>
<p>Taylor, J. W., Geiser, D. M., Burt, A., &amp; Koufopanou, V. (1999). The evolutionary biology and population genetics underlying fungal strain typing. <em>Clinical Microbiology Reviews</em>, <em>12</em>(1), 126–146.</p>
</div>
<div>
<p>Tibayrenc, M. (1995). Population genetics of parasitic protozoa and other microorganisms. In <em>Advances in parasitology volume 36</em> (pp. 47–115). Elsevier BV. <a href="https://doi.org/10.1016/s0065-308x(08)60490-x" class="uri">https://doi.org/10.1016/s0065-308x(08)60490-x</a></p>
</div>
<div>
<p>Tibayrenc, M. (1996). Towards a unified evolutionary genetics of microorganisms. <em>Annual Review of Microbiology</em>, <em>50</em>(1), 401–429. <a href="https://doi.org/10.1146/annurev.micro.50.1.401" class="uri">https://doi.org/10.1146/annurev.micro.50.1.401</a></p>
</div>
<div>
<p>Tibayrenc, M., Kjellberg, F., &amp; Ayala, F. J. (1990). A clonal theory of parasitic protozoa: the population structures of <em>Entamoeba</em>, <em>Giardia</em>, <em>Leishmania</em>, <em>Naegleria</em>, <em>Plasmodium</em>, <em>Trichomonas</em>, and <em>Trypanosoma</em> and their medical and taxonomical consequences. <em>Proceedings of the National Academy of Sciences</em>, <em>87</em>(7), 2414–2418.</p>
</div>
<div>
<p>Vercauteren, A., Dobbelaere, I. D., Grünwald, N. J., Bonants, P., van Bockstaele, E., Maes, M., &amp; Heungens, K. (2010).  Clonal expansion of the Belgian <em>Phytophthora ramorum</em> populations based on new microsatellite markers. <em>Molecular Ecology</em>, <em>19</em>(1), 92–107. <a href="https://doi.org/10.1111/j.1365-294x.2009.04443.x" class="uri">https://doi.org/10.1111/j.1365-294x.2009.04443.x</a></p>
</div>
<div>
<p>Vercauteren, A., Larsen, M., Goss, E., Grünwald, N. J., Maes, M., &amp; Heungens, K. (2011). Identification of new polymorphic microsatellite markers in the NA1 and NA2 lineages of <em>Phytophthora ramorum</em>. <em>Mycologia</em>, <em>103</em>(6), 1245–1249. <a href="https://doi.org/10.3852/10-420" class="uri">https://doi.org/10.3852/10-420</a></p>
</div>
<div>
<p>Weir, B. S., &amp; Cockerham, C. (1996). Genetic data analysis II: Methods for discrete population genetic data.</p>
</div>
<div>
<p>Werres, S., Marwitz, R., Man In’t veld, W. A., de Cock, A. W., Bonants, P. J., de Weerdt, M., Themann, K., Ilieva, E., &amp; Baayen, R. P. (2001). <em>Phytophthora ramorum sp. nov.</em>, a new pathogen on <em>Rhododendron</em> and <em>Viburnum</em>. <em>Mycological Research</em>, <em>105</em>(10), 1155–1165. <a href="https://doi.org/10.1016/s0953-7562(08)61986-3" class="uri">https://doi.org/10.1016/s0953-7562(08)61986-3</a></p>
</div>
<div>
<p>Wickham, H. (2009). <em>ggplot2: Elegant Graphics for Data Analysis</em>. Springer-Verlag New York. Retrieved from <a href="http://ggplot2.org" class="uri">http://ggplot2.org</a></p>
</div>
<div>
<p>Wickham, H., &amp; Chang, W. (2015). <em>Devtools: Tools to make developing R packages easier</em>. Retrieved from <a href="http://CRAN.R-project.org/package=devtools" class="uri">http://CRAN.R-project.org/package=devtools</a></p>
</div>
<div>
<p>Wilson, G., Aruliah, D., Brown, C. T., Hong, N. P. C., Davis, M., Guy, R. T., Haddock, S. H., Huff, K. D., Mitchell, I. M., Plumbley, M. D., &amp; others. (2014). Best practices for scientific computing. <em>PLoS Biology</em>, <em>12</em>(1), e1001745.</p>
</div>
<div>
<p>Wilson, G., Bryan, J., Cranston, K., Kitzes, J., Nederbragt, L., &amp; Teal, T. K. (2016). Good enough practices in scientific computing (Version 2). Retrieved from <a href="http://arxiv.org/abs/1609.00037v2" class="uri">http://arxiv.org/abs/1609.00037v2</a></p>
</div>
<div>
<p>Winter, D. J. (2012). mmod: An R library for the calculation of population differentiation statistics. <em>Molecular Ecology Resources</em>, <em>12</em>(6), 1158–1160. <a href="https://doi.org/10.1111/j.1755-0998.2012.03174.x" class="uri">https://doi.org/10.1111/j.1755-0998.2012.03174.x</a></p>
</div>
<div>
<p>Winton, L. M., &amp; Hansen, E. M. (2001). Molecular diagnosis of <em>Phytophthora lateralis</em> in trees, water, and foliage baits using multiplex polymerase chain reaction. <em>Forest Pathology</em>, <em>31</em>(5), 275–283. <a href="https://doi.org/10.1046/j.1439-0329.2001.00251.x" class="uri">https://doi.org/10.1046/j.1439-0329.2001.00251.x</a></p>
</div>
<div>
<p>Yoshida, K., Schuenemann, V. J., Cano, L. M., Pais, M., Mishra, B., Sharma, R., Lanz, C., Martin, F. N., Kamoun, S., Krause, J., Thines, M., Weigel, D., &amp; Burbano, H. A. (2013). The rise and fall of the <em>Phytophthora infestans</em> lineage that triggered the Irish potato famine. <em>eLife</em>, <em>2</em>. <a href="https://doi.org/10.7554/elife.00731" class="uri">https://doi.org/10.7554/elife.00731</a></p>
</div>
<div>
<p>Ziemann, M., Eren, Y., &amp; El-Osta, A. (2016). Gene name errors are widespread in the scientific literature. <em>Genome Biology</em>, <em>17</em>(1). <a href="https://doi.org/10.1186/s13059-016-1044-7" class="uri">https://doi.org/10.1186/s13059-016-1044-7</a></p>
</div>
</div>
</div>









            </section>

          </div>
        </div>
      </div>
<a href="6-conclusions-2.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>


<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["http://hdl.handle.net/1957/60042", "PDF"], ["dissertation.tex", "TEX"], ["https://github.com/zkamvar/dissertation", "SOURCE CODE"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
