<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Development and Application of Tools for Genetic Analysis of Clonal Populations</title>
  <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
  <meta name="description" content="Development and Application of Tools for Genetic Analysis of Clonal Populations">
  <meta name="generator" content="bookdown 0.1.18 and GitBook 2.6.7">

  <meta property="og:title" content="Development and Application of Tools for Genetic Analysis of Clonal Populations" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Development and Application of Tools for Genetic Analysis of Clonal Populations" />
  
  
  

<meta name="author" content="Zhian N. Kamvar">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  

<link rel="next" href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html">

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>


  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#population-genetics-and-clonality"><i class="fa fa-check"></i><b>1.1</b> Population Genetics and Clonality</a></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#population-genetics-under-the-lens-of-plant-pathology"><i class="fa fa-check"></i><b>1.2</b> Population Genetics Under the Lens of Plant Pathology</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#the-need-for-reproducible-research"><i class="fa fa-check"></i><b>1.3</b> The Need For Reproducible Research</a></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#overview"><i class="fa fa-check"></i><b>1.4</b> Overview</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-2"><i class="fa fa-check"></i><b>1.4.1</b> Summary of Chapter 2</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-3"><i class="fa fa-check"></i><b>1.4.2</b> Summary of Chapter 3</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-4"><i class="fa fa-check"></i><b>1.4.3</b> Summary of Chapter 4</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-introduction.html"><a href="1-introduction.html#summary-of-chapter-5"><i class="fa fa-check"></i><b>1.4.4</b> Summary of Chapter 5</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><i class="fa fa-check"></i><b>2</b> <em>Poppr</em>: an R Package For Genetic Analysis of Populations With Clonal, Partially Clonal, and/or Sexual Reproduction</a><ul>
<li class="chapter" data-level="2.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#abstract"><i class="fa fa-check"></i><b>2.1</b> Abstract</a></li>
<li class="chapter" data-level="2.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#introduction-1"><i class="fa fa-check"></i><b>2.2</b> Introduction</a></li>
<li class="chapter" data-level="2.3" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#materials-and-methods"><i class="fa fa-check"></i><b>2.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="2.3.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#data-import"><i class="fa fa-check"></i><b>2.3.1</b> Data import</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#data-analysis"><i class="fa fa-check"></i><b>2.3.2</b> Data analysis</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#visualizations"><i class="fa fa-check"></i><b>2.3.3</b> Visualizations</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#performance"><i class="fa fa-check"></i><b>2.3.4</b> Performance</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#citation-of-methods-implemented-in-poppr"><i class="fa fa-check"></i><b>2.4</b> Citation of methods implemented in <em>poppr</em></a></li>
<li class="chapter" data-level="2.5" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#results-and-discussion"><i class="fa fa-check"></i><b>2.5</b> Results and Discussion</a><ul>
<li class="chapter" data-level="2.5.1" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#new-functionalities"><i class="fa fa-check"></i><b>2.5.1</b> New functionalities</a></li>
<li class="chapter" data-level="2.5.2" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#performance-1"><i class="fa fa-check"></i><b>2.5.2</b> Performance</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#conclusions"><i class="fa fa-check"></i><b>2.6</b> Conclusions</a></li>
<li class="chapter" data-level="2.7" data-path="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html"><a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html#acknowledgements"><i class="fa fa-check"></i><b>2.7</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><i class="fa fa-check"></i><b>3</b> Novel R Tools For Analysis of Genome-Wide Population Genetic Data With Emphasis on Clonality</a><ul>
<li class="chapter" data-level="3.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#abstract-1"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#introduction-2"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#implementations-and-examples"><i class="fa fa-check"></i><b>3.3</b> Implementations and Examples</a><ul>
<li class="chapter" data-level="3.3.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#clonal-identification"><i class="fa fa-check"></i><b>3.3.1</b> Clonal identification</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#minimum-spanning-networks-with-reticulation"><i class="fa fa-check"></i><b>3.3.2</b> Minimum Spanning Networks with Reticulation</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#bootstrapping"><i class="fa fa-check"></i><b>3.3.3</b> Bootstrapping</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#genotype-accumulation-curve"><i class="fa fa-check"></i><b>3.3.4</b> Genotype Accumulation Curve</a></li>
<li class="chapter" data-level="3.3.5" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#index-of-association"><i class="fa fa-check"></i><b>3.3.5</b> Index of association</a></li>
<li class="chapter" data-level="3.3.6" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#data-format-updates-population-strata-and-hierarchies"><i class="fa fa-check"></i><b>3.3.6</b> Data format updates: population strata and hierarchies</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#availability"><i class="fa fa-check"></i><b>3.4</b> Availability</a><ul>
<li class="chapter" data-level="3.4.1" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#requirements"><i class="fa fa-check"></i><b>3.4.1</b> Requirements</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#installation"><i class="fa fa-check"></i><b>3.4.2</b> Installation</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
<li class="chapter" data-level="3.6" data-path="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html"><a href="3-novel-r-tools-for-analysis-of-genome-wide-population-genetic-data-with-emphasis-on-clonality.html#acknowledgements-1"><i class="fa fa-check"></i><b>3.6</b> Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><i class="fa fa-check"></i><b>4</b> Spatial and Temporal Analysis of Populations of the Sudden Oak Death Pathogen in Oregon Forests</a><ul>
<li class="chapter" data-level="4.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#abstract-2"><i class="fa fa-check"></i><b>4.1</b> Abstract</a></li>
<li class="chapter" data-level="4.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#introduction-3"><i class="fa fa-check"></i><b>4.2</b> Introduction</a></li>
<li class="chapter" data-level="4.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#materials-and-methods-1"><i class="fa fa-check"></i><b>4.3</b> Materials and Methods</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#location"><i class="fa fa-check"></i><b>4.3.1</b> Location</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#sampling"><i class="fa fa-check"></i><b>4.3.2</b> Sampling</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#isolation-identification-and-dna-extraction"><i class="fa fa-check"></i><b>4.3.3</b> Isolation, identification and DNA extraction</a></li>
<li class="chapter" data-level="4.3.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#genotyping-data-validation-and-harmonization"><i class="fa fa-check"></i><b>4.3.4</b> Genotyping, data validation, and harmonization</a></li>
<li class="chapter" data-level="4.3.5" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#nursery-populations"><i class="fa fa-check"></i><b>4.3.5</b> Nursery populations</a></li>
<li class="chapter" data-level="4.3.6" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#data-analysis-1"><i class="fa fa-check"></i><b>4.3.6</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#results"><i class="fa fa-check"></i><b>4.4</b> Results</a><ul>
<li class="chapter" data-level="4.4.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#demographic-pattern-and-genetic-diversity"><i class="fa fa-check"></i><b>4.4.1</b> Demographic pattern and genetic diversity</a></li>
<li class="chapter" data-level="4.4.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#spatial-correlation"><i class="fa fa-check"></i><b>4.4.2</b> Spatial Correlation</a></li>
<li class="chapter" data-level="4.4.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#population-differentiation"><i class="fa fa-check"></i><b>4.4.3</b> Population differentiation</a></li>
<li class="chapter" data-level="4.4.4" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#clustering-of-forest-with-nursery-populations"><i class="fa fa-check"></i><b>4.4.4</b> Clustering of forest with nursery populations</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#discussion-1"><i class="fa fa-check"></i><b>4.5</b> Discussion</a></li>
<li class="chapter" data-level="4.6" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#acknowledgements-2"><i class="fa fa-check"></i><b>4.6</b> Acknowledgements</a></li>
<li class="chapter" data-level="4.7" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-material"><i class="fa fa-check"></i><b>4.7</b> Supplementary Material</a><ul>
<li class="chapter" data-level="4.7.1" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#text:S1"><i class="fa fa-check"></i><b>4.7.1</b> Supplementary Text</a></li>
<li class="chapter" data-level="4.7.2" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-figures"><i class="fa fa-check"></i><b>4.7.2</b> Supplementary Figures</a></li>
<li class="chapter" data-level="4.7.3" data-path="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html"><a href="4-spatial-and-temporal-analysis-of-populations-of-the-sudden-oak-death-pathogen-in-oregon-forests.html#supplementary-tables"><i class="fa fa-check"></i><b>4.7.3</b> Supplementary Tables</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><i class="fa fa-check"></i><b>5</b> Factors Influencing Inference of Clonality in Diploid Populations</a><ul>
<li class="chapter" data-level="5.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#abstract-3"><i class="fa fa-check"></i><b>5.1</b> Abstract</a></li>
<li class="chapter" data-level="5.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#introduction-4"><i class="fa fa-check"></i><b>5.2</b> Introduction</a></li>
<li class="chapter" data-level="5.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#methods"><i class="fa fa-check"></i><b>5.3</b> Methods</a><ul>
<li class="chapter" data-level="5.3.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#simulating-microsatellite-loci"><i class="fa fa-check"></i><b>5.3.1</b> Simulating Microsatellite Loci</a></li>
<li class="chapter" data-level="5.3.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#simulating-snp-loci"><i class="fa fa-check"></i><b>5.3.2</b> Simulating SNP Loci</a></li>
<li class="chapter" data-level="5.3.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#mating"><i class="fa fa-check"></i><b>5.3.3</b> Mating</a></li>
<li class="chapter" data-level="5.3.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#analysis-of-microsatellite-data"><i class="fa fa-check"></i><b>5.3.4</b> Analysis of Microsatellite Data</a></li>
<li class="chapter" data-level="5.3.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#analysis-of-snp-data"><i class="fa fa-check"></i><b>5.3.5</b> Analysis of SNP Data</a></li>
<li class="chapter" data-level="5.3.6" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#power-analysis"><i class="fa fa-check"></i><b>5.3.6</b> Power Analysis</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#results-1"><i class="fa fa-check"></i><b>5.4</b> Results</a><ul>
<li class="chapter" data-level="5.4.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#clone-correction-negatively-impacts-clonal-inference"><i class="fa fa-check"></i><b>5.4.1</b> Clone Correction Negatively Impacts Clonal Inference</a></li>
<li class="chapter" data-level="5.4.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#negative-values-of-barr_d-in-clonal-populations-correlated-with-low-allelic-diversity"><i class="fa fa-check"></i><b>5.4.2</b> Negative Values of <span class="math inline">\(\bar{r}_d\)</span> in Clonal Populations Correlated With Low Allelic Diversity</a></li>
<li class="chapter" data-level="5.4.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#power-to-detect-clonal-reproduction-in-snp-data"><i class="fa fa-check"></i><b>5.4.3</b> Power To Detect Clonal Reproduction in SNP Data</a></li>
<li class="chapter" data-level="5.4.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#common-features-of-microsatellite-and-snp-markers"><i class="fa fa-check"></i><b>5.4.4</b> Common Features of Microsatellite and SNP Markers</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#discussion-2"><i class="fa fa-check"></i><b>5.5</b> Discussion</a><ul>
<li class="chapter" data-level="5.5.1" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#clone-correction-increases-false-negatives"><i class="fa fa-check"></i><b>5.5.1</b> Clone Correction Increases False Negatives</a></li>
<li class="chapter" data-level="5.5.2" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#mutation-rates-and-barr_d"><i class="fa fa-check"></i><b>5.5.2</b> Mutation Rates and <span class="math inline">\(\bar{r}_d\)</span></a></li>
<li class="chapter" data-level="5.5.3" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#allelic-diversity-influences-the-value-of-barr_d-in-clonal-populations"><i class="fa fa-check"></i><b>5.5.3</b> Allelic Diversity Influences the value of <span class="math inline">\(\bar{r}_d\)</span> in Clonal Populations</a></li>
<li class="chapter" data-level="5.5.4" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#linkage-in-snp-data-increases-barr_d-unilaterally"><i class="fa fa-check"></i><b>5.5.4</b> Linkage in SNP Data Increases <span class="math inline">\(\bar{r}_d\)</span> Unilaterally</a></li>
<li class="chapter" data-level="5.5.5" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#limitations-of-this-study"><i class="fa fa-check"></i><b>5.5.5</b> Limitations of This Study</a></li>
<li class="chapter" data-level="5.5.6" data-path="5-factors-influencing-inference-of-clonality-in-diploid-populations.html"><a href="5-factors-influencing-inference-of-clonality-in-diploid-populations.html#conclusions-1"><i class="fa fa-check"></i><b>5.5.6</b> Conclusions</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-conclusions-2.html"><a href="6-conclusions-2.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a></li>
<li class="chapter" data-level="7" data-path="7-bibliography.html"><a href="7-bibliography.html"><i class="fa fa-check"></i><b>7</b> Bibliography</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Development and Application of Tools for Genetic Analysis of Clonal Populations</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Development and Application of Tools for Genetic Analysis of Clonal Populations</h1>
<h4 class="author"><em>Zhian N. Kamvar</em></h4>
<h4 class="date"><em>December 6, 2016</em></h4>
</div>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<p>Dobzhansky’s famous quote “Nothing in biology makes sense except in the light of evolution” has become a cliché, but it is hard to find a statement that more elegantly captures the essence of biological study <span class="citation">(<a href="#ref-dobzhansky2013nothing">1973</a>)</span>. In a more reductionist sense, it is perhaps appropriate to state that nothing in evolution makes sense except in the light of mutation; that is to say that mutation in an organism’s genetic code is the seed upon which evolution acts <span class="citation">(Lynch et al., <a href="#ref-lynch2016genetic">2016</a>)</span>. Of course, a seed left alone will not grow; it needs the right environment and conditions. In turn, a mutation needs a population in order to thrive. A population can be defined as a group of interbreeding individuals, coexisting temporally and spatially (though we will see later that this is not always the best definition) <span class="citation">(Milgroom &amp; Fry, <a href="#ref-milgroom1997contributions">1997</a>)</span>. In a population, if the conditions are right, a new mutation can grow in frequency and become an established allele. The study of population genetics seeks to understand how evolution acts upon these alleles to shape populations as they grow, shrink, split, and adapt to their environment. Ultimately, these populations are what give rise to species.</p>
<div id="population-genetics-and-clonality" class="section level2">
<h2><span class="header-section-number">1.1</span> Population Genetics and Clonality</h2>
<p>The classical definition of a population is reliant upon the assumption that the individuals within the populations are panmictic, that is that they are randomly mating so that the individuals in the next generation will have a random assortment of alleles inherited from their parents <span class="citation">(Hartl &amp; Clark, <a href="#ref-hartl1997principles">2007</a>; Nielsen &amp; Slatkin, <a href="#ref-nielsen2013introduction">2013</a>; Rice, <a href="#ref-rice2002evolution">2002</a>)</span>. The assumption that sex occurs applies to most major organismal groups <span class="citation">(Heitman et al., <a href="#ref-heitman2012evolution">2012</a>; Rice, <a href="#ref-rice2002evolution">2002</a>)</span>. However, sex does not occur in all organisms as many plants, fungi, protists, bacteria, animals, and viruses are known to exclusively reproduce clonally <span class="citation">(Arnaud-Hanod et al., <a href="#ref-arnaud2007standardizing">2007</a>; Arnaud-Haond et al., <a href="#ref-arnaud2012implications">2012</a>; Heitman et al., <a href="#ref-heitman2012evolution">2012</a>; Orive, <a href="#ref-orive1993effective">1993</a>; Rice, <a href="#ref-rice2002evolution">2002</a>)</span>. Clonal reproduction here means reproduction that results in offspring that are genetically identical to the parents (note that this can include organisms derived from the selfing of a haploid organism). Thus, the definition of a population is perhaps best rephrased to simply indicate a group of individuals of the same species<a href="#fn1" class="footnoteRef" id="fnref1"><sup>1</sup></a> coexisting spatially and temporally <span class="citation">(Milgroom &amp; Fry, <a href="#ref-milgroom1997contributions">1997</a>)</span>.</p>
<p>There are several costs and benefits associated with both clonal and sexual reproductive strategies from an evolutionary perspective. For organisms whose main mode of reproduction is sexual, there is the two-fold cost of outcrossing: 1) only half of the available genetic material is inherited by the offspring and 2) the loss of advantageous gene combinations due to recombination <span class="citation">(Heitman et al., <a href="#ref-heitman2012evolution">2012</a>; Nieuwenhuis &amp; James, <a href="#ref-nieuwenhuis2016frequency">2016</a>; Rice, <a href="#ref-rice2002evolution">2002</a>)</span>. These costs are balanced by the benefits that come with recombining genetic material, which allows these organisms to purge deleterious mutations. As clonally reproducing organisms do not undergo regular periods of recombination, they explicitly do not benefit from the previously mentioned advantage of recombination <span class="citation">(Heitman et al., <a href="#ref-heitman2012evolution">2012</a>; Nieuwenhuis &amp; James, <a href="#ref-nieuwenhuis2016frequency">2016</a>)</span>. This leads to a theory known as Muller’s ratchet, which indicates that the overall fitness of small populations of clonally reproducing organisms will decrease over time as the number of deleterious mutations accumulate, eventually leading to extinction <span class="citation">(Felsenstein, <a href="#ref-felsenstein1974evolutionary">1974</a>; Loewe, <a href="#ref-loewe2006quantifying">2006</a>; Lynch &amp; Gabriel, <a href="#ref-lynch1990mutation">1990</a>; Lynch et al., <a href="#ref-lynch1993mutational">1993</a>)</span>. Despite this, clonal organisms still have the advantage of passing on 100% of their genetic material to the next generation.</p>
<p>It’s easy to see the advantage of both of these strategies in the context of genetic adaptation to the environment. Clonal populations that are well adapted to their environment get to keep the combination of genes that allowed them to be so well adapted <span class="citation">(Heitman et al., <a href="#ref-heitman2012evolution">2012</a>)</span>. This strategy provides an advantage when resources are scarce and competition is high, but does not lend itself well to adaptation against changing environments due to low genotypic diversity <span class="citation">(Milgroom, <a href="#ref-milgroom1996recombination">1996</a>)</span>. Conversely, while sexual populations lose the competitive advantage of conserved beneficial allelic combinations, they make up for it with high genotypic diversity, increasing the chances of surviving change in the environment. As we shall see in the following sections, many microbial pathogens have the ability to reproduce both sexually and asexually <span class="citation">(Anderson &amp; Kohn, <a href="#ref-anderson1995clonality">1995</a>; Milgroom, <a href="#ref-milgroom1996recombination">1996</a>; Tibayrenc, <a href="#ref-tibayrenc1995population">1995</a>)</span>.</p>
<p>The question of sexual reproduction and population structure in microbial pathogens is important for the development of rational management strategies for pathogens or beneficial microbes <span class="citation">(Milgroom &amp; Fry, <a href="#ref-milgroom1997contributions">1997</a>; Taylor et al., <a href="#ref-taylor1999evolutionary">1999</a>; Tibayrenc, <a href="#ref-tibayrenc1995population">1995</a>; Tibayrenc et al., <a href="#ref-tibayrenc1990clonal">1990</a>)</span>. If one were developing an antimicrobial agent on a known set of strains, the effectiveness of this agent will be diminished if these strains represent only a fraction of the genotypic diversity due to sexual recombination <span class="citation">(Smith et al., <a href="#ref-smith1993how">1993</a>; Taylor et al., <a href="#ref-taylor1999evolutionary">1999</a>; Tibayrenc et al., <a href="#ref-tibayrenc1990clonal">1990</a>)</span>. Since most inference of population structure is based on neutral models that assume sexual reproduction, it is important to identify whether or not the population in question is reproducing sexually or clonally before one attempts to assess population structure and infer evolutionary processes <span class="citation">(Tibayrenc et al., <a href="#ref-tibayrenc1990clonal">1990</a>)</span>.</p>
<p>Neutral molecular markers are commonly used for population genetic analyses. Before molecular techniques became widely available for characterization of pathogens or microbes, the variation within most pathogenic microorganisms was described using phenotypic traits that reflected their pathogenicity (ability to cause disease), virulence (severity of disease), or morphotype placing them into different phenotypes <span class="citation">(Levin, <a href="#ref-levin1999population">1999</a>; Milgroom &amp; Fry, <a href="#ref-milgroom1997contributions">1997</a>)</span>. When considering markers for population genetic analysis, one of the most important criteria is that these markers are heritable.</p>
<p>A wide variety of heritable molecular markers exist for population genetic analysis <span class="citation">(Halkett et al., <a href="#ref-halkett2005tackling">2005</a>; Tibayrenc, <a href="#ref-tibayrenc1995population">1995</a>)</span>. It is not necessary to expound here on the variety of molecular markers available to us, but it is useful to take note of the common features. Generally, genetic markers come in two major flavors, selectively neutral and selected markers <span class="citation">(McDonald, <a href="#ref-mcdonald1997population">1997</a>; Milgroom &amp; Fry, <a href="#ref-milgroom1997contributions">1997</a>; Tibayrenc, <a href="#ref-tibayrenc1995population">1995</a>)</span>. Selectively neutral markers are used as independent variables that can reflect the evolutionary history of descent. For selectively neutral markers, it is important that they be independent and unlinked within the genome <span class="citation">(McDonald, <a href="#ref-mcdonald1997population">1997</a>)</span>.</p>
<p>Because neutral genetic markers vary independently within a population, these are more appropriate than selected markers to detect sexual reproduction. The theory behind this is simple; sexual reproduction breaks up associations between markers due to recombination and independent, random assortment. This contrasts with clonal reproduction, which transfers the entire genome in tact to the next generation, preserving associations <span class="citation">(Heitman et al., <a href="#ref-heitman2012evolution">2012</a>; McDonald, <a href="#ref-mcdonald1997population">1997</a>; Milgroom &amp; Fry, <a href="#ref-milgroom1997contributions">1997</a>; Nieuwenhuis &amp; James, <a href="#ref-nieuwenhuis2016frequency">2016</a>; Orive, <a href="#ref-orive1993effective">1993</a>; Tibayrenc, <a href="#ref-tibayrenc1995population">1995</a>)</span>. This means that not only will there be significant associations between alleles in a clonal population, but because of their inability to purge mutations, we also expect to see excess heterozygosity <span class="citation">(Tibayrenc, <a href="#ref-tibayrenc1996towards">1996</a>; Tibayrenc et al., <a href="#ref-tibayrenc1990clonal">1990</a>)</span>.</p>
<p>By using these basic principles, core sets of statistical analyses have been proposed to detect sexual reproduction by analyzing diversity within loci (for diploid organisms), linkage among loci, and diversity of multi-locus genotypes within the population <span class="citation">(Arnaud-Hanod et al., <a href="#ref-arnaud2007standardizing">2007</a>; Grünwald et al., <a href="#ref-grunwald2003analysis">2003</a>; McDonald, <a href="#ref-mcdonald1997population">1997</a>; Parks &amp; Werth, <a href="#ref-parks1993study">1993</a>; Smith et al., <a href="#ref-smith1993how">1993</a>; Tibayrenc, <a href="#ref-tibayrenc1996towards">1996</a>)</span>. None of these tests alone can give a definitive answer as to the nature of reproduction in a given population <span class="citation">(de Meeûs &amp; Balloux, <a href="#ref-de2004clonal">2004</a>; Nieuwenhuis &amp; James, <a href="#ref-nieuwenhuis2016frequency">2016</a>; Tibayrenc, <a href="#ref-tibayrenc1995population">1995</a>)</span>. Researchers still need to use biological evidence, such as the observation of sexual reproductive structures, to support the presence of sex.</p>
</div>
<div id="population-genetics-under-the-lens-of-plant-pathology" class="section level2">
<h2><span class="header-section-number">1.2</span> Population Genetics Under the Lens of Plant Pathology</h2>
<p>One of the over-arching themes in both ecology and evolution is that diversity is important for stable population and community structure <span class="citation">(Magurran, <a href="#ref-magurran1988diversity">1988</a>; McDonald &amp; Linde, <a href="#ref-mcdonald2002pathogen">2002</a><a href="#ref-mcdonald2002pathogen">a</a>)</span>. Agricultural systems, however, behave very differently to natural ecosystems <span class="citation">(Milgroom &amp; Fry, <a href="#ref-milgroom1997contributions">1997</a>; Stukenbrock &amp; McDonald, <a href="#ref-stukenbrock2008origins">2008</a>)</span>. Within agricultural systems, the plants and the landscape are homogenized, providing a steady source of food for people, but this homogeneity combined with intensive management practices also provides fertile ground for the evolution of plant pathogens <span class="citation">(Stukenbrock &amp; McDonald, <a href="#ref-stukenbrock2008origins">2008</a>)</span>. McDonald identified five major factors influencing the evolution of plant pathogens in agricultural systems: 1) mutation, 2) genetic drift, 3) gene/genotype flow, 4) mating system, and 5) selection due to host resistance <span class="citation">(<a href="#ref-mcdonald2002pathogen">2002</a><a href="#ref-mcdonald2002pathogen">a</a>)</span>.</p>
<p>An example of how changing one of these five factors influences plant health is clear in the oomycete <em>Phytophthora infestans</em> (Mont.) de Bary. The causal agent of potato late blight (and a poster child for plant pathology). <em>P. infestans</em> has changed the very demographics of human populations across the Atlantic ocean due to its role in the Irish potato famine <span class="citation">(Erwin et al., <a href="#ref-erwin1996phytophthora">1996</a>)</span>. It has a heterothallic (no selfing) life cycle that includes both clonally and sexually derived infectious propagules (Fig.<a href="1-introduction.html#fig:pinflife">1.1</a>). The clonally derived propagules are swimming, short-lived zoospores that can be transmitted via wind or water, whereas the sexually derived propagules are double-walled oospores. While <em>P. infestans</em> can survive as hyphae in infected tubers, the sexually derived oospores can survive for years in the soil, highlighting the added risk presented by sexual reproduction.</p>
<div class="figure" style="text-align: center"><span id="fig:pinflife"></span>
<img src="figure/introduction/Pinfestans_life.png" alt="Life cycle of the oomycete pathogen, *Phytophthora infestans* on potato
[@piepenbring2015infestans]. M! = meiosis, P! = plasmogamy, K! =
karyogamy. Accessed from https://goo.gl/XO7TmS" width="80%" />
<p class="caption">
Figure 1.1: Life cycle of the oomycete pathogen, <em>Phytophthora infestans</em> on potato <span class="citation">(Piepenbring, <a href="#ref-piepenbring2015infestans">2015</a>)</span>. M! = meiosis, P! = plasmogamy, K! = karyogamy. Accessed from <a href="https://goo.gl/XO7TmS" class="uri">https://goo.gl/XO7TmS</a>
</p>
</div>
<p>From the time of the Great Famine to 1980, there was only one single mating type, A1, observed in Europe, indicating that it was reproducing in a strictly clonal fashion <span class="citation">(Goodwin et al., <a href="#ref-goodwin1994panglobal">1994</a>)</span>. Sexual reproduction became possible when the second mating type, A2, was introduced to Europe <span class="citation">(Galindo &amp; Gallegly, <a href="#ref-galindo1960nature">1960</a>)</span>. Because of this risk, population genetic tools and methods were needed to assess whether or not sexual reproduction had occurred. While evidence for sexual reproduction was found <span class="citation">(Sujkowski, <a href="#ref-sujkowski1994increased">1994</a>)</span>, it additionally appeared that <em>P. infestans</em> was still reproducing and spreading clonally <span class="citation">(Goodwin et al., <a href="#ref-goodwin1994panglobal">1994</a>)</span>. <span class="citation">Cooke et al. (<a href="#ref-cooke2012genome">2012</a>)</span> analyzed isolates from 1983 to 2008 and found a rapid shift in clonal population structure, showing a dramatic increase in the prevalence of a particularly aggressive lineage. While sexual reproduction was not found to be prevalent, it was only via population genetic techniques that this question could be addressed.</p>
</div>
<div id="the-need-for-reproducible-research" class="section level2">
<h2><span class="header-section-number">1.3</span> The Need For Reproducible Research</h2>
<p>Computational methods are increasingly important for answering questions crucial to plant pathology, and as more and more of our analyses are becoming computational in nature, we need to ensure that these methods are reproducible. Computational methods are crucial for population genetic analyses in the 21st century <span class="citation">(Excoffier &amp; Heckel, <a href="#ref-excoffier2006computer">2006</a>)</span>. There has been a call for standards in the analysis of clonal organisms for the past twenty years, which resulted in a plethora of computer programs <span class="citation">(Arnaud-Hanod et al., <a href="#ref-arnaud2007standardizing">2007</a>; Tibayrenc, <a href="#ref-tibayrenc1996towards">1996</a>)</span>. It was not uncommon for a researcher to use several programs to answer a single research question, and each program would often require its own custom input format and would not necessarily run on all operating systems <span class="citation">(Excoffier &amp; Heckel, <a href="#ref-excoffier2006computer">2006</a>; Kamvar et al., <a href="#ref-kamvar2014poppr">2014</a><a href="#ref-kamvar2014poppr">b</a>)</span>. The constant reformatting of data, and the inability to automate series of analyses made it not only challenging to communicate the methods, but also increased the chance for error <span class="citation">(Adamack &amp; Gruber, <a href="#ref-adamack2014popgenreport">2014</a>; Excoffier &amp; Heckel, <a href="#ref-excoffier2006computer">2006</a>; Goecks et al., <a href="#ref-goecks2010galaxy">2010</a>; Kamvar et al., <a href="#ref-kamvar2014poppr">2014</a><a href="#ref-kamvar2014poppr">b</a>)</span>.</p>
<p>Biologists often think about reproducibility in terms of wet lab or bench work. Care is taken to ensure that all methods and data are faithfully recorded in a lab notebook, but this is not necessarily extended to computational analyses. The concept of reproducible research in scientific computing is attempting to take the principles of reproducibility we use in the wet lab and extend them to our downstream analyses <em>in silico</em>. This work attempts to introduce tools for reproducible research in clonal population genetics in an open manner. When researchers have open tools that can be easily shared, they can spend more time asking questions and less time formatting data.</p>
</div>
<div id="overview" class="section level2">
<h2><span class="header-section-number">1.4</span> Overview</h2>
<div class="figure" style="text-align: center"><span id="fig:introduction1"></span>
<img src="figure/introduction/dissertation.png" alt="A graphical representation of the three themes governing the work
presented in this dissertation." width="50%" />
<p class="caption">
Figure 1.2: A graphical representation of the three themes governing the work presented in this dissertation.
</p>
</div>
<p>The work presented here offers tools designed to answer questions related to clonal population genetics and plant pathology (Fig. <a href="1-introduction.html#fig:introduction1">1.2</a>). In this context, the term ‘tool’ refers to software code used to apply mathematical models or theory to population genetic data. The merit of this work lies within the context of reproducible science, which ensures that the computational environment used to create results can be replicated <span class="citation">(Buckheit &amp; Donoho, <a href="#ref-buckheit1995wavelab">1995</a>)</span>. The end goal was not simply the development of the software, but rather, the development of software for the goal of analyzing our own data. Serendipitously, by analyzing our own data with this software, we are able to not only demonstrate that it can be used, but also demonstrate that an entire analysis can be conducted in an open and reproducible manner. We demonstrate the usefulness and flexibility of these tools by using them to show evidence for multiple introductions in the Curry County, OR outbreak of the sudden oak death pathogen, <em>Phytophthora ramorum</em>. We additionally assess the power of the multilocus linkage disequilibrium measure <span class="math inline">\(\bar{r}_d\)</span> to detect clonal reproduction.</p>
<div id="summary-of-chapter-2" class="section level3">
<h3><span class="header-section-number">1.4.1</span> Summary of Chapter 2</h3>
<p>To address the lack of tools for reproducible research in clonal population genetics, we present the software package <em>poppr</em> in the R computing language <span class="citation">(Kamvar et al., <a href="#ref-kamvar2014poppr">2014</a><a href="#ref-kamvar2014poppr">b</a>; R Core Team, <a href="#ref-R2016">2016</a>)</span>. Previously, tools necessary for the analysis of clonal populations were available in several stand-alone software programs, each requiring different data input formats. Moreover, each program had different levels of documentation and limited support for all computing platforms. The novelty of <em>poppr</em> was to introduce indices of multilocus genotypic diversity, the index of association, and a fast implementation of Bruvo’s genetic distance, and clone-correction over unlimited levels of user-specified population hierarchies <span class="citation">(Agapow &amp; Burt, <a href="#ref-Agapow_2001">2001</a>; Arnaud-Hanod et al., <a href="#ref-arnaud2007standardizing">2007</a>; Bruvo et al., <a href="#ref-bruvo2004simple">2004</a>)</span>. Because this was implemented in R these analyses can be performed in a reproducible manner on all computing platforms.</p>
</div>
<div id="summary-of-chapter-3" class="section level3">
<h3><span class="header-section-number">1.4.2</span> Summary of Chapter 3</h3>
<p>The initial implementation of <em>poppr</em> contained basic tools for analysis of clonal populations <span class="citation">(Kamvar et al., <a href="#ref-kamvar2014poppr">2014</a><a href="#ref-kamvar2014poppr">b</a>)</span>, but lacked tools for custom definitions of multilocus genotypes and performed poorly with genomic-scale data. Chapter 3 introduces an updated and improved <em>poppr</em> version 2.0. With high throughput sequencing (HTS) data, the amount of missing data and genotyping error increases, and the definition of a multilocus genotype becomes unclear. Moreover, the calculation of the index of association scales poorly with an increase in the number of loci. To address these limitations, we improved <em>poppr</em> with new functionalities to define multilocus genotypes based on genetic distance and calculate the index of association over random samples or windows of SNP loci.</p>
</div>
<div id="summary-of-chapter-4" class="section level3">
<h3><span class="header-section-number">1.4.3</span> Summary of Chapter 4</h3>
<p>A newly-emerged disease of oak—called Sudden Oak Death—spread from California to the Southwest corner of Oregon in 2001. Because of intense management strategies, the epidemic was largely contained to Curry county for the next 15 years. In 2011, an isolated patch of disease appeared in Cape Sebastian, 12 miles from the nearest infected site. With microsatellite genotyping performed across 2 labs and 15 years, we sought to describe the spread of the epidemic in a population genetic context and ask the question of whether or not there was evidence for more than one introduction event. This work provided evidence supporting at least two introductions to Curry county forests. All the analyses were performed in an open-source and reproducible manner using R.</p>
</div>
<div id="summary-of-chapter-5" class="section level3">
<h3><span class="header-section-number">1.4.4</span> Summary of Chapter 5</h3>
<p>The index of association is a measure of multilocus linkage disequilibrium, that is, a correlation coefficient across multiple loci. In sexual populations, loci are randomly assorting due to recombination, resulting in a near-zero value of the index of association. In clonal populations, recombination is non-existent, meaning that loci are passed from parent to offspring in a non-independent fashion, resulting in a significantly non-zero value of the index of association. De Meeûs &amp; Balloux <span class="citation">(<a href="#ref-de2004clonal">2004</a>)</span> demonstrated that this index shows high variance with low levels of sexual reproduction, but due to limitations in software, they were not able to perform power analyses. We used <em>poppr</em> to investigate the power of the index of association to detect sexual reproduction in simulated data sets generated with microsatellite and genomic markers. This chapter provides novel insights on the power, sensitivity, and scope of the index of association.</p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-dobzhansky2013nothing">
<p>Dobzhansky, T. (1973). Nothing in biology makes sense except in the light of evolution. <em>The American Biology Teacher</em>, <em>75</em>(2), 87–91.</p>
</div>
<div id="ref-lynch2016genetic">
<p>Lynch, M., Ackerman, M. S., Gout, J.-F., Long, H., Sung, W., Thomas, W. K., &amp; Foster, P. L. (2016). Genetic drift, selection and the evolution of the mutation rate. <em>Nature Reviews Genetics</em>, <em>17</em>(11), 704–714. <a href="https://doi.org/10.1038/nrg.2016.104" class="uri">https://doi.org/10.1038/nrg.2016.104</a></p>
</div>
<div id="ref-milgroom1997contributions">
<p>Milgroom, M., &amp; Fry, W. (1997). Contributions of population genetics to plant disease epidemiology and management. In <em>Advances in botanical research</em> (pp. 1–30). Elsevier BV. <a href="https://doi.org/10.1016/s0065-2296(08)60069-5" class="uri">https://doi.org/10.1016/s0065-2296(08)60069-5</a></p>
</div>
<div id="ref-hartl1997principles">
<p>Hartl, D. L., &amp; Clark, A. G. (2007). <em>Principles of population genetics</em>. Sunderland, MA, USA: Sinauer Associates.</p>
</div>
<div id="ref-nielsen2013introduction">
<p>Nielsen, R., &amp; Slatkin, M. (2013). <em>An introduction to population genetics: Theory and applications</em>. Sinauer Associates, Incorporated. Retrieved from <a href="http://books.google.com/books?id=Iy08kgEACAAJ" class="uri">http://books.google.com/books?id=Iy08kgEACAAJ</a></p>
</div>
<div id="ref-rice2002evolution">
<p>Rice, W. R. (2002). Evolution of sex: Experimental tests of the adaptive significance of sexual recombination. <em>Nature Reviews Genetics</em>, <em>3</em>(4), 241–251. <a href="https://doi.org/10.1038/nrg760" class="uri">https://doi.org/10.1038/nrg760</a></p>
</div>
<div id="ref-heitman2012evolution">
<p>Heitman, J., Sun, S., &amp; James, T. Y. (2012). Evolution of fungal sexual reproduction. <em>Mycologia</em>, <em>105</em>(1), 1–27. <a href="https://doi.org/10.3852/12-253" class="uri">https://doi.org/10.3852/12-253</a></p>
</div>
<div id="ref-arnaud2007standardizing">
<p>Arnaud-Hanod, S., Duarte, C. M., Alberto, F., &amp; Serrão, E. A. (2007). Standardizing methods to address clonality in population studies. <em>Molecular Ecology</em>, <em>16</em>(24), 5115–5139. <a href="https://doi.org/10.1111/j.1365-294X.2007.03535.x" class="uri">https://doi.org/10.1111/j.1365-294X.2007.03535.x</a></p>
</div>
<div id="ref-arnaud2012implications">
<p>Arnaud-Haond, S., Duarte, C. M., Diaz-Almela, E., Marbà, N., Sintes, T., &amp; Serrão, E. A. (2012). Implications of Extreme Life Span in Clonal Organisms: Millenary Clones in Meadows of the Threatened Seagrass <em>Posidonia oceanica</em>. <em>PLoS ONE</em>, <em>7</em>(2), e30454. <a href="https://doi.org/10.1371/journal.pone.0030454" class="uri">https://doi.org/10.1371/journal.pone.0030454</a></p>
</div>
<div id="ref-orive1993effective">
<p>Orive, M. E. (1993). Effective population size in organisms with complex life-histories. <em>Theoretical Population Biology</em>, <em>44</em>(3), 316–340. <a href="https://doi.org/10.1006/tpbi.1993.1031" class="uri">https://doi.org/10.1006/tpbi.1993.1031</a></p>
</div>
<div id="ref-nieuwenhuis2016frequency">
<p>Nieuwenhuis, B. P. S., &amp; James, T. Y. (2016). The frequency of sex in fungi. <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em>, <em>371</em>(1706), 20150540. <a href="https://doi.org/10.1098/rstb.2015.0540" class="uri">https://doi.org/10.1098/rstb.2015.0540</a></p>
</div>
<div id="ref-felsenstein1974evolutionary">
<p>Felsenstein, J. (1974). The evolutionary advantage of recombination. <em>Genetics</em>, <em>78</em>(2), 737–756. Retrieved from <a href="http://www.genetics.org/content/78/2/737" class="uri">http://www.genetics.org/content/78/2/737</a></p>
</div>
<div id="ref-loewe2006quantifying">
<p>Loewe, L. (2006). Quantifying the genomic decay paradox due to Muller’s ratchet in human mitochondrial DNA. <em>Genetical Research</em>, <em>87</em>(02), 133. <a href="https://doi.org/10.1017/s0016672306008123" class="uri">https://doi.org/10.1017/s0016672306008123</a></p>
</div>
<div id="ref-lynch1990mutation">
<p>Lynch, M., &amp; Gabriel, W. (1990). Mutation load and the survival of small populations. <em>Evolution</em>, 1725–1737.</p>
</div>
<div id="ref-lynch1993mutational">
<p>Lynch, M., Bürger, R., Butcher, D., &amp; Gabriel, W. (1993). The mutational meltdown in asexual populations. <em>Journal of Heredity</em>, <em>84</em>(5), 339–344. Retrieved from <a href="http://jhered.oxfordjournals.org/content/84/5/339.abstract" class="uri">http://jhered.oxfordjournals.org/content/84/5/339.abstract</a></p>
</div>
<div id="ref-milgroom1996recombination">
<p>Milgroom, M. G. (1996). Recombination and the multilocus structure of fungal populations. <em>Annual Review of Phytopathology</em>, <em>34</em>(1), 457–477.</p>
</div>
<div id="ref-anderson1995clonality">
<p>Anderson, J. B., &amp; Kohn, L. M. (1995). Clonality in soilborne, plant-pathogenic fungi. <em>Annual Review of Phytopathology</em>, <em>33</em>(1), 369–391. <a href="https://doi.org/10.1146/annurev.py.33.090195.002101" class="uri">https://doi.org/10.1146/annurev.py.33.090195.002101</a></p>
</div>
<div id="ref-tibayrenc1995population">
<p>Tibayrenc, M. (1995). Population genetics of parasitic protozoa and other microorganisms. In <em>Advances in parasitology volume 36</em> (pp. 47–115). Elsevier BV. <a href="https://doi.org/10.1016/s0065-308x(08)60490-x" class="uri">https://doi.org/10.1016/s0065-308x(08)60490-x</a></p>
</div>
<div id="ref-taylor1999evolutionary">
<p>Taylor, J. W., Geiser, D. M., Burt, A., &amp; Koufopanou, V. (1999). The evolutionary biology and population genetics underlying fungal strain typing. <em>Clinical Microbiology Reviews</em>, <em>12</em>(1), 126–146.</p>
</div>
<div id="ref-tibayrenc1990clonal">
<p>Tibayrenc, M., Kjellberg, F., &amp; Ayala, F. J. (1990). A clonal theory of parasitic protozoa: the population structures of <em>Entamoeba</em>, <em>Giardia</em>, <em>Leishmania</em>, <em>Naegleria</em>, <em>Plasmodium</em>, <em>Trichomonas</em>, and <em>Trypanosoma</em> and their medical and taxonomical consequences. <em>Proceedings of the National Academy of Sciences</em>, <em>87</em>(7), 2414–2418.</p>
</div>
<div id="ref-smith1993how">
<p>Smith, J. M., Smith, N. H., O’Rourke, M., &amp; Spratt, B. G. (1993). How clonal are bacteria? <em>Proceedings of the National Academy of Sciences</em>, <em>90</em>(10), 4384–4388. <a href="https://doi.org/10.1073/pnas.90.10.4384" class="uri">https://doi.org/10.1073/pnas.90.10.4384</a></p>
</div>
<div id="ref-levin1999population">
<p>Levin, B. R. (1999). Population biology, evolution, and infectious disease: Convergence and synthesis. <em>Science</em>, <em>283</em>(5403), 806–809. <a href="https://doi.org/10.1126/science.283.5403.806" class="uri">https://doi.org/10.1126/science.283.5403.806</a></p>
</div>
<div id="ref-halkett2005tackling">
<p>Halkett, F., Simon, J., &amp; Balloux, F. (2005). Tackling the population genetics of clonal and partially clonal organisms. <em>Trends in Ecology &amp; Evolution</em>, <em>20</em>(4), 194–201. <a href="https://doi.org/10.1016/j.tree.2005.01.001" class="uri">https://doi.org/10.1016/j.tree.2005.01.001</a></p>
</div>
<div id="ref-mcdonald1997population">
<p>McDonald, B. A. (1997). The population genetics of fungi: tools and techniques. <em>Phytopathology</em>, <em>87</em>(4), 448–453.</p>
</div>
<div id="ref-tibayrenc1996towards">
<p>Tibayrenc, M. (1996). Towards a unified evolutionary genetics of microorganisms. <em>Annual Review of Microbiology</em>, <em>50</em>(1), 401–429. <a href="https://doi.org/10.1146/annurev.micro.50.1.401" class="uri">https://doi.org/10.1146/annurev.micro.50.1.401</a></p>
</div>
<div id="ref-grunwald2003analysis">
<p>Grünwald, N. J., Goodwin, S. B., Milgroom, M. G., &amp; Fry, W. E. (2003). Analysis of genotypic diversity data for populations of microorganisms. <em>Phytopathology</em>, <em>93</em>(6), 738–746. <a href="https://doi.org/10.1094/phyto.2003.93.6.738" class="uri">https://doi.org/10.1094/phyto.2003.93.6.738</a></p>
</div>
<div id="ref-parks1993study">
<p>Parks, J. C., &amp; Werth, C. R. (1993). A Study of Spatial Features of Clones in a Population of Bracken Fern, <em>Pteridium aquilinum</em> (Dennstaedtiaceae). <em>American Journal of Botany</em>, <em>80</em>(5), 537. <a href="https://doi.org/10.2307/2445369" class="uri">https://doi.org/10.2307/2445369</a></p>
</div>
<div id="ref-de2004clonal">
<p>de Meeûs, T., &amp; Balloux, F. (2004). Clonal reproduction and linkage disequilibrium in diploids: A simulation study. <em>Infection, Genetics and Evolution</em>, <em>4</em>(4), 345–351. <a href="https://doi.org/10.1016/j.meegid.2004.05.002" class="uri">https://doi.org/10.1016/j.meegid.2004.05.002</a></p>
</div>
<div id="ref-magurran1988diversity">
<p>Magurran, A. E. (1988). Why diversity? In <em>Ecological diversity and its measurement</em> (pp. 1–5). Dordrecht: Springer Netherlands. <a href="https://doi.org/10.1007/978-94-015-7358-0_1" class="uri">https://doi.org/10.1007/978-94-015-7358-0_1</a></p>
</div>
<div id="ref-mcdonald2002pathogen">
<p>McDonald, B. A., &amp; Linde, C. (2002a). Pathogen population genetics, evolutionary potential, and durable resistance. <em>Annual Review of Phytopathology</em>, <em>40</em>(1), 349–379. <a href="https://doi.org/10.1146/annurev.phyto.40.120501.101443" class="uri">https://doi.org/10.1146/annurev.phyto.40.120501.101443</a></p>
</div>
<div id="ref-stukenbrock2008origins">
<p>Stukenbrock, E. H., &amp; McDonald, B. A. (2008). The origins of plant pathogens in agro-ecosystems. <em>Annual Review of Phytopathology</em>, <em>46</em>(1), 75–100. <a href="https://doi.org/10.1146/annurev.phyto.010708.154114" class="uri">https://doi.org/10.1146/annurev.phyto.010708.154114</a></p>
</div>
<div id="ref-erwin1996phytophthora">
<p>Erwin, D. C., Ribeiro, O. K., &amp; others. (1996). <em>Phytophthora diseases worldwide</em>. St. Paul, Minnesota, USA: American Phytopathological Society (APS Press).</p>
</div>
<div id="ref-piepenbring2015infestans">
<p>Piepenbring, M. (2015). Biologische Schemata, gezeichnet und freigegeben von M. Piepenbring. Retrieved from <a href="https://goo.gl/XO7TmS" class="uri">https://goo.gl/XO7TmS</a></p>
</div>
<div id="ref-goodwin1994panglobal">
<p>Goodwin, S. B., Cohen, B. A., &amp; Fry, W. E. (1994). Panglobal distribution of a single clonal lineage of the Irish potato famine fungus. <em>Proceedings of the National Academy of Sciences</em>, <em>91</em>(24), 11591–11595.</p>
</div>
<div id="ref-galindo1960nature">
<p>Galindo, A., &amp; Gallegly, M. (1960). The nature of sexuality in <em>Phytophthora infestans</em>. <em>Phytopathology</em>, <em>50</em>, 123–28.</p>
</div>
<div id="ref-sujkowski1994increased">
<p>Sujkowski, L. S. (1994). Increased genotypic diversity via migration and possible occurrence of sexual reproduction of <em>Phytophthora infestans</em> in Poland. <em>Phytopathology</em>, <em>84</em>(2), 201. <a href="https://doi.org/10.1094/phyto-84-201" class="uri">https://doi.org/10.1094/phyto-84-201</a></p>
</div>
<div id="ref-cooke2012genome">
<p>Cooke, D. E. L., Cano, L. M., Raffaele, S., Bain, R. A., Cooke, L. R., Etherington, G. J., Deahl, K. L., Farrer, R. A., Gilroy, E. M., Goss, E. M., Grünwald, N. J., Hein, I., MacLean, D., McNicol, J. W., Randall, E., Oliva, R. F., Pel, M. A., Shaw, D. S., Squires, J. N., Taylor, M. C., Vleeshouwers, V. G. A. A., Birch, P. R. J., Lees, A. K., &amp; Kamoun, S. (2012). Genome analyses of an aggressive and invasive lineage of the irish potato famine pathogen. <em>PLoS Pathogens</em>, <em>8</em>(10), e1002940. <a href="https://doi.org/10.1371/journal.ppat.1002940" class="uri">https://doi.org/10.1371/journal.ppat.1002940</a></p>
</div>
<div id="ref-excoffier2006computer">
<p>Excoffier, L., &amp; Heckel, G. (2006). Computer programs for population genetics data analysis: A survival guide. <em>Nature Reviews Genetics</em>, <em>7</em>(10), 745–758. <a href="https://doi.org/10.1038/nrg1904" class="uri">https://doi.org/10.1038/nrg1904</a></p>
</div>
<div id="ref-kamvar2014poppr">
<p>Kamvar, Z. N., Tabima, J. F., &amp; Grünwald, N. J. (2014b).  Poppr : an R package for genetic analysis of populations with clonal, partially clonal, and/or sexual reproduction. <em>PeerJ</em>, <em>2</em>, e281. <a href="https://doi.org/10.7717/peerj.281" class="uri">https://doi.org/10.7717/peerj.281</a></p>
</div>
<div id="ref-adamack2014popgenreport">
<p>Adamack, A. T., &amp; Gruber, B. (2014). PopGenReport: Simplifying basic population genetic analyses in R. <em>Methods in Ecology and Evolution</em>, <em>5</em>(4), 384–387. <a href="https://doi.org/10.1111/2041-210x.12158" class="uri">https://doi.org/10.1111/2041-210x.12158</a></p>
</div>
<div id="ref-goecks2010galaxy">
<p>Goecks, J., Nekrutenko, A., Taylor, J., &amp; Team, T. G. (2010). Galaxy: A comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences. <em>Genome Biology</em>, <em>11</em>(8), R86. <a href="https://doi.org/10.1186/gb-2010-11-8-r86" class="uri">https://doi.org/10.1186/gb-2010-11-8-r86</a></p>
</div>
<div id="ref-buckheit1995wavelab">
<p>Buckheit, J. B., &amp; Donoho, D. L. (1995). WaveLab and reproducible research. In <em>Wavelets and statistics</em> (pp. 55–81). Springer. <a href="https://doi.org/10.1007/978-1-4612-2544-7_5" class="uri">https://doi.org/10.1007/978-1-4612-2544-7_5</a></p>
</div>
<div id="ref-R2016">
<p>R Core Team. (2016). <em>R: A language and environment for statistical computing</em>. Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="https://www.R-project.org/" class="uri">https://www.R-project.org/</a></p>
</div>
<div id="ref-Agapow_2001">
<p>Agapow, P.-M., &amp; Burt, A. (2001). Indices of multilocus linkage disequilibrium. <em>Molecular Ecology Notes</em>, <em>1</em>(1-2), 101–102. <a href="https://doi.org/10.1046/j.1471-8278.2000.00014.x" class="uri">https://doi.org/10.1046/j.1471-8278.2000.00014.x</a></p>
</div>
<div id="ref-bruvo2004simple">
<p>Bruvo, R., Michiels, N. K., D’Souza, T. G., &amp; Schulenburg, H. (2004). A simple method for the calculation of microsatellite genotype distances irrespective of ploidy level. <em>Molecular Ecology</em>, <em>13</em>(7), 2101–2106.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>This depends on your definition of a species, which is outside of the scope of this dissertation and therefore best debated elsewhere.<a href="1-introduction.html#fnref1">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="2-poppr-an-r-package-for-genetic-analysis-of-populations-with-clonal-partially-clonal-andor-sexual-reproduction.html" class="navigation navigation-next navigation-unique" aria-label="Next page""><i class="fa fa-angle-right"></i></a>

<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["http://hdl.handle.net/1957/60042", "PDF"], ["dissertation.tex", "TEX"], ["https://github.com/zkamvar/dissertation", "SOURCE CODE"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    if (location.protocol !== "file:" && /^https?:/.test(script.src))
      script.src  = script.src.replace(/^https?:/, '');
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
